# A translation control module coordinates germline stem cell differentiation with ribosome biogenesis during *Drosophila* oogenesis

Elliot Martin^1^\*, Patrick Blatt^1^\*, Elaine Ngyuen^2^, Roni Lahr^2^,
Sangeetha Selvam^1^, Hyun Ah M. Yoon^1,3^, Tyler Pocchiari^1,4^, Shamsi
Emtenani^5^, Daria E. Siekhaus^5^, Andrea Berman^2^, Gabriele Fuchs^1â€ ^
and Prashanth Rangan^1â€ ^

^1^Department of Biological Sciences/RNA Institute, University at Albany
SUNY, Albany, NY 12202

^2^Department of Biological Sciences, University of Pittsburgh,
Pittsburgh, PA 15260

^3^Albany Medical College, Albany, NY 12208

^4^SUNY Upstate Medical University, Syracuse, NY 13210-2375

^5^Institute of Science and Technology Austria, Klosterneuburg, Austria

\*These authors contributed equally to this work

^â€ ^Co-corresponding authors

Email: [[gfuchs\@albany.edu]{.underline}](mailto:gfuchs@albany.edu),
[prangan\@albany.edu]{.underline}

**Summary:** Ribosomal defects perturb stem cell differentiation,
causing diseases called ribosomopathies. How ribosome levels control
stem cell differentiation is not fully known. Here, we discovered three
RNA helicases are required for ribosome biogenesis and for *Drosophila*
oogenesis. Loss of these helicases, which we named Aramis, Athos and
Porthos, lead to aberrant stabilization of p53, cell cycle arrest and
stalled GSC differentiation. Unexpectedly, Aramis is required for
efficient translation of a cohort of mRNAs containing a
5\'-Terminal-Oligo-Pyrimidine (TOP)-motif, including mRNAs that encode
ribosomal proteins and a conserved p53 inhibitor, [No]{.underline}vel
[N]{.underline}ucleolar protein 1 (Non1). The TOP-motif co-regulates the
translation of growth-related mRNAs in mammals. As in mammals, the
La-related protein co-regulates the translation of TOP-motif containing
RNAs during *Drosophila* oogenesis. Thus, a previously unappreciated
TOP-motif in *Drosophila* responds to reduced ribosome biogenesis to
co-regulate the translation of ribosomal proteins and a p53 repressor,
thus coupling ribosome biogenesis to GSC differentiation.

## Introduction

All life depends on the ability of ribosomes to translate mRNAs into
proteins. Despite this universal requirement, ribosome biogenesis is not
universally equivalent. Stem cells, the unique cell type that underlies
the generation and expansion of tissues, in particular have a distinct
ribosomal requirement [@gabutRibosomeTranslationalControl2020;@Sanchez2016h;@woolnoughRegulationRRNAGene2016;@zahradkalRegulationRibosomeBiogenesis1991;@Zhang2014d]. Ribosome
production and levels are dynamically regulated to maintain higher
amounts in stem cells [@Fichelson2009a;@gabutRibosomeTranslationalControl2020;@Sanchez2016h;@woolnoughRegulationRRNAGene2016;@zahradkalRegulationRibosomeBiogenesis1991;@Zhang2014d]. For
example, ribosome biogenesis components are often differentially
expressed, as observed during differentiation of embryonic stem cells,
osteoblasts, and myotubes [@gabutRibosomeTranslationalControl2020;@WatanabeSusaki2014a;@zahradkalRegulationRibosomeBiogenesis1991]. In
some cases, such as during *Drosophila* germline stem cell (GSC)
division, ribosome biogenesis factors asymmetrically segregate during
asymmetric cell division, such that a higher pool of ribosome biogenesis
factors is maintained in the stem cell compared to the daughter cell
[@blattPosttranscriptionalGeneRegulation2020;@Fichelson2009a;@Zhang2014d]. Reduction of ribosome levels in stem cells causes
differentiation defects. In *Drosophila,* perturbations that reduce
ribosome levels in the GSCs result in differentiation defects causing
infertility [@Sanchez2016h]. Similarly, humans with reduced ribosome
levels are afflicted with clinically distinct diseases known as
ribosomopathies, such as Diamond-Blackfan anemia, that often result from
loss of proper differentiation of tissue-specific progenitor cells
[@Armistead2014a;@Barlow2010a;@Brooks2014b;@Higa-Nakamine2012o;@liptonDefectiveErythroidProgenitor1986;@Mills2017c]. However, the
mechanisms by which ribosome biogenesis is coupled to proper stem cell
differentiation remain incompletely understood.

Ribosome production requires the transcription of ribosomal RNAs (rRNAs)
and of mRNAs encoding ribosomal proteins [@Bousquet-Antonelli2000a;@delacruzFunctionsRibosomalProteins2015;@Granneman2011;@Granneman2006;@Tafforeau2013a;@Venema1997]. Several factors, such
as helicases and endonucleases, transiently associate with maturing
rRNAs to facilitate rRNA processing, modification, and folding
[@Granneman2011;@Sloan2017e;@Tafforeau2013a;@Watkins2012b].
Ribosomal proteins are imported into the nucleus, where they assemble
with rRNA to form the small 40S and large 60S ribosome subunits, which
are then exported to the cytoplasm [@Baxter-Roshek2007f;@Decatur2002b;@Granneman2011;@Granneman2006;@Kos2010;@nerurkarEukaryoticRibosomeAssembly2015;@Tafforeau2013a;@Zemp2007]. Loss of RNA Polymerase I transcription factors, helicases,
exonucleases, large or small subunit ribosomal proteins, or other
processing factors all compromise ribosome biogenesis and trigger
diverse stem cell-related phenotypes [@Brooks2014b;@Calo2018a;@Mills2017c;@Sanchez2016h;@Yelick2015a;@Zhang2014d].

Nutrient availability influences the demand for *de novo* protein
synthesis and thus ribosome biogenesis
[@anthonyOrallyAdministeredLeucine2000;@hongEvaluationNutrientSensingMTOR2012;@Mayer2006a;@shuNutrientControlMRNA2020]. In mammals, nearly all of the mRNAs that
encode the ribosomal proteins contain a Terminal Oligo Pyrimidine (TOP)
motif within their 5\' untranslated region (UTR), which regulates their
translation in response to nutrient levels [@Fonseca2015a;@Hong2017a;@Lahr2017b;@Tcherkezian2014b]. Under growth-limiting
conditions, La related protein 1 (Larp1) binds to the TOP sequences and
to mRNA caps to inhibit translation of ribosomal proteins
[@Fonseca2015a;@jiaMTORC1PromotesTOP2021;@Lahr2017b;@philippeLarelatedProteinLARP12018]. When growth conditions are
suitable, Larp1 is phosphorylated by the nutrient/redox/energy sensor
TOR complex 1 (TORC1), and does not efficiently bind the TOP sequence,
thus allowing for translation of ribosomal proteins [@Fonseca2015a;@fonsecaLARP1MajorPhosphorylation2018;@Hong2017a;@jiaMTORC1PromotesTOP2021]. In some instances, Larp1 binding can also
stabilize TOP-containing mRNAs [@aokiLARP1SpecificallyRecognizes2013;@bermanControversiesFunctionLARP12020;@gentilellaAutogenousControlTOP2017;@ogamiLARP1FacilitatesTranslational2020], linking mRNA translation
with mRNA stability to promote ribosome biogenesis
[@aokiLARP1SpecificallyRecognizes2013;@bermanControversiesFunctionLARP12020;@Fonseca2015a;@fonsecaLARP1MajorPhosphorylation2018;@Hong2017a;@Lahr2017b;@ogamiLARP1FacilitatesTranslational2020;@philippeLarelatedProteinLARP12018]. Cellular nutrient levels are
known to affect stem cell differentiation and oogenesis in *Drosophila*
[@hsuDietControlsNormal2008], however whether TOP motifs exist in
*Drosophila* to coordinate ribosome protein synthesis is unclear. The
*Drosophila* ortholog of Larp1, La related protein (Larp) is required
for proper cytokinesis and meiosis in *Drosophila* testis as well as for
female fertility, but its targets remain undetermined [@Blagden2009f;@Ichihara2007a].

Germline depletion of ribosome biogenesis factors manifests as a
stereotypical GSC differentiation defect during *Drosophila* oogenesis
[@Sanchez2016h]. Female *Drosophila* maintain 2-3 GSCs in the
germarium (**Figure 1A**) [@Kai2005;@Twombly1996d;@Xie2000b;@Xie1998d;@Xie2007a]. Asymmetric cell division of GSCs produces a
self-renewing daughter GSC, and a differentiating daughter, called the
cystoblast (CB) [@Chen2003q;@McKearin1995b]. This asymmetric
division is unusual: following mitosis, the abscission of the GSC and CB
is not completed until the following G2 phase (**Figure 1A\'**)
[@DeCuevas1998f;@hsuDietControlsNormal2008]. The GSC is marked by a
round structure called the spectrosome, which elongates and eventually
bridges the GSC and CB, similar to the fusomes that connect
differentiated cysts (**Figure 1A\'**). During abscission the extended
spectrosome structure is severed and a round spectrosome is established
in the GSC and the CB [@DeCuevas1998f;@hsuDietControlsNormal2008].
Ribosome biogenesis defects result in failed GSC-CB abscission, causing
cells to accumulate as interconnected cysts marked by a fusome-like
structure called "stem cysts" (**Figure 1A\'**) [@Mathieu2013d;@Sanchez2016h]. In contrast with differentiated cysts
[@McKearin1995b;@McKearin1990e;@ohlsteinEctopicExpressionDrosophila1997], these stem cysts lack
expression of the differentiation factor Bag of Marbles (Bam), do not
differentiate, and typically die, resulting in sterility
[@Sanchez2016h]. How proper ribosome biogenesis promotes GSC
abscission and differentiation is not known.

By characterizing three RNA helicases that promote ribosome biogenesis,
we identified a translational control module that is sensitive to proper
ribosome biogenesis and coordinates ribosome levels with GSC
differentiation. When ribosome biogenesis is optimal, ribosomal proteins
and a p53 repressor are both efficiently translated allowing for proper
GSC cell cycle progression and its differentiation. However, when
ribosome biogenesis is perturbed, we observe diminished translation of
both ribosomal proteins and the p53 repressor. As a consequence, p53 is
stabilized, cell cycle progression is blocked and GSC differentiation is
stalled. Thus, our work reveals an elegant tuning mechanism that links
ribosome biogenesis with cell cycle progression checkpoint and thus stem
cell differentiation. Given that ribosome biogenesis defects in humans
result in ribosomopathies, which often result from stem cell
differentiation defects, our data lay the foundation for understanding
the etiology of developmental defects that arise due to ribosomopathies.

**Results**

**Three conserved RNA helicases are required in the germline for GSC
differentiation**

We performed a screen to identify RNA helicases that are required for
female fertility in *Drosophila*, and identified three predicted RNA
helicases with previously uncharacterized functions, *CG5589*, *CG4901,*
and *CG9253* (**Figure 1B-C**) (**Supplemental Table 1**)
[@blattRNADegradationSculpts2020]. We named these candidate genes
*aramis*, *athos*, and *porthos,* respectively, after Alexandre Dumas\'
three musketeers who fought in service of their queen. To further
investigate how these helicases promote fertility, we depleted *aramis*,
*athos*, and *porthos* in the germline using the germline-driver
*nanos*-*GAL4* (*nosGAL4*) in combination with RNAi lines. We detected
the germline and spectrosomes/fusomes in ovaries by immunostaining for
Vasa and 1B1, respectively. In contrast to controls, *aramis*, *athos*,
and *porthos* germline RNAi flies lacked spectrosome-containing cells,
and instead displayed cells with fusome-like structures proximal to the
self-renewal niche **(Figure 1D-H; Figure S1A-A\'\'\'**). The cells in this
cyst-like structure contained ring canals, a marker of cytoplasmic
bridges, suggesting that they are indeed interconnected (**Figure
S1B-B\'\'\'**) [@Zhang2014d]. In addition to forming cysts in an
aberrant location, the *aramis*, *athos*, and *porthos* germline RNAi
ovaries failed to form egg chambers **(Figure 1D-H)**.

Aberrant cyst formation proximal to the niche could reflect stem cysts
with GSCs that divide to give rise to CBs but fail to undergo
cytokinesis or differentiated cysts that initiate differentiation but
cannot progress further to form egg chambers. To discern between these
possibilities, first we detected the expression of a marker of GSCs,
phosphorylated Mothers against decapentaplegic (pMad). We observed pMad
expression in the cells closest to the niche, but not elsewhere in the
germline cysts of *aramis*, *athos*, and *porthos* germline RNAi flies
(**Figure S1C-F\')** [@kaiEmptyDrosophilaStem2003]. Additionally, none
of the cells connected to the GSCs in *aramis*, *athos*, and *porthos*
germline RNAi flies expressed the differentiation reporter *bamGFP*
(**Figure 1D-G\'\')** [@McKearin1995b]*.* Thus, loss of *aramis*,
*athos*, or *porthos* in the germline results in the formation of stem
cysts, however with variable severity. This variability could be due to
a differential requirement for these genes or different RNAi
efficiencies. Overall, we infer that Aramis, Athos, and Porthos are
required for proper GSC cytokinesis to produce a stem cell and
differentiating daughter.

**Athos, Aramis, and Porthos are required for ribosome biogenesis**

We found that Aramis, Athos, and Porthos are conserved from yeast to
humans (**Figure 1B**). The closest orthologs of Aramis, Athos, and
Porthos are Rok1, Dhr2, and Rrp3 in yeast and DExD-Box Helicase 52
(DDX52), DEAH-Box Helicase 33 (DHX33), and DEAD-Box Helicase 47 (DDX47)
in humans, respectively [@huIntegrativeApproachOrtholog2011]. Both
the yeast and human orthologs have been implicated in rRNA biogenesis
[@bohnsackQuantitativeAnalysisSnoRNA2008;@khoshnevisDEADboxProteinRok12016;@martinPreribosomalRNAInteraction2014;@oday8SRRNAProcessing1996;@sekiguchiNOP132RequiredProper2006;@Tafforeau2013a;@venemaProcessingPreribosomalRNA1995;@Venema1997;@vincentSSUProcessomeInteractome2017;@zhangIdentificationDHX33Mediator2011]. In addition, the
GSC-cytokinesis defect that we observed in *aramis*, *athos*, and
*porthos* RNAi flies is a hallmark of reduced ribosome biogenesis in the
germline [@Sanchez2016h]. Based on these observations, we
hypothesized that Aramis, Athos, and Porthos could enhance ribosome
biogenesis to promote proper GSC differentiation.

Many factors involved in rRNA biogenesis localize to the nucleolus and
interact with rRNA [@arabiCMycAssociatesRibosomal2005;@grandoriCMycBindsHuman2005;@Henras2008c;@Karpen1988a]. To detect
the subcellular localization of Aramis and Athos, we used available
lines that express Aramis::GFP::FLAG or Athos::GFP::FLAG fusion proteins
under endogenous control. For Porthos, we expressed a Porthos::FLAG::HA
fusion under the control of UASt promoter in the germline using a
previously described approach [@delucaEfficientExpressionGenes2018].
We found that in the germline, Aramis, Athos and Porthos colocalized
with Fibrillarin, which marks the nucleolus, the site of rRNA synthesis
(**Figure 2A-C\'\'\'**) [@ochsFibrillarinNewProtein1985]. Aramis was
also in the cytoplasm of the germline and somatic cells of the gonad. To
determine if Aramis, Athos, and Porthos directly interact with rRNA, we
performed immunoprecipitation (IP) followed by RNA-seq. We found that
rRNA immunopurified with Aramis, Athos, and Porthos **(Figure 2D-D\'\',
Figure S2A-A\'\').** Thus, Aramis, Athos, and Porthos are present in the
nucleolus and interact with rRNA, suggesting that they might regulate
rRNA biogenesis.

Nucleolar size, and in particular nucleolar hypotrophy, is associated
with reduced ribosome biogenesis and nucleolar stress [@Neumuller2008;@zhangIdentificationDHX33Mediator2011]. If Aramis, Athos, and Porthos
promote ribosome biogenesis, then their loss would be expected to cause
nucleolar stress and a reduction in mature ribosomes. Indeed,
immunostaining for Fibrillarin revealed hypotrophy of the nucleolus in
*aramis, athos,* and *porthos* germline RNAi flies compared to in
control flies, consistent with nucleolar stress (**Figure S2B-C**).
Next, we used polysome profile analysis to evaluate the ribosomal
subunit ratio and translation status of ribosomes in S2 cells depleted
of *aramis*, *athos*, or *porthos*
[@boamahPolyADPRibosePolymerase2012;@ounapHumanWBSCR22Protein2013].
We found that upon the depletion of all three helicases, the heights of
the polysome peaks were reduced (**Figure 2E-E\'\'**). We found that
depletion of *aramis* and *porthos* diminished the height of the 40S
subunit peak compared to the 60S subunit peak, characteristic of
defective 40S ribosomal subunit biogenesis (**Figure 2E, E\'\'**, **Figure
S2D)** [@chengSmallLargeRibosomal2019], whereas *athos* depletion
diminished the height of the 60S subunit peak compared to the 80S peaks,
characteristic of a 60S ribosomal subunit biogenesis defect (**Figure
2E\', Figure S2D\'**) [@chengSmallLargeRibosomal2019]**.**
RNAi-mediated depletion of the orthologs of these helicases in HeLa
cells similarly affected the polysome profiles (**Figure 2F\'-F\'\', Figure
S2E-G**). Taken together our findings indicate that these helicases
promote ribosome biogenesis in *Drosophila* and mammalian cells.

**Aramis promotes cell cycle progression via p53 repression**

Our data so far indicate that Aramis, Athos and Porthos promote ribosome
biogenesis, which is known to be required for GSC abscission
[@Sanchez2016h]. Yet the connections between ribosome biogenesis and
GSC abscission are poorly understood. To explore the connection, we
further examined the *aramis* germline RNAi line, as its defect was
highly penetrant but maintained sufficient germline for analysis
(**Figure 1E**). First, we compared the mRNA profiles of *aramis*
germline RNAi ovaries to *bam* germline RNAi to determine if genes that
are known to be involved in GSC abscission have altered expression. We
used *bam* depletion as a control because it leads to the accumulation
of stem cell daughters [@Flora2018l;@gilboaGermLineStem2003;@McKearin1995b;@ohlsteinEctopicExpressionDrosophila1997]..

We performed RNA-seq and found that 607 RNAs were downregulated and 673
RNAs were upregulated in *aramis* germline RNAi versus *bam* germline
RNAi (cut-offs for differential gene expression were log~2~(foldchange)
\>\|1.5\|, FDR \< 0.05) (**Figure S3A, Supplemental Table 2**). Gene
Ontology (GO) analysis for biological processes on these genes encoding
these differentially expressed mRNAs
[@thomasPANTHERLibraryProtein2003] revealed that the genes that were
downregulated upon *aramis* germline depletion were enriched for GO
terms related to the cell cycle, whereas the upregulated genes were
enriched for GO terms related to stress response (**Figure 3A, Figure
S3B**). The downregulated genes included *Cyclin A*, which is required
for cell cycle progression, *Cyclin B* (*CycB*) and *aurora B*, which
are required for both cell cycle progression and cytokinesis; in
contrast the housekeeping gene *Actin 5C* was unaffected (**Figure 3B-C,
Figure S3C-C\'**) [@Mathieu2013d;@Matias2015g]. We confirmed that
CycB was reduced in the ovaries of *aramis* germline RNAi flies compared
to *bam* germline RNAi flies by immunofluorescence (**Figure 3D-F**).
These results suggest that *aramis* is required for the proper
expression of key regulators of GSC abscission.

CycB is expressed during G2 phase after asymmetric cell division to
promote GSC abscission [@Flora2018l;@Mathieu2013d]. To test if the
loss of germline *aramis* leads to GSC abscission defects due to
diminished expression of CycB, we attempted to express a functional
CycB::GFP fusion protein in the germline under the control of a UAS/GAL4
system (**Figure S3D-D\'**) [@Mathieu2013d]. Unexpectedly, the
CycB::GFP fusion protein was not expressed in the *aramis-*depleted
germline, unlike the wild type (WT) germline (**Figure S3E-E\'**)
[@glotzerCyclinDegradedUbiquitin1991;@Mathieu2013d;@Zielke2014a].
We considered the possibility that progression into G2 is blocked in the
absence of *aramis*, precluding expression of CycB. To monitor the cell
cycle, we used theÂ FluorescenceÂ Ubiquitin-basedÂ CellÂ CycleÂ Indicator
(FUCCI) system. *Drosophila* FUCCI utilizes a GFP-tagged degron from
E2f1 to mark G2, M, and G1 phases and an RFP-tagged degron from CycB to
mark S, G2, and M phases [@Zielke2014a]. We observed cells in
different cell cycle stages in both WT and *bam*-depleted germaria, but
the *aramis*-depleted germaria did not express GFP nor RFP (**Figure
S3F-H\'\'**). Double negative reporter expression is thought to indicate
early S phase, when expression of E2f1 is low and CycB is not expressed
[@hinnantTemporalRemodelingCell2017]. The inability to express FPs is
not due to a defect in translation as *aramis*-depleted germline can
express GFP that is not tagged with the degron (**Figure S3I-I\'**).
Taken together, we infer that loss of *aramis* blocks cell cycle
progression around late G1 phase/early S phase and prevents progression
to G2 phase, when GSCs abscise from CBs.

In mammals, cells defective for ribosome biogenesis stabilize p53, which
is known to impede the G1 to S transition {Formatting Citation}.
Therefore, we hypothesized that the reduced ribosome biogenesis in the
*aramis*-depleted germline leads to p53 stabilization in
undifferentiated cells, driving cell cycle arrest and GSC abscission
defects. To test this hypothesis, we detected p53 and Vasa in the
germline by immunostaining. A hybrid dysgenic cross that expresses p53
in undifferentiated cells was utilized as a positive control, and *p53*
null flies were used as negative controls (**Figure S3J-K**)
[@Moon2018a]. In WT, we observed p53 expression in the meiotic stages
of germline but p53 expression in GSCs and CBs was attenuated as
previously reported (**Figure 3G-G\'\'**)
[@luMeioticRecombinationProvokes2010]. However, compared to WT, we
observed p53 expression in the stem cysts of the *aramis-*depleted
germline (**Figure 3G-I**). Similarly, we observed p53 expression in the
stem cysts of *athos-* and *porthos-*depleted germlines **(Figure
S3L-M)**, further supporting that reduced ribosome biogenesis stabilizes
p53. To determine if p53 stabilization is required for the cell cycle
arrest in *aramis-*depleted germline cysts, we depleted *aramis* in the
germline of *p53* mutants. We observed a partial but significant
alleviation of the cyst phenotype, such that spectrosomes were restored
(**Figure 3J-L**). This finding indicates that p53 contributes to
cytokinesis failure upon loss of *aramis*, but that additional factors
are also involved. Taken together, we find that *aramis-*depleted germ
cells display reduced ribosome biogenesis, aberrant expression of p53
protein and a block in cell cycle progression. Reducing p53 partially
alleviates the cell cycle block and GSC cytokinesis defect.

**Aramis promotes translation of Non1, a negative regulator of p53,
linking ribosome biogenesis to the cell cycle**

Although p53 protein levels were elevated upon loss of *aramis* in the
germline, *p53* mRNA levels were not significantly altered (log~2~ fold
change: -0.49; FDR: 0.49). Given that ribosome biogenesis is affected,
we considered that translation of p53 or one of its regulators was
altered in *aramis*-depleted germlines. To test this hypothesis, we
performed polysome-seq of gonads depleted for *aramis* or *bam* in the
germline [@Flora2018k]. We plotted the ratios of polysome-associated
RNAs to total RNAs (**Figure 4A-A\'\', Supplemental Table 3**) and
identified 87 mRNAs with a reduced ratio upon depletion of *aramis*,
suggesting that they were translated less efficiently. Loss of *aramis*
reduced the levels of these 87 downregulated transcripts in polysomes,
without significantly affecting their total mRNA levels (**Figure 4B,
Figure S4A-A\'**). These 87 transcripts encode proteins mostly associated
with translation including Ribosomal proteins (**Figure 4C).** To
validate that Aramis regulates translation of these target mRNAs, we
utilized a reporter line for the aramis-regulated transcript encoding
Ribosomal protein S2 (RpS2) that is expressed in the context of the
endogenous promoter and regulatory sequences
[@buszczakCarnegieProteinTrap2007;@Zhang2014d]. We observed reduced
levels of RpS2::GFP in germlines depleted of *aramis* but not in those
depleted of *bam* **(Figure 4D-F)**. To ensure that reduced RpS2::GFP
levels did not reflect a global decrease in translation, we visualized
nascent translation using O-propargyl-puromycin (OPP). OPP is
incorporated into nascent polypeptides and can be detected using
Click-chemistry [@Sanchez2016h]. We observed that global translation
in the germlines of ovaries depleted of *aramis* was not reduced
compared to *bam* **(Figure 4G-I)**. Thus, loss of *aramis* results in
reduced translation of a subset of transcripts.

None of these 87 translational targets have been implicated in directly
controlling abscission [@Mathieu2013d;@Matias2015g]. However, we
noticed that the mRNA encoding Novel Nucleolar protein 1 (Non1/CG8801)
was reduced in polysomes upon loss of *aramis* in the germline (**Figure
4C**). The human ortholog of Non1 is GTP Binding Protein 4 (GTPBP4), and
these proteins are known to physically interact with p53 in both
*Drosophila* and human cells and have been implicated in repressing p53
(mentioned as CG8801 in Lunardi et al.) [@Li2018a;@Lunardi2010a].
To determine if translation of Non1 is reduced upon depletion of
*aramis,* we monitored the abundance of Non1::GFP, a transgene that is
under endogenous control [@sarovGenomewideResourceAnalysis2016], and
found that Non1::GFP was expressed in the undifferentiated GSCs and CBs
(**Figure 5A-A\'\')**. Non1::GFP levels were reduced in the
*aramis-*depleted stem cysts compared to the CBs that accumulated upon
*bam*-depletion (**Figure 5B-D**), suggesting that Aramis and ribosome
biogenesis promote efficient translation of Non1.

During normal oogenesis, p53 is expressed in cyst stages in response to
recombination-induced double strand breaks
[@luMeioticRecombinationProvokes2010]. We found that Non1 was highly
expressed at undifferentiated stages and in two- and four-cell cysts
when p53 levels were low, whereas its expression was attenuated at
eight- and 16-cell cyst stages when p53 levels were high (**Figure
5A-A\'\'**, **Figure S5A-B\'**). Non1 was highly expressed in egg chambers,
which express low levels of p53. To determine if Non1 regulates GSC
differentiation and p53, we depleted *Non1* in the germline. We found
that germline-depletion of *Non1* results in stem cyst formation and
loss of later stages, as well as increased p53 expression, phenocopying
germline-depletion of *aramis*, *athos*, and *porthos* (**Figure 5E-F,
H, Figure S5C-E**). In addition, we found that loss of *p53* from
*Non1-*depleted germaria partially suppressed the phenotype (**Figure
5F-H**). Thus, *Non1* is regulated by *aramis* and is required for p53
suppression, cell cycle progression, and GSC abscission.

To determine if Aramis promotes GSC differentiation via translation of
Non1, we restored *Non1* expression in germ cells depleted of *aramis*.
Briefly, we cloned *Non1* with heterologous UTR elements under the
control of the UAS/GAL4 system (see Methods)
[@rorthGal4DrosophilaFemale1998]. We found that restoring *Non1*
expression in the *aramis-*depleted germline significantly attenuated
the stem cysts and increased the number of cells with spectrosomes
(**Figure 5I-K**). Taken together, we conclude that Non1 can partially
suppress the cytokinesis defect caused by germline *aramis* depletion.

**Aramis-regulated targets contain a TOP motif in their 5\'UTR**

We next asked how *aramis* and efficient ribosome biogenesis promote the
translation of a subset of mRNAs, including Non1, to regulate GSC
differentiation. We hypothesized that the 87 mRNA targets share a
property that make them sensitive to rRNA and ribosome levels. To
identify shared characteristics, we performed *de novo* motif discovery
of target genes compared to non-target genes
[@heinzSimpleCombinationsLineagedetermining2010] and identified a
polypyrimidine motif in the 5\'UTRs of most target genes (UCUUU; E-value:
6.6e^-094^). This motif resembles the previously described TOP motif at
the 5\' end of mammalian transcripts
[@philippeLarelatedProteinLARP12018;@thoreenUnifyingModelMTORC1mediated2012]. Although the existence of
TOP-containing mRNAs in *Drosophila* has been speculated, to our
knowledge their presence has not been explicitly demonstrated
[@chenComprehensiveAnalysisNucleocytoplasmic2017;@qinGlobalAnalysesMRNA2007a]. This observation motivated us to
precisely determine the 5\' end of transcripts, so we analyzed previously
published [c]{.underline}ap [a]{.underline}nalysis of [g]{.underline}ene
[e]{.underline}xpression sequencing (CAGE-seq) data that had determined
transcription start sites (TSS) in total mRNA from the ovary (**Figure
6A, Figure S6A-A\'**) [@boleyNavigatingMiningModENCODE2014;@chenComparativeValidationMelanogaster2014;@dossantosFlyBaseIntroductionDrosophila2015]. Of the 87 target genes,
76 had sufficient expression in the CAGE-seq dataset to define their
TSS. We performed motif discovery using the CAGE-seq data and found that
72 of 76 Aramis-regulated mRNAs have a polypyrimidine motif that starts
within the first 50 nt of their TSS (**Figure 6B-C**). In mammals, it
was previously thought that the canonical TOP motif begins with an
invariant \'C\' [@meyuhasSynthesisTranslationalApparatus2000;@philippeGlobalAnalysisLARP12020]. However, systematic
characterization of the sequence required in order for an mRNA to be
regulated as a TOP containing mRNA revealed that TOP mRNAs can start
with either a \'C\' or a \'U\' [@philippeGlobalAnalysisLARP12020]. Thus,
mRNAs whose efficient translation is dependent on *aramis* share a
terminal polypyrimidine-rich motif in their 5\'UTR that resembles a TOP
motif.

In vertebrates, most canonical TOP-regulated mRNAs encode ribosomal
proteins and translation initiation factors that are coordinately
upregulated in response to growth cues mediated by the Target of
Rapamycin (TOR) pathway and the TOR complex 1 (TORC1)
[@Hornstein2001a;@iadevaiaMTORC1SignalingControls2014;@Kim2008b;@meyuhasRaceDecipherTop2015;@Pallares-Cartes2012a] Indeed, 76 of the
87 Aramis targets were ribosomal proteins, and 9 were known or putative
translation factors, consistent with TOP-containing RNAs in vertebrates
(**Figure 4C, Supplemental Table 4**). To determine if the putative TOP
motifs that we identified are sensitive to TORC1 activity, we designed
"TOP reporter" constructs. Specifically, the germline-specific *nanos*
promoter was employed to drive expression of an mRNA with 1) the 5\'UTR
of the *aramis* target RpL30, which contains a putative TOP motif, 2)
the coding sequence for a GFP-HA fusion protein and 3) a 3\'UTR (K10)
that is not translationally repressed [@Flora2018k], referred to as
the WT-TOP reporter (**Figure S6B**). As a control, we created a
construct in which the polypyrimidine sequence was mutated to a
polypurine sequence referred to as the Mut-TOP reporter (**Figure
S6B**).

In *Drosophila*, TORC1 activity increases in 8- and 16-cell cysts
[@hongEvaluationNutrientSensingMTOR2012;@kimSpatialActivationTORC12017]. We found that the WT-TOP reporter
displayed peak expression in 8 cell cysts, whereas the Mutant-TOP
reporter did not **(Figure 6D-E\'\')**, suggesting that the WT-TOP
reporter is sensitive to TORC1 activity. Moreover, depletion of
*Nitrogen permease regulator-like 3* (*Nprl3*), an inhibitor of mTORC1
[@Wei2014b], led to a significant increase in expression of the
WT-TOP reporter but not the Mutant-TOP reporter **(Figure S6C-G)**.
Additionally, to attenuate mTORC1 activity, we depleted
*regulatory-associated protein of mTOR* (*raptor)*, one of the subunits
of the mTORC1 complex [@hongEvaluationNutrientSensingMTOR2012;@loewithTargetRapamycinTOR2011]. Here we found that the WT-TOP
reporter had a significant decrease in reporter expression while the
Mutant-TOP reporter did not show a decrease in expression **(Figure
S6H-L)**. Taken together, our data suggest that Aramis-target
transcripts contain TOP motifs that are sensitive to TORC1 activity.
However, we note that our TOP reporter did not recapitulate the pattern
of Non1::GFP expression, suggesting that Non1 may have additional
regulators that modulate its protein levels in the cyst stages.

TOP mRNAs show increased translation in response to TOR signaling,
leading to increased ribosome biogenesis
[@jefferiesRapamycinSuppressesTOP1997;@jiaMTORC1PromotesTOP2021;@powersRegulationRibosomeBiogenesis1999;@thoreenUnifyingModelMTORC1mediated2012]. However, to our knowledge,
whether reduced ribosome biogenesis can coordinately diminish the
translation of TOP mRNAs to balance and lower ribosome protein
production and thus balance the levels of the distinct components needed
for full ribosome assembly is not known. To address this question, we
crossed the transgenic flies carrying the WT-TOP reporter and Mutant-TOP
reporter into *bam* and *aramis* germline RNAi backgrounds. We found
that the expression from the WT-TOP reporter was reduced by 2.9-fold in
the germline of *aramis* RNAi ovaries compared to *bam* RNAi ovaries
**(Figure 6F-G, J)**. In contrast, the Mutant-TOP reporter was only
reduced by 1.6-fold in the germline of *aramis* RNAi ovaries compared to
*bam* RNAi ovaries **(Figure 6H-J)**. This suggests that the TOP
motif-containing mRNAs are sensitive to ribosome biogenesis.

**Larp binds TOP sequences in *Drosophila***

Next, we sought to determine how TOP-containing mRNAs are regulated
downstream of Aramis. In mammalian cells, Larp1 is a critical negative
regulator of TOP-containing RNAs during nutrient deprivation
[@bermanControversiesFunctionLARP12020;@Fonseca2015a;@Hong2017a;@philippeGlobalAnalysisLARP12020;@Tcherkezian2014b]. Therefore, we
hypothesized that *Drosophila* Larp reduces the translation of
TOP-containing mRNAs when rRNA biogenesis is reduced upon loss of
*aramis*. First, using an available gene-trap line in which endogenous
Larp is tagged with GFP and 3xFLAG, we confirmed that Larp was robustly
expressed throughout all stages of oogenesis including in GSCs **(Figure
S7A-A\')**.

Next, we performed electrophoretic mobility shift assays (EMSA) to
examine protein-RNA interactions with purified *Drosophila* Larp-DM15,
the conserved domain that binds to TOP sequences in vertebrates
[@Lahr2017b]. As probes, we utilized capped 42-nt RNAs corresponding
to the 5\'UTRs of *RpL30* and *Non1*, including their respective TOP
sequences. We observed a gel shift with these RNA oligos in the presence
of increasing concentrations of Larp-DM15 **(Figure 7A-A\', Figure
S7B)**, and this shift was abrogated when the TOP sequences were mutated
to purines **(Figure S7C-C\').** To determine if Larp interacts with
TOP-containing mRNAs *in vivo*, we immunopurified Larp::GFP::3xFLAG from
the ovaries of the gene-trap line and performed RNA-seq (**Figure
S7D**). We uncovered 156 mRNAs that were bound to Larp, and 84 of these
were among the 87 *aramis* translation targets, including *Non1*,
*RpL30*, and *RpS2* (**Figure 7B-C, Supplemental Table 5**). Thus,
*Drosophila* Larp binds to TOP sequences *in vitro* and TOP-containing
mRNAs *in vivo*.

To test our hypothesis that *Drosophila* Larp inhibits the translation
of TOP-containing mRNAs upon loss of *aramis*, we immunopurified
Larp::GFP::3xFLAG from germline *bam* RNAi ovaries and germline *aramis*
RNAi ovaries. Larp protein is not expressed at higher levels in *aramis*
RNAi compared to developmental control *bam* RNAi (**Figure S7E-G**). We
found that Larp binding to *aramis* target mRNAs *Non1* and *RpL30* was
increased in *aramis* RNAi ovaries compared to *bam* RNAi ovaries
(**Figure 7D, Figure S7H**). In contrast, a non-target mRNA that does
not contain a TOP motif, *alpha-tubulin* mRNA, did not have a
significant increase in binding to Larp in *aramis* RNAi ovaries
compared to *bam* RNAi ovaries. Overall, these data suggest that reduced
rRNA biogenesis upon loss of *aramis* increases Larp binding to the
TOP-containing mRNAs *Non1* and *RpL30*.

If loss of *aramis* inhibits the translation of TOP-containing mRNAs due
to increased Larp binding, then overexpression of Larp would be expected
to phenocopy germline depletion of *aramis*. Unphosphorylated Larp binds
to TOP motifs more efficiently, but the precise phosphorylation sites of
*Drosophila* Larp, to our knowledge, are currently unknown
[@Hong2017a]. To circumvent this issue, we overexpressed the DM15
domain of Larp which we showed binds the RpL30 and Non1 TOP motifs *in
vitro* (**Figure 7A-A\'**), and, based on homology to mammalian Larp1,
lacks majority of the putative phosphorylation sites
[@jiaMTORC1PromotesTOP2021;@Lahr2017b;@philippeLarelatedProteinLARP12018]. We found that overexpression of a
Larp-DM15::GFP fusion in the germline resulted in fusome-like structures
extending from the niche (**Figure 7E-F\'**)**.** Additionally, ovaries
overexpressing Larp-DM15 had 32-cell egg chambers, which were not
observed in control ovaries (**Figure S7I-I\'**). The presence of 32-cell
egg chambers is emblematic of cytokinesis defects that occur during
early oogenesis [@Mathieu2013d;@Matias2015g;@Sanchez2016h]. Our
findings indicate that these cells are delayed in cytokinesis and that
over expression of Larp partially phenocopies depletion of *aramis*.

**Discussion**

During *Drosophila* oogenesis, efficient ribosome biogenesis is required
in the germline for proper GSC cytokinesis and differentiation. The
outstanding questions that needed to be addressed were: 1) Why does
disrupted ribosome biogenesis impair GSC abscission and differentiation?
and 2) How does the GSC monitor and couple ribosome abundance to
differentiation? Our results suggest that germline ribosome biogenesis
defect stalls the cell cycle, resulting a loss of differentiation and
the formation of stem cysts. We discovered that proper ribosome
biogenesis is monitored through a translation control module that allows
for co-regulation of ribosomal proteins and a p53 repressor. Loss of
*aramis*, *athos* and *porthos* reduces ribosome biogenesis and inhibits
translation of a p53 repressor, leading to p53 stabilization, cell cycle
arrest and loss of stem cell differentiation (**Figure 7G**).

**Aramis, Athos, and Porthos are required for efficient ribosome
biogenesis in *Drosophila***

We provide evidence that Aramis, Athos and Porthos play a role in
ribosome biogenesis in *Drosophila*, similar to their orthologs in yeast
[@bohnsackQuantitativeAnalysisSnoRNA2008;@Granneman2006;@khoshnevisDEADboxProteinRok12016;@oday8SRRNAProcessing1996] and
mammals [@sekiguchiNOP132RequiredProper2006;@Tafforeau2013a;@zhangIdentificationDHX33Mediator2011]. Their role in ribosome
biogenesis is likely a direct function of these helicases as they
physically interact with precursor rRNA. In yeast, Rok1, the ortholog of
Aramis, binds to several sites on pre-rRNA, predominantly in the 18S
region [@bohnsackQuantitativeAnalysisSnoRNA2008;@khoshnevisDEADboxProteinRok12016;@martinPreribosomalRNAInteraction2014]. This is consistent with the
small subunit ribosome biogenesis defect we observe upon loss of
*aramis* in *Drosophila* (**Figure 2E**). Rrp3, the yeast ortholog of
Porthos, promotes proper cleavage of pre-rRNA and is required for proper
18S rRNA production [@Granneman2006;@oday8SRRNAProcessing1996].
DDX47, the mammalian ortholog of Porthos, binds to early rRNA precursors
as well as proteins involved in ribosome biogenesis
[@sekiguchiNOP132RequiredProper2006]. Consistent with these findings,
we find that Aramis and Porthos promote 40S ribosome biogenesis. DHX33,
the mammalian ortholog of Athos, has been implicated in facilitating
rRNA synthesis [@zhangIdentificationDHX33Mediator2011]. In contrast,
we find that Athos promotes 60S ribosome biogenesis by directly
interacting with rRNA. However, we cannot rule out that Athos also
affects transcription of rRNA in *Drosophila* as it does in mammals
[@zhangIdentificationDHX33Mediator2011]. Overall, we find that each
mammalian ortholog of Aramis, Athos, and Porthos has consistent ribosome
subunit defects, suggesting that the function of these helicases is
conserved from flies to mammals. Intriguingly, DDX52 (Aramis) is one of
the 15 genes deleted in 17q12 syndrome
[@hendrixPrenatallyDiagnosed17q122012]. 17q12 syndrome results in
delayed development, intellectual disability, and, more rarely,
underdevelopment of organs such as the uterus
[@bernardiniRecurrentMicrodeletion17q122009;@hendrixPrenatallyDiagnosed17q122012]. Our finding that Aramis
disrupts stem cell differentiation could explain some of the poorly
understood defects in 17q12 syndrome.

## Ribosome biogenesis defects leads to cell cycle defects mediated by p53

Here we report that three RNA helicases, *aramis*, *athos*, and
*porthos,* that promote proper ribosome biogenesis in *Drosophila* are
required in the germline for fertility. Loss of *aramis*, *athos*, and
*porthos* causes formation of a "stem cyst" and loss of later stage
oocytes. Stem cysts are a characteristic manifestation of ribosomal
biogenesis deficiency wherein GSCs are unable to complete cytokinesis
and fail to express the differentiation factor Bam, which in GSCs is
initiated at G2 of the cell cycle [@Sanchez2016h;@Zhang2014d]. Our
RNA seq and cell cycle analysis indicates that depletion of *aramis*
blocks the cell cycle at G1, and that failure to progress to G2 prevents
abscission and expression of Bam. Thus, our results suggest that
ribosome biogenesis defects in the germline stall the cell cycle,
resulting in formation of stem cysts and sterility.

In most tissues in *Drosophila,* p53 primarily activates apoptosis,
however, in the germline p53 is activated during meiosis and does not
cause cell death [@fanDualRolesDrosophila2010;@luMeioticRecombinationProvokes2010]. Furthermore, p53 activation in
the germline is required for germline repopulation and GSC survival
after genetic insult, implicating p53 as a potential cell cycle
regulator [@maDNADamageinducedLok2016;@Tasnim2018a]. Our
observation that reduction of *p53* partially rescues a stem cyst defect
caused by ribosome deficiency due to germline depletion of *aramis*
indicates that the G1 block in GSCs is, in part, mediated by p53
activation. Thus, in the *Drosophila* GSCs, p53 blocks the GSC cell
cycle and is sensitive to rRNA production. The developmental
upregulation of p53 during GSC differentiation concomitant with lower
ribosome levels parallels observations in disease states, such as
ribosomopathies [@Calo2018a;@Deisenroth2010e;@Pereboom2011a;@Yelick2015a].

We find that p53 levels in GSCs are regulated by conserved p53 regulator
Non1. In mammalian cells, increased free RpS7 protein due to nucleolar
stress binds and sequesters MDM2, a repressor of p53, freeing p53,
resulting in G1 cell cycle arrest [@Deisenroth2010e;@zhangSignalingP53Ribosomal2009]. *Drosophila* have no identified
homolog to MDM2. It is not fully known how ribosome levels are monitored
in *Drosophila* in the absence of MDM2 and how this contributes to cell
cycle progression. In *Drosophila,* Non1 levels are high in the GSCs and
p53 is low, and reciprocally Non1 levels are low during meiosis, but p53
is expressed. Our finding that loss of Aramis leads to diminished Non1
and elevated p53, and that either loss of p53 or elevated Non1 suppress
differentiation defects caused by loss of Aramis, suggests that, in the
female germline, Non1 may fulfill the function of Mdm2 by promoting p53
degradation during *Drosophila* oogenesis. While Non1 has been shown to
directly interact with p53, how it regulates p53 levels in both humans
and *Drosophila* is not known [@Li2018a;@Lunardi2010a]. Overall,
our data place Non1 downstream of ribosome biogenesis and upstream of
p53 in controlling cell cycle progression and GSC differentiation.
However, our data do not rule out that Non1 may also act upstream of or
in parallel to Aramis.

The vertebrate ortholog of Non1, GTPBP4, also controls p53 levels and is
upregulated in some cancers [@Li2018a;@Lunardi2010a;@yuUpregulationGTPBP4Colorectal2016]. This suggests that there may be
parallel pathways for monitoring ribosome levels via p53 in different
tissue types. Unlike *Drosophila* Non1, its ortholog, GTPBP4 has not
been identified as a TOP mRNA, so if it similarly acts as a mediator
between ribosome biogenesis and the cell cycle it is likely activated in
a somewhat different manner [@philippeGlobalAnalysisLARP12020].
Mammalian Larp1 is required for proper cell cycle progression and
cytokinesis [@burrowsRNABindingProtein2010;@Tcherkezian2014b].
Excitingly several differentiation and cell cycle regulation genes in
mammals are TOP mRNAs regulated by Larp1, including Tumor Protein,
Translationally-Controlled 1 (TPT1) and Nucleosome Assembly Protein 1
Like 1 (NAP1L1) [@philippeGlobalAnalysisLARP12020]. TPT1 is a cancer
associated factor that has been implicated in activating pluripotency
[@burrowsRNABindingProtein2010;@qiaoNap1l1ControlsEmbryonic2018].
Similarly, NAP1L1, a nucleosome assembly protein, is required to
maintain proper cell cycle control as loss of NAP1L1 results in cell
cycle exit and premature differentiation. Overall, although the specific
targets of Larp1 in mammals may differ from those in *Drosophila*, the
mechanism by which Larp modulates cell cycle and differentiation may be
conserved.

**Ribosome biogenesis defects leads to repression of TOP-containing
mRNA**

TOP-containing mRNAs are known to be coregulated to coordinate ribosome
production in response to nutrition or other environmental cues
[@kimballRegulationGlobalSpecific2002;@meyuhasRaceDecipherTop2015;@tangAminoAcidInducedTranslation2001]. Surprisingly, our observation
that loss of *aramis* reduces translation of a cohort of TOP-containing
mRNAs, including Non1, suggests that the TOP motif also sensitizes their
translation to lowered levels of rRNA. This notion is supported by TOP
reporter assays demonstrating that reduced translation upon loss of
*aramis* requires the TOP motif. We hypothesize that limiting TOP mRNA
translation lowers ribosomal protein production to maintain a balance
with reduced rRNA production. This mechanism would prevent the
production of excess ribosomal proteins that cannot be integrated into
ribosomes and the ensuing harmful aggregates
[@tyeProteotoxicityAberrantRibosome2019]. Additionally, it would
coordinate rRNA production and ribosomal protein translation during
normal germline development, where it is known that the level of
ribosome biogenesis and of global translation are dynamic
[@blattPosttranscriptionalGeneRegulation2020;@Fichelson2009a;@Sanchez2016h;@Zhang2014d].

**Larp transduces growth status to ribosome biogenesis targets**

Recent work has shown that the translation and stability of
TOP-containing mRNAs are mediated by Larp1 and its phosphorylation
[@bermanControversiesFunctionLARP12020;@Hong2017a;@jiaMTORC1PromotesTOP2021]. We found that perturbing rRNA production
and thus ribosome biogenesis, without directly targeting ribosomal
proteins, similarly results in deregulation of TOP mRNAs. Our data show
that *Drosophila* Larp binds the *RpL30* and *Non1* 5\'UTR in a
TOP-dependent manner *in vitro* and to nearly all of the translation
targets we identified *in vivo.* Together these data suggest that rRNA
production regulates TOP mRNAs via Larp. Furthermore, the cytokinesis
defect caused by overexpression of Larp-DM15 in the germline suggests
that Larp regulation could maintain the homeostasis of ribosome
biogenesis more broadly by balancing the expression of ribosomal protein
production with the rate of other aspects of ribosome biogenesis, such
as rRNA processing, during development.

Previous studies indicate that unphosphorylated Larp1 binds to and
represses its targets more efficiently than phosphorylated Larp1
[@fonsecaLARP1MajorPhosphorylation2018;@Hong2017a;@jiaMTORC1PromotesTOP2021]. Thus, although we do not know the identity
of the kinase that phosphorylates Larp in *Drosophila*, we hypothesize
that Larp is not phosphorylated upon loss of *aramis, athos* and
*porthos*, when ribosome biogenesis is perturbed. We propose that until
ribosome biogenesis homeostasis is reached, this kinase will remain
inactive, continuously increasing the pool of dephosphorylated Larp. In
this scenario, as dephosphorylated Larp accumulates, it begins to bind
its targets. Initially, it will bind its highest affinity targets,
presumably encoding ribosomal proteins and repress their translation to
rebalance ribosomal protein production with rRNA production. Consistent
with this model, the TOP motif in *RpL30* is bound by Larp even more
tightly with a nearly 9-fold higher affinity compared to the *Non1* TOP
site (**Figure S7B**). We propose that such differences in affinity may
allow Larp to repress ribosomal protein translation to facilitate
cellular homeostasis without immediately causing cell cycle arrest.
However, if homeostasis cannot be achieved and sufficient
dephosphorylated Larp accumulates, Larp will also bind and repress the
translation of lower affinity targets. Repression of Non1 in this manner
would result in cell cycle arrest and block differentiation as occurs
upon *aramis* depletion.

**Ribosome biogenesis in stem cell differentiation** **and
ribosomopathies**

Ribosomopathies arise from defects in ribosomal components or ribosome
biogenesis and include a number of diseases such as Diamond-Blackfan
anemia, Treacher Collins syndrome, Shwachman-Diamond syndrome, and
5q-myelodysplastic syndrome [@Armistead2014a;@draptchinskaiaGeneEncodingRibosomal1999;@mcgowanReducedRibosomalProtein2011;@valdezTreacherCollinsSyndrome2004;@warrenMolecularBasisHuman2018].
Despite the ubiquitous requirement for ribosomes and translation,
ribosomopathies cause tissue-specific disease [@Armistead2014a]. The
underlying mechanisms of tissue specificity remain unresolved.

In this study we demonstrate that loss of helicases involved in rRNA
processing lead to perturbed ribosome biogenesis and, ultimately, cell
cycle arrest. Given that *Drosophila* germ cells undergo an atypical
cell cycle program as a normal part of their development it may be that
this underlying cellular program in the germline leads to the
tissue-specific symptom of aberrant cyst formation [@McKearin1990e].
This model implies that other tissues would likewise exhibit unique
tissue-specific manifestations of ribosomopathies due to their
underlying cell state and underscores the need to further explore
tissue-specific differentiation programs and development to shed light
not only on ribosomopathies but on other tissue-specific diseases
associated with ubiquitous processes. Although it is also possible that
phenotypic differences arise from a common molecular cause, our data
suggests two sources of potential tissue specificity: 1) tissues express
different cohort of mRNAs, such as *Non1*, that are sensitive to
ribosome levels. For example, we find that in *Drosophila* macrophages,
RNAs that regulate the metabolic state of macrophages and influence
their migration require increased levels of ribosomes for their
translation [@emtenaniGeneticProgramBoosts2021]. 2) p53 activation,
as has been previously described, is differentially tolerated in
different developing tissues [@bowenRoleP53Developmental2019;@Calo2018a;@jonesPreventionNeurocristopathyTreacher2008]. Together,
both mechanisms could begin to explain the tissue-specific nature of
ribosomopathies and their link to differentiation.

**[Acknowledgements]{.underline}**

We are grateful to all members of the Rangan and Fuchs labs for their
discussion and comments on the manuscript. We also thanks Dr. Sammons,
Dr. Marlow, Life Science Editors, for their thoughts and comments the
manuscript Additionally, we thank the Bloomington Stock Center, the
Vienna *Drosophila* Resource Center, the BDGP Gene Disruption Project,
and Flybase for fly stocks, reagents, and other resources. P.R. is
funded by the NIH/NIGMS (R01GM111779-06 and RO1GM135628-01), G.F. is
funded by NSF MCB-2047629 and NIH RO3 AI144839, D.E.S. was funded by
Marie Curie CIG 334077/IRTIM and the Austrian Science Fund (FWF) grant
ASI\_FWF01\_P29638S, and A.B is funded by NIH R01GM116889 and American
Cancer Society RSG-17-197-01-RMC.

**[Author Contributions]{.underline}**

Conceptualization, E.T.M., P.B., G.F., and P.R.; Methodology, E.T.M.,
P.B., G.F., and P.R.; Investigation, E.T.M., P.B., E.N., R.L., S.S.,
H.Y., T.P., and S.E.; Writing -- Original Draft, E.T.M., D.E.S., and
P.R.; Writing -- Review & Editing, E.T.M., P.B., D.E.S, A.B., G.F., and
P.R.; Funding Acquisition, G.F. and P.R.; Visualization, E.T.M., E.N.;
Supervision, G.F. and P.R.

**Figure Legends**

**Figure 1: RNA helicases Aramis, Athos and Porthos** **are required for
GSC differentiation.** (**A**) Schematic of *Drosophila* germarium.
Germline stem cells are attached to the somatic niche (dark red). The
stem cells divide and give rise to a stem cell and a cystoblast (CB)
that expresses the differentiation factor Bag-of-marbles (Bam). GSCs and
CBs are marked by spectrosomes. The CB undergoes four incomplete mitotic
divisions giving rise to a 16-cell cyst (blue). Cysts are marked by
branched spectrosome structures known as fusomes (red). One cell of the
16-cell cyst is specified as the oocyte. The 16-cell cyst is
encapsulated by the surrounding somatic cells giving rise to an egg
chamber (green). (**A\'**) Ribosome biogenesis promotes GSC cytokinesis
and differentiation. Disruption of ribosome biogenesis results in
undifferentiated stem cyst accumulation. (**B**) Representation of
conserved protein domains for three RNA helicases in *Drosophila*
compared to *H. sapiens* and *S. cerevisiae* orthologs. Percentage
values represent similarity to *Drosophila* orthologs. (**C**) Egg
laying assay after germline RNAi knockdown of *aramis*, *athos* or
*porthos* indicating a loss of fertility compared to *nosGAL4*, driver
control (n=3 trials). \*\*\* = p \< 0.001, Tukey\'s post-hoc test after
one-way ANOVA, p \< 0.001. Error bars represent standard error (SE).
(**D-G\'\'**) Confocal micrographs of control (**D-D\'\'**) and germline
RNAi depletion targeting (**E-E\'\'**) *aramis*, (**F-F\'\'**) *athos* or
(**G-G\'\'**) *porthos* stained for 1B1 (red, middle grayscale), Vasa
(green), and Bam-GFP (blue, right grayscale). Depletion of these genes
results in a characteristic phenotype in which early germ cells are
connected marked by a 1B1 positive, fusome-like structure highlighted by
a yellow dotted line (**E-G\'\'**) in contrast to the single cells present
in controls (white arrow) or differentiating cysts (yellow dashed line)
(**D-D\'\'**). Bam expression, if present, is followed by loss of the
germline. (**H**) Phenotype quantification of ovaries depleted of
*aramis*, *athos* or *porthos* compared to control ovaries (n=50
ovarioles, df=2, \*\*\* = p \< 0.001, Fisher\'s exact tests with
Holm-Bonferroni correction). Scale bars are 15 micron.

**Figure 2. Athos, Aramis, and Porthos are required for efficient
ribosome biogenesis.** (**A-C\'\'**) Confocal images of ovariole
immunostained for Fibrillarin (red, right grayscale), Vasa (blue),
(**A-A\'\'**) Aramis::GFP, (**B-B\'\'**) Athos::GFP and (**C-C\'\'**)
Porthos::HA (green, middle grayscale). (**A\'\'\'-C\'\'\'**) Fluorescence
intensity plot generated from a box of averaged pixels centered around
the punctate of Fibrillarin in the white box. R values denote Spearman
correlation coefficients between GFP and Fibrillarin from plot profiles
generated using Fiji, taken from the nucleolus denoted by the white box.
Aramis, Athos and Porthos are expressed throughout oogenesis and
localize to the nucleolus. Aramis also localizes to the cytoplasm and
Athos is also present throughout the nucleus (**D-D\'\'**) RNA IP-seq of
(**D**) Aramis, (**D\'**) Athos, and (**D\'\'**) Porthos aligned to rDNA
displayed as genome browser tracks. Bar height represents log scaled
rRNA reads mapping to rDNA normalized to input and spike-in. Grey boxes
outline rRNA precursors that are significantly enriched in the IP
compared to the IgG control (bootstrapped paired t-tests, n=3, \* =
p-value \< 0.05). (**E-E\'\'**) Polysome traces from *Drosophila* S2 cells
treated with dsRNA targeting (**E**) *aramis*, (**E\'**) *athos*,
(**E\'\'**) *porthos* (red line) compared to a mock control (black line).
*aramis* and *porthos* are required to maintain a proper 40S/60S
ribosomal subunit ratio compared to control and have a smaller 40S/60S
ratio. *athos* is required to maintain a proper 40S/60S ribosomal
subunit ratio compared to control and has a larger 40S/60S ratio.
Additionally, *aramis*, *athos*, and *porthos* are required to maintain
polysome levels. (**F-F\'\'**) Polysome preparations from HeLa cells
depleted of *DDX52*, *DHX33*, *DDX47*, and control siRNA treated cells.
*DDX52*, *DHX33*, and *DDX47* are required to maintain a proper 40S/60S
ribosomal subunit ratio. Additionally, all three are required to
maintain polysome levels. Scale bar for all images is 15 micron.

**Figure 3. Athos, Aramis, and Porthos are required for cell cycle
progression during early oogenesis.** (**A**) Bar plot representing the
most significant Biological Process GO terms of downregulated genes in
ovaries depleted of *aramis* compared to *bam* RNAi control (FDR = False
Discovery Rate from p-values using a Fisher\'s exact test). (**B-C**)
Genome browser tracks representing the gene locus of (**B**) *Cyclin B*
and (**C**) a*urora B* in ovaries depleted of *aramis* compared to the
developmental control, *bam* RNAi. Y-axis represents the number of reads
mapping to the locus in bases per million (BPM). (**D-E\'\'**) Confocal
images of germaria stained for Cyclin B (red, middle grayscale) and Vasa
(blue, right grayscale) in (**D-D\'\'\'**) *bam* RNAi control ovaries and
(**E-E\'\'\'**) *aramis* germline RNAi. (**F**) Boxplot of Cyclin B
intensity in the germline normalized to Cyclin B intensity in the soma
in *bam* RNAi and *aramis* RNAi (n=12-14 germaria per sample, \*\*\* =
p\< 0.001, Welch t-test. (**G-H\'\'**) Confocal images of germaria stained
for p53 (red, middle grayscale) and Vasa (blue, right grayscale) in
(**G-G\'\'**) *nosGAL4*, driver control ovaries and (**H-H\'\'**) germline
depletion of *aramis*. Cells highlighted by a dashed yellow circle
represent cell shown in the inset. Driver control *nosGAL4* ovaries
exhibit attenuated p53 expression in GSCs and CBs, but higher expression
in cyst stages as previously reported, while p53 punctate are visible in
the germline of *aramis* RNAi in the undifferentiated cells. (**I**) Box
plot of percentage of pixel area exceeding the background threshold for
p53 in GSCs and CBs in driver control *nosGAL4* ovaries and the germline
of *aramis* RNAi indicates p53 expression is elevated in the germline
over the GSCs/CBs of control ovaries. (n=10 germaria per sample, \*\*\*
= p \< 0.001, Welch\'s t-test. (**J-K\'\'**) Confocal images of germaria
stained for 1B1 (red, middle grayscale) and Vasa (blue, right grayscale)
in (**J-J\'\'**) germline *aramis* RNAi in a wild type background and
(**K-K\'\'**) germline *aramis* RNAi with a mutant, null, *p53^5-A-14^*
background showing presence of spectrosomes upon loss of p53. (**L**)
Quantification of stem cyst phenotypes demonstrates a significant rescue
upon of loss of *p53^5-A-14^* in *aramis* germline depletion compared to
the wild type control (n=43-55 germaria per genotype, df=2, Fisher\'s
exact test p\< 0.05). Scale bar for main images is 15 micron, scale bar
for insets is 3.75 micron.

**Figure 4.** **Aramis is required for efficient translation of a subset
of mRNAs.** (**A-A\'\'**) Biplots of poly(A)+ mRNA Input versus polysome
associated mRNA from (**A**) ovaries genetically enriched for GSCs
(*UAS-tkv*), (**A\'**) Undifferentiated GSC daughter cells (*bam* RNAi)
or (**A\'\'**) germline *aramis* RNAi ovaries. (**B**) Boxplot of
translation efficiency of target genes in *UAS-tkv*, *bam* RNAi, and
*aramis* RNAi samples (ANOVA p\<0.001, post-hoc Welch\'s t-test, n=87,
\*\*\* = p \< 0.001). (**C**) Summary of downregulated target genes
identified from polysome-seq. (**D-E\'**) Confocal images of germaria
stained for 1B1 (red), RpS2::GFP (green, grayscale), and Vasa (blue) in
(**D-D\'**) *bam* RNAi control and (**E-E\'**) *aramis* RNAi (yellow
dashed line marks approximate region of germline used for
quantification). (**F**) A.U. quantification of germline RpS2::GFP
expression normalized to RpS2::GFP expression in the surrounding soma in
undifferentiated daughter cells of *bam* RNAi compared to *aramis* RNAi.
RpS2::GFP expression is significantly lower in *aramis* RNAi compared to
control (n=14 germaria per sample, Welch\'s t-test, \*\*\* = p \< 0.001).
(**G-H\'**) Confocal images of germaria stained for 1B1 (red), OPP
(green, grayscale), and Vasa (blue) in (**G-G\'**) *bam* RNAi and
(**H-H\'**) *aramis* RNAi (yellow dashed line marks approximate region of
germline used for quantification). (**I**) A.U. quantification of OPP
intensity in undifferentiated daughter cells in *bam* RNAi and *aramis*
RNAi (n=11-17 germaria per genotype, Welch\'s t-test, \*\*\* = p \<
0.001). OPP intensity is not downregulated in *aramis* RNAi compared to
the control. Scale bar for all images is 15 micron.

**Figure 5. Non1 represses p53 expression to allow for
differentiation.** (**A-A\'**) Confocal images of Non1::GFP germaria
stained for 1B1 (red), GFP (green, grayscale), and Vasa (blue).
(**A\'\'\'**) Boxplot of Non1::GFP expression over germline development in
GSCs, CBs and Cyst (CC) stages (\* = p \< 0.05, \*\* = p \< 0.01, ANOVA
with Welch\'s post-hoc tests). (**B-C\'**) Confocal images of (**B-B\'**)
*bam* RNAi and (**C-C\'**) *aramis* RNAi germaria both carrying non1::GFP
transgene stained for 1B1 (red), Vasa (blue), and Non1::GFP (green,
grayscale). (**D**) Boxplot of Non1::GFP expression in the germline
normalized to somatic Non1::GFP expression in *bam* RNAi and *aramis*
RNAi (n=24 germaria per genotype, Welch\'s t-test, \*\*\* = p \< 0.001).
Non1 expression is significantly lower in the germline of *aramis* RNAi
compared to *bam* RNAi control. (**E-G\'**) Confocal images of germaria
stained for 1B1 (red), and Vasa (blue) in (**E-E\'**) *nosGAL4*, driver
control ovaries, (**F-F\'**) germline *non1* RNAi, and (**G-G\'**)
germline *non1* RNAi in a *p53^5-A-1-4^* background. Arrow marks the
presence of a single cell (**E, G**), yellow dashed line marks cyst
emanating from the niche (**F-F\'**) or the presence of proper cysts
(**E-E\'**). (**H**) Quantification of percentage of germaria with no
defect (black), presence of single cell (salmon), presence of a cyst
emanating from the niche (brown-red), or germline loss (dark red)
demonstrates a significant rescue of stem cyst formation upon of loss of
*Non1* in *p53^5-A-14^* compared to the *p53* wild type control (n=35-55
germaria per genotype, df=3, Fisher\'s exact test with Holm-Bonferroni
correction \*\* = p\< 0.01, \*\*\* = p\< 0.001). (**I-J\'**) Confocal
images of germaria stained for 1B1 (red), and Vasa (blue) in (**I-I\'**)
*aramis* germline RNAi exhibiting stem cyst phenotype (yellow dashed
line) and (**J-J**\') *aramis* germline RNAi with *non1* overexpression
exhibiting single cells (arrow). (**K**) Phenotypic quantification of
*aramis* RNAi with *non1* overexpression demonstrates a significant
alleviation of the stem cyst phenotype (n=33-57 germaria per genotype,
df=2, Fisher\'s exact test, \*\* = p\< 0.01). Scale bar for all images is
15 micron.

**Figure 6. Aramis regulated mRNAs contain a TOP motif**. (**A**) Genome
browser tract of *RpL30* locus in ovary CAGE-seq data showing the
proportion of transcripts that are produced from a given TSS (orange).
Predominant TSSs are shown in orange and putative TOP motif indicated
with a green box. The bottom blue and red graph represents sequence
conservation of the locus across *Diptera*. The dominant TSS initiates
with a canonical TOP motif. (**B**) Sequence logo generated from *de
novo* motif discovery on the first 200 bases downstream of CAGE derived
TSSs of *aramis* translation target genes resembles a canonical TOP
motif. (**C**) Histogram representing the location of the first 5-mer
polypyrimidine sequence from each CAGE based TSS of *aramis* translation
target genes demonstrates that the TOP motifs occur proximal to the TSS
(n=76 targets). (**D-E\'\'**) Confocal images and quantifications of
*WT-TOP-GFP* (**D-D\'**) and *Mut-TOP-GFP* (**E-E\'**) reporter expression
stained for 1B1 (red), GFP (green), and Vasa (blue). Yellow dotted-line
marks increased reporter expression in 8-cell cysts of *WT-TOP-GFP* but
not in *Mut-TOP-GFP*. Reporter expression was quantified over germline
development for *WT-TOP-GFP* (**D\'\'**) and *Mut-TOP-GFP* reporter
expression (**E\'\'**) and normalized to expression in the GSC reveals
dynamic expression based on the presence of a TOP motif. (**F-G\'**)
Confocal images of *WT-TOP-GFP* reporter ovarioles showing 1B1 (red),
GFP (green), and Vasa (blue) in *bam* germline depletion as a
developmental control (**F-F\'**) and *aramis* germline depleted ovaries
(**G-G\'**). (**H-I\'**) Confocal images of *Mut-TOP-GFP* reporter
expression showing 1B1 (red), GFP (green), and Vasa (blue) in *bam* RNAi
(**H-H\'**) and *aramis* germline RNAi. (**I-I\'**) Yellow dotted-lines
indicates germline. (**J**) A.U. quantification of WT and Mutant TOP
reporter expression in undifferentiated daughter cells in *bam* RNAi
compared *aramis* RNAi demonstrates that the *WT-TOP-GFP* reporter shows
significantly lower expression in *aramis* RNAi than the *Mut-TOP-GFP*
relative to the expression of the respective reporters in *bam* RNAi
(n=17-25 germaria per genotype, with Welch\'s t-test \*\*\* = p\<0.001).
Scale bar for all images is 15 micron.

**Figure 7. Larp binds to TOP mRNAs and binding is regulated by
Aramis.** (**A-A\'**) EMSA of Larp-DM15 and the leading 42 nucleotides of
*RpL30* (**A**) and *Non1* (**A\'**) with increasing concentrations of
Larp-DM15 from left to right indicates that both RNAs bind to Larp-DM15.
(**B**) Volcano plot of mRNAs in Larp::GFP::3xFLAG IP compared to input.
Blue points represent mRNAs significantly enriched in Larp::GFP::3xFLAG
compared to input, but not enriched in an IgG control compared to input.
(**C**) Venn diagram of overlapping Larp IP targets and *aramis* RNAi
polysome seq targets indicates that Larp physically associates with
mRNAs that are translationally downregulated in germline *aramis* RNAi
(p \< 0.001, Hypergeometric Test). (**D**) Bar plot representing the
fold enrichment of mRNAs from Larp RNA IP in germline *aramis* RNAi
relative to matched *bam* RNAi ovaries as a developmental control
measured with qPCR (n=3, \* = p\<0.5, \*\* = p\<0.01, NS =
nonsignificant, One-sample t-test, mu=1) indicates that more of two
*aramis* translation targets *Non1* and *RpL30* are bound by Larp in
*aramis* RNAi. (**E-F\'\'**) Confocal images of *nosGAL4*, driver control
(**E-E\'\'**) and ovaries overexpressing the DM15 region of Larp in the
germline (**F-F\'\'**) ovaries stained for 1B1 (red), Vasa (blue), and
Larp-DM15::GFP (green). Overexpression of Larp results in an
accumulation of extended 1B1 structures (highlighted with a dotted
yellow line), marking interconnected cells when Larp-DM15 is
overexpressed compared to *nosGAL4*, driver control ovaries. (**G**) In
conditions with normal ribosome biogenesis Non1 is efficiently
translated, downregulating p53 levels allowing for progression through
the cell cycle. When ribosome biogenesis is perturbed Non1 is not
translated to sufficient levels, resulting in the accumulation of p53
and cell cycle arrest. Scale bar for all images is 15 micron.

**Supplemental Figure 1. Aramis, Athos, and Porthos are required for
proper cytokinesis and differentiation, related to Figure 1.**
(**A-A\'\'\'**) Confocal images of *nosGAL4*, driver control (**A**) and
germline RNAi knockdown using additional RNAis for *aramis* (**A\'**) and
*athos* (**A\'\'**) stained for 1B1 (red) and Vasa (green). (**A\'\'\'**)
Quantification of percentage of germaria with no defect (black), stem
cysts (salmon), or germline loss (dark red) in ovaries depleted of
*athos*, *aramis*, or *porthos* compared to control ovaries
recapitulates the phenotypes with independent RNAi lines (n=50, df=2,
\*\*\* = p\<0.001, Fisher\'s exact test with Holm-Bonferroni correction).
(**B-B\'\'\'**) Confocal images of germaria stained for 1B1 (red) and
Phospho-tyrosine (green). Ring canals, marked by Phosopho-tyrosine, are
present between the interconnected cells of ovaries depleted of *athos*,
*aramis*, and *porthos* with 1B1 positive structures going through the
ring canals. (**C-F\'**) Confocal images of germaria stained for pMad
(red) and Vasa (green). In control ovaries (**C**) nuclear pMad staining
occurs in cells proximal to the niche marking GSCs. Nuclear pMad
staining in ovaries depleted of (**D**) *athos*, (**E**) *aramis*, and
(**F**) *porthos* demonstrates that the observed cysts are not composed
of GSCs. Scale bar for main images is 15 micron, scale bar for insets is
3.75 micron.

**Supplemental Figure 2. Athos, Aramis, and Porthos are required for
efficient ribosome biogenesis., related to Figure 2.** (**A-A\'\'**)
Western blots of immunoprecipitations from ovaries for FLAG-tagged
Aramis (**A**), Athos, (**A\'**), and Porthos (**A\'\'**). (**B-B\'\'\'**)
Confocal images of (**B**) *nosGAL4*, driver control, (**B\'**) *aramis*
(**B\'\'**) *athos* and *porthos* (**B\'\'\'**) germline RNAi germaria
stained for Fibrillarin (red), DAPI (blue), and Vasa (green). (**C**)
Quantification of nucleolar volume in GSCs of *aramis*, *athos*, and
*porthos* RNAi, compared to control normalized to somatic nucleolar
volume indicates loss of each helicase results in nucleolar stress (n=24
GSCs per genotype, One-way ANOVA, p\<0.001, with Welch\'s t-test, \* =
p\<0.05, \*\* = p\<0.01). (**D-D\'**) Polysome preparations from
*Drosophila* S2 cells in cells treated with dsRNA targeting *RpS19a*
(**D**) or *RpL30* (**D\'**). (**E-G**) Western blot against proteins
targeted for depletion by siRNA in HeLa cells. The human homologs of
Aramis (DDX52) (**E**), Athos (DHX33) (**F**), and Porthos (DDX47)
(**G**) are efficiently depleted with siRNA treatment after 72 hours
(n=3, Welch\'s t-test, \* = p\<0.05). Scale bar for all images is 15
micron.

**Supplemental Figure 3**. **Aramis is required to maintain proper cell
cycle progression, related to Figure 3**. (**A**) Volcano plot of mRNA
expression in *aramis* RNAi compared to *bam* RNAi. Blue points
represent mRNAs significantly upregulated *aramis* RNAi compared to
*bam* RNAi, red points represent mRNAs significantly downregulated
*aramis* RNAi compared to *bam* RNAi. (**B**) Bar plot representing the
most significant Biological Process GO terms of upregulated genes in
ovaries depleted of *aramis* compared to the developmental control,
*bam* RNAi. (**C-C\'**) Genome browser tracks of mRNA expression at the
*Cyclin* *A* (**C**) and *Actin 5C* (**C\'**) loci indicate that the
RNAseq target gene *Cyclin A* expression is downregulated, while a
non-target, *Actin 5C* is not downregulated. (**D-E\'**) Confocal images
of germaria stained for 1B1 (red), DAPI (blue), and Cyclin B::GFP
(green) in control (**D-D\'**) and germline depletion of *aramis*
(**E-E\'**) demonstrates that functional Cyclin B::GFP cannot be
efficiently expressed in germline depleted of *aramis*. (**F-H\'\'**)
Confocal images of germaria that express Fly-FUCCI in the germline
stained for Vasa (blue). GFP-E2f1^degron^ (green, right greyscale) and
RFP-CycB^degron^ (red, left greyscale) *nosGAL4*, driver control ovaries
(**F-F\'\'**), *bam* RNAi as a developmental control (**G-G\'\'**), and
ovaries with germline depletion *of* *aramis* (**H-H\'\'**) demonstrates
that the germline of *aramis* RNAi germline depleted ovaries are
negative for both G1 and G2 cell cycle markers. (**I-I\'**) Confocal
images of *aramis* germline RNAi expressing GFP indicates productive
translation of transgenes still occurs. (**J-M**) Confocal images of
germaria stained for p53 (red) and Vasa (blue) in hybrid dysgenic,
Harwich, ovaries (**J**) and p53^11-B1^ ovaries (**K**) demonstrate the
expected p53 staining patterns. (**L-M**) Confocal images of germaria
immunostained for p53 (red) and Vasa (blue) in ovaries depleted of
*athos* (**L**) or *porthos* (**M**) in the germline exibit p53 punctate
staining. Cells highlighted by a dashed yellow circle represent cells
shown in the inset. Scale bar for main images is 15 micron, scale bar
for insets is 3.75 micron.

**Supplemental Figure 4. The mRNA levels of Aramis polysome-seq targets
are not significantly changing, related to Figure 4.** (**A-A\'**)
Volcano plot of mRNA expression from poly(A)+ mRNA Input libraries in
germline *aramis* RNAi compared to germline driven *UAS*-*tkv* (**A**)
and *bam* RNAi (**A\'**) of targets identified from polysome-seq. No
target genes identified from polysome-seq meet the differential
expression cutoff for mRNA in *UAStkv* compared to *aramis* RNAi or
*bam* RNAi compared to *aramis* RNAi input libraries.

**Supplemental Figure 5. Non1 and p53 expression are inversely related,
related to Figure 5.** (**A-B\'**) Confocal images of ovarioles
expressing Non1::GFP stained for p53 (red), Vasa (blue), and Non1::GFP
(green). Quantifications of staining (**B-B\'**), peak Non1 expression in
control ovaries occurs in GSC-4 cell cyst stages and 16-cell cyst-region
2b stages where p53 expression is low. (**C-D\'**) Confocal images of
*nosGAL4*, driver control (**C-C\'**) and germline *non1* RNAi germaria
stained for p53 (red) and Vasa (blue). (**E**) Quantification of p53
punctate area above cutoff are markedly brighter in the germline of
*Non1* RNAi depleted ovaries compared to the control. Cells highlighted
by a dashed yellow circle represent cells shown in the inset. Scale bar
for main images is 15 micron, scale bar for insets is 3.75 micron.

**Supplemental Figure 6. mTorc1 activity positively regulates TOP
expression, related to Figure 6.** (**A-A\'**) Genome browser tracks of
the *Non1* (**A**) and RpS2 (**A\'**) loci in ovary CAGE-seq data showing
the proportion of transcripts that are produced from a given TSS
(orange). Predominant TSSs are shown in orange and putative TOP motif
beginning at the dominant TSS is indicated with a green box. The bottom
blue and red graph represents sequence conservation of the locus across
*Diptera*. The dominant TSS of *Non1* initiates with a canonical TOP
motif and the *RpS2* TSS initiates at a sequence resembling a TOP motif.
(**B**) Diagram of the *WT* and *Mut-TOP-GFP* reporter constructs
indicating the TOP sequence that is mutated by transversion in the
Mutant reporter (blue). (**C-D\'**) Confocal images of *WT-TOP* reporter
expression stained for 1B1 (red), GFP (green), and Vasa (blue) in
*nosGAL4*, driver control ovaries (**C-C\'**) and ovaries depleted of
*Nprl3* (**D-D\'**) in the germline. (**E-F\'**) Confocal images of
*Mut-TOP-GFP* reporter expression stained for 1B1 (red), GFP (green),
and Vasa (blue) in *nosGAL4*, driver control ovaries (**E-E\'**) and
ovaries depleted of *Nprl3* (**F-F\'**) in the germline. (**G**) A.U.
quantification of WT and Mutant TOP reporter expression in GSCs of
*nosGAL4*, driver control ovaries and GSCs of *Nprl3* germline depleted
ovaries normalized to Vasa expression indicate that the relative
expression of the *WT-TOP-GFP* reporter is higher than the *Mut-TOP-GFP*
reporter (n=9-11 germaria per genotype, Welch\'s t-test, \* = p\<0.05,
\*\* = p\<0.01, \*\*\* = p\<0.001). (**H-I\'**) Confocal images of
*WT-TOP* reporter expression stained for 1B1 (red), GFP (green), and
Vasa (blue) in *nosGAL4*, driver control ovaries (**H-H\'**) and ovaries
depleted of *raptor* (**I-I\'**) in the germline. (**J-K\'**) Confocal
images of *Mut-TOP-GFP* reporter expression stained for 1B1 (red), GFP
(green), and Vasa (blue) in *nosGAL4*, driver control ovaries (**J-J\'**)
and ovaries depleted of *raptor* (**K-K\'**) in the germline. (**L**)
A.U. quantification of WT and Mutant TOP reporter expression in GSCs of
*nosGAL4*, driver control ovaries and GSCs of *raptor* germline depleted
ovaries normalized to Vasa expression indicate that the relative
expression of the *WT-TOP-GFP* reporter is lower than the *Mut-TOP-GFP*
reporter (n=10 germaria per genotype, Welch\'s t-test, \* = p\<0.05, \*\*
= p\<0.01). Scale bar for images is 15 micron.

**Supplemental Figure 7. Larp binds specifically to TOP containing mRNAs
and regulates cytokinesis, related to Figure 7.** (**A-A\'**) Confocal
images of germaria stained for 1B1 (red), Vasa (blue), and *Larp
GFP-3xFLAG* (green, greyscale) indicates Larp is expressed throughout
early oogenesis. (**B**) Quantification of EMSAs and summary of K~d~ of
the protein-RNA interactions. (**C-C\'**) EMSA of Larp-DM15 and the
leading 42 nucleotides of *RpL30* (**B**) and Non1 (**B\'**) with their
TOP sequence mutated to purines as a negative control with increasing
concentrations of Larp-DM15 from left to right indicates that Larp-DM15
requires a leading TOP sequence for its binding. (**D**) Western of
representative IP of Larp::GFP::FLAG from ovary tissue used for RNA
IP-seq. (**E-F\'**) Confocal images of *Larp::GFP::FLAG* reporter
expression stained for 1B1 (red), GFP (green, greyscale), and Vasa
(blue) in *bam* (**E-E\'**) and *aramis* depleted germaria (**F-F\'**).
(**G**) A.U. quantification of Larp::GFP::FLAG reporter expression in
the germline of *bam* RNAi and *aramis* RNAi demonstrates that the
germline expression of Larp is not elevated in *aramis* germline RNAi
compared to *bam* germline RNAi as a developmental control (n=10,
p\>0.05, Welch\'s t-test). (**H**) Western of representative IP of
Larp::GFP::FLAG from ovary tissue used for RNA IP qPCR. (**I-I\'**)
Confocal images of *nosGAL4*, driver control (**H**) and ovaries
overexpressing the DM15 region of Larp in the germline (**I\'**) ovaries
stained for 1B1 (red), Vasa (blue), and Larp-DM15::GFP (green).
Overexpression of Larp-DM15 results in the production of 32-cell egg
chambers which indicates it causes a cytokinesis defect. Scale bar for
all images is 15 micron.

**Supplemental Table 1. Results of germline helicase RNAi screen on
ovariole morphology.**

Results of screen of RNA helicases depleted from the germline. Reported
is the majority phenotype from n=50 ovarioles.

**Supplemental Table 2. Differential expression analysis from** **RNAseq
of ovaries depleted of aramis in the germline compared to a
developmental control**. DEseq2 output from RNAseq of ovaries depleted
of *aramis* in the germline compared to ovaries depleted of bam in the
germline as a developmental control. Sheet 1 (Downregulated Genes)
contains genes and corresponding DEseq2 output meeting the cutoffs to be
considered downregulated in aramis RNAi compared to bam RNAi. Sheet 2
(Upregulated Genes) contains genes and corresponding DEseq2 output
meeting the cutoffs to be considered upregulated in aramis RNAi compared
to bam RNAi. Sheet 3 (All Genes) contains DEseq2 output for all genes in
the dm6 assembly.

**Supplemental Table 3. Analysis of polysome-seq of ovaries depleted of
aramis in the germline compared to developmental controls**. Results of
polysome-seq from ovaries depleted of *aramis* in the germline, ovaries
depleted of *bam*, and ovaries overexpressing Tkv in the germline as
developmental controls. Sheet 1 (Downregulated Genes) contains genes and
corresponding polysome/input ratio values and values representing the
difference in the polysome/input ratios between *aramis* RNAi and the
developmental controls meeting the cutoffs to be considered
downregulated in *aramis* RNAi. Sheet 2 (Upregulated Genes) contains
genes and corresponding polysome/input ratio values and values
representing the difference in the polysome/input ratios between
*aramis* RNAi and the developmental controls meeting the cutoffs to be
considered upregulated in *aramis* RNAi. Sheet 3 (All Genes) contains
DEseq2 output for all genes in the dm6 assembly.

**Supplemental Table 4. Aramis translation targets contain TOP
sequences.** List of *aramis* RNAi polysome downregulated targets and
the position and sequence of the first instance of a 5-mer pyrimidine
sequence downstream of the CAGE-defined TSS of each gene.

**Supplemental Table 5. Enrichment analysis of** **Larp RNA IP
mRNA-seq**. Results of Larp::GFP::FLAG IP/IgG/Input mRNAseq. Each sheet
contains the output of DEseq2. Sheet 1 (Larp Targets) contains Larp IP
targets as defined in methods. Sheet 2 (IPvsIn Enriched) contains genes
significantly enriched in the Larp IP samples compared to the input
samples. Sheet 3 (IgGvsIn Enriched) contains genes significantly
enriched (see methods) in the IgG samples compared to the input samples.
Sheet 4 (IPvsIn All Genes) contains the DEseq2 output of all genes in
the Larp IP samples compared to the input samples. Sheet 5 (IgGvsIn All
Genes) contains the DEseq2 output of all genes in the IgG samples
compared to the input samples.