
\setstretch{1.0}

(ref:RiBi-F2-1-short) **RNA helicases Aramis, Athos and Porthos are required for GSC differentiation.**

(ref:RiBi-F2-2-short) **Aramis, Athos, and Porthos are required for proper cytokinesis and differentiation, related to Figure 2.1.**

(ref:RiBi-F2-3-short) **Athos, Aramis, and Porthos are required for efficient ribosome biogenesis.**

(ref:RiBi-F2-4-short) **Athos, Aramis, and Porthos are required for efficient ribosome biogenesis., related to Figure 2.3.**

(ref:RiBi-F2-5-short) **Athos, Aramis, and Porthos are required for cell cycle progression during early oogenesis.**

(ref:RiBi-F2-6-short) **Aramis is required for proper cell cycle progression, related to Figure 2.5.**

(ref:RiBi-F2-7-short) **Aramis is required for efficient translation of a subset of mRNAs.**

(ref:RiBi-F2-8-short) **The mRNA levels of Aramis polysome-seq targets are not significantly changing, related to Figure 2.7.**

(ref:RiBi-F2-9-short) **Non1 represses p53 expression to allow for GSC differentiation.**

(ref:RiBi-F2-10-short) **Non1 and p53 are inversely expressed, related to Figure 2.9.**

(ref:RiBi-F2-11-short) **Aramis regulated mRNAs contain a TOP motif.**

(ref:RiBi-F2-12-short) **TORC1 activity, Athos, and Porthos, regulate TOP expression in the germarium, related to Figure 2.11.**

(ref:RiBi-F2-13-short) **Larp binds to TOP mRNAs and binding is regulated by Aramis.**

(ref:RiBi-F2-14-short) **Larp binds specifically to TOP containing mRNAs and regulates cytokinesis, related to Figure 2.13.**

<!-- **Figure 2.1: RNA helicases Aramis, Athos and Porthos are required for GSC differentiation.**  -->

(ref:RiBi-F2-1-long) \setlength\parindent{0pt}(**A**) Schematic of *Drosophila* germarium. Germline stem cells are attached to the somatic niche (dark red). The stem cells divide and give rise to a stem cell and a cystoblast (CB) that expresses the differentiation factor Bag-of-marbles (Bam). GSCs and CBs are marked by spectrosomes. The CB undergoes four incomplete mitotic divisions giving rise to a 16-cell cyst (blue). Cysts are marked by branched spectrosome structures known as fusomes (red). One cell of the 16-cell cyst is specified as the oocyte. The 16-cell cyst is encapsulated by the surrounding somatic cells giving rise to an egg chamber. (**A\'**) Ribosome biogenesis promotes GSC cytokinesis and differentiation. Disruption of ribosome biogenesis results in undifferentiated stem cyst accumulation. (**B**) Conservation of *aramis*, *athos*, and *porthos* between *H. sapiens*, *D. melanogaster*, and *S. cerevisiae* (left), trees are drawn to scale, with branch lengths measured in the number of substitutions per site. Representation of conserved protein domains for three RNA helicases in *Drosophila* compared to *H. sapiens* and *S. cerevisiae* orthologs (right). Percentage values represent similarity to *Drosophila* orthologs. (**C**) Egg laying assay after germline RNAi knockdown of *aramis*, *athos* or *porthos* indicating a loss of fertility compared to *nosGAL4*, driver control (n=3 trials). \*\*\* = p \< 0.001, Tukey\'s post-hoc test after one-way ANOVA, p \< 0.001. Error bars represent standard error (SE). (**D-G\'\'**) Confocal micrographs of ovaries from control, *UAS-Dcr2; nosGAL4; bam-GFP* (**D-D\'\'**) and germline RNAi depletion targeting (**E-E\'\'**) *aramis*, (**F-F\'\'**) *athos* or (**G-G\'\'**) *porthos* stained for 1B1 (red, left grayscale), Vasa (green), and Bam-GFP (blue, right grayscale). Depletion of these genes (**E-G\'\'**) results in a characteristic phenotype in which early germ cells are connected marked by a 1B1 positive, fusome-like structure highlighted by a yellow dotted line in contrast to the single cells present in (**D-D\'\'**) controls (white arrow) or differentiating cysts (yellow dashed line). Bam expression, if present, is followed by loss of the germline. (**H**) Phenotype quantification of ovaries depleted of *aramis*, *athos* or *porthos* compared to control ovaries (n=50 ovarioles, df=2, \*\*\* = p \< 0.001, Fisher\'s exact tests with Holm-Bonferroni correction). Scale bars are 15 micron.

<!-- **Figure 2.2. Aramis, Athos, and Porthos are required for proper cytokinesis and differentiation, related to Figure 2.1.**  -->

(ref:RiBi-F2-2-long) \setlength\parindent{0pt}(**A**) qPCR targeting *aramis*, *athos*, *porthos* relative to $\alpha$*-tub84B* to measure knockdown efficiency of *aramis*, *athos*, and *porthos* RNAi in the germline (n=3, Welch\'s t-test with Holm-Bonferroni correction, \* =p\<0.05, \*\*=p\<0.01). (**B-B\'**) Confocal images of (**B**) Aramis::GFP::3XFLAG and (**B\'**) Aramis::GFP::3XFLAG in conjunction with *aramis* germline knockdown stained for Vasa (blue), and GFP (green). (**C-C\'**) Confocal images of (**C**) Athos::GFP::3XFLAG and (**C\'**) Athos::GFP::3XFLAG in conjunction with *athos* germline knockdown stained for Vasa (blue), and GFP (green), early germline is outlined in yellow, representative somatic cell indicated by arrow. (**D**) Quantification of germline expression of GFP normalized to somatic GFP expression in Aramis::GFP::3XFLAG ovaries compared to Aramis::GFP::3XFLAG in conjunction with *aramis* germline knockdown and Athos::GFP::3XFLAG ovaries compared to Athos::GFP::3XFLAG in conjunction with *athos* germline knockdown showing that *aramis* RNAi and *athos* RNAi efficiently knockdown their targets in the germline. (n=14-15, Welch\'s t-test with Holm-Bonferroni correction, \*\*\*=p\<0.001) (**E-E\'\'\'**) Confocal images of (**E**) *nosGAL4*, driver control and germline RNAi knockdown using additional RNAi lines for (**E\'**) *aramis* and (**E\'\'**) *athos* stained for 1B1 (red) and Vasa (green). (**E\'\'\'**) Quantification of percentage of germaria with no defect (black), stem-cysts (salmon), or germline loss (dark red) in ovaries depleted of *athos*, *aramis*, or *porthos* compared to control ovaries recapitulates the phenotypes with independent RNAi lines (n=50, df=2, \*\*\* = p\<0.001, Fisher\'s exact test with Holm-Bonferroni correction). (**F-F\'\'\'**) Confocal images of germaria stained for 1B1 (red) and Phospho-tyrosine (green). Ring canals, marked by Phosopho-tyrosine, connect differentiating cysts in (**F**) control *nosGAL4* ovaries and in between the interconnected cells of ovaries depleted of (**F\'**) *aramis*, (**F\'\'**) *athos*, or (**F\'\'\'**) *porthos* with 1B1 positive structures going through the ring canals. (**G-G\'\'\'**) Confocal images of ovarioles stained for 1B1 (red) and Vasa (green). Control, *nosGAL* ovaries have egg chambers while ovaries depleted of (**G\'**) *aramis*, (**G\'\'**) *athos*, or (**G\'\'\'**) *porthos* lack egg chambers. (**H-H\'\'\'**) Confocal images of germaria stained for pMad (red, grayscale) and Vasa (green). In (**H**) control ovaries nuclear pMad staining occurs in cells proximal to the niche marking GSCs. Nuclear pMad staining in ovaries depleted of (**H\'**) *aramis*, (**H\'\'**) *athos*, and (**H\'\'\'**) *porthos* demonstrates that the observed cysts are not composed of GSCs. Scale bar for main images is 15 micron, scale bar for insets is 3.75 micron.

<!-- **Figure 2.3. Athos, Aramis, and Porthos are required for efficient ribosome biogenesis.**  -->

(ref:RiBi-F2-3-long) \setlength\parindent{0pt}(**A-C\'\'**) Confocal images of ovariole immunostained for Fibrillarin (red, right grayscale), Vasa (blue), (**A-A\'\'**) Aramis::GFP, (**B-B\'\'**) Athos::GFP and (**C-C\'\'**) Porthos::HA (green, left grayscale). (**A\'\'\'-C\'\'\'**) Fluorescence intensity plot generated from a box of averaged pixels centered around the punctate of Fibrillarin in the white box. The white box also indicates a nucleus, while the yellow dotted outline indicates the divide between soma and germline, with the germline on the interior of the outline and soma on the exterior. R values denote Spearman correlation coefficients between GFP and Fibrillarin from plot profiles generated using Fiji, taken from the nucleolus denoted by the white box. Aramis, Athos and Porthos are expressed throughout oogenesis and localize to the nucleolus. Aramis is also present in the cytoplasm. (**D-D\'\'**) RNA IP-seq of (**D**) Aramis, (**D\'**) Athos, and (**D\'\'**) Porthos aligned to rDNA locus displayed as genome browser tracks. Bar height represents log scaled rRNA reads mapping to rDNA normalized to input and spike-in. Grey boxes outline ETS (external spacers) and ITS (internal spacers) which are only present in pre-rRNA that are significantly enriched in the IP compared to the IgG control (bootstrapped paired t-tests, n=3, \* = p-value \< 0.05). (**E-E\'\'**) Polysome traces from *Drosophila* S2 cells treated with dsRNA targeting *aramis*, (**E\'**) *athos*, (**E\'**) *porthos* (red line) compared to a mock transfection control (black line). *aramis*, athos and porthos are required to maintain a proper 40S/60S ribosomal subunit ratio compared to control and have a smaller 40S/60S ratio. *athos* is required to maintain a proper 40S/60S ribosomal subunit ratio compared to control and has a larger 40S/60S ratio. Additionally, *aramis*, *athos*, and *porthos* are required to maintain polysome levels. All three helicases are required to maintain polysome levels. Scale bar for all images is 15 micron.

<!-- **Figure 2.4. Athos, Aramis, and Porthos are required for efficient ribosome biogenesis., related to Figure 2.3.**  -->

(ref:RiBi-F2-4-long) \setlength\parindent{0pt}(**A-A\'\'**) Western blots of immunoprecipitations from ovaries for FLAG-tagged (**A**) Aramis, (**A\'**) Athos, and (**A\'\'**) Porthos. (**B-B\'\'\'**) Confocal images of (**B**) *nosGAL4*, driver control, (**B\'**) *aramis* (**B\'\'**) *athos* and (**B\'\'\'**) *porthos* germline RNAi germaria stained for Fibrillarin (green), and Vasa (blue). (**C-C\'**) Quantification of (**C**) raw nucleolar volume or (**C\'**) normalized to cell volume nucleolar volume in GSCs of *aramis*, *athos*, and *porthos* RNAi, compared to control indicates loss of each helicase results in nucleolar stress. (n=12-16 GSCs per genotype, One-way ANOVA, p\<0.001, with Welch\'s t-test, Holm-Bonferroni corrected, \* = p\<0.05, \*\* = p\<0.01, \*\*\* = p\<0.001). (**D-D\'**) Polysome preparations from *Drosophila* S2 cells in cells treated with dsRNA targeting (**D**) *RpS19a* or (**D\'**) *RpL30* (red line) compared to a mock transfection control (black line). (**E-K**) Confocal images of heterozygous (**E**) *shrb*^O3^/+, (**F**) *aramis*^f06152^/+ mutant ovaries, (**G**) *shrb*^O3^/+; *aramis*^f06152^/+ transheterozygous, (**H**) *athos* deficiency/+ mutant ovaries, (**I**) *athos* deficiency/*shrb*^O3^ transheterozygous, heterozygous (**J**) *porthos*^KG05120^/+ mutant ovaries and (**K**) *porthos*^KG05120^/*shrb*^O3^ transheterozygous ovaries stained for 1B1 (red) and Vasa (blue). (**L**) Quantification of proportion of germaria with presence of a stem-cyst or single cells from the indicated genotypes (pairwise one-sided Fisher\'s tests, Holm-Bonferroni corrected, \* = p\<0.05, \*\* = p\<0.01, \*\*\* = p\<0.001). Scale bar for main images is 15 micron, scale bar for insets is 3.75 micron.

<!-- **Figure 2.5. Athos, Aramis, and Porthos are required for cell cycle progression during early oogenesis.**  -->

(ref:RiBi-F2-5-long) \setlength\parindent{0pt}(**A**) Bar plot representing the most significant Biological Process GO-terms of downregulated genes in ovaries depleted of *aramis* compared to *bam* RNAi control (FDR = False Discovery Rate from p-values using a Fisher\'s exact test). (**B-C**) Genome browser tracks representing the gene locus of (**B**) *CycB* and (**C**) *aurora B* in ovaries depleted of *aramis* compared to the developmental control, *bam* RNAi. Y-axis represents the number of reads mapping to the locus in bases per million (BPM). (**D-E\'\'**) Confocal images of germaria stained for CycB (red, left grayscale) and Vasa (blue, right grayscale) in (**D-D\'\'\'**) *bam* RNAi control ovaries and (**E-E\'\'**) *aramis* germline RNAi. (**F**) Boxplot of CycB intensity in the germline normalized to Cyc B intensity in the soma in *bam* RNAi and *aramis* RNAi (n=12-14 germaria per sample, \*\*\* = p\< 0.001, Welch t-test. (**G-H\'\'**) Confocal images of germaria stained for p53 (red, left grayscale), GFP (green), and Vasa (blue, right grayscale) in (**G-G\'\'**) *nosGAL4*, driver control ovaries and (**H-H\'\'**) germline depletion of *aramis*. Cells highlighted by a dashed yellow circle represent cell shown in the inset. Driver control *nosGAL4* ovaries exhibit attenuated p53 expression in GSCs and CBs, but higher expression in cyst stages as previously reported, while p53 punctate are visible in the germline of *aramis* RNAi in the undifferentiated cells. (**I**) Box plot of percentage of pixel area exceeding the background threshold for p53 in GSCs and CBs in driver control *nosGAL4* ovaries and the germline of *aramis* RNAi indicates p53 expression is elevated in the germline over the GSCs/CBs of control ovaries. (n=10 germaria per sample, \*\*\* = p \< 0.001, Welch\'s t-test. (**J-K\'\'**) Confocal images of germaria stained for 1B1 (red, left grayscale) and Vasa (blue, right grayscale) in (**J-J\'\'**) germline *aramis* RNAi in a wildtype background and (**K-K\'\'**) germline *aramis* RNAi with a mutant, null, *p53^5-A-14^* background showing presence of spectrosomes upon loss of p53. (**L**) Quantification of stem-cyst phenotypes demonstrates a significant rescue upon of loss of *p53^5-A-14^* in *aramis* germline depletion compared to the control (n=43-55 germaria per genotype, df=2, Fisher\'s exact test p\< 0.05). (**M-N**) Confocal images of ovaries stained for 1B1 (red) and Vasa (blue) in *nosGAL4* ovaries (**M**) and ovaries overexpressing *p53* in the germline. Cysts are denoted by a dotted yellow line, single cells by a white arrow. (**N**). We observed that 84% of germaria from *p53* overexpression ovaries had loss of germline while 12% of germaria contained a cyst, marked by an extended spectrosome structure connecting germline cells together, and an additional 4% of germaria contained an accumulation of single cells (n=55 germaria, Fisher\'s exact test, p\<0.001). Scale bar for main images is 15 micron, scale bar for insets is 3.75 micron

<!-- **Figure 2.6 Aramis is required for proper cell cycle progression, related to Figure 2.5.**  -->

(ref:RiBi-F2-6-long) \setlength\parindent{0pt}(**A**) Biplot of mRNA levels in *aramis* RNAi compared to *bam* RNAi. Blue points represent mRNAs significantly upregulated *aramis* RNAi compared to *bam* RNAi, red points represent mRNAs significantly downregulated *aramis* RNAi compared to *bam* RNAi. (**B**) Bar plot representing the most significant Biological Process GO-terms of upregulated genes in ovaries depleted of *aramis* compared to the developmental control, *bam* RNAi. (**C-C\'**) Genome browser tracks of mRNA levels at the (**C**) *Cyclin* *A* and (**C\'**) *Actin 5C* loci indicate that the RNAseq target gene *Cyclin A* expression is downregulated, while a non-target, *Actin 5C* is not downregulated. (**D-E\'**) Confocal images of germaria stained for 1B1 (red, greyscale) and Vasa (blue) (**D-D\'**) from ovaries with germline depletion of *bam* and (**E-E\'**) germline depletion of *bam* and *aramis* simultaneously demonstrates that simultaneous depletion of *bam* and *aramis* results in a stem-cyst similar to depletion of *aramis* alone. (**F-G**) Confocal images of germaria stained for 1B1 (red), Vasa (blue), and Cyclin B::GFP (green) in (**F**) control and (**G**) germline depletion of *aramis* demonstrates that functional Cyclin B::GFP cannot be efficiently expressed in germline depleted of *aramis*. (**H-J\'\'**) Confocal images of germaria that express Fly-FUCCI in the germline stained for Vasa (blue). GFP-E2f1^degron^ (green, right grayscale) and RFP-CycB^degron^ (red, left grayscale) in (**H-H\'\'**) *nosGAL4*, driver control ovaries, (**I-I\'\'**) *bam* RNAi as a developmental control, and (**J-J\'\'**) ovaries with germline depletion *of* *aramis* demonstrates that the germline of *aramis* RNAi germline depleted ovaries are negative for both G1 and G2 cell cycle markers. (**K**) Confocal images of *aramis* germline RNAi expressing GFP in the germline, stained for 1B1 (red), Vasa (blue), and GFP (green) indicates productive translation of transgenes still occurs. (**L-M**) Confocal images of germaria stained for p53 (red) and Vasa (blue) in (**L**) hybrid dysgenic, Harwich, ovaries and (**M**) p53^11-B-1^ ovaries stained for p53 (red) and Vasa (blue) demonstrate the expected p53 staining patterns. (**N-O**) Confocal images of germaria stained for p53 (red) and Vasa (blue) in ovaries depleted of (**N**) *athos* or (**O**) *porthos* in the germline exhibit p53 punctate staining. Cells highlighted by a dashed yellow circle represent cells shown in the inset. (**P-Q**) Confocal images of germaria stained for 1B1 (red) and Vasa (blue) in (**P**) germline *athos* RNAi in a wildtype background and (**Q**) germline *athos* RNAi with a mutant, null, *p53^5-A-14^* background showing presence of spectrosomes upon loss of p53. (**R-S**) Confocal images of germaria stained for 1B1 (red) and Vasa (blue) in (**R**) germline *porthos* RNAi in a wildtype background and (**S**) germline *porthos* RNAi with a mutant, heterozygous, *p53^5-A-14^*/+ background showing presence of spectrosomes upon reduction of p53 (**T**) Phenotypic quantification of demonstrates a significant rescue upon of loss of *p53^5-A-14^* in germline *athos* (n=63 *athos*, n=47 *athos* RNAi, *p53^5-A-14^*, Fisher\'s exact test p \> 0.001) or *porthos*-depletion (n=34 *porthos* RNAi; n=41 *porthos* RNAi, *p53^5-A-14^*/+, Fisher\'s exact test p \> 0.001) compared to the respective control.

<!-- **Figure 2.7.** **Aramis is required for efficient translation of a subset of mRNAs.**  -->

(ref:RiBi-F2-7-long) \setlength\parindent{0pt}(**A-A\'\'**) Biplots of poly(A)+ mRNA Input versus polysome associated mRNA from (**A**) ovaries genetically enriched for GSCs (*UAS-tkv*), (**A\'**) Undifferentiated GSC daughter cells (*bam* RNAi) or (**A\'\'**) germline *aramis* RNAi ovaries. (**B**) Boxplot of translation efficiency of target genes in *UAS*-*tkv*, *bam* RNAi, and *aramis* RNAi samples (ANOVA p\<0.001, post-hoc Welch\'s t-test, n=87, \*\*\* = p \< 0.001). (**C**) Summary of downregulated target genes identified from polysome-seq. (**D-E\'**) Confocal images of germaria stained for 1B1 (red), RpS2::GFP (green, grayscale), and Vasa (blue) in (**D-D\'**) *bam* RNAi control and (**E-E\'**) *aramis* RNAi (yellow dashed line marks approximate region of germline used for quantification). (**F**) A.U. quantification of germline RpS2::GFP expression normalized to RpS2::GFP expression in the surrounding soma in undifferentiated daughter cells of *bam* RNAi compared to *aramis* RNAi. RpS2::GFP expression is significantly lower in *aramis* RNAi compared to control (n=14 germaria per sample, Welch\'s t-test, \*\*\* = p \< 0.001). (**G-I\'**) Confocal images of germaria stained for 1B1 (red), OPP (green, grayscale), and Vasa (blue) in (**G-G\'**) *nosGAL4*, (**H-H\'**) *bam* RNAi, and (**I-I\'**) *aramis* RNAi (yellow dashed line marks approximate region of germline used for quantification). (**J**) A.U. quantification of OPP intensity in single cells of nosGAL4 control germeria and undifferentiated daughter cells in *bam* RNAi as controls and *aramis* RNAi (n = 10 germaria per genotype, Welch\'s t-test, NS = p \> 0.05). OPP intensity is not significantly downregulated in *aramis* RNAi compared to either control. Scale bar for all images is 15 micron.

<!-- **Figure 2.8. The mRNA levels of Aramis polysome-seq targets are not significantly changing, related to Figure 2.7.**  -->

(ref:RiBi-F2-8-long) \setlength\parindent{0pt}(**A-A\'**) Volcano plot of mRNA levels from poly(A)+ mRNA input libraries in germline *aramis* RNAi compared to (**A**) germline driven *UAS*-*tkv* and (**A\'**) *bam* RNAi of targets identified from polysome-seq. No target genes identified from polysome-seq meet the differential expression cutoff for mRNA in *UAS*-*tkv* compared to *aramis* RNAi or *bam* RNAi compared to *aramis* RNAi input libraries. (**B-C\'**) Confocal images of germaria stained for 1B1 (red) RpS2::GFP (green, greyscale) and Vasa (blue) in (**B-B\'**) germline *bam* RNAi as a control and (**C-C\'**) germline *bam* RNAi; *aramis* RNAi. Yellow dotted outline denotes the germline on the interior of the outline with the soma on the exterior of the outline. (**D**) Quantification of RpS2::GFP expression in the germline normalized to the soma in germline *bam* RNAi compared to *bam* RNAi; *aramis* RNAi demonstrates a significant decrease in RpS2::GFP expression when *aramis* is knocked down in a germline *bam* RNAi background (Welch\'s t-test, \*\*\* = p\<0.001, n=15).

<!-- **Figure 2.9. Non1 represses p53 expression to allow for GSC differentiation.**  -->

(ref:RiBi-F2-9-long) \setlength\parindent{0pt}(**A-A\'**) Confocal images of Non1::GFP germaria stained for 1B1 (red), GFP (green, grayscale), and Vasa (blue). (**A\'\'**) Boxplot of Non1::GFP expression over germline development in GSCs, CBs and Cyst (CC) stages (n=5-25 cysts of each type, \* = p \< 0.05, \*\* = p \< 0.01, ANOVA with Welch\'s post-hoc tests). (**B-C\'**) Confocal images of (**B-B\'**) *bam* RNAi and (**C-C\'**) *aramis* RNAi germaria both carrying *Non1::GFP* transgene stained for 1B1 (red), Vasa (blue), and Non1::GFP (green, grayscale). Yellow dashed line marks region of germline used for quantification. (**D**) Boxplot of Non1::GFP expression in the germline normalized to somatic Non1::GFP expression in *bam* RNAi and *aramis* RNAi (n=24 germaria per genotype, Welch\'s t-test, \*\*\* = p \< 0.001). Non1 expression is significantly lower in the germline of *aramis* RNAi compared to *bam* RNAi control. (**E-G\'**) Confocal images of germaria stained for 1B1 (red, grayscale) and Vasa (blue) in (**E-E\'**) *nosGAL4*, driver control ovaries, (**F-F\'**) germline *Non1* RNAi, and (**G-G\'**) germline *Non1* RNAi in a *p53^5-A-1-4^* background. Arrow marks the presence of a single cell (**E, G**), yellow dashed line marks a stem-cyst emanating from the niche (**F-F\'**) or the presence of proper cysts (**E-E\'**). (**H**) Quantification of percentage of germaria with no defect (black), presence of single cell (salmon), presence of a stem-cyst emanating from the niche (brown-red), or germline loss (dark red) demonstrates a significant rescue of stem-cyst formation upon of loss of *Non1* in *p53^5-A-14^* compared to the *p53* wildtype control (n=35-55 germaria per genotype, df=3, Fisher\'s exact test with Holm-Bonferroni correction \*\* = p\< 0.01, \*\*\* = p\< 0.001). (**I-K\'**) Confocal images of germaria stained for 1B1 (red, grayscale) and Vasa (blue) in ovaries with (**I-I\'**) germline Non1 overexpression, (**J-J\'**) *aramis* germline RNAi exhibiting stem-cyst phenotype (yellow dashed line) and (**K-K**\') *aramis* germline RNAi with *Non1* overexpression exhibiting single cells (arrow). (**L**) Phenotypic quantification of *aramis* RNAi with *Non1* overexpression demonstrates a significant alleviation of the stem-cyst phenotype (n=33-57 germaria per genotype, df=2, Fisher\'s exact test, \*\* = p\< 0.01). Scale bar for all images is 15 micron.

<!-- **Figure 2.10. Non1 and p53 are inversely expressed, related to Figure 2.9.**  -->

(ref:RiBi-F2-10-long) \setlength\parindent{0pt}(**A-A\'\'**) Confocal images of ovarioles expressing Non1::GFP stained for p53 (red, right grayscale), Vasa (blue), and Non1::GFP (green, left grayscale). (**B-B\'**) Quantifications of staining, (**B**) peak Non1 expression in control ovaries occurs in GSC-4 cell cyst stages and 16-cell cyst-region 2b stages where (**B\'**) p53 expression is low. (**C-E\'**) Confocal images of germline (**C-C\'**) *bam* RNAi, (**D-D\'**) germline *athos* RNAi, or (**E-E\'**) germline *porthos* RNAi germaria stained for 1B1 (red), *Non1::GFP* (green, greyscale), and Vasa (blue). (**F**) Quantification of Non1:GFP expression in germline *athos, or porthos* RNAi compared to germline *bam* RNAi as a developmental control demonstrates there is significantly lower Non1::GFP expression in *athos, or porthos* RNAi compared to *bam* RNAi (n=10 germaria per genotype, Welch\'s t-test, \*\*\* = p\<0.001) (**G-H\'**) Confocal images of (**G-G\'**) *nosGAL4*, driver control and germline (**H-H\'**) *Non1* RNAi germaria stained for p53 (red, grayscale) and Vasa (blue). (**I**) Quantification of p53 punctate area above cutoff are markedly brighter in the germline of *Non1* RNAi depleted ovaries compared to the control (n=10 germaria per genotype, Welch\'s t-test, \* = p\<0.05). Cells highlighted by a dashed yellow circle represent cells shown in the inset. (**J-M**) Confocal images of germaria stained for 1B1 (red, grayscale), and Vasa (blue) in (**J**) *athos* germline RNAi exhibiting stem-cyst phenotype (yellow dashed line) and (**K**) *athos* germline RNAi with *Non1* overexpression exhibiting single cells (arrow), (**L**) *porthos* germline RNAi exhibiting stem-cyst phenotype (yellow dashed line) and (**M**) *porthos* germline RNAi with *Non1* overexpression exhibiting single cells (arrow). (**N**) Phenotypic quantification of germline *athos* RNAi or *porthos* RNAi with *Non1* overexpression demonstrates a significant alleviation of the stem-cyst phenotype compared to germline *athos* or *porthos* RNAi alone (n=26-33 germaria per genotype, df=2, Fisher\'s exact test, \* = p\< 0.05, \*\* = p\< 0.01). Scale bar for main images is 15 micron, scale bar for insets is 3.75 micron.

<!-- Aramis regulated mRNAs contain a TOP motif. -->

(ref:RiBi-F2-11-long) \setlength\parindent{0pt}(**A**) Genome browser tract of *RpL30* locus in ovary CAGE-seq data showing the proportion of transcripts that are produced from a given TSS (orange). Predominant TSSs are shown in orange and putative TOP motif indicated with a green box. The bottom blue and red graph represents sequence conservation of the locus across *Diptera*. The dominant TSS initiates with a canonical TOP motif. (**B**) Sequence logo generated from *de novo* motif discovery on the first 200 bases downstream of CAGE derived TSSs of *aramis* translation target genes resembles a canonical TOP motif. (**C**) Histogram representing the location of the first 5-mer polypyrimidine sequence from each CAGE based TSS of *aramis* translation target genes demonstrates that the TOP motifs occur proximal to the TSS (n=76 targets). (**D**) Diagram of the *WT* and *Mut-TOP-GFP* reporter constructs indicating the TOP sequence that is mutated by transversion in the Mutant reporter (blue). (**E-F\'\'**) Confocal images and quantifications of (**E-E\'**) *WT-TOP-GFP* and (**F-F\'**) *Mut-TOP-GFP* reporter expression stained for 1B1 (red), GFP (green, grayscale), and Vasa (blue). Yellow dotted-line marks increased reporter expression in 8-cell cysts of *WT-TOP-GFP* but not in *Mut-TOP-GFP*. Reporter expression was quantified over germline development for (**E\'\'**) *WT-TOP-GFP* and (**F\'\'**) *Mut-TOP-GFP* reporter expression and normalized to expression in the GSC reveals dynamic expression based on the presence of a TOP motif. (**G-H\'**) Confocal images of *WT-TOP-GFP* reporter ovarioles showing 1B1 (red), GFP (green, grayscale), and Vasa (blue) in (**G-G\'**) *bam* germline depletion as a developmental control and (**H-H\'**) *aramis* germline depleted ovaries. Yellow dotted lines indicate germline. (**I-J\'**) Confocal images of *Mut-TOP-GFP* reporter expression showing 1B1 (red), GFP (green, grayscale), and Vasa (blue) in (**I-I\'**) *bam* RNAi and (**J-J\'**) *aramis* germline RNAi. Yellow dotted lines indicate germline. (**K**) A.U. quantification of WT and Mutant TOP reporter expression in undifferentiated daughter cells in *bam* RNAi compared *aramis* RNAi demonstrates that the *WT-TOP-GFP* reporter shows significantly lower expression in *aramis* RNAi than the *Mut-TOP-GFP* relative to the expression of the respective reporters in *bam* RNAi indicating that the presence of a TOP motif sensitizes transcripts to regulation (n=17-25 germaria per genotype, with Welch\'s t-test \*\*\* = p\<0.001). Scale bar for all images is 15 micron.

<!-- **Figure 2.12. TORC1 activity, Athos, and Porthos, regulate TOP expression in the germarium, related to Figure 2.11.** -->

(ref:RiBi-F2-12-long) \setlength\parindent{0pt}(**A-B**) Confocal images of *WT-TOP* reporter expression stained for 1B1 (red), GFP (green), and Vasa (blue) in (**A**) *nosGAL4*, driver control ovaries and (**B**) ovaries depleted of *Nprl3* in the germline. (**C-D**) Confocal images of *Mut-TOP-GFP* reporter expression stained for 1B1 (red), GFP (green, grayscale), and Vasa (blue) in (**C**) *nosGAL4*, driver control ovaries and (**D**) ovaries depleted of *Nprl3* in the germline. (**E**) A.U. quantification of WT and Mutant TOP reporter expression in GSCs of *nosGAL4*, driver control ovaries and GSCs of *Nprl3* germline depleted ovaries normalized to Vasa expression indicate that the relative expression of the *WT-TOP-GFP* reporter is higher than the *Mut-TOP-GFP* reporter (n=9-11 germaria per genotype, Welch\'s t-test, \* = p\<0.05, \*\* = p\<0.01, \*\*\* = p\<0.001). (**F-G**) Confocal images of *WT-TOP* reporter expression stained for 1B1 (red), GFP (green), and Vasa (blue) in (**F**) *nosGAL4*, driver control ovaries and (**G**) ovaries depleted of *raptor* in the germline. (**H-I**) Confocal images of *Mut-TOP-GFP* reporter expression stained for 1B1 (red), GFP (green), and Vasa (blue) in (**H**) *nosGAL4*, driver control ovaries and (**I**) ovaries depleted of *raptor* in the germline. (**J**) A.U. quantification of WT and Mutant TOP reporter expression in GSCs of *nosGAL4*, driver control ovaries and GSCs of *raptor* germline depleted ovaries normalized to Vasa expression indicate that the relative expression of the *WT-TOP-GFP* reporter is lower than the *Mut-TOP-GFP* reporter (n=10 germaria per genotype, Welch\'s t-test, \* = p\<0.05, \*\* = p\<0.01). (**K-M\'**) Confocal images of *WT-TOP-GFP* reporter ovarioles showing 1B1 (red), GFP (green, grayscale), and Vasa (blue) in (**K-K\'**) *bam* germline depletion as a developmental control, (**L-L\'**) *athos* germline depleted ovaries, and (**M-M\'**) *porthos* germline depleted ovaries. Yellow dotted lines indicate germline. (**N-P\'**) Confocal images of *Mut-TOP-GFP* reporter expression showing 1B1 (red), GFP (green, grayscale), and Vasa (blue) in (**N-N\'**) *bam* RNAi, (**O-O\'**) *athos* germline RNAi, and (**P-P\'**) *porthos* germline depleted ovaries. Yellow dotted lines indicate germline. (**Q**) A.U. quantification of WT and Mutant TOP reporter expression in undifferentiated daughter cells in *bam* RNAi compared *athos* or *porthos* RNAi demonstrates that the *WT-TOP-GFP* reporter shows significantly lower expression in *athos* and *porthos* RNAi than the *Mut-TOP-GFP* relative to the expression of the respective reporters in *bam* RNAi indicating that the presence of a TOP motif sensitizes transcripts to regulation (n=17-25 germaria per genotype, with Welch\'s t-test with , \*\* = p\<0.01, \*\*\* = p\<0.001). Scale bar for images is 15 micron.

<!-- **Figure 2.13. Larp binds to TOP mRNAs and binding is regulated by Aramis.**  -->

(ref:RiBi-F2-13-long) \setlength\parindent{0pt}(**A-A\'**) EMSA of Larp-DM15 and the leading 42 nucleotides of (**A**) *RpL30* and (**A\'**) *Non1* with increasing concentrations of Larp-DM15 from left to right indicates that both RNAs bind to Larp-DM15. (**B**) Volcano plot of mRNAs in Larp::GFP::3XFLAG IP compared to input. Blue points represent mRNAs significantly enriched in Larp::GFP::3XFLAG compared to input, but not enriched in an IgG control compared to input. (**C**) Venn diagram of overlapping Larp IP targets and *aramis* RNAi polysome seq targets indicates that Larp physically associates with mRNAs that are translationally downregulated in germline *aramis* RNAi (p \< 0.001, Hypergeometric Test). (**D**) Bar plot representing the fold enrichment of mRNAs from Larp RNA IP in germline *aramis* RNAi relative to matched *bam* RNAi ovaries as a developmental control measured with qPCR (n=3, \* = p\<0.5, \*\* = p\<0.01, NS = nonsignificant, One-sample t-test, mu=1) indicates that more of two *aramis* translation targets *Non1* and *RpL30* are bound by Larp in *aramis* RNAi. (**E-F\'\'**) Confocal images of (**E-E\'\'**) *nosGAL4*, driver control and (**F-F\'\'**) ovaries overexpressing the DM15 region of Larp in the germline ovaries stained for 1B1 (red, left grayscale), Vasa (blue), and Larp-DM15::GFP (green, right grayscale). Overexpression of Larp results in an accumulation of extended 1B1 structures (highlighted with a dotted yellow line), marking interconnected cells when Larp-DM15 is overexpressed compared to *nosGAL4*, driver control ovaries. (**G**) In conditions with normal ribosome biogenesis Non1 is efficiently translated, downregulating p53 levels allowing for progression through the cell cycle. When ribosome biogenesis is perturbed Non1 is not translated to sufficient levels, resulting in the accumulation of p53 and cell cycle arrest. Dotted lines indicate indirect affects. Scale bar for all images is 15 micron.

<!-- **Figure 2.14. Larp binds specifically to TOP containing mRNAs and regulates cytokinesis, related to Figure 2.13.** -->

(ref:RiBi-F2-14-long) \setlength\parindent{0pt}(**A-A\'**) Confocal images of germaria stained for 1B1 (red), Vasa (blue), and *Larp GFP-3XFLAG* (green, grayscale) indicates Larp is expressed throughout early oogenesis. (**B**) Quantification of EMSAs and summary of K~d~ of the protein-RNA interactions. (**C-C\'**) EMSA of Larp-DM15 and the leading 42 nucleotides of (**B**) *RpL30* and (**B\'**) *Non1* with their TOP sequence mutated to purines as a negative control with increasing concentrations of Larp-DM15 from left to right indicates that Larp-DM15 requires a leading TOP sequence for its binding. (**D**) Western of representative IP of Larp::GFP::3XFLAG from ovary tissue used for RNA IP-seq. (**E-F\'**) Confocal images of Larp::GFP::3XFLAG reporter expression stained for 1B1 (red), GFP (green, grayscale), and Vasa (blue) in (**E-E\'**) *bam* and (**F-F\'**) *aramis*-depleted germaria. (**G**) A.U. quantification of Larp::GFP::3XFLAG reporter expression in the germline of *bam* RNAi and *aramis* RNAi normalized to germline Vasa intensity demonstrates that the germline expression of Larp is not elevated in *aramis* germline RNAi compared to *bam* germline RNAi as a developmental control (n=10, NS = p>0.05, Welch\'s t-test). (**H-H\'**) Confocal images of Larp::GFP::3XFLAG reporter expression stained for 1B1 (red), GFP (green), and Vasa (blue) in (**H**) *bam* and (**H\'**) *bam*; *aramis*-depleted germaria. (**I**) A.U. quantification of Larp::GFP::3XFLAG reporter expression in the germline of *bam* RNAi and *bam* RNAi ; *aramis* RNAi demonstrates that the germline expression of Larp normalized to somatic Larp expression is not elevated in *bam*; *aramis* germline RNAi compared to *bam* germline RNAi as a developmental control (n=10, NS = p>0.05, Welch\'s t-test). (**J**) Western of representative IP of Larp::GFP::3XFLAG from ovary tissue used for RNA IP qPCR. (**K-K\'**) Confocal images of (**K**) *nosGAL4*, driver control and (**K\'**) ovaries overexpressing the DM15 region of Larp in the germline ovaries stained for DAPI (green) and Vasa (blue). Overexpression of Larp-DM15 results in the production of 32-cell egg chambers which indicates it causes a cytokinesis defect. Scale bar for all images is 15 micron.

# A translation control module coordinates germline stem cell differentiation with ribosome biogenesis during *Drosophila* oogenesis

Elliot T. Martin^1^\*, Patrick Blatt^1^\*, Elaine Nguyen^2^, Roni Lahr^2^, Sangeetha Selvam^1^, Hyun Ah M. Yoon^1,3^, Tyler Pocchiari^1,4^, Shamsi Emtenani^5^, Daria E. Siekhaus^5^, Andrea Berman^2^, Gabriele Fuchs^1†^ and Prashanth Rangan^1†^

^1^Department of Biological Sciences/RNA Institute, University at Albany SUNY, Albany, NY 12202

^2^Department of Biological Sciences, University of Pittsburgh, Pittsburgh, PA 15260

^3^Albany Medical College, Albany, NY 12208

^4^SUNY Upstate Medical University, Syracuse, NY 13210-2375

^5^Institute of Science and Technology Austria, Klosterneuburg, Austria

\*These authors contributed equally to this work

^†^Co-corresponding authors

Email: gfuchs\@albany.edu, prangan\@albany.edu

\setstretch{2.0}

## Summary:

Ribosomal defects perturb stem cell differentiation, causing ribosomopathies. How ribosome levels control stem cell differentiation is not fully known. Here we discovered that three RNA helicases govern ribosome biogenesis and *Drosophila* oogenesis. Loss of these helicases, which we named Aramis, Athos and Porthos, aberrantly stabilized p53, arrested the cell cycle and stalled GSC differentiation. Aramis controls cell cycle progression by regulating translation of mRNAs containing a Terminal Oligo Pyrimidine (TOP) motif in their 5\'-UTRs; we find TOP motifs confer sensitivity to ribosome levels mediated by La-related protein (Larp). One such TOP-containing mRNA codes for [No]{.ul}vel [N]{.ul}ucleolar protein 1 (Non1), a conserved p53 destabilizing protein. Upon a sufficient ribosome concentration, Non1 is expressed and promotes GSC cell cycle progression via p53 degradation. Thus, a previously unappreciated TOP-motif in *Drosophila* responds to reduced ribosome biogenesis to co-regulate the translation of ribosomal proteins and a p53 repressor, coupling ribosome biogenesis to GSC differentiation.

## Introduction

All life depends on the ability of ribosomes to translate mRNAs into proteins. Despite this universal requirement, perturbations in ribosome biogenesis affects some cell types more than others. Stem cells, the unique cell type that underlies the generation and expansion of tissues, in particular have an increased ribosomal requirement [@gabutRibosomeTranslationalControl2020; @Sanchez2016h; @woolnoughRegulationRRNAGene2016; @zahradkalRegulationRibosomeBiogenesis1991; @Zhang2014d]. Ribosome production and levels are dynamically regulated to maintain higher amounts in stem cells [@Fichelson2009a; @gabutRibosomeTranslationalControl2020; @Sanchez2016h; @woolnoughRegulationRRNAGene2016; @zahradkalRegulationRibosomeBiogenesis1991; @Zhang2014d]. For example, ribosome biogenesis components are often differentially expressed, as observed during the differentiation of embryonic stem cells, osteoblasts, and myotubes [@gabutRibosomeTranslationalControl2020; @WatanabeSusaki2014a; @zahradkalRegulationRibosomeBiogenesis1991]. In some cases, such as during *Drosophila* germline stem cell (GSC) division, ribosome biogenesis factors asymmetrically segregate during asymmetric cell division, such that a higher pool of ribosome biogenesis factors is maintained in the stem cell compared to the daughter cell [@blattPosttranscriptionalGeneRegulation2020; @Fichelson2009a; @Zhang2014d]. Reduction of ribosome levels in several stem cell systems can cause differentiation defects [@corsiniCoordinatedControlMRNA2018; @fortierHaploinsufficiencyScreenHighlights2015; @Khajuria2018a; @Zhang2014d]. In *Drosophila,* perturbations that reduce ribosome levels in the GSCs result in differentiation defects causing infertility [@Sanchez2016h]. Similarly, humans with reduced ribosome levels are afflicted with clinically distinct diseases known as ribosomopathies, such as Diamond-Blackfan anemia, that often result from loss of proper differentiation of tissue-specific progenitor cells [@Armistead2014a; @Barlow2010a; @Brooks2014b; @Higa-Nakamine2012o; @liptonDefectiveErythroidProgenitor1986; @Mills2017c]. However, the mechanisms by which ribosome biogenesis is coupled to proper stem cell differentiation remain incompletely understood.

Ribosome production requires the transcription of ribosomal RNAs (rRNAs) and of mRNAs encoding ribosomal proteins [@Bousquet-Antonelli2000a; @delacruzFunctionsRibosomalProteins2015; @Granneman2011; @Granneman2006; @Tafforeau2013a; @Venema1997]. Hundreds of factors including helicases and endonucleases, transiently associate with maturing rRNAs to facilitate rRNA processing, modification, and folding [@Granneman2011; @Sloan2017e; @Tafforeau2013a; @Watkins2012b]. Ribosomal proteins are imported into the nucleus, where they assemble with rRNAs to form precursors to the 40S and 60S ribosomal subunits, which are then exported to the cytoplasm [@Baxter-Roshek2007f; @Decatur2002b; @Granneman2011; @Granneman2006; @Kos2010; @nerurkarEukaryoticRibosomeAssembly2015; @Tafforeau2013a; @Zemp2007]. Loss of RNA Polymerase I transcription factors, helicases, exonucleases, large or small subunit ribosomal proteins, or other processing factors all compromise ribosome biogenesis and trigger diverse stem cell-related phenotypes [@Brooks2014b; @Calo2018a; @Sanchez2016h; @Yelick2015a; @Zhang2014d].

Nutrient availability influences the demand for *de novo* protein synthesis and thus ribosome biogenesis [@anthonyOrallyAdministeredLeucine2000; @hongEvaluationNutrientSensingMTOR2012; @Mayer2006a; @shuNutrientControlMRNA2020]. In mammals, mRNAs that encode the ribosomal proteins contain a Terminal Oligo Pyrimidine (TOP) motif within their 5\' untranslated region (UTR), which regulates their translation in response to nutrient levels [@Fonseca2015a; @Hong2017a; @Lahr2017b; @Tcherkezian2014b]. Under growth-limiting conditions, La related protein 1 (Larp1) binds to the TOP sequences and to mRNA caps to inhibit translation of ribosomal proteins [@Fonseca2015a; @jiaMTORC1PromotesTOP2021; @Lahr2017b; @philippeLarelatedProteinLARP12018]. When growth conditions are suitable, Larp1 is phosphorylated by the nutrient/redox/energy sensor mammalian Target of rapamycin (mTOR) complex 1 (mTORC1), and does not efficiently bind the TOP sequence, thus allowing for translation of ribosomal proteins [@Fonseca2015a; @fonsecaLARP1MajorPhosphorylation2018; @Hong2017a; @jiaMTORC1PromotesTOP2021]. In some instances, Larp1 binding can also stabilize TOP-containing mRNAs [@aokiLARP1SpecificallyRecognizes2013; @bermanControversiesFunctionLARP12020; @gentilellaAutogenousControlTOP2017; @ogamiLARP1FacilitatesTranslational2020], linking mRNA translation with mRNA stability to promote ribosome biogenesis [@aokiLARP1SpecificallyRecognizes2013; @bermanControversiesFunctionLARP12020; @Fonseca2015a; @fonsecaLARP1MajorPhosphorylation2018; @Hong2017a; @Lahr2017b; @ogamiLARP1FacilitatesTranslational2020; @philippeLarelatedProteinLARP12018]. Cellular nutrient levels are known to affect stem cell differentiation and oogenesis in *Drosophila* [@hsuDietControlsNormal2008], however whether TOP motifs exist in *Drosophila* to coordinate ribosome protein synthesis is unclear. The *Drosophila* ortholog of Larp1, La related protein (Larp) is required for proper cytokinesis and meiosis in *Drosophila* testis as well as for female fertility, but its targets remain undetermined [@Blagden2009f; @Ichihara2007a].

Germline depletion of ribosome biogenesis factors manifests as a stereotypical GSC differentiation defect during *Drosophila* oogenesis [@Sanchez2016h]. Female *Drosophila* maintain 2-3 GSCs in the germarium (**Figure 1A**) [@Kai2005; @Twombly1996d; @Xie2000b; @Xie1998d; @Xie2007a]. Asymmetric cell division of GSCs produces a self-renewing daughter GSC and a differentiating daughter, called the cystoblast (CB) (**Figure 1A**) [@Chen2003q; @McKearin1995b]. This asymmetric division is unusual: following mitosis, the abscission of the GSC and CB is not completed until the following G2 phase (**Figure 1A\'**) [@DeCuevas1998f; @hsuDietControlsNormal2008]. The GSC is marked by a round structure called the spectrosome, which elongates and eventually bridges the GSC and CB, similar to the fusomes that connect differentiated cysts (**Figure 1A-A\'**). During abscission the extended spectrosome structure is severed and a round spectrosome is established in the GSC and the CB (**Figure 1A\'**)[@DeCuevas1998f; @hsuDietControlsNormal2008]. Ribosome biogenesis defects result in failed GSC-CB abscission, causing cells to accumulate as interconnected cysts called "stem-cysts" that are marked by a fusome-like structure (**Figure 1A\'**) [@Mathieu2013d; @Sanchez2016h]. In contrast with differentiated cysts [@McKearin1995b; @McKearin1990e; @ohlsteinEctopicExpressionDrosophila1997], these stem-cysts do not express the differentiation factor Bag of Marbles (Bam), do not differentiate, and typically die, resulting in sterility (**Figure 1A\'**) [@Sanchez2016h]. How proper ribosome biogenesis promotes GSC abscission and differentiation is not known.

By characterizing three RNA helicases and showing that they promote ribosome biogenesis, we identified a translational control module, which coordinates ribosome levels with GSC differentiation. When ribosome biogenesis is optimal, ribosomal proteins and a p53 repressor are both efficiently translated allowing for proper GSC cell cycle progression and its differentiation. However, when ribosome biogenesis is perturbed, we observe diminished translation of both ribosomal proteins and the p53 repressor. As a consequence, p53 is stabilized, cell cycle progression is blocked and GSC differentiation is stalled. Thus, our work reveals an elegant tuning mechanism that links ribosome biogenesis with a cell cycle progression checkpoint and thus stem cell differentiation. Given that ribosome biogenesis defects in humans result in ribosomopathies, which often result from stem cell differentiation defects, our data lay the foundation for understanding the etiology of developmental defects that arise due to ribosomopathies.

## Results

### Three conserved RNA helicases are required in the germline for GSC differentiation

We performed a screen to identify RNA helicases that are required for female fertility in *Drosophila*, and identified three predicted RNA helicases with previously uncharacterized functions, *CG5589*, *CG4901*, and *CG9253* (**Figure 2.1B-C**) (**Supplemental Table 2.1**) [@blattRNADegradationSculpts2020]. We named these candidate genes *aramis*, *athos*, and *porthos,* respectively, after Alexandre Dumas\' three musketeers who fought in service of their queen. We evaluated the efficiency of RNAi in ovaries using qPCR and found that *aramis*, *athos*, and *porthos* was significantly downregulated **(Figure 2.2A)**. We additionally drove RNAi of *aramis* and *athos* in the germline of flies expressing GFP::3XFLAG tagged versions of each gene respectively and performed immunostaining and found that the expression of Aramis and Athos was reduced specifically in the germline in contrast to the soma **(Figure 2.2B-E)**. To further investigate how these helicases promote fertility, we depleted *aramis*, *athos*, and *porthos* in the germline using the germline-driver *nanos*-*GAL4* (*nosGAL4*) in combination with RNAi lines. We detected the germline and spectrosomes/fusomes in ovaries by immunostaining for Vasa and 1B1, respectively. In contrast to controls, *aramis*, *athos*, and *porthos* germline RNAi flies lacked spectrosome-containing cells, and instead displayed cells with fusome-like structures proximal to the self-renewal niche **(Figure 2.1D-H; Figure 2.2E-E\'\'\'**). The cells in this cyst-like structurecontained ring canals, a marker of cytoplasmic bridges, suggesting that they are indeed interconnected (**Figure 2.2F-F\'\'\'**) [@Zhang2014d]. In addition to forming cysts in an aberrant location, the *aramis*, *athos*, and *porthos* germline RNAi ovaries failed to form egg chambers **(Figure 2.2G-G\'\'\')**.

```{r, fig.align="center", include=TRUE, out.width="100%", fig.align="center", fig.cap="(ref:RiBi-F2-1-short)", fig.scap="(ref:RiBi-F2-1-short)", echo=FALSE}
knitr::include_graphics("./figure/Ribosome Biogenesis/Ribosome Biogenesis 1.png")
```

\setstretch{1.0}

(ref:RiBi-F2-1-long)

\setstretch{2.0}
\setlength\parindent{24pt}

**\
**

Aberrant cyst formation proximal to the niche could reflect stem cysts with GSCs that divide to give rise to CBs but fail to undergo cytokinesis or differentiated cysts that initiate differentiation but cannot progress further to form egg chambers. To discern between these possibilities, first we examined the expression of a marker of GSCs, phosphorylated Mothers against decapentaplegic (pMad). We observed pMad expression in the cells closest to the niche, but not elsewhere in the germline cysts of *aramis*, *athos*, and *porthos* germline RNAi flies (**Figure 2.2H-H\')** [@kaiEmptyDrosophilaStem2003]. Additionally, none of the cells connected to the GSCs in *aramis*, *athos*, and *porthos* germline RNAi flies expressed the differentiation reporter *bamGFP* (**Figure 2.1D-G\'\')** [@McKearin1995b]. Thus, loss of *aramis*, *athos*, or *porthos* in the germline results in the formation of stem cysts, however with variable severity. This variability could be due to a differential requirement for these genes or different RNAi efficiencies. Overall, we infer that Aramis, Athos, and Porthos are required for proper GSC cytokinesis to produce a stem cell and differentiating daughter.

```{r, fig.align="center", include=TRUE, fig.fullwidth=TRUE, out.width="100%", fig.align="center", fig.cap="(ref:RiBi-F2-2-short)", fig.scap="(ref:RiBi-F2-2-short)", echo=FALSE}
knitr::include_graphics("./figure/Ribosome Biogenesis/Ribosome Biogenesis 1S.png")
```

\setstretch{1.0}

(ref:RiBi-F2-2-long)

\setstretch{2.0}
\setlength\parindent{24pt}

**\
**

### Athos, Aramis, and Porthos are required for ribosome biogenesis

We found that Aramis, Athos, and Porthos are conserved from yeast to humans (**Figure 2.1B**). The closest orthologs of Aramis, Athos, and Porthos are Rok1, Dhr2, and Rrp3 in yeast and DExD-Box Helicase 52 (DDX52), DEAH-Box Helicase 33 (DHX33), and DEAD-Box Helicase 47 (DDX47) in humans, respectively [@huIntegrativeApproachOrtholog2011]. Both the yeast and human orthologs have been implicated in rRNA biogenesis [@bohnsackQuantitativeAnalysisSnoRNA2008; @khoshnevisDEADboxProteinRok12016; @martinPreribosomalRNAInteraction2014; @oday8SRRNAProcessing1996; @sekiguchiNOP132RequiredProper2006; @Tafforeau2013a; @venemaProcessingPreribosomalRNA1995; @Venema1997; @vincentSSUProcessomeInteractome2017; @zhangIdentificationDHX33Mediator2011]. In addition, the GSC-cytokinesis defect that we observed in *aramis*, *athos*, and *porthos* RNAi flies is a hallmark of reduced ribosome biogenesis in the germline [@Sanchez2016h]. Based on these observations, we hypothesized that Aramis, Athos, and Porthos could enhance ribosome biogenesis to promote proper GSC differentiation.

Many factors involved in rRNA biogenesis localize to the nucleolus and interact with rRNA [@arabiCMycAssociatesRibosomal2005; @grandoriCMycBindsHuman2005; @Henras2008c; @Karpen1988a]. To detect the subcellular localization of Aramis and Athos, we used available lines that express Aramis::GFP::FLAG or Athos::GFP::FLAG fusion proteins under endogenous control. For Porthos, we expressed a Porthos::FLAG::HA fusion under the control of UASt promoter in the germline using a previously described approach [@delucaEfficientExpressionGenes2018]. We found that in the germline, Aramis, Athos and Porthos colocalized with Fibrillarin, which marks the nucleolus, the site of rRNA synthesis (**Figure 2.3A-C\'\'\'**) [@ochsFibrillarinNewProtein1985]. Aramis was also in the cytoplasm of the germline and somatic cells of the gonad. To determine if Aramis, Athos, and Porthos directly interact with rRNA, we performed immunoprecipitation (IP) followed by RNA-seq. We found that rRNA immunopurified with Aramis, Athos, and Porthos **(Figure 2.3D-D\'\', Figure 2.4A-A\'\')**. Thus, Aramis, Athos, and Porthos are present in thenucleolus and interact with rRNA, suggesting that they might regulate rRNA biogenesis.

```{r, fig.align="center", include=TRUE, out.width="100%", fig.align="center", fig.cap="(ref:RiBi-F2-3-short)", fig.scap="(ref:RiBi-F2-3-short)", echo=FALSE}
# (8.3/8.5)*6.5
knitr::include_graphics("./figure/Ribosome Biogenesis/Ribosome Biogenesis 2.png")
```

\setstretch{1.0}

(ref:RiBi-F2-3-long)

\setstretch{2.0}
\setlength\parindent{24pt}

**\
**

Nucleolar size, and in particular nucleolar hypotrophy, is associated with reduced ribosome biogenesis and nucleolar stress [@Neumuller2008; @zhangIdentificationDHX33Mediator2011]. If Aramis, Athos, and Porthos promote ribosome biogenesis, then their loss would be expected to cause nucleolar stress and a reduction in mature ribosomes. Indeed, immunostaining for Fibrillarin revealed hypotrophy of the nucleolus in *aramis, athos,* and *porthos* germline RNAi flies compared to in control flies, consistent with nucleolar stress (**Figure 2.3B-C\'**). Next, we used polysome profile analysis to evaluate the ribosomal subunit ratio and translation status of ribosomes in S2 cells depleted of *aramis*, *athos*, or *porthos* [@boamahPolyADPRibosePolymerase2012; @ounapHumanWBSCR22Protein2013]. We found that upon the depletion of all three helicases, the heights of the polysome peaks were reduced (**Figure 2.3E-E\'\'**). We found that depletion of *aramis* and *porthos* diminished the height of the 40S subunit peak compared to the 60S subunit peak, characteristic of defective 40S ribosomal subunit biogenesis (**Figure 2.3E, E\'\'**, **Figure 2.4D)** [@chengSmallLargeRibosomal2019], whereas *athos* depletion diminished the height of the 60S subunit peak compared to the 40S peaks, characteristic of a 60S ribosomal subunit biogenesis defect (**Figure 2.3E\', Figure 2.4D\'**) [@chengSmallLargeRibosomal2019]. Previous work indicates that the stem-cyst that arises from depletion of genes involved in ribosome biogenesis in the germline genetically interacts with Shrub (shrb) a member of the Escrt-III complex. To further determine if *aramis*, *athos*, and *porthos* regulates ribosome biogenesis, we performed trans-heterozygous crosses between *aramis*, *athos*, and *porthos* and *shrb*. We observed the presence of stem-cyst structures even heterozygotes mutants of *shrb*, consistent with previous observations [@Matias2015g; @Sanchez2016h], and in *aramis*, or *athos*, *porthos*. We found that in trans-heterozygous germaria of a *shrb* mutant our genes of interest result in a more frequent occurrence of stem-cysts than in their respective heterozygous background consistent with their role in ribosome biogenesis (**Figure S2.4E-L**). Taken together our findings indicate that these helicases promote ribosome biogenesis in *Drosophila*.

```{r, fig.align="center", include=TRUE, out.width="100%", fig.align="center", fig.cap="(ref:RiBi-F2-4-short)", fig.scap="(ref:RiBi-F2-4-short)", echo=FALSE}
knitr::include_graphics("./figure/Ribosome Biogenesis/Ribosome Biogenesis 2S.png")
```

\setstretch{1.0}

(ref:RiBi-F2-4-long)

\setstretch{2.0}
\setlength\parindent{24pt}

**\
**

### Aramis promotes cell cycle progression via p53 repression

Our data so far indicate that Aramis, Athos and Porthos promote ribosome biogenesis, which is known to be required for GSC abscission [@Sanchez2016h]. Yet the connections between ribosome biogenesis and GSC abscission are poorly understood. To explore the connection, we further examined the *aramis* germline RNAi line, as its defect was highly penetrant but maintained sufficient germline for analysis (**Figure 2.1E, H**). First, we compared the mRNA profiles of *aramis* germline RNAi ovaries to *bam* germline RNAi to determine if genes that are known to be involved in GSC abscission have altered expression. We used germline *bam* depletion as a control because it leads to the accumulation of CBs with no abscission defects [@Flora2018l; @gilboaGermLineStem2003; @McKearin1995b; @ohlsteinEctopicExpressionDrosophila1997], whereas loss of *aramis* resulted in accumulation of CBs that do not abscise from the GSCs.

We performed RNA-seq and found that 607 RNAs were downregulated and 673 RNAs were upregulated in *aramis* germline RNAi versus *bam* germline RNAi (cut-offs for differential gene expression were log~2~(foldchange) \>\|1.5\|, FDR \< 0.05) (**Figure 2.10A, Supplemental Table 2.2**). Gene Ontology (GO) analysis for biological processes on these genes encoding these differentially expressed mRNAs [@thomasPANTHERLibraryProtein2003] revealed that the genes that were downregulated upon *aramis* germline depletion were enriched for GO terms related to the cell cycle, whereas the upregulated genes were enriched for GO terms related to stress response (**Figure 2.5A, Figure 2.6B**). The downregulated genes included *Cyclin A*, which is required for cell cycle progression, *Cyclin B* (*CycB*) and *aurora B*, which are required for both cell cycle progression and cytokinesis; in contrast the housekeeping gene *Actin 5C* was unaffected (**Figure 2.5B-C, Figure 2.6C-C\'**) [@Mathieu2013d; @Matias2015g]. We confirmed that CycB was reduced in the ovaries of *aramis* germline RNAi flies compared to *bam* germline RNAi flies by immunofluorescence (**Figure 2.5D-F**). These changes to genes that promote cell cycle and cytokinesis were also seen *aramis; bam* double depletions as the Biological Process GO-terms we identified from targets downregulated in *aramis* RNAi were very similar to those we identified from in *bam* RNAi; *aramis* RNAi compared to their controls (**Figure 2.5A, Supplemental Table 2.3**). Similarly, all of the GO-terms we identified in from upregulated genes are also enriched GO terms from the double-knockdown upregulated targets (**Figure 2.6B, Supplemental Table 2.3**). Crucially, all the genes we refer to in the manuscript such as *CycB*, *AurB,* and *CycA* are also targets in *bam* RNAi; *aramis* RNAi. (**Figure 2.6B, Supplemental Table 2.3**). These results suggest that *aramis* is required for the proper expression of key regulators of GSC abscission.

CycB is expressed during G2 phase after asymmetric cell division to promote GSC abscission [@Flora2018l; @Mathieu2013d]. To test if the loss of germline *aramis* leads to GSC abscission defects due to diminished expression of CycB, we attempted to express a functional CycB::GFP fusion protein in the germline under the control of a UAS/GAL4 system (**Figure 2.6F-G\'**) [@Mathieu2013d]. Unexpectedly, the CycB::GFP fusion protein was not expressed in the *aramis-*depleted germline, unlike the wild type (WT) germline (**Figure 2.6F-G**) [@glotzerCyclinDegradedUbiquitin1991; @Mathieu2013d; @Zielke2014a]. We considered the possibility that progression into G2 is blocked in the absence of *aramis*, precluding expression of CycB. To monitor the cell cycle, we used the Fluorescence Ubiquitin-based Cell Cycle Indicator (FUCCI) system. *Drosophila* FUCCI utilizes a GFP-tagged degron from E2f1 to mark G2, M, and G1 phases and an RFP-tagged degron from CycB to mark S, G2, and M phases [@Zielke2014a]. We observed cells in different cell cycle stages in both WT and *bam*-depleted germaria, but the *aramis*-depleted germaria expressed neither GFP nor RFP (**Figure 2.6F-H\'\'**). Double negative reporter expression is thought to indicate early S phase, when expression of E2f1 is low and CycB is not expressed [@hinnantTemporalRemodelingCell2017]. The inability to express FPs is not due to a defect in translation as *aramis*-depleted germline can express GFP that is not tagged with the degron (**Figure 2.6K**). Taken together, we infer that loss of *aramis* blocks cell cycle progression around late G1 phase/early S phase and prevents progression to G2 phase, when GSCs abscise from CBs.

```{r, fig.align="center", include=TRUE, out.width="100%", fig.align="center", fig.cap="(ref:RiBi-F2-5-short)", fig.scap="(ref:RiBi-F2-5-short)", echo=FALSE}
knitr::include_graphics("./figure/Ribosome Biogenesis/Ribosome Biogenesis 3.png")
```

\setstretch{1.0}

(ref:RiBi-F2-5-long)

\setstretch{2.0}
\setlength\parindent{24pt}

**\
**

In mammals, cells defective for ribosome biogenesis stabilize p53, which is known to impede the G1 to S transition [@Agarwal1995a; @senturkP53CellCycle2013]. Therefore, we hypothesized that the reduced ribosome biogenesis in the *aramis*-depleted germline leads to p53 stabilization in undifferentiated cells, driving cell cycle arrest and GSC abscission defects. To test this hypothesis, we detected p53 and Vasa in the germline by immunostaining. A hybrid dysgenic cross that expresses p53 in undifferentiated cells was utilized as a positive control, and *p53* null flies were used as negative controls (**Figure 2.6L-M**) [@Moon2018a]. In WT, we observed p53 expression in the meiotic stages of germline but p53 expression in GSCs and CBs was attenuated as previously reported (**Figure 2.5G-G\'\'**) [@luMeioticRecombinationProvokes2010]. However, compared to WT GSCs/CBs, we observed p53 expression in the stem cysts of the *aramis-*depleted germline (**Figure 2.5G-I**). Similarly, we observed p53 expression in the stem cysts of *athos-* and *porthos-*depleted germlines (**Figure 2.6N-O**), further supporting that reduced ribosome biogenesis stabilizes p53. To determine if p53 stabilization is required for the cell cycle arrest in *aramis*, *athos*, and *porthos-*depleted germline cysts, we depleted *aramis*, *athos* and *porthos* in the germline of *p53* mutants using germline specific knockdown. We observed a partial but significant alleviation of the cyst phenotype, such that spectrosomes were restored (**Figure 2.5J-L, Figure 2.6P-T**). This finding indicates that p53 contributes to cytokinesis failure upon loss of *aramis*, but that additional factors are also involved.

To determine if p53 stabilization is required for the cell cycle arrest in *aramis-*, *athos-*, and *porthos-*depleted germline cysts, we depleted *aramis*, *athos* and *porthos* in the germline of *p53* mutants using germline specific knockdowns. We observed a partial but significant alleviation of the cyst phenotype, such that spectrosomes were restored (*Figure 2.5M-N*), This finding indicates that p53 contributes to cytokinesis failure upon loss of *aramis, athos* and *porthos* but that additional factors are also involved. To determine if aberrant expression of p53 is sufficient to cause the formation of stem-cysts, we overexpressed p53 in the germline under the control of a UAS/GAL4 system. While 84% of germaria had a complete loss of germline as previously reported [@bakhratDrosophilaChk2P532010], excitingly in 12% of germaria the germline cells were connected by a fusome-like structure, phenocopying loss of *aramis*, *athos*, or *porthos* (**Figure 3M-N**), and in the rest, we observed several single germline cells, compared to the control (n=55, Fisher\'s exact test, p\<0.001). Taken together, we find that *aramis-, athos-, and porthos-* depleted germ cells display reduced ribosome biogenesis, aberrant expression of p53 protein, and a block in cell cycle progression. Reducing p53 partially alleviates the cell cycle block and GSC cytokinesis defect, while inappropriate p53 expression results in loss of germline and cytokinesis defects in the GSCs., while inappropriate p53 expression results in loss of germline and cytokinesis defects.

```{r, fig.align="center", include=TRUE, out.width="100%", fig.align="center", fig.cap="(ref:RiBi-F2-6-short)", fig.scap="(ref:RiBi-F2-6-short)", echo=FALSE}
knitr::include_graphics("./figure/Ribosome Biogenesis/Ribosome Biogenesis 3S.png")
```

\setstretch{1.0}

(ref:RiBi-F2-6-long)

\setstretch{2.0}
\setlength\parindent{24pt}

**\
**

**Aramis promotes translation of Non1, a negative regulator of p53, linking ribosome biogenesis to the cell cycle**

Although p53 protein levels were elevated upon loss of *aramis* in the germline, *p53* mRNA levels were not significantly altered (log~2~ fold change: -0.49; FDR: 0.49). Given that ribosome biogenesis is affected, we considered that translation of p53 or one of its regulators was altered in *aramis*-depleted germlines. To test this hypothesis, we performed polysome-seq of gonads enriched for GSCs or CBs as developmental controls, as well as gonads depleted for *aramis* in the germline [@Flora2018k]. We plotted the ratios of polysome-associated RNAs to total RNAs (**Figure 2.7A-A\'\', Supplemental Table 2.4**) and identified 87 mRNAs with a reduced ratio upon depletion of *aramis*, suggesting that they were translated less efficiently compared to developmental controls. Loss of *aramis* reduced the levels of these 87 downregulated transcripts in polysomes, without significantly affecting their total mRNA levels (**Figure 2.7B, Figure 2.8A-A\'**). These 87 transcripts encode proteins mostly associated with translation including ribosomal proteins (**Figure 2.7C)**. To validate that Aramis regulates translation of these target mRNAs, we utilized a reporter line for the Aramis-regulated transcript encoding Ribosomal protein S2 (RpS2) that is expressed in the context of the endogenous promoter and regulatory sequences [@buszczakCarnegieProteinTrap2007; @Zhang2014d]. We observed reduced levels of RpS2::GFP in germlines depleted of *aramis* but not in those depleted of *bam* (**Figure 2.7D-F**). RpS2::GFP expression is also reduced in *bam* RNAi, *aramis* RNAi double germline knockdown compared to *bam* RNAi (**Figure 2.8B-D**). To ensure that reduced RpS2::GFP levels did not reflect a global decrease in translation, we visualized nascent translation using O-propargyl-puromycin (OPP). OPP is incorporated into nascent polypeptides and can be detected using click-chemistry [@Sanchez2016h]. We observed that global translation in the germlines of ovaries depleted of *aramis* was not reduced compared to single cells of control ovaries or *bam* **(Figure 2.7G-J)**. Notably, the regulation of these genes do not appear to be directly mediated by Aramis as none of the target genes are enriched from Aramis::GFP::3XFLAG RNA IP-seq (**Supplemental Table 2.5**). Thus, loss of *aramis* results in reduced translation of a subset of transcripts compared to the rest of the transcriptome.

```{r, fig.align="center", include=TRUE, out.width="100%", fig.align="center", fig.cap="(ref:RiBi-F2-7-short)", fig.scap="(ref:RiBi-F2-7-short)", echo=FALSE}
knitr::include_graphics("./figure/Ribosome Biogenesis/Ribosome Biogenesis 4.png")
```

\setstretch{1.0}

(ref:RiBi-F2-7-long)

\setstretch{2.0}
\setlength\parindent{24pt}

**\
**

Previous work has indicated that excess ribosomal protein expression may be deleterious to cellular functions and induce cell stress. We hypothesized that the decrease in translation we observe may be the result of a regulatory mechanism to mitigate such affects. To test if overexpression of ribosomal proteins is deleterious to germline development, we overexpressed several ribosomal proteins in the germline. We found no obvious phenotype as a result of overexpression of single ribosomal proteins (**Figure 2.8E-G\'**). We reason that individual overexpression may not be as detrimental as the excess expression of nearly all ribosomal proteins that we observed in our experiments, which is not technically possible to recapitulate through exogenous expression lines.

```{r, fig.align="center", include=TRUE, out.width="100%", fig.align="center", fig.cap="(ref:RiBi-F2-8-short)", fig.scap="(ref:RiBi-F2-8-short)", echo=FALSE}
knitr::include_graphics("./figure/Ribosome Biogenesis/Ribosome Biogenesis 4S.png")
```

\setstretch{1.0}

(ref:RiBi-F2-8-long)

\setstretch{2.0}
\setlength\parindent{24pt}
**\
**

None of these 87 translational targets have been implicated in directly controlling abscission [@Mathieu2013d; @Matias2015g]. However, we noticed that the mRNA encoding Novel Nucleolar protein 1 (Non1/CG8801) was reduced in polysomes upon loss of *aramis* in the germline (**Figure 2.7C**). The human ortholog of Non1 is GTP Binding Protein 4 (GTPBP4), and these proteins are known to physically interact with p53 in both *Drosophila* and human cells and have been implicated in repressing p53 (mentioned as CG8801 in Lunardi et al.) [@Li2018a; @Lunardi2010a]. To determine if the protein level of Non1 is reduced upon depletion of *aramis,* we monitored the abundance of Non1::GFP, a transgene that is under endogenous control [@sarovGenomewideResourceAnalysis2016], and found that Non1::GFP was expressed in the undifferentiated GSCs and CBs (**Figure 2.9A-A\'\')**. Non1::GFP levels were reduced in the *aramis*, *athos* or *porthos-*depleted stem cysts compared to the CBs that accumulated upon *bam*-depletion (**Figure 2.9B-D, Figure 2.10C-F**), suggesting that efficient ribosome biogenesis promotes efficient translation of Non1.

During normal oogenesis, p53 protein is expressed in cyst stages in response to recombination-induced double strand breaks [@luMeioticRecombinationProvokes2010]. We found that Non1 was highly expressed at undifferentiated stages and in two- and four-cell cysts when p53 protein levels were low, whereas its expression was attenuated at eight- and 16-cell cyst stages when p53 protein levels were high (**Figure 2.9A-A\'\'**, **Figure 2.10A-B\'**). Non1 was highly expressed in egg chambers, which express low levels of p53 protein suggesting that Non1 could regulate p53 protein levels. To determine if Non1 regulates GSC differentiation and p53, we depleted *Non1* in the germline. We found that germline-depletion of *Non1* results in stem cyst formation and loss of later stages, as well as increased p53 expression, phenocopying germline-depletion of *aramis*, *athos*, and *porthos* (**Figure 2.9E-F, H, Figure 2.10G-I**). In addition, we found that loss of *p53* from *Non1-*depleted germaria partially suppressed the phenotype (**Figure 5F-H**). Thus, *Non1* is regulated by *aramis* and is required for p53 suppression, cell cycle progression, and GSC abscission.

```{r, fig.align="center", include=TRUE, out.width="100%", fig.align="center", fig.cap="(ref:RiBi-F2-9-short)", fig.scap="(ref:RiBi-F2-9-short)", echo=FALSE}
knitr::include_graphics("./figure/Ribosome Biogenesis/Ribosome Biogenesis 5.png")
```

\setstretch{1.0}

(ref:RiBi-F2-9-long)

\setstretch{2.0}
\setlength\parindent{24pt}

**\
**

To determine if Aramis, Athos, and Porthos promotes GSC differentiation via translation of Non1, we restored *Non1* expression in germ cells depleted of *aramis*, *athos*, or *porthos*. Briefly, we cloned *Non1* with heterologous UTR elements under the control of the UAS/GAL4 system (see Methods) [@rorthGal4DrosophilaFemale1998]. We found that restoring *Non1* expression in the *aramis*, *athos*, or *porthos -*depleted germline significantly attenuated the stem cysts and increased the number of cells with spectrosomes, however overexpression of Non1 alone did not cause any noticeable defect (**Figure 2.9I-K, Figure 2.10J-N**). Taken together, we conclude that Non1 can partially suppress the cytokinesis defect caused by germline *aramis* depletion.

```{r, fig.align="center", include=TRUE, out.width="100%", fig.align="center", fig.cap="(ref:RiBi-F2-10-short)", fig.scap="(ref:RiBi-F2-10-short)", echo=FALSE}
knitr::include_graphics("./figure/Ribosome Biogenesis/Ribosome Biogenesis 5S.png")
```

\setstretch{1.0}

(ref:RiBi-F2-10-long)

\setstretch{2.0}
\setlength\parindent{24pt}

**\
**

### Aramis-regulated targets contain a TOP motif in their 5\'UTR

We next asked how *aramis* and efficient ribosome biogenesis promote the translation of a subset of mRNAs, including *Non1*, to regulate GSC differentiation. We hypothesized that the 87 mRNA targets share a property that make them sensitive to rRNA and ribosome levels. To identify shared characteristics, we performed *de novo* motif discovery of target genes compared to non-target genes [@Bailey2006a] and identified a polypyrimidine motif in the 5\'UTRs of most target genes (UCUUU; E-value: 6.6e^-094^). This motif resembles the previously described TOP motif at the 5\' end of mammalian transcripts [@philippeLarelatedProteinLARP12018; @thoreenUnifyingModelMTORC1mediated2012]. Although the existence of TOP-containing mRNAs in *Drosophila* has been speculated, to our knowledge their presence has not been explicitly demonstrated [@chenComprehensiveAnalysisNucleocytoplasmic2017; @qinGlobalAnalysesMRNA2007a]. This observation motivated us to precisely determine the 5\' end of transcripts, so we analyzed previously published Cap Analysis of Gene Expression sequencing (CAGE-seq) data that had determined transcription start sites (TSS) in total mRNA from the ovary (**Figure 6A**) [@boleyNavigatingMiningModENCODE2014; @chenComparativeValidationMelanogaster2014; @dossantosFlyBaseIntroductionDrosophila2015]. Of the 87 target genes, 76 had sufficient expression in the CAGE-seq dataset to define their TSS. We performed motif discovery using the CAGE-seq data and found that 72 of 76 Aramis-regulated mRNAs have a polypyrimidine motif that starts within the first 50 nt of their TSS (**Figure 2.11B-C**). In mammals, it was previously thought that the canonical TOP motif begins with an invariant \'C\' [@meyuhasSynthesisTranslationalApparatus2000; @philippeGlobalAnalysisLARP12020]. However, systematic characterization of the sequence required in order for an mRNA to be regulated as a TOP containing mRNA revealed that TOP mRNAs can start with either a \'C\' or a \'U\' [@philippeGlobalAnalysisLARP12020]. Thus, mRNAs whose efficient translation is dependent on *aramis* share a terminal polypyrimidine-rich motif in their 5\'UTR that resembles a TOP motif.

In vertebrates, most canonical TOP-regulated mRNAs encode ribosomal proteins and translation initiation factors that are coordinately upregulated in response to growth cues mediated by the Target of Rapamycin (TOR) pathway and the TOR complex 1 (mTORC1) [@Hornstein2001a; @iadevaiaMTORC1SignalingControls2014; @Kim2008b; @meyuhasRaceDecipherTop2015; @Pallares-Cartes2012a] Indeed, 76 of the 87 Aramis targets were ribosomal proteins, and 9 were known or putative translation factors, consistent with TOP-containing RNAs in vertebrates (**Figure 2.7C, Supplemental Table 2.6**). To determine if the putative TOP motifs that we identified are sensitive to TORC1 activity, we designed "TOP reporter" constructs. Specifically, the germline-specific *nanos* promoter was employed to drive expression of an mRNA with 1) the 5\'UTR of the *aramis* target *RpL30*, which contains a putative TOP motif, 2) the coding sequence for a GFP-HA fusion protein and 3) a 3\'UTR (K10) that is not translationally repressed [@Flora2018k; @seranoElementTransformationVectors1994], referred to as the WT-TOP reporter (**Figure 2.11D**). As a control, we created a construct in which the polypyrimidine sequence was mutated to a polypurine sequence referred to as the Mut-TOP reporter (**Figure 2.11D**).

In *Drosophila*, TORC1 activity increases during meiotic cyst stages [@Wei2018a; @Wei2014b]. We found that the WT-TOP reporter displayed peak expression in 8-cell cysts, whereas the Mutant-TOP reporter did not **(Figure 2.11E-F\'\')**, suggesting that the WT-TOP reporter is sensitive to TORC1 activity. Moreover, depletion of *Nitrogen permease regulator-like 3* (*Nprl3*), an inhibitor of TORC1 [@Wei2014b], led to a significant increase in expression of the WT-TOP reporter but not the Mutant-TOP reporter **(Figure 2.12A-E)**. Additionally, to attenuate TORC1 activity, we depleted *raptor*, one of the subunits of the TORC1 complex [@hongEvaluationNutrientSensingMTOR2012; @loewithTargetRapamycinTOR2011]. Here we found that the WT-TOP reporter had a significant decrease in reporter expression while the Mutant-TOP reporter did not show a decrease in expression **(Figure 2.12F-J)**. Taken together, our data suggest that Aramis-target transcripts contain TOP motifs that are sensitive to TORC1 activity. However, we note that our TOP reporter did not recapitulate the pattern of Non1::GFP expression, suggesting that Non1 may have additional regulators that modulate its protein levels in the cyst stages.

```{r, fig.align="center", include=TRUE, out.width="100%", fig.align="center", fig.cap="(ref:RiBi-F2-11-short)", fig.scap="(ref:RiBi-F2-11-short)", echo=FALSE}
knitr::include_graphics("./figure/Ribosome Biogenesis/Ribosome Biogenesis 6.png")
```

\setstretch{1.0}

(ref:RiBi-F2-11-long)

\setstretch{2.0}
\setlength\parindent{24pt}

**\
**

TOP mRNAs show increased translation in response to TOR signaling, leading to increased ribosome biogenesis [@jefferiesRapamycinSuppressesTOP1997; @jiaMTORC1PromotesTOP2021; @powersRegulationRibosomeBiogenesis1999; @thoreenUnifyingModelMTORC1mediated2012]. However, to our knowledge, whether reduced ribosome biogenesis can coordinately diminish the translation of TOP mRNAs to balance and lower ribosome protein production and thus balance the levels of the distinct components needed for full ribosome assembly is not known. To address this question, we crossed the transgenic flies carrying the WT-TOP reporter and Mutant-TOP reporter into *bam* and *aramis*, *athos*, and *porthos* germline RNAi backgrounds. We found that the expression from the WT-TOP reporter was reduced by 2.9-fold in the germline of *aramis* RNAi ovaries compared to *bam* RNAi ovaries **(Figure 2.11F-G, J)**. In contrast, the Mutant-TOP reporter was only reduced by 1.6-fold in the germline of *aramis* RNAi ovaries compared to *bam* RNAi ovaries **(Figure 2.11H-J)**. We observed the same trend for *athos* and *porthos* (**Figure 2.12K-Q**). This suggests that the TOP motif-containing mRNAs are sensitive to ribosome biogenesis.

```{r, fig.align="center", include=TRUE, out.width="100%", fig.align="center", fig.cap="(ref:RiBi-F2-12-short)", fig.scap="(ref:RiBi-F2-12-short)", echo=FALSE}
knitr::include_graphics("./figure/Ribosome Biogenesis/Ribosome Biogenesis 6S.png")
```

\setstretch{1.0}

(ref:RiBi-F2-12-long)

\setstretch{2.0}
\setlength\parindent{24pt}

**\
**

### Larp binds TOP sequences in *Drosophila*

Next, we sought to determine how TOP-containing mRNAs are regulated downstream of Aramis. In mammalian cells, Larp1 is a critical negative regulator of TOP-containing RNAs during nutrient deprivation [@bermanControversiesFunctionLARP12020; @Fonseca2015a; @Hong2017a; @philippeGlobalAnalysisLARP12020; @Tcherkezian2014b]. Therefore, we hypothesized that *Drosophila* Larp reduces the translation of TOP-containing mRNAs when rRNA biogenesis is reduced upon loss of *aramis*. First, using an available gene-trap line in which endogenous Larp is tagged with GFP and 3XFLAG, we confirmed that Larp was robustly expressed throughout all stages of oogenesis including in GSCs **(Figure 2.14A-A\')**.

Next, we performed electrophoretic mobility shift assays (EMSA) to examine protein-RNA interactions with purified *Drosophila* Larp-DM15, the conserved domain that binds to TOP sequences in vertebrates [@Lahr2017b]. As probes, we utilized capped 42-nt RNAs corresponding to the 5\'UTRs of *RpL30* and *Non1*, including their respective TOP sequences. We observed a gel shift with these RNA oligos in the presence of increasing concentrations of Larp-DM15 **(Figure 2.13A-A\', Figure 2.14B)**, and this shift was abrogated when the TOP sequences were mutated to purines **(Figure 2.14C-C\').** To determine if Larp interacts with TOP-containing mRNAs *in vivo*, we immunopurified Larp::GFP::3XFLAG from the ovaries of the gene-trap line and performed RNA-seq (**Figure 2.14D**). We uncovered 156 mRNAs that were bound to Larp, and 84 of these were among the 87 *aramis* translationally regulated targets, including *Non1*, *RpL30*, and *RpS2* (**Figure 2.13B-C, Supplemental Table 2.7**). Thus, *Drosophila* Larp binds to TOP sequences *in vitro* and TOP-containing mRNAs *in vivo*.

```{r, fig.align="center", include=TRUE, out.width="100%", fig.align="center", fig.cap="(ref:RiBi-F2-13-short)", fig.scap="(ref:RiBi-F2-13-short)", echo=FALSE}
knitr::include_graphics("./figure/Ribosome Biogenesis/Ribosome Biogenesis 7.png")
```

\setstretch{1.0}

(ref:RiBi-F2-13-long)

\setstretch{2.0}
\setlength\parindent{24pt}

**\
**

To test our hypothesis that *Drosophila* Larp inhibits the translation of TOP-containing mRNAs upon loss of *aramis*, we immunopurified Larp::GFP::3XFLAG from germline *bam* RNAi ovaries and germline *aramis* RNAi ovaries. Larp protein is not expressed at higher levels in *aramis* RNAi compared to developmental control *bam* RNAi (**Figure 2.14E-G**). Larp protein is also not expressed at higher levels in *bam* RNAi; *aramis* RNAi germline knockdown compared to *bam* RNAi as a control (**Figure 2.14H-I**). We found that Larp binding to *aramis* target mRNAs *Non1* and *RpL30* was increased in *aramis* RNAi ovaries compared to *bam* RNAi ovaries (**Figure 2.13D, Figure 2.14J**). In contrast, a non-target mRNA that does not contain a TOP motif, *alpha-tubulin* mRNA, did not have a significant increase in binding to Larp in *aramis* RNAi ovaries compared to *bam* RNAi ovaries. Overall, these data suggest that reduced rRNA biogenesis upon loss of *aramis* increases Larp binding to the TOP-containing mRNAs *Non1* and *RpL30*.

If loss of *aramis* inhibits the translation of TOP-containing mRNAs due to increased binding of Larp to its targets, then overexpression of Larp would be expected to phenocopy germline depletion of *aramis*. Unphosphorylated Larp binds to TOP motifs more efficiently, but the precise phosphorylation sites of *Drosophila* Larp, to our knowledge, are currently unknown [@Hong2017a]. To circumvent this issue, we overexpressed the DM15 domain of Larp which we showed binds the *RpL30* and *Non1* TOP motifs *in vitro* (**Figure 2.13A-A\'**), and, based on homology to mammalian Larp1, lacks majority of the putative phosphorylation sites [@jiaMTORC1PromotesTOP2021; @Lahr2017b; @philippeLarelatedProteinLARP12018]. We found that overexpression of a Larp-DM15::GFP fusion in the germline resulted in fusome-like structures extending from the niche (**Figure 2.13E-F\'**)**.** Additionally, ovaries overexpressing Larp-DM15 had 32-cell egg chambers, which were not observed in control ovaries (**Figure 2.14K-K\'**). The presence of 32-cell egg chambers is emblematic of cytokinesis defects that occur during early oogenesis [@Mathieu2013d; @Matias2015g; @Sanchez2016h]. Our findings indicate that these cells are delayed in cytokinesis and that over expression of Larp partially phenocopies depletion of *aramis*.

```{r, fig.align="center", include=TRUE, out.width="100%", fig.align="center", fig.cap="(ref:RiBi-F2-14-short)", fig.scap="(ref:RiBi-F2-14-short)", echo=FALSE}
knitr::include_graphics("./figure/Ribosome Biogenesis/Ribosome Biogenesis 7S.png")
```

\setstretch{1.0}

(ref:RiBi-F2-14-long)

\setstretch{2.0}
\setlength\parindent{24pt}

**\
**

## Discussion

During *Drosophila* oogenesis, efficient ribosome biogenesis is required in the germline for proper GSC cytokinesis and differentiation. The outstanding questions that needed to be addressed were: 1) Why does disrupted ribosome biogenesis impair GSC abscission and differentiation? And 2) How does the GSC monitor and couple ribosome abundance to differentiation? Our results suggest that a germline ribosome biogenesis defect stalls the cell cycle, resulting a loss of differentiation and the formation of stem-cysts. We discovered that proper ribosome biogenesis is monitored through a translation control module that allows for co-regulation of ribosomal proteins and a p53 repressor. Loss of *aramis*, *athos* and *porthos* reduces ribosome biogenesis and inhibits translation of a p53 repressor, leading to p53 stabilization, cell cycle arrest and loss of stem cell differentiation (**Figure 2.13G**).

### Aramis, Athos, and Porthos are required for efficient ribosome biogenesis in *Drosophila*

We provide evidence that Aramis, Athos and Porthos play a role in ribosome biogenesis in *Drosophila*, similar to their orthologs in yeast [@bohnsackQuantitativeAnalysisSnoRNA2008; @Granneman2006; @khoshnevisDEADboxProteinRok12016; @oday8SRRNAProcessing1996] and mammals [@sekiguchiNOP132RequiredProper2006; @Tafforeau2013a; @zhangIdentificationDHX33Mediator2011]. Their role in ribosome biogenesis is likely a direct function of these helicases as they physically interact with precursor rRNA. In yeast, Rok1, the ortholog of Aramis, binds to several sites on pre-rRNA, predominantly in the 18S region [@bohnsackQuantitativeAnalysisSnoRNA2008; @khoshnevisDEADboxProteinRok12016; @martinPreribosomalRNAInteraction2014]. This is consistent with the small subunit ribosome biogenesis defect we observe upon loss of *aramis* in *Drosophila*. Rrp3, the yeast ortholog of Porthos, promotes proper cleavage of pre-rRNA and is required for proper 18S rRNA production [@Granneman2006; @oday8SRRNAProcessing1996]. DDX47, the mammalian ortholog of Porthos, binds to early rRNA precursors as well as proteins involved in ribosome biogenesis [@sekiguchiNOP132RequiredProper2006]. Consistent with these findings, we find that Aramis and Porthos promote 40S ribosome biogenesis. DHX33, the mammalian ortholog of Athos, has been implicated in facilitating rRNA synthesis [@zhangIdentificationDHX33Mediator2011]. In contrast, we find that Athos promotes 60S ribosome biogenesis by directly interacting with rRNA. However, we cannot exclude the possibility that Athos also affects transcription of rRNA in *Drosophila* as it does in mammals [@zhangIdentificationDHX33Mediator2011]. Overall, we find that each mammalian ortholog of Aramis, Athos, and Porthos has consistent ribosome subunit defects, suggesting that the function of these helicases is conserved from flies to mammals. Intriguingly, DDX52 (Aramis) is one of the 15 genes deleted in 17q12 syndrome [@hendrixPrenatallyDiagnosed17q122012]. 17q12 syndrome results in delayed development, intellectual disability, and, more rarely, underdevelopment of organs such as the uterus [@bernardiniRecurrentMicrodeletion17q122009; @hendrixPrenatallyDiagnosed17q122012]. Our finding that Aramis disrupts stem cell differentiation could explain some of the poorly understood defects in 17q12 syndrome.

### Ribosome biogenesis defects leads to cell cycle defects mediated by p53

Here we report that three RNA helicases, *aramis*, *athos*, and *porthos,* that promote proper ribosome biogenesis in *Drosophila* are required in the germline for fertility. Loss of *aramis*, *athos*, and *porthos* causes formation of a "stem-cyst" and loss of later stage oocytes. Stem-cysts are a characteristic manifestation of ribosome biogenesis deficiency wherein GSCs are unable to complete cytokinesis and fail to express the differentiation factor Bam, which in GSC daughters is initiated at G2 of the cell cycle [@Sanchez2016h; @Zhang2014d]. Our RNA seq and cell cycle analysis indicates that depletion of *aramis* blocks the cell cycle at G1, and that failure to progress to G2 prevents abscission and expression of Bam. Thus, our results suggest that ribosome biogenesis defects in the germline stall the cell cycle, resulting in formation of stem-cysts and sterility.

In most tissues in *Drosophila,* p53 primarily activates apoptosis, however, in the germline p53 is activated during meiosis and does not cause cell death [@fanDualRolesDrosophila2010; @luMeioticRecombinationProvokes2010]. Furthermore, p53 activation in the germline is required for germline repopulation and GSC survival after genetic insult, implicating p53 as a potential cell cycle regulator [@maDNADamageinducedLok2016; @Tasnim2018a]. Our observation that reduction of *p53* partially rescues a stem-cyst defect caused by ribosome deficiency due to germline depletion of *aramis* indicates that the G1 block in GSCs is, in part, mediated by p53 activation. Thus, in *Drosophila* GSCs, p53 blocks the GSC cell cycle and is sensitive to ribosome biogenesis. Furthermore, while overexpression of p53 causes germline death, it is also sufficient to induce the formation of stem-like cysts demonstrating p53 plays a key regulatory role in GSC cell cycle. The developmental upregulation of p53 during GSC differentiation concomitant with lower ribosome levels parallels observations in disease states, such as ribosomopathies [@Calo2018a; @Deisenroth2010e; @Pereboom2011a; @Yelick2015a].

We find that p53 levels in GSCs are regulated by the conserved p53 regulator Non1. In mammalian cells, increased free RpS7 protein due to nucleolar stress binds and sequesters a repressor of p53, MDM2, freeing p53, resulting in G1 cell cycle arrest [@Deisenroth2010e; @zhangSignalingP53Ribosomal2009]. *Drosophila* have no identified homolog to MDM2. It is not fully known how ribosome levels are monitored in *Drosophila* in the absence of MDM2 and how this contributes to cell cycle progression. In *Drosophila,* Non1 levels are high in the GSCs and p53 is low, and reciprocally Non1 levels are low during meiosis, but p53 is expressed. Our finding that loss of Aramis leads to diminished Non1 and elevated p53, and that either loss of p53 or elevated Non1 suppress differentiation defects caused by loss of Aramis, suggests that, in the female germline, Non1 may fulfill the function of Mdm2 by promoting p53 degradation during *Drosophila* oogenesis. While Non1 has been shown to directly interact with p53, how it regulates p53 levels in both humans and *Drosophila* is not known [@Li2018a; @Lunardi2010a]. Overall, our data place Non1 downstream of ribosome biogenesis and upstream of p53 in controlling cell cycle progression and GSC differentiation. However, our data do not rule out that Non1 may also act upstream of or in parallel to Aramis.

The vertebrate ortholog of Non1, GTPBP4, also controls p53 levels and is upregulated in some cancers [@Li2018a; @Lunardi2010a; @yuUpregulationGTPBP4Colorectal2016]. This suggests that there may be parallel pathways for monitoring ribosome levels via p53 in different tissue types. Unlike *Drosophila* Non1, its ortholog, GTPBP4 has not been identified as a TOP mRNA, so if it similarly acts as a mediator between ribosome biogenesis and the cell cycle it is likely activated in a somewhat different manner [@philippeGlobalAnalysisLARP12020]. However, mammalian Larp1 is required for proper cell cycle progression and cytokinesis [@burrowsRNABindingProtein2010; @Tcherkezian2014b]. Excitingly several differentiation and cell cycle regulation genes in mammals are TOP mRNAs regulated by Larp1, including Tumor Protein, Translationally-Controlled 1 (TPT1) and Nucleosome Assembly Protein 1 Like 1 (NAP1L1) [@philippeGlobalAnalysisLARP12020]. TPT1 is a cancer associated factor that has been implicated in activating pluripotency [@koziolTpt1ActivatesTranscription2007]. Similarly, NAP1L1, a nucleosome assembly protein, is required to maintain proper cell cycle control as loss of NAP1L1 results in cell cycle exit and premature differentiation [@qiaoNap1l1ControlsEmbryonic2018]. Overall, although the specific targets of Larp1 in mammals may differ from those in *Drosophila*, the mechanism by which Larp modulates cell cycle and differentiation may be conserved.

### Ribosome biogenesis defects leads to repression of TOP-containing mRNA

TOP-containing mRNAs are known to be coregulated to coordinate ribosome production in response to nutrition or other environmental cues [@kimballRegulationGlobalSpecific2002; @meyuhasRaceDecipherTop2015; @tangAminoAcidInducedTranslation2001]. Surprisingly, our observation that loss of *aramis* reduces translation, albeit indirectly via regulation of ribosome biogenesis, of a cohort of TOP-containing mRNAs, including Non1, suggests that the TOP motif also sensitizes their translation to lowered levels of ribosome biogenesis. This notion is supported by TOP reporter assays demonstrating that reduced translation upon loss of *aramis* requires the TOP motif. We hypothesize that limiting TOP mRNA translation lowers ribosomal protein production to maintain a balance with reduced rRNA production. This feedback mechanism would prevent the production of excess ribosomal proteins that cannot be integrated into ribosomes and the ensuing harmful aggregates [@tyeProteotoxicityAberrantRibosome2019]. Additionally, it would coordinate rRNA production and ribosomal protein translation during normal germline development, where it is known that the level of ribosome biogenesis and of global translation are dynamic [@blattPosttranscriptionalGeneRegulation2020; @Fichelson2009a; @Sanchez2016h; @Zhang2014d].

### Larp transduces growth status to ribosome biogenesis targets

Recent work has shown that the translation and stability of TOP-containing mRNAs are mediated by Larp1 and its phosphorylation [@bermanControversiesFunctionLARP12020; @Hong2017a; @jiaMTORC1PromotesTOP2021]. We found that perturbing rRNA production and thus ribosome biogenesis, without directly targeting ribosomal proteins, similarly results in dysregulation of TOP mRNAs. Our data show that *Drosophila* Larp binds the *RpL30* and *Non1* 5\'UTR in a TOP-dependent manner *in vitro* and to 100% of the translation targets we identified *in vivo.* Together these data suggest that rRNA production regulates TOP mRNAs via Larp. Furthermore, the cytokinesis defect caused by overexpression of Larp-DM15 in the germline suggests that Larp regulation could maintain the homeostasis of ribosome biogenesis more broadly by balancing the expression of ribosomal protein production with the rate of other aspects of ribosome biogenesis, such as rRNA processing, during development.

Previous studies indicate that unphosphorylated Larp1 binds to and represses its targets more efficiently than phosphorylated Larp1 [@fonsecaLARP1MajorPhosphorylation2018; @Hong2017a; @jiaMTORC1PromotesTOP2021]. In mammalian systems Larp1 has been shown to be phosphorylated by the TORC1 complex, AKT, and CDK1 [@bermanControversiesFunctionLARP12020; @Hong2017a; @jiaMTORC1PromotesTOP2021]. In *Drosophila*, it has been shown that Pink1 can phosphorylates Larp, and the Pink1 dependent phosphorylation sites have been identified in *Drosophila* Larp [@zhangPINK1InhibitsLocal2019]. However, to our knowledge, Larp phosphorylation sites have not been systematically catalogued, nor has it been studied if TORC1, AKT, or CDK1 phosphorylate Larp in *Drosophila*. We have demonstrated that expression of our TOP-reporter is dependent on Raptor and TOP-reporter expression is repressed by Nprl3 (**Figure 2.12A-J**). These results suggest a model where TORC1 either directly or indirectly monitors ribosome biogenesis status by regulating the activity of Larp. Thus, although we do not know the identity of the kinase that phosphorylates Larp in *Drosophila* definitively, we hypothesize that Larp is not phosphorylated upon loss of *aramis, athos* and *porthos*, when ribosome biogenesis is perturbed. We propose that until ribosome biogenesis homeostasis is reached, this kinase will remain inactive, continuously increasing the pool of dephosphorylated Larp. In this scenario, as dephosphorylated Larp accumulates, it begins to bind its targets. Initially, it will bind its highest affinity targets, presumably encoding ribosomal proteins and repress their translation to rebalance ribosomal protein production with rRNA production. Consistent with this model, the TOP motif in *RpL30* is bound by Larp even more tightly with a nearly 9-fold higher affinity compared to the *Non1* TOP site (**Figure 2.14B**). We propose that such differences in affinity may allow Larp to repress ribosomal protein translation to facilitate cellular homeostasis without immediately causing cell cycle arrest. However, if homeostasis cannot be achieved and sufficient dephosphorylated Larp accumulates, Larp will also bind and repress the translation of lower affinity targets. Repression of Non1 in this manner would result in cell cycle arrest and block differentiation as occurs upon *aramis* depletion.

### Ribosome biogenesis in stem cell differentiation and ribosomopathies

Ribosomopathies arise from defects in ribosomal components or ribosome biogenesis and include a number of diseases such as Diamond-Blackfan anemia, Treacher Collins syndrome, Shwachman-Diamond syndrome, and 5q-myelodysplastic syndrome [@Armistead2014a; @draptchinskaiaGeneEncodingRibosomal1999; @mcgowanReducedRibosomalProtein2011; @valdezTreacherCollinsSyndrome2004; @warrenMolecularBasisHuman2018]. Despite the ubiquitous requirement for ribosomes and translation, ribosomopathies cause tissue-specific disease [@Armistead2014a]. The underlying mechanisms of tissue specificity remain unresolved.

In this study we demonstrate that loss of helicases involved in rRNA processing lead to perturbed ribosome biogenesis and, ultimately, cell cycle arrest. Given that *Drosophila* germ cells undergo an atypical cell cycle program as a normal part of their development it may be that this underlying cellular program in the germline leads to the tissue-specific symptom of aberrant stem-cyst formation [@McKearin1990e]. This model implies that other tissues would likewise exhibit unique tissue-specific manifestations of ribosomopathies due to their underlying cell state and underscores the need to further explore tissue-specific differentiation programs and development to shed light not only on ribosomopathies but also on other tissue-specific diseases associated with ubiquitous processes. Although it is also possible that phenotypic differences arise from a common molecular cause, our data suggests two sources of potential tissue specificity: 1) tissues express different cohorts of mRNAs, such as *Non1*, that are sensitive to ribosome levels. For example, we find that in *Drosophila* macrophages, RNAs that regulate the metabolic state of macrophages and influence their migration require increased levels of ribosomes for their translation [@emtenaniGeneticProgramBoosts2021]. 2) p53 activation, as has been previously described, is differentially tolerated in different developing tissues [@bowenRoleP53Developmental2019; @Calo2018a; @jonesPreventionNeurocristopathyTreacher2008]. Together, both mechanisms could begin to explain the tissue-specific nature of ribosomopathies and their link to differentiation.

**Acknowledgements**

We are grateful to all members of the Rangan and Fuchs labs for their discussion and comments on the manuscript. We also thanks Dr. Sammons, Dr. Marlow and Life Science Editors for their thoughts and comments the manuscript Additionally, we thank the Bloomington Stock Center, the Vienna *Drosophila* Resource Center, the BDGP Gene Disruption Project, and Flybase for fly stocks, reagents, and other resources. P.R. is funded by the NIH/NIGMS (R01GM111779-06 and RO1GM135628-01), G.F. is funded by NSF MCB-2047629 and NIH RO3 AI144839, D.E.S. was funded by Marie Curie CIG 334077/IRTIM and the Austrian Science Fund (FWF) grant ASI_FWF01_P29638S, and A.B is funded by NIH R01GM116889 and American Cancer Society RSG-17-197-01-RMC.

**Author Contributions**

Conceptualization, E.T.M., P.B., G.F., and P.R.; Methodology, E.T.M., P.B., G.F., and P.R.; Investigation, E.T.M., P.B., E.N., R.L., S.S., H.Y., T.P., and S.E.; Writing -- Original Draft, E.T.M., D.E.S., and P.R.; Writing -- Review & Editing, E.T.M., P.B., D.E.S, A.B., G.F., and P.R.; Funding Acquisition, G.F. and P.R.; Visualization, E.T.M., E.N.; Supervision, G.F. and P.R.

**\
**

Supplemental Tables can be found along with the original publication here: XXX\

**Supplemental Table 2.1. Results of germline helicase RNAi screen on ovariole morphology.**
Results of screen of RNA helicases depleted from the germline. Reported is the majority phenotype from n=50 ovarioles.

**\
**

**Supplemental Table 2.2. Differential expression analysis from RNAseq of ovaries depleted of *aramis* in the germline compared to a developmental control.** 
DEseq2 output from RNAseq of ovaries depleted of *aramis* in the germline compared to ovaries depleted of bam in the germline as a developmental control. Sheet 1 (Downregulated Genes) contains genes and corresponding Deseq2 output meeting the cutoffs to be considered downregulated in *aramis* RNAi compared to *bam* RNAi. Sheet 2 (Upregulated Genes) contains genes and corresponding Deseq2 output meeting the cutoffs to be considered upregulated in *aramis* RNAi compared to *bam* RNAi. Sheet 3 (All Genes) contains Deseq2 output for all genes in the dm6 assembly.  

**\
**

**Supplemental Table 2.3. Differential expression analysis from RNAseq of ovaries depleted of *bam* and *aramis* in the germline compared to ovaries depleted of *bam* in the germline.** 
DEseq2 output from RNAseq of ovaries depleted of *bam* and *aramis* in the germline compared to ovaries depleted of *bam* in the germline as a control. Sheet 1 (Downregulated Genes) contains genes and corresponding DEseq2 output meeting the cutoffs to be considered downregulated in *bam*; *aramis* RNAi compared to *bam* RNAi. Sheet 2 (Upregulated Genes) contains genes and corresponding DEseq2 output meeting the cutoffs to be considered upregulated in *bam*; *aramis* RNAi compared to *bam* RNAi. Sheet 3 (All Genes) contains DEseq2 output for all genes in the dm6 assembly. Sheet 4(BP GO terms - downregulated) contains the output of a Biological Process PANTHER Overrepresentation Test of significantly enriched GO terms performed on genes identified as downregulated targets from Sheet 1. Sheet 5 (BP GO terms - upregulated) contains the output of a Biological Process PANTHER Overrepresentation Test of significantly enriched GO terms performed on genes identified as upregulated targets from Sheet 2.

**\
**

**Supplemental Table 2.4. Analysis of polysome-seq of ovaries depleted of *aramis* in the germline compared to developmental controls.** 
Results of polysome-seq from ovaries depleted of *aramis* in the germline, ovaries depleted of *bam*, and ovaries overexpressing tkv in the germline as developmental controls. Sheet 1 (Downregulated Genes) contains genes and corresponding polysome/input ratio values and values representing the difference in the polysome/input ratios between *aramis* RNAi and the developmental controls meeting the cutoffs to be considered downregulated in *aramis* RNAi. Sheet 2 (Upregulated Genes) contains genes and corresponding polysome/input ratio values and values representing the difference in the polysome/input ratios between *aramis* RNAi and the developmental controls meeting the cutoffs to be considered upregulated in *aramis* RNAi. Sheet 3 (All Genes) contains DEseq2 output for all genes in the dm6 assembly.

**\
**

**Supplemental Table 2.5. Enrichment analysis of Aramis RNA IPmRNA-seq.** 
Results of Aramis::GFP::FLAG IP/IgG/Input mRNAseq. Each sheet contains the output of results from DEseq2. Sheet 1 (aramis polysome IP Enrichment) contains the enrichment value of all *aramis* polysome targets from Aramis IP. Sheet 2 (aramis polysome IgG Enrichment) contains the enrichment value of all *aramis* polysome targets from Aramis IgG control. Sheet 3 (Aramis IP Targets) contains Aramis IP targets as defined in methods. Sheet 4 (IP vs In Enriched) contains genes significantly enriched in the Aramis IP samples compared to the input samples. Sheet 5 (IgG vs In Enriched) contains genes significantly enriched (see methods) in the IgG samples compared to the input samples. Sheet 6 (IPvsIn All Genes) contains the DEseq2 output of all genes in the Aramis IP samples compared to the input samples. Sheet 7 (IgG vs In All Genes) contains the DEseq2 output of all genes in the IgG samples compared to the input samples.

**\
**

**Supplemental Table 2.6. Aramis translation targets contain TOP sequences.** 
Sheet 1 (aramisRNAi target CAGE 5\'UTRs) contains the CAGE corrected 5\'UTRs of *aramis* RNAi polysome downregulated targets with leading TOP sequences and start codons annotated. Sheet 2 (TOP location) contains a list of *aramis* RNAi polysome downregulated targets and the position and sequence of the first instance of a 5-mer pyrimidine sequence downstream of the CAGE-defined TSS of each gene.

**\
**

**Supplemental Table 2.7. Enrichment analysis of Larp RNA IP mRNA-seq.** 
Results of Larp::GFP::3XFLAG IP/IgG/Input mRNAseq. Each sheet contains the output of DEseq2. Sheet 1 (Larp Targets) contains Larp IP targets as defined in methods. Sheet 2 (IP vs In Enriched) contains genes significantly enriched in the Larp IP samples compared to the input samples. Sheet 3 (IgG vs In Enriched) contains genes significantly enriched (see methods) in the IgG samples compared to the input samples. Sheet 4 (IPvsIn All Genes) contains the DEseq2 output of all genes in the Larp IP samples compared to the input samples. Sheet 5 (IgG vs In All Genes) contains the DEseq2 output of all genes in the IgG samples compared to the input samples.

## Materials and Methods

**Resource Availability**

**Lead Contact:**

Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Prashanth Rangan ([prangan\@albany.edu](mailto:prangan@albany.edu){.email}).

**Materials availability:**

Materials generated during this study are available upon request.

**Data and Code availability:**

Sequencing data generated during this study are available on GEO under the accession [GSE171350](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE171350). Other data generated during this study are available from the lead contact.

#### Fly lines

The following Bloomington Stock Center lines were used in this study: #25751 *UAS-Dcr2;nosGAL4*, #4442 *nosGAL4;MKRS/TM6*, #32334 Aramis RNAi#1 CG5589^HMS00325^, #56977 Athos RNAi#1 CG4901^HMC04417^, #36589 Porthos RNAi#1 CG9253^GL00549^, #36537 UAS-tkv.CA, #33631 bam RNAi^HMS00029^, #6815 p53^5A-1-4^, #4264 Harwich, #6816 p53^11-1B-1^, #55101 FUCCI: UASp-GFP.E2f1.1-230, UASp-mRFP1.CycB.1-266/TM6B, #5431 UAS-EGFP, #18942 aramis^f06152^ Pbac{WH}CG5589f06152/TM6B, Tb1, #9503 athos Df Df(2L)BSC143/CyO, #13988 porthos^KG^ P{SUPor-P}CG9253^KG05120^, #58178 bam RNAi P{TriP.HMJ22155}, #78777 Non1 RNAi P{TriP.HMS05872}, #61790 Larp::GFP::3XFLAG Mi{PT-GFSTF.1}larp^MI06928-GFSTF.1^, #8841 w1118; Df(3R)Hsp70A, Df(3R)Hsp70B, #55384 Nprl3 RNAi P{TriP.HMC04072}attP40, #34814 raptor RNAi P{TriP.HMS00124}attP2

The following Vienna Stock Center lines were used in this study: Aramis RNAi#2 CG5589^v44322^, Athos RNAi#2 CG4901^v34905^, Aramis::GFP Pbac{fTRG01033.sfGFP-TVPTBF}VK00002, Athos::GFP Pbac{fTRG01233.sfGFP-TVPTBF}VK00033, Non1::GFP Pbac{fTRG00617.sfGFP-TVPTBF}VK00033

The following additional fly lines were used in the study: UASp-CycB::GFP ([Mathieu et al., 2013](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5830152/#R25)), *UAS-Dcr2;nosGAL4;bamGFP, If/CyO;nosGAL4* (Lehmann Lab), w1118 (Lehmann lab), *tjGAL4/CyO* [@tanentzapfIntegrindependentAnchoringStemcell2007], UASp-p53 [@bakhratDrosophilaChk2P532010], RpS2::GFP^CB02294^ [@buszczakCarnegieProteinTrap2007; @Zhang2014d], UASt-porthos::3XFLAG::3XHA [@emtenaniGeneticProgramBoosts2021], UASp-Non1 (this study), UASp-Larp-DM15 (this study), WT-TOP-Reporter (this study), Mutant-TOP-Reporter (this study).

#### Antibodies IF

The following antibodies were used for immunoflourenscence: mouse anti-1B1 1:20 (DSHB 1B1), rabbit anti-Vasa 1:833-1:4000 (Rangan Lab), chicken anti-Vasa 1:833-1:4000 (Rangan Lab) [@upadhyayTransposonDysregulationModulates2016], rabbit anti-pTyr 1:500 (Sigma T1235), rabbit anti-pMad 1:200 (Abcam ab52903), rabbit anti-GFP 1:2000 (abcam, ab6556), mouse anti-p53 1:200 (DSHB 25F4), Rabbit anti-CycB 1:200 (Santa Cruz Biotechnology, 25764), Rabbit anti-Fibrillarin 1:200 (Abcam ab5821), Mouse anti-Fibrillarin 1:50 (Fuchs Lab) [@McCarthy2018h]. Alexa 488 (Molecular Probes), Cy3 and Cy5 (Jackson Labs) were used at a dilution of 1:500.

#### Antibodies Western/IP

Mouse anti-FLAG-HRP 1:5000 (Sigma Aldrich, A8592)

Mouse anti-FLAG (Sigma Aldrich, F1804)

Anti-GAPDH-HRP 1:10,000 (Cell Signaling, 14C10)

Rabbit anti-DDX52 1:5000 (Bethyl, A303-053A)

Rabbit anti-DHX33 1:5000 (Bethyl, A300-800A)

Rabbit anti-DDX47 1:1000 (Bethyl, A302-977A)

#### Protein Domain Analysis

Protein domain figures were adapted from: The Pfam protein families database in 2019: S. El-Gebali et al. Nucleic Acids Research (2019). Protein Similarity values were obtained from the DRSC/TRiP Functional Genomics Resources.

#### Protein Conservation Analysis

Evolutionary trees were generated using MEGA. The evolutionary history was inferred by using the Maximum Likelihood method and JTT matrix-based model. The tree with the highest log likelihood is shown. Initial tree(s) for the heuristic search were obtained automatically by applying Neighbor-Join and BioNJ algorithms to a matrix of pairwise distances estimated using a JTT model, and then selecting the topology with superior log likelihood value. Trees are drawn to scale, with branch lengths measured in the number of substitutions per site.

#### TOP Reporter Cloning

gBlocks (see primer list for details) were cloned into pCasper2 containing a Nos promoter, HA-tag, GFP-tag, and K10 3\'UTR. PCR was used in order to amplify the gBlock and to remove the 5\'-end of the RpL30 5\'UTR in order to generate the 5\'-UTR discovered via CAGE-seq. In order to clone the Nos promoter followed by the RpL30 5\'UTR without an intervening restriction site, the portion of the plasmid 5\' of the 5\'UTR consisting of a portion of the plasmid backbone, a NotI restriction site, and the Nos Promoter was amplified from the pCasper plasmid using PCR. HiFi cloning was performed on the amplified fragments. The backbone was cut with NotI and SpeI and HiFi cloning was performed according to the manufactures\' instructions except the HiFi incubation was performed for 1 hour to increase cloning efficiency. Colonies were picked and cultured and plasmids were purified using standard techniques. Sequencing was performed by Eton Bioscience Inc. to confirm the correct sequence was present in the final plasmids. Midi-prep scale plasmid was prepared using standard methods and plasmids were sent to BestGene Inc. for microinjection.

#### Gateway Cloning

Gateway cloning was performed as described according to the manufacture\'s manual. Briefly, primers containing the appropriate Gateway *attb* sequence on the 5\'-ends and gene specific sequences on the 3\'-ends (see primer list for sequences) were used to PCR amplify each gene of interest. PCR fragments were BP cloned into pEntr221 as detailed in the Thermofisher Gateway Cloning Manual and used to transform Invitrogen One Shot OmniMAX 2 T1 Phage-Resistant Cells. Resulting clones were picked and used to perform LR cloning into either pPGW or pPWG as appropriate. Cloning was carried out according to the Thermofisher Gateway Cloning Manual except the LR incubation was carried out up to 16 hours. Colonies were picked and cultured and plasmids were purified using standard techniques. Sequencing was performed by Eton Bioscience Inc. to confirm the correct sequence was present in the final plasmids. Midi-prep scale plasmid was prepared using standard methods and plasmids were sent to BestGene Inc. for microinjection.

#### Egg Laying Test

Newly eclosed flies were collected and fattened overnight on yeast. Six female flies were crossed to 4 male controls and kept in cages at 25°C. Flies were allowed to lay for three days, and plates were changed and counted daily. Total number of eggs laid over the three day laying periods were determined and averaged between three replicate crosses for control and experimental crosses.

#### Immunostaining

Ovaries were dissected and teased apart with mounting needles in cold PBS and kept on ice for subsequent dissections. All incubations were performed with nutation. Ovaries were fixed for 10-15 min in 5% methanol-free formaldehyde in PBS. Ovaries were washed with PBT (1x PBS, 0.5% Triton X-100, 0.3% BSA) once quickly, twice for 5 min, and finally for 15 min. Ovaries were incubated overnight, up to 72 hours in PBT with the appropriate primary antibodies. Ovaries were again washed with PBT once quickly, twice for 5 min, and finally for 15 min. Ovaries were then incubated with the appropriate secondary antibodies in PBT overnight up to 72 hours at 4°C. Ovaries were washed once quickly, twice for 5 min, and finally for 15 min in PBST (1x PBS, 0.2% Tween 20 Ovaries). Ovaries were mounted with Vectashield with 4\',6-diamidino-2-phenylindole (DAPI) (Vector Laboratories) and imaged on a Zeiss 710. All gain, laser power, and other relevant settings were kept constant for any immunostainings being compared. Image processing was performed in Fiji, gain was adjusted, and images were cropped in Photoshop CC 2018.

#### Florescent Imaging

Tissues were visualized and imaged were acquired using a Zeiss LSM-710 confocal microscope under the 20x--- and 40x--- oil objectives.

#### Measurement of global protein synthesis

OPP (Thermo Fisher, C10456) treatment was performed as in McCarthy [@mccarthyMSL3CoordinatesTranscriptional2019]. Briefly, ovaries were dissected in Schneider\'s media (Thermo Fisher, 21720024) and incubated in 50 µM of OPP reagent for 30 minutes. Tissue was washed in 1x PBS and fixed for 10 minutes in 1x PBS plus 5% methanol-free formaldehyde. Tissue was permeabilized with 1% Triton X-100 in 1x PBST (1x PBS, 0.2% Tween 20) for 30 minutes. Samples were washed with 1x PBS and incubated with Click-iT reaction cocktail, washed with Click-iT reaction rinse buffer according to manufacturer\'s instructions. Samples were then immunostained according to previously described procedures.

#### Image Quantifications

All quantifications were performed on images using the same confocal settings. A.U. quantifications were performed in Fiji on images taken with identical settings using the "Measure" function. Intensities were normalized as indicated in the figure legends, boxplots of A.U. measurements were plotted using R and statistics were calculated using R.

Quantification of nucleolar size was measured in Fiji by measuring the diameter of the nucleolus using the measure tool in Fiji. Volumes were calculated using the formula for a sphere.

Quantification of p53 area of expression was performed from control, *nosGAL4* and nosGAL4>*aramis* RNAi germaria. A manual threshold was set based off of qualitative assessment of a "punctate". For control ovaries, cells proximal to the niche consisting of GSCs/CBs were outlined and for *aramis* RNAi the entire germline proximal to the niche was outlined and a Fiji script was used to determine the number of pixels above the threshold and the total number of pixels. Data from each slice for each replicate was summed prior to plotting and statistical analysis.

Colocalization analysis of helicases with Fibrillarin was performed in Fiji using the Plot Profile tool. A selection box was drawn over a Fibrillarin punctate of interest (indicated with a box in the images) and Plot Profiles was acquired for each channel of interest. Data was plotted and Spearman correlations calculated using R.

Quantification of Non1-GFP expression and p53 expression over development was calculated in Fiji using the Auto Threshold tool with the Yen method [@sezginSurveyImageThresholding2004] to threshold expression. Quantifications were performed on 3 merged slices and egg chambers were cropped out of quantified images prior to thresholding to prevent areas outside of the germarium from influencing the thresholding algorithm. Areas of germline with "high" and "low" expression of Non1-GFP were outlined manually and a custom Fiji script was used in order to quantify the proportion of pixels in the selected marked as positive for expression for either Non1-GFP or p53, staging was inferred from the results of the Non1-GFP quantification performed using 1B1 to determine the stages of peak Non1 expression. Percent area was plotted with ggplot2 as boxplots in a custom R script.

#### RNA Extraction from Ovaries

RNA extraction was performed using standard methods. Ovaries were dissected into PBS and transferred to microcentrifuge tubes. PBS was removed and 100ul of Trizol was added and ovaries were flash frozen and stored at -80°C. Ovaries were lysed in the microcentrifuge tube using a plastic disposable pestle. Trizol was added to 1 mL total volume and sample was vigorously shaken and incubated for 5 min at RT. The samples were centrifuged for x min at \>13,000 g at 4°C and the supernatant was transferred to a fresh microcentrifuge tube. 500 ul of chloroform was added and the samples were vigorously shaken and incubated for 5 minutes at RT. Samples were spun at max speed for 10 minutes at 4°C. The supernatant was transferred to a fresh microcentrifuge tube and ethanol precipitated. Sodium acetate was added equaling 10% of the volume transferred and 2-2.5 volumes of 100% ethanol were added. The samples were shaken thoroughly and left to precipitate at -20°C overnight. The samples were centrifuged at max speed at 4°C for 15 min to pellet the RNA. The supernatant was discarded and 500 ul of 75% ethanol was added to wash the pellet. The samples were vortexed to dislodge the pellet to ensure thorough washing. The samples were spun at 4°C for 5 min and the supernatant was discarded. The pellets were left for 10-20 min until dry. The pellets were resuspended in 20-50ul of RNAse free water and the absorbance at 260 was measured on a nanodrop to measure the concentration of each sample.

#### S2 Cell RNAi

DRSC-S2 cells (Stock #181, DGRC) were cultured according to standard methods in M3+BPYE media supplemented with 10% heat-inactivated FBS. dsRNA for RNAi was prepared as described by the SnapDragon manual. Briefly, template was prepared from S2 cell cDNA using the appropriate primers (see primer list) designed using SnapDragon (<https://www.flyrnai.org/snapdragon>). Template was either used directly for *in-vitro* transcription or TA-cloned into the pCR2.1-TOPO vector (K450002) followed by transformation into TOP-10 cells (K450002), plasmid purified, and digested with *Eco*R I prior to *in-vitro* transcription. For *in-vitro* transcription the T7 Megascript kit (AM1334) was used following manufacturer\'s instructions and in-vitro transcriptions were incubated overnight at 37°C. The RNA was treated with DNAse according to the T7 Megascript manual and the RNA was purified using acid-phenol chloroform extraction and ethanol precipitated. The resulting RNA was annealed by heating at 65°C for 5 minutes and slow cooling to 37°C for an hour. S2 cell RNAi was performed essentially as previously described using Effectine [@zhouInducingRNAiDrosophila2013]. 1.0x10^6^ cells were seeded 30 minutes prior to transfection and allowed to attach. After 30 minutes, just prior to transfection, the media was changed for 500 µl of fresh media. 500µl of transfection complexes using 1 µg of dsRNA was prepared per well of a 6-well plate and pipetted dropwise onto seeded cells. After 24 hours an additional 1 mL of media was added to each well. After an additional 24 hours cells were passaged to 10 cm dishes. After an additional 3 days cells were harvested for further analysis.

#### Polysome-profiling

Polysome-profiling in S2 cells was performed as in Fuchs et al. [@fuchsProteomicAnalysisRibosomes2011] with minor modifications. S2 cells were resuspended by pipetting, pelleted by centrifugation at 800g for one minute, and washed in cold PBS. Cells were again pelleted and resuspended in 400 µl of lysis buffer (300 mM NaCl, 15 mM Tris-HCl, pH 7.5, 15 mM EDTA, 100 g/mL cycloheximide, 1% Triton X-100). Cells were then allowed to continue to lyse for 15 min on ice. Lysate was cleared by centrifugation at 8500g for 5 min at 4°C. Cleared lysate was loaded onto 10%-50% sucrose gradients (300 mM NaCl, 15 mM Tris-HCl, pH 7.5, 15 mM MgCl2, 100 g/mL cycloheximide) and centrifuged in an SW41 rotor at 35,000 RPM, for 3 hours. Gradients were fractionated on a Density Gradient Fractionation System (Brandel, #621140007) at 0.75 mL/min. Data generated from gradients were plotted using R.

#### Western Blot

HeLa cells were harvested for Western by in RIPA buffer by scraping. Western blotting were performed according to standard methods, briefly, each sample was loaded onto a 4-20% commercial, precast gels and run at 100V for 60-90m depending on the size of the protein of interest. Gels were transferred to nitrocellulose membranes at 100V for 1hr at 4°C. Blot was blocked in 1% milk in PBS and washed 3 times with PBS-T for 5 minutes. Primary antibodies were diluted in PBS-T+5% BSA and incubated overnight. Blot was washed once quickly, once for 5m, and once for 10m in PBS-T. Blot was subsequently imaged with ECL for conjugated primaries. For unconjugated primaries, the appropriate secondary was diluted 1:10,000 in 5% milk and incubated for 2-4 hours at RT. Blot was washed once quickly, once for 5m, and once for 10m in PBS-T and imaged. Images were quantified using Fiji.

#### mRNAseq Library Preparation and Analysis

Libraries were prepared with the Biooscientific kit (Bioo Scientific Corp., NOVA-5138-08) according to manufacturer\'s instructions with minor modifications. Briefly, RNA was prepared with Turbo DNAse according to manufacturer\'s instructions (TURBO DNA-free Kit, Life Technologies, AM1907), and incubated at 37°C for 30 min. DNAse was inactivated using the included DNAse Inactivation reagent and buffer according to manufactures instructions. The RNA was centrifuged at 1000 g for 1.5 min and 19 µl of supernatant was transferred into a new 1.5 mL tube. This tube was again centrifuged at 1000 g for 1.5 min and 18 µl of supernatant was transferred to a new tube to minimize any Inactivation reagent carry-over. RNA concentration was measured on a nanodrop. Poly-A selection was performed on a normalized quantity of RNA dependent on the lowest amount of RNA in a sample, but within the manufacturer\'s specifications for starting material. Poly-A selection was performed according to manufacturer\'s instructions (Bioo Scientific Corp., 710 NOVA-512991). Following Poly-A selection mRNA libraries were generated according to manufactures instructions (Bioo Scientific Corp., NOVA-5138-08) except RNA was incubated for 13 min at 95°C to generate optimal fragment sizes. Library quantity was assessed via Qubit according to manufacturer\'s instructions and library quality was assessed with a Bioanalyzer or Fragment Analyzer according to manufacturer\'s instructions to assess the library size distribution. Sequencing was performed on biological duplicates from each genotype on an Illumina NextSeq500 by the Center for Functional Genomics (CFG) to generate single end 75 base pair reads. Reads were aligned to the dm6.01 assembly of the Drosophila genome using HISAT v2.1.0. Reads were counted using featureCounts v1.4.6.p5. UCSC genome browser tracks were generated using the bam coverage module of deeptools v3.1.2.0.0. Differential expression analysis was performed using DEseq2 (v1.24.0) and data was plotted using R. Differentially expressed genes were those with log~2~(foldchange) \> \|1.5\| and FDR \< 0.05 in the *aramis* RNAi versus *bam* RNAi experiment and foldchange \> \|1.5\| and FDR \< 0.05 in the *bam* RNAi; *aramis* RNAi versus *bam* RNAi experiment. GO-term analysis of GO biological processes was performed on differentially expressed genes using PANTHER via <http://geneontology.org/>. Fisher\'s exact test was used to calculate significance and FDR was used to correct for multiple testing. GO-term analysis results were plotted using R.

#### Polysome-seq

Polysome-seq was performed as in Flora et al. [@Flora2018k] with minor modifications. Ovaries were dissected in PBS and transferred to a microcentrifuge tube in liquid nitrogen. Ovaries were lysed in 300µl of lysis buffer (300 mM NaCl, 15 mM Tris-HCl, pH 7.5, 15 mM EDTA, 100 µg/mL cycloheximide, 1% Triton X-100) and allowed to lyse for 15 min on ice. Lysate was cleared by centrifugation at 8500g for 5 min at 4°C. 20% of the lysate was reserved as input, 1 mL of Trizol (Invitrogen, 15596026) was added and RNA was stored at -80°C. Cleared lysate was loaded onto 10%-50% sucrose gradients (300 mM NaCl, 15 mM Tris-HCl, pH 7.5, 15 mM MgCl2, 100 g/mL cycloheximide) and centrifuged in an SW41 rotor at 35,000 RPM, for 3 hours. Gradients were fractionated on a Density Gradient Fractionation System (Brandel, #621140007) at 0.75 mL/min, 20 µl of 20% SDS, 8 µl of 0.5 M pH 8 EDTA, and 16 µl of proteinase K (NEB, P8107S) was added to each polysome fraction. Fractions were incubated for 30m at 37°C. Standard acid phenol chloroform purification followed by ethanol precipitation was performed on each fraction. The RNA from polysome fractions was pooled and RNAseq libraries were prepared.

#### Polysome-seq Data Analysis

Reads were checked for quality using FastQC. Reads were mapped to the *Drosophila* genome (dm6.01) using Hisat version 2.1.0. Mapped reads were assigned to features using featureCount version v1.6.4. Translation efficiency was calculated as in (Flora et al., 2018; Kronja et al., 2014) using an R script. Briefly, TPMs (transcripts per million) values were calculated. Any gene having zero reads in any library was discarded from further analysis. The log~2~ ratio of CPMs between the polysome fraction and total mRNA was calculated and averaged between replicates. This ratio represents the TE. TE of each replicate was averaged. Targets were defined as transcripts falling greater or less than two standard deviations from the median TE in aramis RNAi for upregulated and downregulated genes respectively, but not in either of the two developmental controls (Nos-GAL4 UAS-*tkv* or Nos-GAL4 UAS-*bam* RNAi). Additionally, genes were only considered targets if their mean TE value in Nos-GAL4 UAS-*aramis* RNAi was higher (for upregulated targets) or lower (for downregulated targets) than their mean TE values in both of the two developmental controls. Finally, only targets meeting a conservative expression cutoff of log~2~(TPM) expression greater than five were considered to exclude more lowly expressed genes as they are highly influenced by noise in polysome-seq in both controls.

#### CAGE-seq Tracks

CAGE-seq tracks were visualized using the UCSC Genome Browser after adding the publicly available track hub \'EPD Viewer Hub\'.

#### CAGE-seq Data Reanalysis

Publicly available genome browser tracks were obtained of CAGE-seq data (generated by Chen et al. (2014) and viewed through the UCSC Genome Browser. The original CAGE-seq data from ovaries was obtained from SRA under the accession number SRR488282. Reads were aligned to the dm6.01 assembly of the *Drosophila* genome using HISAT v2.1.0. cageFightR was used to determine the dominant TSS for every gene with sufficient expression in from the aligned dataset according to its documentation with default parameters excepting the following: For getCTSS, a mappingQualityThreshold of 10 was used. For normalizeTagCount the method used was "simpleTPM". For clusterCTSS the following parameters were used; threshold = 1, thresholdIsTPM = TRUE, nrPassThreshold = 1, method = "paraclu", maxDist = 20, removeSingletons = TRUE, keepSingletonsAbove = 5. R was used to obtain genome sequence information downstream of the TSS of each gene identified.

To generate a table of *aramis* polysome-seq target 5\'UTRs adjusted using CAGE-seq data, bigwig files of CAGE-seq from ovaries were obtained from EPD Viewer Hub. The most highly expressed TSS within a CAGE cluster (obtained as described in this section) was used to determine the new 5\'-end coordinate associated with each *aramis* polysome-seq target gene at the transcript level. These coordinates were used to obtain the corrected 5\'UTR using R and transcripts with identical sequences were discarded.

#### Motif Enrichment Analysis

Motif enrichment analysis was performed using Homer [@heinzSimpleCombinationsLineagedetermining2010] using the findmotifs.pl module, supplying Homer with the first 200 nucleotides downstream of the TSS as determined by CAGE-seq for polysome-seq targets and non-targets as a background control with the following parameters "-rna -nogo -p 6 -len 6". Only motifs not marked as potential false positives were considered. The position of the putative TOP motifs was determined using a custom R script by searching for the first instance of any five pyrimidines in a row within the first 200 nucleotides of the TSS using the Biostrings package [@pagesBiostringsEfficientManipulation2019]. Results were plotted as a histogram in R.

#### RNA Immunoprecipitation (RNA IP)

All RIPs were performed with biological triplicates. 50-60 ovary pairs were dissected for each sample in RNase free PBS and dissected ovaries were kept on ice during subsequent dissections. After dissection, ovaries were washed with 500µl of PBS to remove any debris. This PBS was removed, and ovaries were lysed in 100µl of RIPA buffer (10 mM Tris-Cl Buffer (pH 8.0), 1 mM EDTA, 1% Triton X-100,0.1% Sodium deoxycholate, 0.1% SDS, 140 mM NaCl, 1 mM PMSF, 1 cOmplete, EDTA-free Protease Inhibitor/10mLbuffer (Roche, 11873580001), RNase free H2O) supplemented with 8µl of RNase Out. Following lysis an additional 180µl of RIPA was added to each sample. Lysate was cleared with centrifugation at 14,000g for 20m at 4°C. Cleared lysate was transferred to a new 1.5 mL tube. 10% of this lysate was reserved for RNA input and 5% was reserved as a protein input. To the RNA input 100µl of Trizol was added and the input was stored at -80°C. To the protein input SDS loading buffer was added to a 1X working concentration and the sample was heated at 95°C for 5m and stored at -20°C. The remaining lysate was equally divided into two new 1.5 mL tubes. To one tube 3µg of mouse anti-FLAG antibody was added and to the other tube 3µg of mouse IgG was added. These samples were incubated for 3 hours with nutation at 4°C. NP40 buffer was diluted to a 1X working concentration from a 10X stock (10x NP40 Buffer: 50 mM Tris-Cl Buffer (pH 8.0), 150 mM NaCl, 10% NP-40, 1 cOmplete, EDTA-free Protease Inhibitor Cocktail Pill/10mL buffer, RNase free H2O). 30µl of Protein-G beads per RIP were pelleted on a magnetic stand and supernatant was discarded. 500µl of 1X NP40 buffer was used to resuspend Protein-G beads by nutation. Once beads were resuspended, they were again pelleted on the magnetic stand. This washing process was repeated a total of 5 times. Washed Protein-G beads were added to each lysate and incubated overnight. The next day fresh 1X NP40 buffer was prepared. Lysates were pelleted on a magnetic stand at 4°C and supernatant was discarded. 300µl of 1X NP40 buffer was added to each sample and samples were resuspended by nutation at 4°C. Once samples were thoroughly resuspended, they were pelleted on a magnetic stand. These washing steps were repeated 6 times. Following the final washing steps, beads were resuspended in 25 ul of 1X NP40 Buffer. 5µl of beads were set aside for Western and the remaining beads were stored at -80°C in 100µl of Trizol. SDS loading buffer was added was added to a 1X working concentration and the sample was heated at 95°C for 5m and stored at -20°C or used for Western (refer to Western Blot section).

#### Helicase RNA IPseq

RNA was purified as previously described. RNA yield was quantified using Qubit or nanodrop according to manufactures instructions. RNA was run on a Fragment Analyzer according to manufactures instructions to assess quality. Inputs were diluted 1:50 to bring them into a similar range as the IgG and IP samples. To each sample 0.5 ng of Promega Luciferase Control RNA was added as a spike-in. Libraries were prepared as previously described except Poly(A) selection steps were skipped and library preparation was started with between 1-100 ng of total RNA. Reads were mapped to the M21017.1 NCBI *Drosophila* rRNA sequence record and the sequence of Luciferase obtained from Promega. All further analysis was performed using custom R scripts. Reads were assigned to features using featureCounts based off of a custom GTF file assembled based off of the Flybase record of rRNA sequences. Reads mapping to rRNA were normalized to reads mapping to the Luciferase spike-in control. Reads were further normalized to the reads from the corresponding input library to account for differences in input rRNA concentration between replicates and replicates were subsequently averaged. Tracks were visualized using the R package \'ggplot2\', with additional formatting performed using \'scales\' and \'egg\'. The rRNA GTF was read into R using \'rtracklayer\' and visualized using \'gggenes\'. Average reads mapping to rRNA from IgG control and IP was plotted and a one-sided bootstrapped paired t-test for was performed on regions on rRNA that appeared to be enriched in the IP samples compared to the IgG control as it is a non-parametric test suitable for use with low n using R with 100,000 iterations.

#### Larp Gel Shifts

##### Cloning, Protein expression and purification

The Larp-DM15 protein expression construct (amino acids 1330-1481 corresponding to isoform D) was cloned into a modified pET28a vector by PCR using cDNA corresponding to accession ID NP_733244.5. The resulting fusion protein has an N-fHis~10~-maltose binding protein (MBP)-tobacco etch virus (TEV) protease recognition site tag. Protein expression and purification were performed as described previously [@lahrLarelatedProtein1specific2015]. Briefly, plasmid was transformed into BL21(DE3) *E. coli* cells and plated onto kanamycin-supplemented agar plates. A confluent plate was used to inoculate 500 mL of autoinduction media [@studierProteinProductionAutoinduction2005]. Cells were grown for three hours at 37ºC and induced overnight at 18ºC. Cells were harvested, flash frozen, and stored at -80ºC.

Cells were resuspended in lysis buffer (50 mM Tris, pH 8, 400 mM NaCl, 10 mM imidazole, 10% glycerol) supplemented with aprotinin (Gold Bio), leupeptin (RPI Research), and PMSF (Sigma) protease inhibitors. Cells were lysed via homogenization. Lysate was clarified by centrifugation and incubated with Ni-NTA resin (ThermoScientific) for batch purification. Resin was washed with lysis buffer supplemented with 35 mM imidazole to remove non-specific interactions. His~10~-MBP-DM15 was eluted with 250 mM imidazole. The tag was removed via proteolysis using TEV protease and simultaneously dialyzed overnight (3 mg TEV to 40 mL protein elution). Larp-DM15 was further purified by tandem anion (GE HiTrap Q) and cation exchange (GE HiTrap SP) chromatography using an AKTA Pure (GE) to remove nucleic acid and protein contaminants. The columns were washed with in buffer containing 50 mM Tris, pH 7, 175 mM NaCl, 0.5 mM EDTA, and 10% glycerol and eluted with a gradient of the same buffer containing higher salt (1 M NaCl). Fractions containing Larp-DM15 were pooled, and 3 M ammonium sulfate was added to a final concentration of 1 M. A butyl column (GE HiTrap Butyl HP) was run to remove TEV contamination. The wash buffer contained 50 mM Tris, pH 7, 1 M ammonium sulfate, and 5% glycerol, and the elution buffer contained 50 mM Tris pH 7 and 2 mM DTT. Fractions containing Larp-DM15 were buffer exchanged into storage buffer (50 mM Tris pH, 7.5, 250 mM NaCl, 2 mM DTT, 25% glycerol), flash frozen in liquid nitrogen, and stored at -80°C. The purification scheme and buffer conditions were the same as with *Hs*DM15 [@lahrLarelatedProtein1specific2015], except cation and anion exchange buffers were at pH 7, as noted above.

##### RNA preparation

5\'-triphosphorylated *RpL30* and *Non1* 42-mers were synthesized (ChemGenes). Purine-substituted controls were synthesized by *in vitro* transcription using homemade P266L T7 RNAP polymerase [@guillerezMutationT7RNA2005]. The transcription reaction containing 40 mM Tris, pH 8, 10 mM DTT, 5 mM spermidine, 2 mM NTPs, and 10-15 mM MgCl~2~ was incubated at 37°C for 4 hours. Transcripts were subsequently purified from an 8% polyacrylamide/6M urea/1XTBE denaturing gel, eluted passively using 10 mM sodium cacodylate, pH 6.5, and concentrated using spin concentrators (Millipore Amicon). All oligos were radioactively capped using Vaccinia virus capping system (NEB) and $\alpha^{32}$-GTP (Perkin-Elmer). Labelled oligos were purified using a 10% polyacrylamide/6M urea/1XTBE denaturing gel, eluted with 10 mM sodium cacodylate, pH 6.5, and concentrated by ethanol precipitation.

The RNA sequences used were:

RpL30: 
**\
**
**CUUUU**GCCAUUGUCAGCCGACGAAGUGCUUUAACCCAAACUA

Non1:
**\
**
**CUUUUU**GGAAUACGAAGCUGACACCGCGUGGUGUUUUUGCUU

\*Purine-substituted RPL30 control: 
**\
**
**GAAAAG**CCAUUGUCAGCCGACGAAGUGCUUUAACCCAAACUA

\*Purine-substituted Non1 control: 
**\
**
**GAAAAAG**GAAUACGAAGCUGACACCGCGUGGUGUUUUUGCUU

\setstretch{1.0}

Oligos used for run-off transcription
**\
**

+-------------------------------+--------------------------------------+
| **DNA oligo**                 | **Sequence (5' to 3')**              |
+===============================+======================================+
| \*\*RpL30 control             | GCGCGCGAATTCTAATACGACTCACTATA        |
|                               |                                      |
| gene block (with 3' HDV)      | [GAAAAGCCATTGTCAGCCGACGAAGTG]{.ul}   |
|                               |                                      |
|                               | [CTTTAACCCAAACTA]{.ul}GGGTCGGCATGG   |
|                               |                                      |
|                               | CATCTCCACCTCCTCGCGGTCCGACCTG         |
|                               |                                      |
|                               | GGCTACTTCGGTAGGCTAAGGGAGAAG          |
|                               |                                      |
|                               | CTTGGCACTGGCCGTCGTTTTGGCACTG         |
|                               |                                      |
|                               | GCCGTCGTTT                           |
+-------------------------------+--------------------------------------+
|                               |                                      |
+-------------------------------+--------------------------------------+
| Non1 control                  | GCGCGCGAATTCTAATACGACTCACTATA        |
|                               |                                      |
| Forward                       | [GGAAAAAGGAATACGAAG]{.ul}            |
|                               |                                      |
|                               | [CTGACA]{.ul}                        |
+-------------------------------+--------------------------------------+
|                               |                                      |
+-------------------------------+--------------------------------------+
| Non1 control                  | [AAGCAAAAACACCACGCGGTGTCAGCTT]{.ul}  |
|                               |                                      |
| Reverse                       | [CGTATTCCTTTTTC]{.ul}CTATAGTGAG      |
+-------------------------------+--------------------------------------+
|                               |                                      |
+-------------------------------+--------------------------------------+
| 5' GEN amp                    | GCGCGCGAATTCTAATACGACTCA             |
+-------------------------------+--------------------------------------+
|                               |                                      |
+-------------------------------+--------------------------------------+
| RpL30 amp Reverse             | TAGTTTGGGTTAAAGCACTTCGTCGGC          |
+-------------------------------+--------------------------------------+
|                               |                                      |
+-------------------------------+--------------------------------------+
| Non1 amp Reverse              | AAGCAAAAACACCACGCGGTGTCA             |
+-------------------------------+--------------------------------------+

\* These RNAs were synthesized using run-off transcription.

\setstretch{2.0}

##### Electrophoretic mobility shift assays (EMSAs)**

Each binding reaction contained 125 total radioactive counts with final reaction conditions of: 20 mM Tris-HCl, pH 8, 150 mM NaCl, 10% glycerol, 1 mM DTT, 0.5 µg tRNA (Ambion), 1 µg BSA (Invitrogen), and \<90 pM RNA. To anneal RNA, oligos were snap-cooled by heating at 95°C for 1 min and cooled on ice for 1 hour. For capped RpL30 shifts and capped purine-substituted controls, final concentrations of 0, 0.001, 0.003, 0.01, 0.03, 0.1, 0.3, 1, 3, 10, 30, and 100 nM Larp-DM15 were titrated. For capped Non1 shifts, final concentrations of 0, 0.01, 0.03, 0.1, 0.3, 1, 3, 10, 30, 100, 300, and 1000 nM Larp-DM15 were titrated. Native 7% polyacrylamide 0.5X TBE gels were pre-run on ice at 120 V for 30 min. Binding reactions were run at 120 V on ice for 45-52 min. Gels were dried for 30 min and allowed to expose overnight using a phosphor screen (GE). Screens were imaged using GE Amersham Typhoon. Bands were quantified using ImageQuant TL (GE). Background subtraction was first done using the rolling ball method and then subtracting the signal from the zero-protein lane from each of the shifted bands. Fraction shifted was determined by dividing the background-corrected intensity of the shifted band by total intensity of bands in each lane. Three independent experiments were done for each oligo, with the average plotted and standard deviation shown.

#### mRNA IPseq

IPs of Larp and Aramis were performed as described in the RNA IP-seq section above in triplicate. mRNA libraries were prepared as described in mRNAseq Library Preparation and Data Processing using a constant volume of RNA from each sample with input samples having been diluted 1:50. Data was processed as described as in the mRNAseq Library Preparation and Data Processing section. Targets are defined as genes with \>2 fold enrichment and an adjusted p-value \<0.05 in the Larp-IP libraries compared to input libraries, but not meeting those criteria in the IgG libraries compared to input.

#### Larp RNA IP qPCR

Larp RNA IP was performed as described in the Larp RNA IPseq section with the following modifications. As the ovaries used were small, they were flash frozen in order to accumulate 40-50 ovaries for each biological replicate. Additionally, 5% input was taken for both RNA and protein samples. Once RNA was purified all of the RNA was treated with Turbo DNAse as in the **mRNAseq Library Preparation and Analysis** section. Reverse transcription (RT) was performed using Superscript II according to the manufacture\'s protocol with equivalent volumes of RNA for each sample. cDNA was diluted 1:8 before performing qPCR using Syber Green. Each reaction consisted of 5ul Syber Green master mix, 0.4 ul water, 0.3 ul of each primer, and 4 ul of diluted cDNA. For each sample 3 biological and 3 technical replicates were performed. Oulier values of technical replicates were removed using a Dixon test with a cutoff of p\<0.05. Remaining technical replicates were averaged, and the IP Input Ct value, the log~2~ of the Input dilution (20) was also subtracted to account for the Input being 5% of the total sample as follows:

$\Delta Ct[normalized\ IP] = (Average\ Ct[IP]/(Average\ Ct[Input] - log~2~(Input\ Dilution\ Factor))$

Next, RNA recovery was normalized using the spike-in control for each sample as follows:

$\Delta \Delta Ct = \Delta Ct[normalized\ IP]-\Delta\ Ct[Luciferase]$

Next, Each sample was normalized to it\'s matched *bam* RNAi control as follows:

$bam\ RNAi\ normalized Ct= \Delta \Delta Ct[aramis\ RNAi\ IP] - \Delta \Delta Ct[bam\ RNAi\ IP]$

Finally, fold increase of IP from *aramis* RNAi over *bam* RNAi was calculated as follows:

$Fold Enrichment = 2^{-bam\ RNAi\ normalized\ Ct}$

Fold enrichment was plotted and One-sample t-test performed on *aramis* RNAi samples in R using a mu of 1.
