\setstretch{2.0}
\setlength\parindent{24pt}
# Conclusion

My work has shined a spotlight on the role of post-transcriptional control in GSC differentiation. Specifically, my work has revolved around the idea that the ribosome acts as a nexus for gene expression regulation in three ways. First, the ribosome plays its "traditional" role as a factory to translate genetic instructions into protein products. Although the role of the ribosome in this aspect is "passive" mRNAs must make their way from the nucleus to the ribosome and are subject to degradation, sequestration, and interaction with hundreds or thousands of potential regulators on their journey. Second, the ribosome is the [subject]{.ul} of post-transcriptional gene regulation in order to balance the production of ribosomal proteins with the production of ribosomal RNA, ribosomal proteins are regulated post-transcriptionally. This role is interestingly ouroboric because the mechanism behind this regulation works by inhibiting translation initiation by the ribosome onto ribosomal mRNAs. Finally, less explored by my work, is the role of post-translational modifications on ribosomal proteins and mRNAs and the role that these modifications play in gene regulation. My work has specifically highlighted the importance of the first two facets in controlling GSC differentiation, and future work should seek to understand the role of post-translational modifications is GSC differentiation and stem cell differentiation more broadly.  

Previous work has explored the translation control of ribosomal protein mRNAs, in particular research has focused on the role of Larp1 in this regulatory axis [@Fonseca2015a; @Hong2017a; @Lahr2017b; @Tcherkezian2014b]. However, my work expands those findings in two major ways. First, while previous work characterized the regulation of ribosomal protein mRNAs in cell culture, my work explores this mechanism in a tissue where ribosome production is dynamic and in a tissue where Larp has been demonstrated to play a crucial role [@Ichihara2007a, Blagden2009f]. This provides important context as to why this aspect of regulation is important for development and additional evidence as to the role of Larp in the ovary. Second, previous work principally focuses of modulating this regulatory axis though the use of Torc1 inhibitors, thorough my work, we found that this axis can be modulated by altering the status of ribosome biogenesis [@Fonseca2015a; @Hong2017a; @Lahr2017b; @Tcherkezian2014b]. This establishes a feedback loop that can be used to balance ribosome biogenesis contextualizing the role of Larp.  

Throughout the course of this work I have developed tools to study and described the crucial role of post-transcriptional gene regulation in GSC differentiation and entry into meiosis in *Drosophila*. Work in other systems has underscored the importance of translation control in stem cell differentiation in general [@gabutRibosomeTranslationalControl2020; @Sanchez2016h; @woolnoughRegulationRRNAGene2016; @zahradkalRegulationRibosomeBiogenesis1991; @Zhang2014d]. Historically, study of post-transcriptional control has lagged behind that of control at the level of transcription. This is in part due to the lack of equity in tools and techniques between the two areas of study. One salient example is the lag in single-cell sequencing of mRNA which was first published in 2009 compared to the first published example of single-cell Ribo-seq, the first example of which was published in 2021, a 12-year lag [@vaninsbergheSinglecellRiboseqReveals2021; @tangMRNASeqWholetranscriptomeAnalysis2009]. This exemplifies that those interested in understanding translation control must continue to develop tools and use those tools to better understand developmental systems, regeneration, and disease states. 

With Oo-site I have attempted to fill a small part of this gap by democratizing a trove of RNAseq and polysome-seq data to allow researchers without bioinformatics skills to access data that can provide insight into post-transcriptional gene expression regulation. However, to truly make progress in data equity, researchers must improve their reporting of data and increase its accessibility, especially when dealing with high-throughput methods. 

There are two key areas in which progress must be made in order to close the knowledge-gap between transcriptional and post-transcriptional control. The first is the gap in tools to study post-transcriptional control and the second is to empower researchers to interpret the data that result from these tools. Just as the experimental tools used to study post-transcriptional control lag behind, the data analysis tools aimed at processing post-transcriptional data lag as well as the data processing and interpretation of post-transcriptional data tends to be more complex. Therefore, effort should continue to be focused on further tool development to allow researchers studying post-transcriptional control to probe this control using high-throughput methods. In concert work must also be done to ensure researchers have the ability to interpret the resulting data. Equity in understanding between the domains of transcription control and translation control, gaps in our knowledge will prevent our understanding of fundamental biological questions. However, as our toolkits grow we must also remember that every year we generate more data than the previous, but only a subset of the scientific community has the skills necessary to process that data. Therefore, the scientific community must attempt to democratize access to the high-throughput data we generate to empower the research of others.  

As the tools advance to better study post-transcriptional control, future work should aim to understand the pathways that underlie stem cell differentiation and meiotic entry, in particular the interplay and feedback between the Torc1 pathway and the synthesis of ribosomes. Our work has demonstrated that a feedback loop exists between Larp activity and ribosome biogenesis, but future work in other systems should determine if this loop is conserved and if Torc1 acts upstream of Larp as has been shown in other systems. More importantly, how the balance of the Larp-ribosome axis informs differentiation and what levers upstream of Larp might play a role in the initiation of differentiation are of great interest to better understand differentiation, regeneration, and developmental diseases. Emerging techniques such as single-cell Ribo-seq will no doubt allow for these questions to be studied in complex tissues, including developmental systems and disease states.  

Additionally, future work should examine whether the Larp-ribosome axis is tissue specific. So far we have demonstrated its importance in GSCs, but it remains an open question whether this mechanism acts in other stem cell populations or perhaps in unipotent cells. With our collaborators, we have found certain translational changes related to mitochondrial function that result when perturbing ribosome biogenesis in S2 cells. As Larp as been previously implicated in playing a role in mitochondrial translation in spermatogenesis, this may speak to a tissue specific mode of regulation, however, additional work is required to solidify the linkage between ribosome biogenesis, Larp, and Larp's targets outside of GSCs. These questions are of great interest, as understanding whether Larp may have tissue specific targeting or activity could help explain the tissue specific nature of ribosomeopathies, which has been intensely studied for decades.  
