<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Translation control tunes Drosophila oogenesis</title>
  <meta name="description" content="Translation control tunes Drosophila oogenesis" />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="Translation control tunes Drosophila oogenesis" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Translation control tunes Drosophila oogenesis" />
  
  
  

<meta name="author" content="Elliot T. Martin" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  

<link rel="next" href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"/>
<script src="libs/header-attrs-2.8/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="1" data-path="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html"><a href="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html"><i class="fa fa-check"></i><b>1</b> Post-transcriptional gene regulation instructs germline stem cell to oocyte transition during <em>Drosophila</em> oogenesis</a>
<ul>
<li class="chapter" data-level="1.1" data-path="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html"><a href="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html#abstract"><i class="fa fa-check"></i><b>1.1</b> Abstract</a></li>
<li class="chapter" data-level="1.2" data-path="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html"><a href="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html#introduction"><i class="fa fa-check"></i><b>1.2</b> Introduction</a></li>
<li class="chapter" data-level="1.3" data-path="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html"><a href="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html#alternative-splicing-ensures-accurate-production-of-critical-germline-mrnas-to-regulate-sex-determination-and-differentiation"><i class="fa fa-check"></i><b>1.3</b> Alternative splicing ensures accurate production of critical germline mRNAs to regulate sex determination and differentiation</a></li>
<li class="chapter" data-level="1.4" data-path="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html"><a href="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html#rna-modifications-direct-splicing-of-sex-determinants-and-translation-of-differentiation-promoting-genes-in-the-germline"><i class="fa fa-check"></i><b>1.4</b> RNA modifications direct splicing of sex determinants and translation of differentiation promoting genes in the germline</a></li>
<li class="chapter" data-level="1.5" data-path="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html"><a href="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html#production-of-ribosomes-is-finely-tuned-to-facilitate-differentiation"><i class="fa fa-check"></i><b>1.5</b> Production of ribosomes is finely tuned to facilitate differentiation</a></li>
<li class="chapter" data-level="1.6" data-path="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html"><a href="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html#hand-off-mechanisms-facilitated-by-combinatorial-rna-binding-proteins-dynamically-shape-the-translational-landscape-during-oogenesis"><i class="fa fa-check"></i><b>1.6</b> Hand off mechanisms facilitated by combinatorial RNA binding proteins dynamically shape the translational landscape during oogenesis</a></li>
<li class="chapter" data-level="1.7" data-path="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html"><a href="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html#summary"><i class="fa fa-check"></i><b>1.7</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><i class="fa fa-check"></i><b>2</b> A translation control module coordinates germline stem cell differentiation with ribosome biogenesis during <em>Drosophila</em> oogenesis</a>
<ul>
<li class="chapter" data-level="2.1" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#summary-1"><i class="fa fa-check"></i><b>2.1</b> Summary:</a></li>
<li class="chapter" data-level="2.2" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#introduction-1"><i class="fa fa-check"></i><b>2.2</b> Introduction</a></li>
<li class="chapter" data-level="2.3" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#results"><i class="fa fa-check"></i><b>2.3</b> Results</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#three-conserved-rna-helicases-are-required-in-the-germline-for-gsc-differentiation"><i class="fa fa-check"></i><b>2.3.1</b> Three conserved RNA helicases are required in the germline for GSC differentiation</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#athos-aramis-and-porthos-are-required-for-ribosome-biogenesis"><i class="fa fa-check"></i><b>2.3.2</b> Athos, Aramis, and Porthos are required for ribosome biogenesis</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#aramis-promotes-cell-cycle-progression-via-p53-repression"><i class="fa fa-check"></i><b>2.3.3</b> Aramis promotes cell cycle progression via p53 repression</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#aramis-regulated-targets-contain-a-top-motif-in-their-5utr"><i class="fa fa-check"></i><b>2.3.4</b> Aramis-regulated targets contain a TOP motif in their 5'UTR</a></li>
<li class="chapter" data-level="2.3.5" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#larp-binds-top-sequences-in-drosophila"><i class="fa fa-check"></i><b>2.3.5</b> Larp binds TOP sequences in <em>Drosophila</em></a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#discussion"><i class="fa fa-check"></i><b>2.4</b> Discussion</a>
<ul>
<li class="chapter" data-level="2.4.1" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#aramis-athos-and-porthos-are-required-for-efficient-ribosome-biogenesis-in-drosophila"><i class="fa fa-check"></i><b>2.4.1</b> Aramis, Athos, and Porthos are required for efficient ribosome biogenesis in <em>Drosophila</em></a></li>
<li class="chapter" data-level="2.4.2" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#ribosome-biogenesis-defects-leads-to-cell-cycle-defects-mediated-by-p53"><i class="fa fa-check"></i><b>2.4.2</b> Ribosome biogenesis defects leads to cell cycle defects mediated by p53</a></li>
<li class="chapter" data-level="2.4.3" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#ribosome-biogenesis-defects-leads-to-repression-of-top-containing-mrna"><i class="fa fa-check"></i><b>2.4.3</b> Ribosome biogenesis defects leads to repression of TOP-containing mRNA</a></li>
<li class="chapter" data-level="2.4.4" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#larp-transduces-growth-status-to-ribosome-biogenesis-targets"><i class="fa fa-check"></i><b>2.4.4</b> Larp transduces growth status to ribosome biogenesis targets</a></li>
<li class="chapter" data-level="2.4.5" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#ribosome-biogenesis-in-stem-cell-differentiation-and-ribosomopathies"><i class="fa fa-check"></i><b>2.4.5</b> Ribosome biogenesis in stem cell differentiation and ribosomopathies</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#materials-and-methods"><i class="fa fa-check"></i><b>2.5</b> Materials and Methods</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><i class="fa fa-check"></i><b>3</b> Oo-site: A dashboard to visualize gene expression during <em>Drosophila</em> oogenesis reveals meiotic entry is regulated post-transcriptionally</a>
<ul>
<li class="chapter" data-level="3.1" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#summary-2"><i class="fa fa-check"></i><b>3.1</b> Summary</a></li>
<li class="chapter" data-level="3.2" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#introduction-2"><i class="fa fa-check"></i><b>3.2</b> Introduction</a></li>
<li class="chapter" data-level="3.3" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#results-1"><i class="fa fa-check"></i><b>3.3</b> Results</a></li>
<li class="chapter" data-level="3.4" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#discussion-1"><i class="fa fa-check"></i><b>3.4</b> Discussion</a></li>
<li class="chapter" data-level="3.5" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#acknowledgements"><i class="fa fa-check"></i><b>3.5</b> Acknowledgements</a></li>
<li class="chapter" data-level="3.6" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#materials-and-methods-1"><i class="fa fa-check"></i><b>3.6</b> Materials and Methods</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-conclusion.html"><a href="4-conclusion.html"><i class="fa fa-check"></i><b>4</b> Conclusion</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-the-first-appendix.html"><a href="A-the-first-appendix.html"><i class="fa fa-check"></i><b>A</b> The First Appendix</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Translation control tunes <em>Drosophila</em> oogenesis</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="header">
<h1 class="title">Translation control tunes <em>Drosophila</em> oogenesis</h1>
<p class="author"><em>Elliot T. Martin</em></p>
<p class="date"><em>May 2022</em></p>
</div>
<div id="post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis" class="section level1" number="1">
<h1><span class="header-section-number">Chapter 1</span> Post-transcriptional gene regulation instructs germline stem cell to oocyte transition during <em>Drosophila</em> oogenesis</h1>
<p>Patrick Blatt, Elliot T. Martin, Shane M. Breznak, Prashanth Rangan*</p>
<p>Department of Biological Sciences/RNA Institute, University at Albany
SUNY, Albany, NY
University at Albany SUNY, 1400 Washington Avenue, Albany, 12222, USA
*Correspondence to: prangan@albany.edu</p>
<div id="abstract" class="section level2" number="1.1">
<h2><span class="header-section-number">1.1</span> Abstract</h2>
<p>During oogenesis, several developmental processes must be traversed to
ensure effective completion of gametogenesis including, stem cell
maintenance and asymmetric division, differentiation, mitosis and
meiosis, and production of maternally contributed mRNAs, making the germ
line a salient model for understanding how cell fate transitions are
mediated. Due to silencing of the genome during meiotic divisions, there
is little instructive transcription, barring a few examples, to mediate
these critical transitions. In <em>Drosophila</em>, several layers of
post-transcriptional regulation ensure that the mRNAs required for these
processes are expressed in a timely manner and as needed during germline
differentiation. These layers of regulation include alternative
splicing, RNA modification, ribosome production, and translational
repression. Many of the molecules and pathways involved in these
regulatory activities are conserved from <em>Drosophila</em> to humans making
the <em>Drosophila</em> germline an elegant model for studying the role of
post-transcriptional regulation during stem cell differentiation and
meiosis.</p>
<p><strong>Key words</strong><br />
<u>Splicing, Translation Control, RNA Modifications, Ribosome Biogenesis,</u><br />
<u>Oogenesis, Drosophila, Germline Stem Cell, RNA regulation, Germline,</u><br />
<u>Differentiation, Gametogenesis, RNA Binding Proteins</u><br />
</p>
</div>
<div id="introduction" class="section level2" number="1.2">
<h2><span class="header-section-number">1.2</span> Introduction</h2>
<p>Gametogenesis gives rise to eggs or sperm in all sexually-reproducing
organisms <span class="citation">(Cinalli, Rangan, &amp; Lehmann, 2008; Ellis &amp; Kimble, 1994; Lesch &amp; Page, 2012; Seydoux &amp; Braun, 2006)</span>.
Thus, understanding how gametogenesis is regulated is critical to
comprehending this essential phenomenon that dictates fertility.
Post-fertilization, the zygote gives rise to an entire organism, thus
understanding how gametogenesis is regulated also has implications for
the field of regeneration <span class="citation">(Lasko, 2012; K.-A. Lee &amp; Lee, 2014; Magnúsdóttir &amp; Surani, 2014; Soldner &amp; Jaenisch, 2018; Tadros &amp; Lipshitz, 2009; Theunissen &amp; Jaenisch, 2017)</span>.
<em>Drosophila melanogaster</em> has been one of the central organisms used to
study heritability and gametogenesis for nearly a century due to its
rapid generation time and genetic tractability <span class="citation">(Mattox, Palmer, &amp; Baker, 1990; Allan C. Spradling, 1993; A. C. Spradling et al., 1997; Allan C. Spradling &amp; Rubin, 1981; Allan Spradling, Fuller, Braun, &amp; Yoshida, 2011; Xie &amp; Li, 2007)</span>. These traits have facilitated the establishment of an
extensive collection of informative and useful mutant and transgenic
flies <span class="citation">(Hales, Korey, Larracuente, &amp; Roberts, 2015)</span>. In addition, many of the gametogenic regulatory
factors described in the <em>Drosophila</em> germ line are conserved to mammals
and also play critical roles in other tissues, such as neurons
<span class="citation">(Goldstrohm, Hall, &amp; McKenney, 2018; H. Lin &amp; Spradling, 1997; Reichardt et al., 2018; Vessey et al., 2010; Zamore, Bartel, Lehmann, &amp; Williamson, 1999; K. Zhang &amp; Smith, 2015)</span>. While both male and female <em>Drosophila</em>
undergo meiosis to give rise to gametes, here we focus on the female
germline as regulation of gametogenesis in males has been reviewed
elsewhere <span class="citation">(Barreau, Benson, Gudmannsdottir, Newton, &amp; White-Cooper, 2008; Fuller, 1998; Allan Spradling, Fuller, Braun, &amp; Yoshida, 2011; Yamashita &amp; Fuller, 2005; G.-Q. Zhao &amp; Garbers, 2002)</span>.</p>
<p>The spatiotemporal stages of <em>Drosophila</em> oogenesis are discrete and can
be easily identified by their morphology and molecular markers
<span class="citation">(Gáspár &amp; Ephrussi, 2017; D. Jia, Xu, Xie, Mio, &amp; Deng, 2016; Allan Spradling, Fuller, Braun, &amp; Yoshida, 2011)</span>. At the anterior end of
the ovary, germline stem cells (GSCs) reside in a structure known as the
germarium and initiate differentiation to give rise to gametes
<span class="citation">(Kai, Williams, &amp; Spradling, 2005; Twombly et al., 1996; Xie &amp; Li, 2007; Xie &amp; Spradling, 1998, 2000)</span>. GSCs
are maintained by signaling from the surrounding somatic niche. GSCs
undergo asymmetric mitotic division, producing a stem cell daughter, or
cystoblast (CB) which will begin the process of differentiation by
expressing the essential differentiation factor <em>bag of marbles</em> (<em>bam</em>)
<span class="citation">(D. Chen &amp; McKearin, 2003b; D. McKearin &amp; Ohlstein, 1995)</span>. The differentiating CB then undergoes
four incomplete mitotic divisions, giving rise to an interconnected
16-cell cyst <span class="citation">(D. M. McKearin &amp; Spradling, 1990; D. McKearin &amp; Ohlstein, 1995)</span>. In this cyst, one
cell is designated to become the oocyte and the other 15 cells take on
the role of nurse cells, which generate proteins and mRNAs that are
provided to the developing oocyte <span class="citation">(Navarro, Puthalakath, Adams, Strasser, &amp; Lehmann, 2004; A. C. Spradling et al., 1997)</span>.
The specified oocyte and its associated nurse cells are then
encapsulated by somatic cells to form an egg chamber that buds off from
the germarium (Figure 1B) <span class="citation">(Gilboa &amp; Lehmann, 2004; Margolis &amp; Spradling, 1995)</span>. The nurse
cells will enter into a unique state in which they undergo a modified
version of the cell cycle without undergoing mitosis, creating polyploid
nuclei capable of fulfilling the high transcriptional demand required to
transcribe all of the mRNAs necessary for the egg <span class="citation">(Lilly &amp; Duronio, 2005; Royzman &amp; Orr-Weaver, 1998)</span>. As this process ensues, the egg chambers and oocyte
increase in size as the supply of mRNAs and proteins is created and
deposited into the mature egg (Figure 1A) <span class="citation">(Lasko, 2012; Richter &amp; Lasko, 2011)</span>.</p>
<p>Oocyte development entails multiple processes that ensure effective
completion of gametogenesis and fertility. Among these are stem cell
maintenance and asymmetric division, differentiation, mitosis and
meiosis, and production of the maternal mRNA contribution, thus the germ
line is a salient model for understanding how cells navigate fate
transitions <span class="citation">(D. Chen &amp; McKearin, 2003b; Fu et al., 2015; Harris, Pargett, Sutcliffe, Umulis, &amp; Ashe, 2011; Lasko, 2012)</span>.
During oogenesis, there is little instructive transcription, barring a
few examples, to mediate these critical transitions <span class="citation">(Cinalli, Rangan, &amp; Lehmann, 2008; P. Rangan, DeGennaro, &amp; Lehmann, 2008)</span>. Instead, the germline relies highly on
post-transcriptional regulatory mechanisms to coordinate gametogenesis
<span class="citation">(Slaidina &amp; Lehmann, 2014)</span>. These include: alternative splicing, RNA
modifications to modulate splicing, protein-RNA interactions, small RNA
biology, and organization of the translation machinery to control the
output of gene expression to mediate cell fate transitions. Here we
focus on post-transcriptional processing of germline mRNAs and
translational regulation both of which are required for successful
oogenesis.</p>
<div class="figure" style="text-align: center"><span id="fig:unnamed-chunk-4"></span>
<img src="figure/Book%20Chapter/Figure1.png" alt="(A) Schematic of Drosophila ovariole. Drosophila females have two ovaries consisting of 16–20 ovarioles, which are assembly lines for producing mature eggs. The germarium, the structure that houses the germline stem cell (GSC), is present at anterior tip of the ovariole. The germline stem cell asymmetrically divides, giving rise to another GSC and a GSC daughter. The daughter cell then will undergo four incomplete rounds of mitosis, giving rise to a 16-cell cyst. Of the 16 cells one will be specified as the egg while the others serve as polyploid nurse cells that support oocyte and egg development. The surrounding somatic cells encapsulate the 16-cell cyst creating egg chambers. As development proceeds, the nurse cells provide mRNAs and proteins allowing the oocyte to grow in size and to eventually become a mature egg. (B) Inset of a germarium showing the developing germline, with the GSC located at the most anterior tip. Upon differentiation, the CB will undergo 4 incomplete mitotic divisions giving rise to a 16-cell cyst. Only one cell of the sixteen cells completes meiosis and is destined to become the oocyte." width="97%" />
<p class="caption">
Figure 1.1: (A) <strong>Schematic of <em>Drosophila</em> ovariole.</strong> <em>Drosophila</em> females have two ovaries consisting of 16–20 ovarioles, which are assembly lines for producing mature eggs. The germarium, the structure that houses the germline stem cell (GSC), is present at anterior tip of the ovariole. The germline stem cell asymmetrically divides, giving rise to another GSC and a GSC daughter. The daughter cell then will undergo four incomplete rounds of mitosis, giving rise to a 16-cell cyst. Of the 16 cells one will be specified as the egg while the others serve as polyploid nurse cells that support oocyte and egg development. The surrounding somatic cells encapsulate the 16-cell cyst creating egg chambers. As development proceeds, the nurse cells provide mRNAs and proteins allowing the oocyte to grow in size and to eventually become a mature egg. (B) Inset of a germarium showing the developing germline, with the GSC located at the most anterior tip. Upon differentiation, the CB will undergo 4 incomplete mitotic divisions giving rise to a 16-cell cyst. Only one cell of the sixteen cells completes meiosis and is destined to become the oocyte.
</p>
</div>
</div>
<div id="alternative-splicing-ensures-accurate-production-of-critical-germline-mrnas-to-regulate-sex-determination-and-differentiation" class="section level2" number="1.3">
<h2><span class="header-section-number">1.3</span> Alternative splicing ensures accurate production of critical germline mRNAs to regulate sex determination and differentiation</h2>
<p>Splicing decisions are crucial during the generation of mature mRNAs
post-transcriptionally and significantly contribute to germline
development. Splicing is mediated by a large ribonucleoprotein catalytic
complex called the spliceosome, the core of which is made up of five
small nuclear RNAs (snRNA), U1, U2, U4, U5 and U6, that work with
spliceosomal proteins to form a small nuclear ribonucleoprotein complex
(snRNP) <span class="citation">(Madhani, Bordonne, &amp; Guthrie, 1990; Wahl, Will, &amp; Lührmann, 2009; Will &amp; Lührmann, 2001, 2011)</span>. This
complex removes introns from newly synthesized pre-mRNAs and links
exonic sequences together <span class="citation">(Wahl, Will, &amp; Lührmann, 2009)</span>. Initially, U1 snRNP recognizes
the donor site, which is located at the 5' end of the intron, and U2
snRNP binds the branch site located at the 3' end, leading to structural
rearrangements of the complex and its associated substrate pre-mRNA
<span class="citation">(Matera &amp; Wang, 2014)</span>. Catalytic actions of pre-mRNA splicing occur in two
main steps. Cleavage at the 5' splice site forms a lariat-like structure
such that a 2'-5' phosphodiester bond is created between the first
nucleotide of the donor site and a conserved adenosine residue at the
branch site <span class="citation">(Rymond &amp; Rosbash, 1985)</span>. Next, a second cleavage event occurs at
the 3' splice site and is followed by ligation of flanking exons to
complete splicing <span class="citation">(Umen &amp; Guthrie, 1995; Wahl, Will, &amp; Lührmann, 2009)</span>.</p>
<p>Alternative splicing is a process by which a single locus can give rise
to many unique mRNA isoforms and their resulting protein variants
<span class="citation">(Black, 2000)</span>. The selection of the splice sites is exquisitely
regulated to determine which exons will be included in the resulting
alternatively spliced transcripts <span class="citation">(Y. Wang et al., 2015)</span>. Alternative splicing
is highly regulated and is critical to germline development
<span class="citation">(Hager &amp; Cline, 1997; Kalsotra &amp; Cooper, 2011)</span>. There are a myriad of RNA targets that
must be differentially spliced, and a complex web of interacting
proteins orchestrate production of their splice variants <span class="citation">(Y. Lee &amp; Rio, 2015)</span>.
One of the first described instances of alternative splicing in
<em>Drosophila</em> females is the splicing of the sex determination gene
<em>sex-lethal</em> (<em>sxl</em>) <span class="citation">(Bell, Maine, Schedl, &amp; Cline, 1988)</span>. <em>sxl</em> is alternatively spliced to
generate isoforms that control sex determination in somatic tissues
<span class="citation">(Chang, Dunham, Nuzhdin, &amp; Arbeitman, 2011)</span>. In females, an autoregulatory loop forms between Sxl
protein, U2AF splicing factor and U1 snRNP <span class="citation">(Nagengast, Stitzinger, Tseng, Mount, &amp; Salz, 2003)</span>. In
<em>Drosophila</em>, the protein component of the U1 and U2 snRNPs are encoded
by a gene called <em>sans fille</em> (<em>snf</em>) <span class="citation">(Cline, Rudner, Barbash, Bell, &amp; Vutien, 1999)</span>. Loss of <em>snf</em>
results in a sterility phenotype in females that specifically affects
germline <em>sxl</em> splicing and leads to a tumor comprised of
undifferentiated cells <span class="citation">(Johnson, Nagengast, &amp; Salz, 2010; Nagengast, Stitzinger, Tseng, Mount, &amp; Salz, 2003)</span>. When
correctly spliced, the resulting Sxl protein recognizes its own
pre-mRNAs by binding both upstream and downstream of Exon 3
<span class="citation">(Penalva &amp; Sánchez, 2003)</span>. In addition, Sxl protein interacts with the U2AF and
U1 snRNP to block the recognition of splice sites at Exon 3
<span class="citation">(Nagengast, Stitzinger, Tseng, Mount, &amp; Salz, 2003)</span>. As a result, exon 3 is spliced out of the pre-mRNA
in the final transcript that is capable of being translated into a fully
functional protein <span class="citation">(Penalva &amp; Sánchez, 2003)</span>. In contrast, males include exon
three in the final <em>sxl</em> transcript. Exon 3 contains a premature stop
codon within the <em>sxl</em> transcript that results in a truncated protein
that lacks the activity of the female-specific variant <span class="citation">(Inoue, Hoshijima, Sakamoto, &amp; Shimura, 1990)</span>.
Thus, <em>sxl</em> is differentially expressed in the male and the female gonad
due to alternative splicing events.</p>
<p>In addition to control of <em>sxl</em> via alternative splicing, <em>sxl</em>
expression is controlled at the level of transcription by several
transcription factors, such as Ovo <span class="citation">(Salles, Mével-Ninio, Vincent, &amp; Payre, 2002)</span>. Ovo is a zinc
finger DNA binding protein that is required in the germline for proper
gametogenesis<span class="citation">(Andrews et al., 2000)</span>. <em>ovo</em> is also alternatively spliced and
each of its isoforms have different implications for <em>sxl</em> expression.
Ovo-A and Ovo-B where the first splice variants of <em>ovo</em> shown to be
expressed in the female germline during oogenesis <span class="citation">(Salles, Mével-Ninio, Vincent, &amp; Payre, 2002)</span>. In
addition to differences due to alternative exon usage, Ovo-A, unlike
Ovo-B, contains a 381 amino acid N-terminal extension which arises due
to alternative transcription start sites <span class="citation">(Andrews et al., 2000)</span>. Use of
these promoters generates distinct Ova isoforms with unique temporal
requirements during oocyte development; Ovo-B was found to be necessary
and sufficient during early oogenesis and Ovo-A is critical in the later
stages of egg development for a fully functional egg. The <em>ovo-B</em> gene
has two characterized isoforms, Ovo+2B and Ovo-2B, which were discovered
through a transposon insertion that disrupts exon splicing of <em>ovo-B</em>.
This transposition event prevents inclusion of the exon 2b extension,
producing a nonfunctional protein that accumulates during oogenesis. In
the absence of retrotransposon insertion, the 178-amino acid extension
encoded by exon 2b is included forming a fully functional Ovo protein,
known as Ovo-+2B <span class="citation">(Salles, Mével-Ninio, Vincent, &amp; Payre, 2002)</span>. Interestingly, Ovo-B promotes
transcription of <em>ovarian tumor</em> (<em>otu</em>), which enhances <em>sxl</em>
expression (Figure 2) <span class="citation">(J. Lu &amp; Oliver, 2001)</span>. The mechanism by which Otu regulates
sxl expression is unknown but various mutations in <em>otu</em> lead to a
myriad of phenotypes such as loss of germ cell proliferation, and
inability to complete the differentiation process. The <em>otu</em> gene
produces two cytoplasmic protein isoforms, a 104-kDA isoform (Otu-104)
and a 98-kDA isoform (Otu-98) <span class="citation">(Tirronen, Lahti, Heino, &amp; Roos, 1995)</span>. Strikingly, only
Otu-104 is capable of rescuing all the <em>otu</em> mutant phenotypes,
indicating its requirement during oogenesis, while Otu-98 is dispensable
during this process <span class="citation">(Tirronen, Lahti, Heino, &amp; Roos, 1995)</span>. Despite the lack of insight into
how the <em>otu</em> splice forms regulate GSC development, its alternative
splicing is critical for oogenesis <span class="citation">(Sass, Comer, &amp; Searles, 1995)</span>. Thus, a cascade of
alternative splicing events regulate production of Sxl in the female
germline to promote oogenesis (Figure 2).</p>
<p>Sxl expression in the female gonad regulates both sex determination as
well as differentiation <span class="citation">(Chau, Kulnane, &amp; Salz, 2012)</span>. One critical task of Sxl is to
represses Tudor domain containing protein 5-like (<em>tdrd5l</em>)
<span class="citation">(Primus, Pozmanter, Baxter, &amp; Van Doren, 2019)</span>. Tdrd5l is present in the cytoplasm of the male
germline, localizing to granules associated with RNA regulation, to
promote male identity and differentiation. Sxl expression the female
gonad represses translation of Tdrd5l to promote female identity
<span class="citation">(Primus, Pozmanter, Baxter, &amp; Van Doren, 2019)</span>. In addition, female Sxl has been found to regulate
transcription of <em>PHD finger protein 7 (phf7)</em>, a key regulator of male
identity <span class="citation">(S. Y. Yang, Baxter, &amp; Van Doren, 2012)</span>. Sxl was found to recruit SETDB1, a chromatin
writer, to deposit trimethylated H3K9 (H3K9me3) repressing transcription
of <em>phf7</em> <span class="citation">(Smolko, Shapiro-Kulnane, &amp; Salz, 2018)</span>. Thus, alternative splicing of <em>sxl</em> results
in different sexes helps promote proper sex determination in the
germline (Figure 2). Sxl also fulfills additional functions outside of
sex determination. Sxl is required in the female germline for germline
stem cell GSC differentiation. Loss of Sxl protein causes an
accumulation of single cells and two cell cysts <span class="citation">(Chau, Kulnane, &amp; Salz, 2009)</span>. It is
thought that Sxl binds <em>nanos</em> (<em>nos</em>) mRNA, an RNA binding protein that
is necessary for GSC self-renewal, using a canonical Sxl binding
sequence in the 3' UTR <span class="citation">(Chau, Kulnane, &amp; Salz, 2012)</span>. Loss of Sxl leads to an
accumulation of excess of Nanos protein, which is thought to limit? GSC
differentiation <span class="citation">(Boerner &amp; Becker, 2016; Chau, Kulnane, &amp; Salz, 2012; Y. Li et al., 2013)</span>. While
regulation by Sxl is beginning to be deciphered, several aspects remain
to be discovered. For example, Sxl, a splicing factor, is predominantly
cytoplasmic in undifferentiated cells but becomes nuclear as
differentiation proceeds <span class="citation">(Chau, Kulnane, &amp; Salz, 2009)</span>, yet, how it works as
translational regulator while in the cytoplasm and how it is transported
to the nucleus to function as splicing factor during differentiation are
not known.</p>
<p>Polypyrimidine tract binding proteins (PTBs) promote splicing by binding
polypyrimidine tracts that are ~10nt long and bring splice sites
together by means of protein dimerization to promote alternative
splicing <span class="citation">(Polydorides, Okano, Yang, Stefani, &amp; Darnell, 2000; Romanelli, Diani, &amp; Lievens, 2013)</span>. A PTB, <em>half pint</em>
(<em>hfp</em>), a homolog of human PUF60, is important for oogenesis
<span class="citation">(Maniatis &amp; Tasic, 2002)</span>. Loss of <em>hfp</em> results in missplicing of the <em>otu</em>
transcripts described above <span class="citation">(Van Buskirk &amp; Schüpbach, 2002)</span>. In addition, <em>hfp</em>
also regulates alternative splicing of <em>eukaryotic initiation factor 4E</em>
(<em>eIF4E</em>) during development through 3' splice site selection
<span class="citation">(Reyes &amp; Izquierdo, 2008)</span>. Hfp is required to increase the relative abundance of
the longer <em>eIF4E</em> transcript <span class="citation">(Van Buskirk &amp; Schüpbach, 2002)</span>. Lastly, <em>hfp</em> also
regulates splicing of <em>gurken</em>, a critical regulator of dorsal-ventral
patterning <span class="citation">(Kalifa, Armenti, &amp; Gavis, 2009)</span>. Thus, sex determination, differentiation
and production of the determinants of embryonic patterning for the next
generation are all regulated by mechanisms involving alternative
splicing in the female germline.</p>
<div class="figure" style="text-align: center"><span id="fig:unnamed-chunk-5"></span>
<img src="figure/Book%20Chapter/Figure2.png" alt="Schematic of the pathway that promotes alternative splicing of sxl to generate the female sex determining variant in the germline. Ovo-B promotes the transcription of otu, which enhances splicing of sxl. The female-specific splice form of sxl is further enhanced by RNA modification by the m6A writer. Formation of the female-specific form generates a functional Sxl protein. Sxl represses Tdrd5l, a protein that promotes male identify. Additionally, Sxl post-transcriptionally represses nhp2 to promote cyst formation during differentiation." width="97%" />
<p class="caption">
Figure 1.2: <strong>Schematic of the pathway that promotes alternative splicing of <em>sxl</em> to generate the female <em>sex determining</em> variant in the germline.</strong> Ovo-B promotes the transcription of <em>otu,</em> which enhances splicing of <em>sxl</em>. The female-specific splice form of <em>sxl</em> is further enhanced by RNA modification by the m6A writer. Formation of the female-specific form generates a functional Sxl protein. Sxl represses Tdrd5l, a protein that promotes male identify. Additionally, Sxl post-transcriptionally represses <em>nhp2</em> to promote cyst formation during differentiation.
</p>
</div>
</div>
<div id="rna-modifications-direct-splicing-of-sex-determinants-and-translation-of-differentiation-promoting-genes-in-the-germline" class="section level2" number="1.4">
<h2><span class="header-section-number">1.4</span> RNA modifications direct splicing of sex determinants and translation of differentiation promoting genes in the germline</h2>
<p>Post transcriptional RNA modifications are abundant and conserved in all
branches of life <span class="citation">(Yi &amp; Pan, 2011)</span>. There have been over 100 described RNA
modifications that can alter stability, function and splicing of RNAs
<span class="citation">(Licht &amp; Jantsch, 2016; Roundtree, Evans, Pan, &amp; He, 2017)</span>. A well-known example of an mRNA
modification is the 5' methylguanosine cap that is added to all mRNAs to
promote their stability and aid in translation initiation
<span class="citation">(Mitchell et al., 2010; Mukherjee et al., 2012)</span>. A variety of RNA modifications have
been linked to developmental transitions, such as those affecting GSC
fate <span class="citation">(Batista et al., 2014; Roundtree, Evans, Pan, &amp; He, 2017)</span>. Specifically during oogenesis,
N6A-methyladenosine (m<sup>6</sup>A) has been shown to be important for
differentiation of germline stem cell daughter cells in females by
ensuring proper female-specific splicing of <em>sxl</em> <span class="citation">(Haussmann et al., 2016)</span>.
Additionally, the H/ACA box complex, an RNP complex responsible for
depositing pseudouridine on rRNA, has been suggested to be regulated by
Sxl during the germline stem cell to daughter cell transition and is
required for proper cyst differentiation <span class="citation">(T. Kiss, Fayet-Lebaron, &amp; Jády, 2010; Morita, Ota, &amp; Kobayashi, 2018)</span>.</p>
<p>m<sup>6</sup>A is prevalent on mRNA and is mediated by a methyltransferase
complex that deposits a methyl-group at the sixth nitrogen on adenosine
<span class="citation">(Y.-G. Y. Yang, Hsu, Chen, &amp; Yang, 2018)</span>. In <em>Drosophila,</em> m<sup>6</sup>A is placed by a m<sup>6</sup>A writer
complex consisting of Xio, Virilizer (Vir), Spenito (Nito), female
lethal d (fl(2)d), Methyltransferase like 3 (Mettl3) and
Methyltransferase like 14 (Mettl14) <span class="citation">(Yan &amp; Perrimon, 2015)</span>. Some described roles
of m<sup>6</sup>A involve modulating RNA-structure, facilitating mRNA
degradation, promoting translation initiation and mediating alternative
splicing <span class="citation">(Roundtree, Evans, Pan, &amp; He, 2017)</span>. Interestingly, the m<sup>6</sup>A writer complex
has been linked to <em>sxl</em> splicing during <em>Drosophila</em> oogenesis
<span class="citation">(Kan et al., 2017)</span>. miCLIP data revealed that m<sup>6</sup>A must be placed at
intergenic regions of the <em>sxl</em> mRNA in order to produce the
female-specific isoform <span class="citation">(Kan et al., 2017)</span>. Accordingly. loss of m<sup>6</sup>A
complex members such as <em>spenito</em> result in expression of the male
specific isoform of <em>sxl</em>, and tumors of undifferentiated cells, similar
to loss of <em>sxl</em> <span class="citation">(Mattox, Palmer, &amp; Baker, 1990)</span>; <span class="citation">(Kan et al., 2017)</span>. This suggests that
m<sup>6</sup>A enables proper splicing of female-specific <em>sxl</em>, which allows for
proper differentiation of germline stem cells into cystoblast daughter
cells (Figure 2).</p>
<p>Pseudouridine is one of the most abundant RNA modifications
<span class="citation">(B. S. Zhao &amp; He, 2015)</span>. Although most commonly found on tRNAs, pseudouridine is
also found on mRNAs as well as rRNA <span class="citation">(Penzo &amp; Montanaro, 2018)</span>. Unlike the
canonical nucleoside uridine which is attached to the sugar via a
nitrogen-carbon bond, pseudouridine is a uridine isomer attached through
a carbon-carbon bond <span class="citation">(Cohn, 1960)</span>. Pseudouridine can be placed by two
different classes of enzymes; either by a sequence specific
pseudouridine synthase or a small RNA guided complex called the box
H/ACA ribonucleoprotein <span class="citation">(De Zoysa &amp; Yu, 2017)</span>. Depletion of the H/ACA box
complex member Nucleolar Protein Family A Member 2 (NHP2) in the
germline leads to an accumulation of 4- and 8- cell cysts that do not
transition to the 16-cell cyst stage <span class="citation">(Morita, Ota, &amp; Kobayashi, 2018)</span>. Interestingly,
the accumulation of single cells due to loss of <em>sxl</em> is partially
rescued by loss of <em>NHP2 indicating that this sxl phenotype is due to
excess NHP2</em> <span class="citation">(Morita, Ota, &amp; Kobayashi, 2018)</span>. Consistent with this notion, Sxl
interacts with <em>nhp2</em> mRNA suggesting that Sxl may impose a regulatory
function, in this case likely repression of <em>nhp2</em> to allow initiation
of the differentiation program (Figure 2) <span class="citation">(Morita, Ota, &amp; Kobayashi, 2018)</span>. Thus,
although it is clear that RNA modifications help to ensure proper
splicing of sex determination factors, but the pathway, mechanism, and
direct targets remain unresolved.</p>
</div>
<div id="production-of-ribosomes-is-finely-tuned-to-facilitate-differentiation" class="section level2" number="1.5">
<h2><span class="header-section-number">1.5</span> Production of ribosomes is finely tuned to facilitate differentiation</h2>
<p>While splicing mediates proper mRNA production, access of the mature
mRNAs to ribosomes controls their translation. Once mRNAs are gated for
translation, proper ribosome levels control protein production. The
levels of ribosomes during early oogenesis are strictly regulated and
shockingly dynamic. Ribosome biogenesis is the process of transcribing
and processing the ribosomal RNA (rRNA) components, as well as
transcribing and translating the protein constituents of the ribosome
<span class="citation">(Granneman &amp; Baserga, 2004; Nazar, 2004; Teng, Thomas, &amp; Mercer, 2013; Yelick &amp; Trainor, 2015)</span>. This
process is exquisitely regulated as ribosome biogenesis is one of the
most energy intensive tasks of maintaining cell homeostasis and is even
more crucial in proliferative cells <span class="citation">(Phipps, Charette, &amp; Baserga, 2011)</span>. In addition to
the high energy requirement of ribosome biogenesis, all of the
components of the ribosome must be coordinated in their production. The
process of ribosome biogenesis involves a series of coordinated steps of
processing and assembly that involve dozens of non-coding RNAs and
proteins and the molecular details of this process have been thoroughly
covered in detail in several recent reviews <span class="citation">(Granneman &amp; Baserga, 2004; Yelick &amp; Trainor, 2015; You, Park, &amp; Kim, 2015)</span>. Briefly, ribosomal DNA (rDNA) is present in
multicopy stretches within the genome; these areas of DNA are localized
to a subnuclear organelle called the nucleolus <span class="citation">(Karpen, Schaefer, &amp; Laird, 1988; Ritossa &amp; Spiegelman, 1965; Schwarzacher &amp; Wachtler, 1993)</span>. rDNA is transcribed into rRNA in
the nucleolus and processing steps begin cotranscriptionally
<span class="citation">(Koš &amp; Tollervey, 2010)</span> to remove internal and external spacers found in immature
rRNA <span class="citation">(Granneman &amp; Baserga, 2004; Granneman, Petfalski, Tollervey, &amp; Hurt, 2011; Schäfer, Strauß, Petfalski, Tollervey, &amp; Hurt, 2003; Tafforeau et al., 2013)</span>. As these processing steps occur, the rRNA is
covalently modified and ribosomal proteins begin to interact with the
partially processed rRNA <span class="citation">(Agalarov, Sridhar, Funke, Stout, &amp; Williamson, 2000; Deshmukh, Tsay, Paulovich, &amp; Woolford, 1993; Gumienny et al., 2017; Jády &amp; Kiss, 2001; A. M. Kiss, Jady, Bertrand, &amp; Kiss, 2004)</span>. When the rRNA is mostly
mature it is exported from the nucleus to the cytoplasm where the small
and large subunits of the ribosome fully mature and assemble
<span class="citation">(Lo et al., 2010; Schäfer, Strauß, Petfalski, Tollervey, &amp; Hurt, 2003; Sloan et al., 2017; Tschochner &amp; Hurt, 2003; Zemp &amp; Kutay, 2007)</span>. Errors at any of these steps can result in ribosome
biogenesis defects which in humans result in disease states known as
ribosomopathies <span class="citation">(Armistead &amp; Triggs-Raine, 2014; Barlow et al., 2010; Brooks et al., 2014; Higa-Nakamine et al., 2012; Mills &amp; Green, 2017; Sloan et al., 2017)</span>.</p>
<p>Curiously, despite the presence of ribosomes across cell types and
sharing similar molecular origins, ribosomopathies manifest as tissue
specific defects rather than pleiotropic phenotypes <span class="citation">(Brooks et al., 2014; Higa-Nakamine et al., 2012; Mills &amp; Green, 2017; Pereboom, van Weele, Bondt, &amp; MacInnes, 2011; Yelick &amp; Trainor, 2015)</span>.
The reasons behind the unique, tissue-specific manifestations are still
being investigated but in several cases it seems that stem cells may be
particularly sensitive to perturbations in ribosome biogenesis
<span class="citation">(Brooks et al., 2014; Morgado-Palacin, Llanos, &amp; Serrano, 2012; Pereboom, van Weele, Bondt, &amp; MacInnes, 2011; Watanabe-Susaki et al., 2014)</span>. Indeed, a growing body of evidence is beginning
to suggest that <em>Drosophila</em> GSCs not only have a specific requirement
for ribosome biogenesis, but also that ribosome biogenesis, as well as
global translation, vary greatly over the course of GSC differentiation
and are uncoupled during early oogenesis <span class="citation">(Sanchez et al., 2016; Q. Zhang, Shalaby, &amp; Buszczak, 2014)</span>. These attributes make <em>Drosophila</em> oogenesis an
excellent system to address how perturbations of ribosome levels affects
stem cell differentiation.</p>
<p>In order to maintain stem cell fate, GSCs asymmetrically partition
factors required for ribosome biogenesis by retaining more of this
machinery than they pass on to daughter cells <span class="citation">(Fichelson et al., 2009; Q. Zhang, Shalaby, &amp; Buszczak, 2014)</span>. In particular, Underdeveloped (Udd), an rRNA
transcription factor segregates asymmetrically to the GSC during mitosis
and seems to promote a high rate of rRNA synthesis within the GSC
<span class="citation">(Q. Zhang, Shalaby, &amp; Buszczak, 2014)</span>. Furthermore, Wicked (Wcd), a U3 snoRNP complex member
required for rRNA maturation, is also asymmetrically partitioned to GSCs
and associates with the original spectrosome, an ER rich organelle found
in GSCs and CBs <span class="citation">(A. C. Spradling et al., 1997)</span>, of the dividing GSC. How GSCs
carry out this specialized cellular division requires further
investigation, however, asymmetric stem cell division is crucial for
proper differentiation <span class="citation">(D. Chen &amp; McKearin, 2003a; D. Chen &amp; McKearin, 2003b; H. Lin &amp; Spradling, 1997)</span>.Consistent with this loss of <em>wcd</em> results in premature
differentiation of GSCs <span class="citation">(Fichelson et al., 2009)</span>. Nascent rRNA production,
measured by BrUTP incorporation, and presumably ribosomes, are produced
at high levels in GSCs but this production drops in CBs and in
subsequent stages (Figure 3) <span class="citation">(Q. Zhang, Shalaby, &amp; Buszczak, 2014)</span>. Additionally, it has been
observed that certain ribosome biogenesis components are expressed at
high levels specifically in the germline <span class="citation">(Kai, Williams, &amp; Spradling, 2005)</span>. In particular,
RNA exonuclease 5 (Rexo5) is an RNA exonuclease that facilitates
ribosome biogenesis by trimming snoRNAs as well as rRNAs
<span class="citation">(Gerstberger et al., 2017)</span>. Depletion of <em>rexo5</em> in the germline results in
an accumulation of egg chambers that bud off from the germarium, but do
not grow in size, and causes defects in GSC proliferation
<span class="citation">(Gerstberger et al., 2017)</span>. These observations suggest that the machinery for
ribosome biogenesis is not only critical for germline development but is
also dynamically regulated.</p>
<p>Sanchez et al. demonstrated that the dynamic nature of rRNA
transcription during germline development is not simply a consequence of
the differentiation process. Instead, lowering ribosome biogenesis is
required for timely differentiation, but severe loss of ribosome
biogenesis causes formation of stem-cysts, a product of perturbed
cytokinesis of GSC daughters <span class="citation">(Mathieu et al., 2013; Matias, Mathieu, &amp; Huynh, 2015; Sanchez et al., 2016)</span> . Somewhat surprisingly, despite their increased
retention of ribosome biogenesis components, GSCs exhibit a lower rate
of translation compared to daughter cells and cyst stages (Figure 3).
This finding invokes the hypothesis that despite the GSCs elevated
capacity for ribosome biogenesis, GSCs do not intrinsically require
higher ribosome levels for translation. Instead, the data is suggestive
of the possibility that GSCs produce high levels of ribosomes in order
to pass them on to and facilitate differentiation of their daughter
cells. We thus hypothesize that a ribosome biogenesis checkpoint could
couple ribosome production to cell cycle progression to ensure a
sufficient ribosome concentration is passed from the GSC to the daughter
CB. Conversely, increasing ribosome biogenesis via overexpression of
TIF-IA, an RNA Pol I transcription initiation factor that is required
for rRNA synthesis <span class="citation">(Grewal, Evans, &amp; Edgar, 2007)</span>, results in a failure of germ cells
to differentiate, causing a marked overproliferation of undifferentiated
GSC daughters <span class="citation">(Q. Zhang, Shalaby, &amp; Buszczak, 2014)</span>. This overproliferation may be caused by
bypassing or rapid progression through the proposed ribosome biogenesis
checkpoint such that the cell cycle is hastened in response to elevated
ribosome biogenesis. The overproliferation of undifferentiated germ
cells when ribosome levels are elevated is consistent with observations
that high ribosome levels lead to rapidly growing cancers
<span class="citation">(Belin et al., 2009; Deisenroth &amp; Zhang, 2010; Vlachos &amp; Muir, 2010)</span>.</p>
<p>Although reducing ribosome biogenesis tends to result in the formation
of a stem-cyst as previously described, some factors that play a role in
ribosome biogenesis have a less severe phenotypes. For example, some
mutants of the ribosomal protein S2 (rps2) gene have a repeating
egg-chamber mid-oogenesis defect, wherein ovarian development halts at
stage 5 and successive egg chambers do not grow in size and eventually
die, resulting in sterility <span class="citation">(Cramton &amp; Laski, 1994)</span>. This phenotype may be
the consequence of incomplete loss of function as the allele that
results in the repeating egg chamber phenotype reduces mRNA expression
of <em>rps2</em>, incompletely, by 60-70%, while other allelic combinations
result in embryonic lethality <span class="citation">(Cramton &amp; Laski, 1994)</span>. Incomplete loss of
function alleles for another ribosomal protein, <em>ribosomal protein S3</em>,
result in a similar repeating egg chamber phenotype
[Sæbøe-Larssen1998]. These observations suggest that partial loss of
ribosome biogenesis during oogenesis may be tolerated during
differentiation but results in phenotypes at a later phase of egg
production, consistent with the model that high levels of biogenesis in
early stages supply the ribosomes for subsequent differentiation and
development.</p>
<p>Not only do ribosome levels vary but a class of ribosomal protein
paralogs are enriched specifically in early germ cells <span class="citation">(Xue &amp; Barna, 2012)</span>.
Several variant ribosomal proteins such as <em>ribosomal proteins S5b</em>
(<em>rps5b</em>), <em>s10a</em>, <em>s19b</em>, and <em>l22</em>-like are enriched in the germline
and others are enriched during early oogenesis <span class="citation">(Kai, Williams, &amp; Spradling, 2005)</span>. The role
of these ribosomal proteins has not been thoroughly explored, but their
presence indicates either a role for specialized ribosomes early during
germline development or as a way to further increase the availability of
ribosomal proteins to facilitate the high level of ribosome production
in GSCs. One of these ribosomal protein paralogs, RpS5b, has recently
been characterized <span class="citation">(Kong et al., 2019)</span>. <em>rps5b</em> is most highly expressed in
ovaries in contrast to its paralog, <em>ribosomal protein S5a</em> (<em>rps5a</em>),
which is expressed at high levels ubiquitously <span class="citation">(Kong et al., 2019)</span>. Loss of
<em>rps5a</em> in the germline does not cause a germline phenotype, however,
loss of <em>rps5b</em> results in a mid-oogenesis defect that is further
exacerbated when <em>rps5a</em> is depleted in a <em>rps5b</em> mutant background
<span class="citation">(Kong et al., 2019)</span>. This could suggest that RpS5a and RpS5b are functionally
similar and that the RpS5b phenotype results from lowering the overall
amount of RpS5 available during oogenesis. However, RpS5b was also found
to interact preferentially with mRNAs that encode proteins involved in
mitochondrial electron transport, in contrast to RpS5a which binds mRNAs
from a broad spectrum of gene categories <span class="citation">(Kong et al., 2019)</span>. In accordance
with the binding data, <em>rps5b</em> depleted ovaries expressed lower levels
of proteins involved in oxidative phosphorylation and mitochondrial
respiration <span class="citation">(Kong et al., 2019)</span>. This evidence suggests that the expression
of ribosomal protein paralogs may be a part of specialized ribosomes
that translate specific groups of mRNAs; however, these ribosomal
protein paralogs must be carefully analyzed to determine if they make up
bonafide special ribosomes or instead have ribosome independent
functions <span class="citation">(Dinman, 2016)</span>.</p>
<p>What regulates ribosome biogenesis to allow for it to be dynamic during
early <em>Drosophila</em> germline development? The best understood regulator
of ribosome biogenesis is the Target of Rapamycin (TOR) pathway
<span class="citation">(Chymkowitch, Aanes, Robertson, Klungland, &amp; Enserink, 2017; Magnuson, Ekim, &amp; Fingar, 2012; Yuehua Wei &amp; Zheng, 2009; Yerlikaya et al., 2016)</span>
TOR is a kinase that is part of two distinct subcomplexes, TOR complex 1
(TORC1) and TOR complex 2 (TORC2) <span class="citation">(Wullschleger, Loewith, &amp; Hall, 2006)</span>. These
complexes have distinct biological roles. TORC2 has been shown to
function as an important regulator of the cytoskeleton
<span class="citation">(Wullschleger, Loewith, &amp; Hall, 2006)</span>. Whereas, TORC1 receives and integrates several
different signals including nutritional and growth factors and its
activity promotes pro-proliferative activities such as global
translation, ribosomal protein translation, and cell cycle progression
<span class="citation">(E. Kim, Goraksha-Hicks, Li, Neufeld, &amp; Guan, 2008; Magnuson, Ekim, &amp; Fingar, 2012; Texada et al., 2019)</span>. TORC1 activity also helps
to coordinate the transcription and translation of the components
required for ribosome biogenesis <span class="citation">(Grewal, Evans, &amp; Edgar, 2007; Magnuson, Ekim, &amp; Fingar, 2012; D. E. Martin, Powers, &amp; Hall, 2006)</span>. In <em>Drosophila,</em> TORC1 activity is high in GSCs through
the 4-cell cyst, but TORC1 activity dips in 8 and 16 cell cysts and
subsequently increases after the cyst stages <span class="citation">(Youheng Wei, Bettedi, Kim, Ting, &amp; Lilly, 2019)</span>.
Interestingly, the landscape of TORC1 activity resembles the landscape
of ribosome biogenesis, but not global translation (Figure 3)
<span class="citation">(Sanchez et al., 2016; Q. Zhang, Shalaby, &amp; Buszczak, 2014)</span>. However, loss of TORC1 components does
not phenocopy perturbation of ribosome biogenesis <span class="citation">(Sanchez et al., 2016)</span>.
This is possibly because TORC1 plays a broader role in early oogenesis
given the myriad of regulatory functions TORC1 is known to play in other
systems <span class="citation">(E. Kim, Goraksha-Hicks, Li, Neufeld, &amp; Guan, 2008; S. Li, Zhang, Takemori, Zhou, &amp; Xiong, 2009; Moreno-Torres, Jaquenoud, &amp; De Virgilio, 2015; Noda, 2017; Yuehua Wei &amp; Zheng, 2009)</span>. A downstream effector of mTORC1, La related protein 1
(Larp1) is known to silence ribosomal protein translation in mammals
through binding to terminal oligopyrimidine tracts in the 5'UTR of its
targets <span class="citation">(Bruno D. Fonseca et al., 2015; Hong et al., 2017; Roni M. Lahr et al., 2017; Tcherkezian et al., 2014)</span>; however, the same has yet to be
demonstrated for the <em>Drosophila</em> ortholog, La related protein (Larp).
Tantalizingly, Larp is required for male and female fertility in
<em>Drosophila</em>, but details of Larp's precise role in the female and
oogenesis are lacking <span class="citation">(Blagden et al., 2009; Ichihara, Shimizu, Taguchi, Yamaguchi, &amp; Inoue, 2007)</span>. In contrast,
in males Larp is required for proper spindle pole formation as well as
proper cytokinesis <span class="citation">(Blagden et al., 2009)</span>. Given the regulatory role Larp
plays in ribosome biogenesis in mammals and the data from <em>Drosophila</em>
spermatogenesis, Larp could facilitate the dynamic nature of ribosome
biogenesis during GSC differentiation and meiosis. However, further
study is required to understand the role of Larp during GSC
differentiation and oogenesis to determine its function in this context.</p>
<p>The process of differentiation requires major cellular reprogramming.
Surprisingly, despite being required for cell viability ribosome
biogenesis and global translation are two key programs that are
modulated to shape GSC differentiation<span class="citation">(Sanchez et al., 2016; Q. Zhang, Shalaby, &amp; Buszczak, 2014)</span>.
When ribosome production is improperly modulated during GSC
differentiation it results in characteristic phenotypes, accumulation of
single cells if biogenesis components are overexpressed and formation of
a stem-like cyst if ribosome biogenesis components are knocked down in
the germline <span class="citation">(Sanchez et al., 2016; Q. Zhang, Shalaby, &amp; Buszczak, 2014)</span>. Additionally, several
ribosomal protein variants are highly enriched in ovaries and they may
perform special functions, however, these variants are just beginning to
be studied. Additionally, based on what we know of the mechanisms and
networks that control ribosome biogenesis in <em>Drosophila</em> oocytes, the
dynamic nature of ribosome biogenesis seems likely to be conserved;
however, further investigation is required to determine and compare the
basis of ribosome biogenesis control.</p>
<div class="figure" style="text-align: center"><span id="fig:unnamed-chunk-6"></span>
<img src="figure/Book%20Chapter/Figure3.png" alt="Global translation rate, rRNA transcription rate, and mTorc1 activity during development. Schematic representing the germarium and plots representing relative changes in global translation rate, rRNA transcription rate, and mTorc1 activity during development at the developmental stages indicated. As germline stem cell differentiation occurs rRNA production decreases, while global translation initially increases as differentiation occurs then falls off post differentiation. A global regulator of both translation and rRNA production, mTorc1 activity decreases during differentiation and increases post differentiation." width="4.536585 in" height="6 in" />
<p class="caption">
Figure 1.3: <strong>Global translation rate, rRNA transcription rate, and mTorc1 activity during development.</strong> Schematic representing the germarium and plots representing relative changes in global translation rate, rRNA transcription rate, and mTorc1 activity during development at the developmental stages indicated. As germline stem cell differentiation occurs rRNA production decreases, while global translation initially increases as differentiation occurs then falls off post differentiation. A global regulator of both translation and rRNA production, mTorc1 activity decreases during differentiation and increases post differentiation.
</p>
</div>
</div>
<div id="hand-off-mechanisms-facilitated-by-combinatorial-rna-binding-proteins-dynamically-shape-the-translational-landscape-during-oogenesis" class="section level2" number="1.6">
<h2><span class="header-section-number">1.6</span> Hand off mechanisms facilitated by combinatorial RNA binding proteins dynamically shape the translational landscape during oogenesis</h2>
<p>While some mRNAs are translated post-transcriptionally, other critical
mRNAs are translationally regulated. For efficient translation of mRNAs,
it is thought that the mRNAs must be circularized - bringing their 5'
cap and 3' poly A tail in close proximity to each other <span class="citation">(Fukao et al., 2009; Martineau et al., 2008; Preiss &amp; Hentze, 1998)</span>. This interaction is mediated by cap
binding proteins such as eukaryotic initiation factor 4E (eIF4E) and the
poly-A binding protein (PABP)<span class="citation">(Eichhorn et al., 2016; Kronja et al., 2014; Subtelny, Eichhorn, Chen, Sive, &amp; Bartel, 2014; Tarun Jr, Wells, Deardorff, &amp; Sachs, 1997)</span>. A longer poly-A tail and uninhibited
access to the 5' cap for eIF4E is believed to promote efficient
translation <span class="citation">(Jalkanen, Coleman, &amp; Wilusz, 2014)</span>. A major mode of translational
regulation is that RNA binding proteins (RBPs) recognize cognate
sequences in the 3' UTRs of their target mRNAs <span class="citation">(Harvey et al., 2018)</span>. The
binding of the RBP prevents circularization of the mRNA and inhibits
efficient translation initiation, leading to reduced translation
<span class="citation">(Mazumder, Seshadri, Imataka, Sonenberg, &amp; Fox, 2001)</span>. RBP binding to the 3' UTR can mediate translation
inhibition by recruiting cofactors to inhibit circularization
<span class="citation">(Szostak &amp; Gebauer, 2013)</span>.This inhibition of circularization can be achieved by
RBP binding to the cap and competing with eIF4E, removal of the cap by
the decapping machinery, or recruitment of factors such as the CCR4-Not
complex to shorten poly-A tail length <span class="citation">(Rissland, 2017)</span>. In some
cases, RBPs can both block initiation as well as mediate shortening of
the poly-A tail <span class="citation">(Neve, Patel, Wang, Louey, &amp; Furger, 2017)</span>.</p>
<p>As mentioned in the germline several developmental processes such as
stem cell maintenance, differentiation, mitosis and meiosis are
coordinated and successful transition through these diverse programs
relies on precise translational control (Figure 4) <span class="citation">(Joshi, Riddle, Djabrayan, &amp; Rothman, 2010; Slaidina &amp; Lehmann, 2014)</span>. As factors that interfere with translation such as
the decapping machinery and the poly-A tail shortening CCR4-Not complex
are expressed continuously during oogenesis, and cannot support dynamic
translational control on their own, a dynamic and diverse landscape of
translational regulators has evolved to allow for fine-scale temporal
control of mRNA translation <span class="citation">(Eichhorn et al., 2016; Flora, Wong-Deyrup, et al., 2018b)</span>. To add an
additional layer of complexity, the expression or abundance of several
RBPs that regulate translational control oscillate as oogenesis
progresses (Figure 4) <span class="citation">(Flora, Wong-Deyrup, et al., 2018b; Prashanth Rangan et al., 2009; Richter &amp; Lasko, 2011)</span>. As
the levels of RBPs decrease, their bound mRNA targets are licensed for
translation <span class="citation">(Flora, Wong-Deyrup, et al., 2018b; Lasko, 2000; Linder &amp; Lasko, 2006)</span>. There are three
major themes that work to control mRNA translation: 1. RBPs collaborate
in a combinatorial manner to regulate mRNAs, 2. Target mRNAs are handed
off from one RBP complex to another as levels oscillate during oogenesis
to consistently repress or promote target mRNA translation, and 3.
Multiple feedback mechanisms operate to mediate each transition (Figure
4) <span class="citation">(Flora, Wong-Deyrup, et al., 2018b)</span>. The feedback mechanism has been extensively
reviewed elsewhere and is not the focus of this chapter <span class="citation">(Flora, Wong-Deyrup, et al., 2018b; Slaidina &amp; Lehmann, 2014)</span>. Here, we outline how RBPs both collaborate as well
hand off mRNAs during the transition from GSC to mature oocyte.</p>
<p>GSCs rely on several factors to maintain self-renewal, two of the main
factors are Pumilio (Pum) and Nanos (Nos), which work in a combinatorial
fashion to repress the translation of differentiation-promoting mRNAs
(Figure 4) <span class="citation">(A. Forbes &amp; Lehmann, 1998; Gilboa &amp; Lehmann, 2004; Joly, Chartier, Rojas-Rios, Busseau, &amp; Simonelig, 2013; H. Lin &amp; Spradling, 1997)</span>.
Pum, a member of the conserved Pum- and Fem-3-binding factor (PUF)
family of proteins, is present at high levels in the undifferentiated
germline cells of the ovary, including GSCs, CBs, and
early-differentiating cysts <span class="citation">(A. Forbes &amp; Lehmann, 1998; Kai, Williams, &amp; Spradling, 2005)</span>. Independent of
other factors, Pum can directly bind mRNA, but it requires the catalytic
activity of other proteins to regulate translation of its targets in the
<em>Drosophila</em> germline <span class="citation">(Sonoda &amp; Wharton, 1999; Tadauchi, Matsumoto, Herskowitz, &amp; Irie, 2001)</span>. Pum is known
to have dynamic interactions with two critical regulators, Nos in GSCs,
and Brain tumor (Brat) in CBs (Figure 4) <span class="citation">(Arvola, Weidmann, Tanaka Hall, &amp; Goldstrohm, 2017; Goldstrohm, Hall, &amp; McKenney, 2018; Harris, Pargett, Sutcliffe, Umulis, &amp; Ashe, 2011; Reichardt et al., 2018; Sonoda &amp; Wharton, 1999, 2001)</span>. Nos, a well conserved RNA binding protein, has the
ability to bind mRNA, albeit at low affinity and requires the presence
of Pum to recognize its targets <span class="citation">(Arvola, Weidmann, Tanaka Hall, &amp; Goldstrohm, 2017)</span>. Nanos directly
interacts with Not1, a member of the CCR4-Not complex, recruiting it to
target mRNAs, such as <em>meiotic P26</em> (<em>mei-p26</em>) and <em>brat</em>, to regulate
their translation <span class="citation">(Bhandari, Raisch, Weichenrieder, Jonas, &amp; Izaurralde, 2014; Raisch et al., 2016; Temme, Simonelig, &amp; Wahle, 2014)</span>.
While in some systems Pum can directly recruit the CCR4-Not complex,
activity of <em>nos</em> is required for this interaction in the <em>Drosophila</em>
germline <span class="citation">(Joly, Chartier, Rojas-Rios, Busseau, &amp; Simonelig, 2013; Temme, Simonelig, &amp; Wahle, 2014)</span>. Upon loss of Pum, Nanos or Twin,
GSCs fail to maintain stem cell fate and differentiate into stem cell
daughters, resulting in the inability to sustain oogenesis as outlined
below.</p>
<p>An example of distinct, stage-specific translational control by
Pum/Nos/CCR4-Not complex in the germline is the mechanism by which
<em>polar granule component</em> (<em>pgc</em>), a germline-specific transcriptional
repressor, is controlled (Figure 4) <span class="citation">(Flora, Wong-Deyrup, et al., 2018b)</span>. Pgc interacts with
the Positive Transcription Elongation Factor (P-TEFb) complex and
inhibits the phosphorylation of the Serine-2 residue that is critical
for transcriptional elongation, resulting in global transcriptional
silencing <span class="citation">(Hanyu-Nakamura, Sonobe-Nojima, Tanigawa, Lasko, &amp; Nakamura, 2008)</span>. A single pulse of expression of Pgc
protein in the CB allows for epigenetic and transcriptomic reprogramming
during differentiation <span class="citation">(Flora, Schowalter, et al., 2018)</span>. While <em>pgc</em> mRNA is expressed
highly and ubiquitously throughout oogenesis, translation of <em>pgc</em> mRNA
is tightly regulated to mitigate the effects of its potent
transcriptional silencing activity. The <em>pgc</em> 3' UTR contains a
conserved consensus sequence that is transiently and sequentially bound
by multiple distinct, developmentally regulated RBPs <span class="citation">(Flora, Wong-Deyrup, et al., 2018b)</span>.
This 3' UTR sequence is required for post-transcriptional control of
<em>pgc</em> as Pgc protein expression is restricted to the CB. In the GSCs,
Pum and Nos bind the <em>pgc</em> 3' UTR and recruit Twin a component of the
CCR4-Not complex to deadenylate <em>pgc</em> mRNA and inhibit its translation
(Figure 4) <span class="citation">(Flora, Wong-Deyrup, et al., 2018b)</span>. In addition to <em>pgc</em>, Pum/Nos and Twin also
regulate Brain tumor (Brat) <span class="citation">(Joly, Chartier, Rojas-Rios, Busseau, &amp; Simonelig, 2013)</span>. Brat is a TRIM-NHL domain
protein expressed in the germline that represses translation by engaging
with d4EHP and competing with the cap-binding protein eIF4E to prevent
translation initiation (Figure 4) <span class="citation">(Arvola, Weidmann, Tanaka Hall, &amp; Goldstrohm, 2017; Harris, Pargett, Sutcliffe, Umulis, &amp; Ashe, 2011; Sonoda &amp; Wharton, 2001)</span>. While <em>brat</em> mRNA is expressed in the GSC, it is
specifically repressed by Nos and Pum . In addition to these targets,
several differentiation promoting mRNAs such as <em>meiP26</em> are also
repressed <span class="citation">(Joly, Chartier, Rojas-Rios, Busseau, &amp; Simonelig, 2013)</span>. Thus, in the GSCs, a combination of Pum, Nos
and CCR4-Not complex are required for repressing translation of several
critical differentiation promoting mRNAs <span class="citation">(Flora, Wong-Deyrup, et al., 2018b; Lasko, 2000, 2012; Slaidina &amp; Lehmann, 2014)</span>.</p>
<p>Subsequent differentiation of the GSC daughters relies on several
factors to repress expression of <em>nos</em> mRNA <span class="citation">(Lasko, 2000, 2012)</span>. Differentiation is initiated upon Bam expression in the
CB, where Bam and its binding partner benign gonial cell neoplasm (Bgcn)
act through a sequence in the <em>nos</em> 3' UTR to its inhibit translation
(Figure 4) <span class="citation">(Y. Li, Minor, Park, McKearin, &amp; Maines, 2009; McCarthy, Deiulio, Martin, Upadhyay, &amp; Rangan, 2018)</span>. This repression mechanism
includes deadenylation activity by Twin, which works in conjunction with
Bam and Bgcn <span class="citation">(Fu et al., 2015)</span>. As Nos protein levels decrease in the CB,
<em>pgc</em> and <em>brat</em> mRNAs are translated <span class="citation">(Flora, Wong-Deyrup, et al., 2018b)</span>. The expressed
Brat protein now partners with Pum to repress translation of GSC
self-renewal genes (Figure 4) <span class="citation">(Harris, Pargett, Sutcliffe, Umulis, &amp; Ashe, 2011)</span>. In addition, expression
of Mei-P26 increases initiating interactions with Bam, Bgcn and Sxl.
Mei-P26 then promotes translational repression of GSC fate promoting
genes such as <em>nos</em>, allowing for further differentiation by cooperating
with Bam and Bgcn <span class="citation">(Y. Li et al., 2013; Reichardt et al., 2018)</span>. As the CB
differentiates into 2-, 4-, 8- and 16- cell cysts, levels of Nanos
protein rebound. However, in spite of the presence of Nos, Pum partners
with Brat to suppress <em>pgc</em> translation in the 4- to 16-cell cyst stages
(Figure 4) <span class="citation">(Flora, Wong-Deyrup, et al., 2018b)</span>. Thus, in CBs, absence of Nos allows for Pum
to complex with a different subset of proteins as well as license
expression of new translational regulators to promote differentiation.</p>
<p>After cyst differentiation, Pum protein levels decrease and expression
of another translational repressor, Bruno (Bru), increases
<span class="citation">(Kim-Ha, Kerr, &amp; Macdonald, 1995; Schupbach &amp; Wieschaus, 1989, 1991; Webster, Liang, Berg, Lasko, &amp; Macdonald, 1997)</span>. Downregulation of Pum expression is
critical for the transition from GSC to an oocyte
<span class="citation">(Carreira-Rosario et al., 2016; A. Forbes &amp; Lehmann, 1998)</span>. Rbfox1, an RBP whose
cytoplasmic isoform regulates the translation of specific mRNAs in the
germline is responsible for repressing Pum translation through binding
of a consensus sequence in the <em>pum</em> 3' UTR (Figure 4)
<span class="citation">(Carreira-Rosario et al., 2016)</span>. Loss of Rbfox1 leads to an expansion of Pum
protein expression and a disruption of differentiation
<span class="citation">(Carreira-Rosario et al., 2016)</span>. Repression of Pum levels by Rbfox1 allows
for Bru expression <span class="citation">(Carreira-Rosario et al., 2016)</span>. Surprisingly, Bru can
bind to a sequence in the 3' UTR that is very similar to Pum binding
sequence (Figure 4)<span class="citation">(Reveal, Garcia, Ellington, &amp; Macdonald, 2011)</span>. Bruno blocks translation
initiation by interacting with Cup, a conserved eIF4E binding protein
<span class="citation">(G. Kim et al., 2015; Nakamura, Sato, &amp; Hanyu-Nakamura, 2004)</span>. In fact, Bru binds the same sequence in
the <em>pgc</em> 3' UTR as Nos/Pum to prevent <em>pgc</em> translation
<span class="citation">(Flora, Wong-Deyrup, et al., 2018b)</span>. This mode of translation repression is not restricted
to <em>pgc</em>, rather a cohort of maternal mRNAs are co-regulated by Pum and
Bru representing a hand-off mechanism for repression of maternal mRNAs
<span class="citation">(Flora, Wong-Deyrup, et al., 2018b)</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:unnamed-chunk-7"></span>
<img src="figure/Book%20Chapter/Figure4.png" alt="Schematic of combinatorial and dynamic translation regulation in the Drosophila germarium. In the GSCs Nos, Pum and Twin form a complex to inhibit the translation of differentiation mRNAs such as pgc, which increases throughout oogenesis. Expression of Bam in the CB initiates differentiation by interacting with its partner Bgcn and Mei-P26 to repress the translation of GSC-expressed mRNAs, specifically nos. As Nos protein levels decrease in the CB, Pum is available to partner with Brat to repress the translation of self-renewal genes and pgc. In cyst stages, Rbfox1 binds the pum 3' UTR to inhibit its translation. Throughout oogenesis Bru and Cup continuously block translation of pgc." width="82%" />
<p class="caption">
Figure 1.4: <strong>Schematic of combinatorial and dynamic translation regulation in the <em>Drosophila</em> germarium.</strong> In the GSCs Nos, Pum and Twin form a complex to inhibit the translation of differentiation mRNAs such as <em>pgc</em>, which increases throughout oogenesis. Expression of Bam in the CB initiates differentiation by interacting with its partner Bgcn and Mei-P26 to repress the translation of GSC-expressed mRNAs, specifically <em>nos</em>. As Nos protein levels decrease in the CB, Pum is available to partner with Brat to repress the translation of self-renewal genes and <em>pgc</em>. In cyst stages, Rbfox1 binds the <em>pum</em> 3' UTR to inhibit its translation. Throughout oogenesis Bru and Cup continuously block translation of <em>pgc</em>.
</p>
</div>
</div>
<div id="summary" class="section level2" number="1.7">
<h2><span class="header-section-number">1.7</span> Summary</h2>
<p>Decades of work using elegant genetics has revealed several paradigms in
which splicing machinery, RNA modifying enzymes, ribosome levels, and
translational regulation mediates the transition from GSC to oocyte
fate. However, several critical details such as the direct targets and
mechanisms still need to be deciphered. Together the advent of
cost-effective sequencing technologies combined with the increasing
ability to easily create mutants in previously uncharacterized genes
will allow us to further elucidate the regulatory logic (underlying or
of) this critical transition.</p>















<!-- **Figure 2.1: RNA helicases Aramis, Athos and Porthos are required for GSC differentiation.**  -->

<!-- **Figure 2.2. Aramis, Athos, and Porthos are required for proper cytokinesis and differentiation, related to Figure 2.1.**  -->

<!-- **Figure 2.3. Athos, Aramis, and Porthos are required for efficient ribosome biogenesis.**  -->

<!-- **Figure 2.4. Athos, Aramis, and Porthos are required for efficient ribosome biogenesis., related to Figure 2.3.**  -->

<!-- **Figure 2.5. Athos, Aramis, and Porthos are required for cell cycle progression during early oogenesis.**  -->

<!-- **Figure 2.6 Aramis is required for proper cell cycle progression, related to Figure 2.5.**  -->

<!-- **Figure 2.7.** **Aramis is required for efficient translation of a subset of mRNAs.**  -->

<!-- **Figure 2.8. The mRNA levels of Aramis polysome-seq targets are not significantly changing, related to Figure 2.7.**  -->

<!-- **Figure 2.9. Non1 represses p53 expression to allow for GSC differentiation.**  -->

<!-- **Figure 2.10. Non1 and p53 are inversely expressed, related to Figure 2.9.**  -->

<!-- Aramis regulated mRNAs contain a TOP motif. -->

<!-- **Figure 2.12. TORC1 activity, Athos, and Porthos, regulate TOP expression in the germarium, related to Figure 2.11.** -->

<!-- **Figure 2.13. Larp binds to TOP mRNAs and binding is regulated by Aramis.**  -->

<!-- **Figure 2.14. Larp binds specifically to TOP containing mRNAs and regulates cytokinesis, related to Figure 2.13.** -->

</div>
</div>
            </section>

          </div>
        </div>
      </div>

<a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html" class="navigation navigation-next navigation-unique" aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.epub", "EPUB"], ["thesis.docx", "Word"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
