<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 3 Oo-site: A dashboard to visualize gene expression during Drosophila oogenesis reveals meiotic entry is regulated post-transcriptionally | Translation control tunes Drosophila oogenesis</title>
  <meta name="description" content="Chapter 3 Oo-site: A dashboard to visualize gene expression during Drosophila oogenesis reveals meiotic entry is regulated post-transcriptionally | Translation control tunes Drosophila oogenesis" />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 3 Oo-site: A dashboard to visualize gene expression during Drosophila oogenesis reveals meiotic entry is regulated post-transcriptionally | Translation control tunes Drosophila oogenesis" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 3 Oo-site: A dashboard to visualize gene expression during Drosophila oogenesis reveals meiotic entry is regulated post-transcriptionally | Translation control tunes Drosophila oogenesis" />
  
  
  

<meta name="author" content="Elliot T. Martin" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"/>
<link rel="next" href="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html"/>
<script src="libs/header-attrs-2.8/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="1" data-path="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html"><a href="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html"><i class="fa fa-check"></i><b>1</b> Post-transcriptional gene regulation instructs germline stem cell to oocyte transition during <em>Drosophila</em> oogenesis</a>
<ul>
<li class="chapter" data-level="1.1" data-path="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html"><a href="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html#abstract"><i class="fa fa-check"></i><b>1.1</b> Abstract</a></li>
<li class="chapter" data-level="1.2" data-path="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html"><a href="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html#introduction"><i class="fa fa-check"></i><b>1.2</b> Introduction</a></li>
<li class="chapter" data-level="1.3" data-path="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html"><a href="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html#alternative-splicing-ensures-accurate-production-of-critical-germline-mrnas-to-regulate-sex-determination-and-differentiation"><i class="fa fa-check"></i><b>1.3</b> Alternative splicing ensures accurate production of critical germline mRNAs to regulate sex determination and differentiation</a></li>
<li class="chapter" data-level="1.4" data-path="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html"><a href="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html#rna-modifications-direct-splicing-of-sex-determinants-and-translation-of-differentiation-promoting-genes-in-the-germline"><i class="fa fa-check"></i><b>1.4</b> RNA modifications direct splicing of sex determinants and translation of differentiation promoting genes in the germline</a></li>
<li class="chapter" data-level="1.5" data-path="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html"><a href="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html#production-of-ribosomes-is-finely-tuned-to-facilitate-differentiation"><i class="fa fa-check"></i><b>1.5</b> Production of ribosomes is finely tuned to facilitate differentiation</a></li>
<li class="chapter" data-level="1.6" data-path="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html"><a href="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html#hand-off-mechanisms-facilitated-by-combinatorial-rna-binding-proteins-dynamically-shape-the-translational-landscape-during-oogenesis"><i class="fa fa-check"></i><b>1.6</b> Hand off mechanisms facilitated by combinatorial RNA binding proteins dynamically shape the translational landscape during oogenesis</a></li>
<li class="chapter" data-level="1.7" data-path="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html"><a href="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html#summary"><i class="fa fa-check"></i><b>1.7</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><i class="fa fa-check"></i><b>2</b> A translation control module coordinates germline stem cell differentiation with ribosome biogenesis during <em>Drosophila</em> oogenesis</a>
<ul>
<li class="chapter" data-level="2.1" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#summary-1"><i class="fa fa-check"></i><b>2.1</b> Summary:</a></li>
<li class="chapter" data-level="2.2" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#introduction-1"><i class="fa fa-check"></i><b>2.2</b> Introduction</a></li>
<li class="chapter" data-level="2.3" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#results"><i class="fa fa-check"></i><b>2.3</b> Results</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#three-conserved-rna-helicases-are-required-in-the-germline-for-gsc-differentiation"><i class="fa fa-check"></i><b>2.3.1</b> Three conserved RNA helicases are required in the germline for GSC differentiation</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#athos-aramis-and-porthos-are-required-for-ribosome-biogenesis"><i class="fa fa-check"></i><b>2.3.2</b> Athos, Aramis, and Porthos are required for ribosome biogenesis</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#aramis-promotes-cell-cycle-progression-via-p53-repression"><i class="fa fa-check"></i><b>2.3.3</b> Aramis promotes cell cycle progression via p53 repression</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#aramis-regulated-targets-contain-a-top-motif-in-their-5utr"><i class="fa fa-check"></i><b>2.3.4</b> Aramis-regulated targets contain a TOP motif in their 5'UTR</a></li>
<li class="chapter" data-level="2.3.5" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#larp-binds-top-sequences-in-drosophila"><i class="fa fa-check"></i><b>2.3.5</b> Larp binds TOP sequences in <em>Drosophila</em></a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#discussion"><i class="fa fa-check"></i><b>2.4</b> Discussion</a>
<ul>
<li class="chapter" data-level="2.4.1" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#aramis-athos-and-porthos-are-required-for-efficient-ribosome-biogenesis-in-drosophila"><i class="fa fa-check"></i><b>2.4.1</b> Aramis, Athos, and Porthos are required for efficient ribosome biogenesis in <em>Drosophila</em></a></li>
<li class="chapter" data-level="2.4.2" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#ribosome-biogenesis-defects-leads-to-cell-cycle-defects-mediated-by-p53"><i class="fa fa-check"></i><b>2.4.2</b> Ribosome biogenesis defects leads to cell cycle defects mediated by p53</a></li>
<li class="chapter" data-level="2.4.3" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#ribosome-biogenesis-defects-leads-to-repression-of-top-containing-mrna"><i class="fa fa-check"></i><b>2.4.3</b> Ribosome biogenesis defects leads to repression of TOP-containing mRNA</a></li>
<li class="chapter" data-level="2.4.4" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#larp-transduces-growth-status-to-ribosome-biogenesis-targets"><i class="fa fa-check"></i><b>2.4.4</b> Larp transduces growth status to ribosome biogenesis targets</a></li>
<li class="chapter" data-level="2.4.5" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#ribosome-biogenesis-in-stem-cell-differentiation-and-ribosomopathies"><i class="fa fa-check"></i><b>2.4.5</b> Ribosome biogenesis in stem cell differentiation and ribosomopathies</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#materials-and-methods"><i class="fa fa-check"></i><b>2.5</b> Materials and Methods</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><i class="fa fa-check"></i><b>3</b> Oo-site: A dashboard to visualize gene expression during <em>Drosophila</em> oogenesis reveals meiotic entry is regulated post-transcriptionally</a>
<ul>
<li class="chapter" data-level="3.1" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#summary-2"><i class="fa fa-check"></i><b>3.1</b> Summary</a></li>
<li class="chapter" data-level="3.2" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#introduction-2"><i class="fa fa-check"></i><b>3.2</b> Introduction</a></li>
<li class="chapter" data-level="3.3" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#results-1"><i class="fa fa-check"></i><b>3.3</b> Results</a></li>
<li class="chapter" data-level="3.4" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#discussion-1"><i class="fa fa-check"></i><b>3.4</b> Discussion</a></li>
<li class="chapter" data-level="3.5" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#acknowledgements"><i class="fa fa-check"></i><b>3.5</b> Acknowledgements</a></li>
<li class="chapter" data-level="3.6" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#materials-and-methods-1"><i class="fa fa-check"></i><b>3.6</b> Materials and Methods</a>
<ul>
<li class="chapter" data-level="3.6.1" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#sequencing-data"><i class="fa fa-check"></i><b>3.6.1</b> Sequencing data</a></li>
<li class="chapter" data-level="3.6.2" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#code-availability"><i class="fa fa-check"></i><b>3.6.2</b> Code Availability</a></li>
<li class="chapter" data-level="3.6.3" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#antibodies"><i class="fa fa-check"></i><b>3.6.3</b> Antibodies</a></li>
<li class="chapter" data-level="3.6.4" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#polysome-seq-1"><i class="fa fa-check"></i><b>3.6.4</b> Polysome-seq</a></li>
<li class="chapter" data-level="3.6.5" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#polysome-seq-data-processing"><i class="fa fa-check"></i><b>3.6.5</b> Polysome-seq data processing</a></li>
<li class="chapter" data-level="3.6.6" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#differential-expression"><i class="fa fa-check"></i><b>3.6.6</b> Differential Expression</a></li>
<li class="chapter" data-level="3.6.7" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#go-term-heatmaps"><i class="fa fa-check"></i><b>3.6.7</b> GO term heatmaps</a></li>
<li class="chapter" data-level="3.6.8" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#fluorescent-in-situ-hybridization"><i class="fa fa-check"></i><b>3.6.8</b> Fluorescent <em>in situ</em> hybridization</a></li>
<li class="chapter" data-level="3.6.9" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#quantification-of-stainings"><i class="fa fa-check"></i><b>3.6.9</b> Quantification of Stainings</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html"><a href="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html"><i class="fa fa-check"></i><b>4</b> Screen of post-translational modifying enzymes for their effects on translation</a>
<ul>
<li class="chapter" data-level="4.1" data-path="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html"><a href="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html#abstract-1"><i class="fa fa-check"></i><b>4.1</b> Abstract</a></li>
<li class="chapter" data-level="4.2" data-path="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html"><a href="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html#introduction-3"><i class="fa fa-check"></i><b>4.2</b> Introduction</a></li>
<li class="chapter" data-level="4.3" data-path="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html"><a href="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html#results-and-conclusions"><i class="fa fa-check"></i><b>4.3</b> Results and Conclusions</a></li>
<li class="chapter" data-level="4.4" data-path="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html"><a href="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html#methods"><i class="fa fa-check"></i><b>4.4</b> Methods</a>
<ul>
<li class="chapter" data-level="4.4.1" data-path="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html"><a href="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html#flylines"><i class="fa fa-check"></i><b>4.4.1</b> Flylines</a></li>
<li class="chapter" data-level="4.4.2" data-path="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html"><a href="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html#cloning"><i class="fa fa-check"></i><b>4.4.2</b> Cloning</a></li>
<li class="chapter" data-level="4.4.3" data-path="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html"><a href="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html#dual-luciferase-assay"><i class="fa fa-check"></i><b>4.4.3</b> Dual luciferase assay</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-conclusion.html"><a href="5-conclusion.html"><i class="fa fa-check"></i><b>5</b> Conclusion</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Translation control tunes <em>Drosophila</em> oogenesis</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally" class="section level1" number="3">
<h1><span class="header-section-number">Chapter 3</span> Oo-site: A dashboard to visualize gene expression during <em>Drosophila</em> oogenesis reveals meiotic entry is regulated post-transcriptionally</h1>
<p>Elliot T. Martin<sup>1*</sup>, Kahini Sarkar<sup>1, 4</sup>, Alicia McCarthy<sup>1</sup>, and
Prashanth Rangan<sup>1,4*</sup></p>
<p><sup>1</sup>Department of Biological Sciences/RNA Institute, University at Albany
SUNY, Albany, NY 12202</p>
<p><sup>4</sup>Black Family Stem Cell Institute, Department of Cell, Developmental,
and Regenerative Biology, Icahn School of Medicine at Mount Sinai, 1
Gustave L. Levy Place, New York, NY 10029, USA</p>
<p><sup>*</sup>Co-corresponding authors</p>
<p>Email: <a href="mailto:etmartin@albany.edu" class="email">etmartin@albany.edu</a> and <a href="mailto:prashanth.rangan@mssm.edu" class="email">prashanth.rangan@mssm.edu</a></p>
<div id="summary-2" class="section level2" number="3.1">
<h2><span class="header-section-number">3.1</span> Summary</h2>
<p>Determining how stem cell differentiation is controlled has important
implications for understanding the etiology of degenerative disease and
designing regenerative therapies. <em>In vivo</em> analyses of stem cell model
systems have revealed regulatory paradigms for stem cell self-renewal
and differentiation. The germarium of the female <em>Drosophila</em> gonad,
which houses both germline and somatic stem cells, is one such model
system. Bulk mRNA sequencing (RNA-seq), single-cell (sc) RNA-seq, and
bulk translation efficiency of mRNAs are available for stem cells and
their differentiating progeny within the <em>Drosophila</em> germarium.
However, visualizing those data is hampered by the lack of a tool to
spatially map gene expression and translational data in the germarium.
Here, we have developed Oo-site (<a href="https://www.ranganlab.com/Oo-site" class="uri">https://www.ranganlab.com/Oo-site</a>), a
tool for visualizing bulk RNA-seq, scRNA-seq, and translational
efficiency data during different stages of germline differentiation,
that makes these data accessible to non-bioinformaticians. Using this
tool, we recapitulated previously reported expression patterns of
developmentally regulated genes and discovered that meiotic genes, such
as those that regulate the synaptonemal complex, are regulated at the
level of translation.</p>
</div>
<div id="introduction-2" class="section level2" number="3.2">
<h2><span class="header-section-number">3.2</span> Introduction</h2>
<p>The <em>Drosophila</em> ovary provides a powerful system to study stem cell
differentiation <em>in vivo</em> <span class="citation">(Bastock &amp; St Johnston, 2008; Eliazer &amp; Buszczak, 2011; Lehmann, 2012; Allan Spradling, Fuller, Braun, &amp; Yoshida, 2011)</span>. The <em>Drosophila</em> ovary consists of two main cell
lineages, the germline, which ultimately gives rise to eggs, and the
soma, which surrounds the germline and plays a supportive role in egg
development <span class="citation">(Eliazer &amp; Buszczak, 2011; Roth, 2001; Schüpbach, 1987; Xie &amp; Spradling, 2000)</span>. Each stage of <em>Drosophila</em> female germline stem cell (GSC)
differentiation is observable and identifiable, allowing GSC development
to be easily studied <span class="citation">(Bastock &amp; St Johnston, 2008; Lehmann, 2012; Xie &amp; Spradling, 1998)</span>. Specifically, female
<em>Drosophila</em> GSCs undergo an asymmetric division, giving rise to another
GSC and a cystoblast (CB) (<strong>Figure 3.1A</strong>) <span class="citation">(D. Chen &amp; McKearin, 2003b; D. McKearin &amp; Ohlstein, 1995; Xie &amp; Spradling, 1998)</span>. The GSC and CB are marked by a round
structure called the spectrosome (<strong>Figure 3.1A</strong>) <span class="citation">(De Cuevas &amp; Spradling, 1998; Zaccai &amp; Lipshitz, 1996)</span>. The CB then undergoes four incomplete divisions
resulting in 2-, 4-, 8-, and finally 16-cell cysts (CC), which are
marked by an extended structure called the fusome (<strong>Figure 3.1A</strong>)
<span class="citation">(D. Chen &amp; McKearin, 2003a; D. Chen &amp; McKearin, 2003b; De Cuevas &amp; Spradling, 1998)</span>. In the 16-CC, one of the
cyst cells is specified as the oocyte, while the other 15 cells become
nurse cells, which provide proteins and mRNAs to support the development
of the oocyte (<strong>Figure 3.1A</strong>) <span class="citation">(Bastock &amp; St Johnston, 2008; Carpenter, 1975; J.-R. Huynh &amp; St Johnston, 2004; J. Huynh &amp; St Johnston, 2000; Navarro, Lehmann, &amp; Morris, 2001; Theurkauf, Alberts, Jan, &amp; Jongens, 1993)</span>. The 16-CC is encapsulated by
somatic cells and buds off from the germarium, forming an egg chamber
(<strong>Figure 3.1A</strong>) <span class="citation">(Bastock &amp; St Johnston, 2008; A. J. Forbes, Lin, Ingham, &amp; Spradling, 1996; Xie &amp; Spradling, 2000)</span>. In each
chamber, the oocyte grows as the nurse cells synthesize and then deposit
mRNAs and proteins into the oocyte, which eventually gives rise to a
mature egg <span class="citation">(Bastock &amp; St Johnston, 2008; J. Huynh &amp; St Johnston, 2000)</span>.</p>
<p>Expression of differentiation factors, including those that regulate
translation, results in progressive differentiation of GSCs to an oocyte
<span class="citation">(Blatt, Martin, Breznak, &amp; Rangan, 2020; Slaidina &amp; Lehmann, 2014)</span>. In
the CB, Bag-of-marbles (Bam) expression promotes differentiation and the
transition from CB to 8-CC stage <span class="citation">(D. Chen &amp; McKearin, 2003a; D. McKearin &amp; Ohlstein, 1995; Ohlstein &amp; McKearin, 1997)</span>. In the 8-CC, RNA-binding
Fox protein 1 (Rbfox1) promotes exit from the mitotic cell cycle into
meiosis <span class="citation">(Carreira-Rosario et al., 2016)</span>. Both the differentiation factors
Bam and Rbfox1 affect the translation of mRNAs to promote
differentiation <span class="citation">(Carreira-Rosario et al., 2016; Y. Li, Minor, Park, McKearin, &amp; Maines, 2009; Tastan, Maines, Li, Mckearin, &amp; Buszczak, 2010)</span>. In addition, in 8-CCs,
recombination is initiated across many cyst cells and then eventually is
restricted to the specified oocyte
<span class="citation">(Hinnant, Merkle, &amp; Ables, 2020; J. Huynh &amp; St Johnston, 2000)</span>. Neither the mRNAs that are translationally
regulated during this progressive differentiation nor how recombination
is temporally regulated is fully understood
<span class="citation">(Cahoon &amp; Hawley, 2016; Carreira-Rosario et al., 2016; Flora, Wong-Deyrup, et al., 2018a; Rubin, Macaisne, &amp; Huynh, 2020; Slaidina &amp; Lehmann, 2014; Tanneti, Landy, Joyce, &amp; McKim, 2011; Youheng Wei et al., 2014)</span>.</p>
<p>Within the germarium, the germline is surrounded by and relies on
distinct populations of somatic cells for signaling, structure, and
organization <span class="citation">(Roth, 2001; Schüpbach, 1987; Xie &amp; Spradling, 1998, 2000)</span>. For example, the
terminal filament, cap, and anterior-escort cells act as a somatic niche
for the GSCs <span class="citation">(Decotto &amp; Spradling, 2005; Haifan Lin &amp; Spradling, 1993; X. Wang &amp; Page-McCaw, 2018; Xie &amp; Spradling, 2000)</span>. Once GSCs divide to give rise to CBs, posterior escort
cells guide CB differentiation by encapsulating the CB and the
early-cyst stages <span class="citation">(Kirilly, Wang, &amp; Xie, 2011; Shi et al., 2021; Upadhyay et al., 2016)</span>. Follicle stem cells
(FSCs), which are present towards the posterior of the germarium, divide
and differentiate to give rise to follicle cells, (FCs) which surround
the late-stage cysts that give rise to egg chambers <span class="citation">(Margolis &amp; Spradling, 1995; Nystul &amp; Spradling, 2010; Rust et al., 2020)</span>. FSCs also give rise to stalk cells
and polar cells which connect the individual egg chambers that comprise
the ovariole <span class="citation">(Margolis &amp; Spradling, 1995; Nystul &amp; Spradling, 2010; Rust et al., 2020; Sahai-Hernandez, Castanieto, &amp; Nystul, 2012)</span>.</p>
<p>While there is a wealth of bulk RNA-seq, single-cell mRNA-seq
(scRNA-seq), and translational efficiency data from polysome-seq
experiments for the cells in the germarium, there are several hurdles
for easy utilization of this data:</p>
<ol style="list-style-type: decimal">
<li><p>scRNA-seq has exquisite temporal resolution but it can miss some
lowly expressed transcripts which are better captured by bulk
RNA-seq <span class="citation">(Lähnemann et al., 2020)</span>. However, there is
no easy way to compare these two data sets.</p></li>
<li><p>While scRNA-seq provides mRNA levels, it does not indicate if these
mRNAs are translated, especially in the germline where translation
control plays an important role
<span class="citation">(Blatt, Martin, Breznak, &amp; Rangan, 2020; Slaidina &amp; Lehmann, 2014)</span>.</p></li>
<li><p>Lastly, there is a barrier to the visualization of the data for
those who are not experienced in bioinformatics.</p></li>
</ol>
<p>Here, we have developed a tool that we call Oo-site which integrates
bulk RNA-seq, scRNA-seq, and polysome-seq data to spatially visualize
gene expression and translational efficiency in the germarium.</p>
</div>
<div id="results-1" class="section level2" number="3.3">
<h2><span class="header-section-number">3.3</span> Results</h2>
<p>To make bulk RNA-, scRNA-, and polysome-, seq data accessible to the
community, we have collated and reprocessed previously published
sequencing datasets of ovaries enriched for GSCs, CBs, cysts, and egg
chambers (<strong>Figure 3.1B</strong>). Notably, each genetically enriched sample had
matched bulk RNA-seq and polysome-seq libraries prepared, allowing for
simultaneous read-out of mRNA level and translation status
(<strong>Figure 3.2A</strong>). One limitation is that the enriched cyst
stages do not resolve each distinct stage of cyst development, instead,
these samples represent a mixture of cyst stages. Therefore to
supplement the enrichment data, we have integrated scRNA-seq data from
Slaidina <em>et al</em>. which provides a more discrete temporal resolution of
the cyst stages <span class="citation">(Slaidina, Gupta, Banisch, &amp; Lehmann, 2021)</span>. We present
these data as a tool called Oo-site
(<a href="https://www.ranganlab.com/Oo-site" class="uri">https://www.ranganlab.com/Oo-site</a>), a collection of interactive
visualizations that allows researchers to easily input a gene or
collection of genes of interest to determine their expression
pattern(s).</p>
<div class="figure" style="text-align: center"><span id="fig:oosite-fig-1"></span>
<img src="figure/Oo_site/Figure1.png" alt="Oo-site integrates and provides an interface for interacting with multi-omic data covering major stages of Drosophila GSC differentiation." width="100%" />
<p class="caption">
Figure 3.1: <strong>Oo-site integrates and provides an interface for interacting with multi-omic data covering major stages of <em>Drosophila</em> GSC differentiation.</strong>
</p>
</div>
<p>(<strong>A</strong>) Schematic illustrating developmental stages of germline development. (<strong>B</strong>) Summary of the samples used for bulk RNA-seq and polysome-seq and the cell types these samples are enriched for. (<strong>C</strong>) Screenshot of Oo-site dashboard, indicating: (1) “Take a Tour!” function, which guides the user through the functionality and operation of Oo-site. (2) The available seq datasets which the user can view, including RNA-seq of ovaries genetically enriched for developmental stages (bulk RNA-seq), polysome-seq of ovaries genetically enriched for developmental stages (Polysome-seq), single-cell seq of germline stages (Single-Cell seq: Germline), and single-cell seq of somatic stages in the germarium (Single-Cell seq: Soma). (3) the available visualizations which the user can use, including viewing the expression of genes over development at the level of a single gene (Developmental Progression), viewing all significantly changing genes as heatmaps (Heatmap), and viewing groups of genes either derived from GO-term categories or supplied by the user (Gene Groups). (4) The control panel, which the user can use to control the current visualization, and (5) the Generate Report Function, which can be used to download a PDF report of either the current visualization or all active visualizations.</p>
<p>Oo-site consists of three modules: ovary-map, ovary-heatmap, and
ovary-violin (<strong>Figure 3.1C</strong>). Each module allows users to visualize
expression from matched mRNA-seq and polysome-seq data of genetically
enriched stages of early GSC differentiation as well as previously
published scRNA-seq data <span class="citation">(Slaidina, Gupta, Banisch, &amp; Lehmann, 2021)</span>.
Additionally, we have integrated scRNA-seq expression data for genes
that cluster in somatic cell populations that reside in the germarium
<span class="citation">(Slaidina, Gupta, Banisch, &amp; Lehmann, 2021)</span>, however, here we focus on the
germline <span class="citation">(Slaidina, Gupta, Banisch, &amp; Lehmann, 2021)</span>. Ovary-map allows
users to visualize the expression of a single gene over the course of
differentiation in the framework of a germarium schematic, which
contextualizes staging for those less familiar with <em>Drosophila</em>
oogenesis. Ovary-heatmap consists of a clustered, interactive heatmap of
genes determined to be differentially expressed that allows users to
explore expression trends over-development (<strong>Figure 3.1B, Figure 3.2B-C'</strong>). Finally, ovary-violin allows users to visualize the
expression of multiple genes over the course of differentiation
(<strong>Figure 3.1C</strong>). These groups of genes can be selected either by a
GO-term of interest or a custom list of genes supplied by the user. The
user can download a spreadsheet of gene expressions corresponding to the
subset of selected or input genes. Finally, Oo-site incorporates a
reporting tool that generates a downloadable report of the
visualization(s) in a standardized format to facilitate their use for
publication (<strong>Figure 3.1C</strong>). Researchers can use these datasets to
enhance hypothesis generation or to confirm expression patterns observed
from other methods.</p>
<div class="figure" style="text-align: center"><span id="fig:oosite-fig-2"></span>
<img src="figure/Oo_site/Supplemental_Figure1.png" alt="Sequencing strategy and clustered heatmaps of differential expression, related to Figure 3.1." width="100%" />
<p class="caption">
Figure 3.2: <strong>Sequencing strategy and clustered heatmaps of differential expression, related to Figure 3.1.</strong>
</p>
</div>
<p>(<strong>A</strong>) Schematic of strategy used to obtain input mRNA samples and matched polysome-seq libraries of ovaries genetically enriched for developmental milestones. (<strong>B-B'</strong>) Clustered heatmaps of (B) bulk RNA-seq and (B') log<sub>2</sub>(TE) from bulk polysome-seq of the developmental milestones indicated on the X-axis. Each row in the heatmap indicates a gene that is differentially expressed in at least one of the milestones compared to all others in a pairwise fashion. Color scale denotes average relative expression. (<strong>C</strong>) scRNA-seq of early germline cells and (<strong>C'</strong>) scRNA-seq of somatic cells in the germarium. X-axis denotes cell-type and each row in the heatmap indicates a gene that is differentially expressed in at least one of the cell-types compared to all others in a pairwise fashion.</p>
<p>Using Oo-site, we first determined if the bulk RNA-seq data that was
acquired by enriching for specific stages of germline development is
representative of the gene expression patterns from purified cell types.
We compared bulk RNA-seq data obtained by enriching for GSC and CB cell
types without purification from somatic cells (<strong>Figure 3.1C</strong>) to the GSC
and CB data from Wilcockson <em>et al.</em> where they included a
fluorescent-assisted cell sorting (FACS) step to eliminate somatic cells
so that a pure population of these germline cells was sequenced
<span class="citation">(Wilcockson &amp; Ashe, 2019)</span>. We analyzed the
expression of genes that Wilcockson <em>et al.</em> identified as 2-fold or
more down- or upregulated with a p-value &lt; 0.01. We found that in the
enriched bulk RNA-seq data these genes followed similar trends as
identified by Wilcockson <em>et al.</em>, indicating that despite the lack of
FACS purification, enrichment of cell types reproduces meaningful mRNA
expression changes over these stages (<strong>Figure 3.4A-A'</strong>).</p>
<div class="figure" style="text-align: center"><span id="fig:oosite-fig-3"></span>
<img src="figure/Oo_site/Figure2.png" alt="Oo-site allows for visualization of dynamically regulated genes." width="100%" />
<p class="caption">
Figure 3.3: <strong>Oo-site allows for visualization of dynamically regulated genes.</strong>
</p>
</div>
<p>(<strong>A-B</strong>) Visualization of expression of <em>RpS19b</em> over germline development from (A) developmentally enriched stages and (B) single-cell seq data indicate that the mRNA level of <em>RpS19b</em> decreases starting in the cysts and is dramatically decreased in early egg chambers. Color indicates relative expression and values indicate the (A) mean TPM±standard error or (B) the normalized expression of <em>RpS19b</em> in each given stage. (<strong>C-C''</strong>) Confocal images of ovaries with <em>in situ</em> hybridization of <em>RpS19b</em> (green, middle greyscale) and stained for DAPI (blue, right greyscale) demonstrate that the mRNA level of <em>RpS19b</em> decreases starting in the cyst stages and are dramatically lower in early egg chambers consistent with the seq data. (<strong>D-D''</strong>) Confocal images of ovaries expressing RpS19b::GFP, visualizing (D') GFP (green, middle greyscale), (D'') Vasa staining (blue, right greyscale), and 1B1 (red) demonstrate that the protein expression of RpS19b::GFP is consistent with its mRNA levels. (<strong>E-E'</strong>) Quantifications of normalized mean intensity of staining, X-axis represents the distance in microns from the niche, Y-axis represents mean intensity normalized to the maximum mean intensity per germarium of (E) <em>RpS19b</em> mRNA or (E') RpS19b::GFP. The line represents fit using a loess regression, shaded area represents the standard error of the fit. (n=5 germaria).</p>
<p>To determine if the bulk RNA-seq data recapitulates genuine changes in
gene expression, we compared the expression of <em>ribosomal small subunit
protein 19b</em> (<em>RpS19b</em>) in bulk RNA-seq to scRNA-seq data. Our bulk
RNA-seq data, as well as the available scRNA-seq data indicated that
<em>RpS19b</em> was highly expressed in GSCs, decreased during differentiation
in the cyst stages and was greatly decreased in expression in early egg
chambers, consistent with previous reports (<strong>Figure 3.3A-B</strong>)
<span class="citation">(McCarthy et al., 2021; Sarkar et al., 2021)</span>. To further validate this
expression pattern, we probed the expression of <em>RpS19b</em> <em>in vivo</em> using
<em>in situ</em> hybridization as well as an RpS19b::GFP line that is under
endogenous control elements <span class="citation">(McCarthy et al., 2021)</span>. We
found that <em>RpS19b</em> was present in the GSCs and diminishes in the cyst
stages both at the mRNA and protein level (<strong>Figure 3.3C-E'</strong>).
Additionally, RpS19b::GFP expression resembled its mRNA expression
indicating that its dynamic expression is achieved primarily through
modulating the mRNA level of <em>RpS19b</em>, consistent with its moderate to
high translational efficiency in early stages (<strong>Figure 3.3C-D, Figure 3.4B</strong>). Thus, enriching for specific germline stages
captures changes to gene expression in the germline. However, we note
that care should be taken in interpreting bulk RNA-seq results as the
data may be influenced by the somatic cells present in the samples.
However, simultaneous comparison with scRNA-seq can alleviate this
problem.</p>
<div class="figure" style="text-align: center"><span id="fig:oosite-fig-4"></span>
<img src="figure/Oo_site/Supplemental_Figure2.png" alt="Bulk RNA-seq recapitulates previously observed expression patterns of gene expression, related to Figure 3.3." width="100%" />
<p class="caption">
Figure 3.4: <strong>Bulk RNA-seq recapitulates previously observed expression patterns of gene expression, related to Figure 3.3.</strong>
</p>
</div>
<p>(<strong>A-A'</strong>) Violin plots of expression from bulk RNA-seq of genes 2-fold or more (A) down or (A') upregulated in bam RNAi germline cells compared to UAS-TKV overexpressing germline cell with a p-value &lt; 0.01 over germline development from Wilcockson <em>et al.</em> demonstrate that bulk RNA-seq identifies similar trends in gene expression compared to the FACS based method employed by Wilcockson <em>et al</em>. Values above plots represent Holm-Bonnferroni adjusted p-values from a Welch's t-test between the indicated genotypes. (<strong>B</strong>) Visualization of expression of <em>RpS19b</em> over germline development from polysome-seq data. Color indicates TE and values indicate the log<sub>2</sub> mean TE±standard error <em>RpS19b</em> TE is relatively consistent during early oogenesis and decreases in the egg chambers.</p>
<p>To determine the groups of genes that change as the GSCs differentiate
into an egg, we used gene ontology (GO)-term analysis to probe for
pathways that change at the level of RNA using bulk RNA-seq data. We did
not identify any significant GO-terms in genes that are differentially
expressed between GSCs and CBs. We found that genes with lower
expression in GSCs compared to differentiating cysts are enriched in the
GO-term polytene chromosome puffing which is consistent with GO-terms
identified in Wilcockson <em>et al</em>. for genes that are expressed at lower
levels in GSCs than in differentiating cysts than GSCs (<strong>Figure 3.5A</strong>).
We also identified the polytene chromosome puffing GO-term in genes
downregulated in CBs compared to cysts. Additionally, we observed that
several GO-terms involving peptidase activity were enriched in genes
upregulated in GSCs and CBs compared to cysts (<strong>Figure 3.5B</strong>). This is
consistent with findings suggesting that peptidases can be actively
regulated during differentiation and can influence stem cell fate
<span class="citation">(Han, Wang, Bachovchin, Zukowska, &amp; Osborn, 2015; Perišić Nanut, Pečar Fonović, Jakoš, &amp; Kos, 2021; Tiaden et al., 2012)</span>. We found that two GO-terms related to
glutathione transferase activity were enriched in genes downregulated in
GSCs and CBs compared to ovaries from young-wildtype (young-WT) flies
and in CBs compared to differentiating cysts, suggesting that metabolic
processes may be altered during GSC differentiation. Additionally,
comparison of CBs and differentiating cysts to young-WT, which contain
early egg chambers, indicated that downregulated genes were enriched in
GO-terms involving vitelline and eggshell coat proteins (<strong>Figure 3.5A</strong>).</p>
<div class="figure" style="text-align: center"><span id="fig:oosite-fig-5"></span>
<img src="figure/Oo_site/Figure3.png" alt="GO-terms enriched from differentially expressed genes between genetically enriched developmental milestones." width="90%" />
<p class="caption">
Figure 3.5: <strong>GO-terms enriched from differentially expressed genes between genetically enriched developmental milestones.</strong>
</p>
</div>
<p>(<strong>A-B</strong>) Heatmaps of top five significant GO-terms by fold enrichment resulting from each pairwise comparison of significantly (A) downregulated or (B) upregulated genes in the first genotype listed relative to the second genotype listed in the x-axis from bulk RNA-seq of each developmentally enriched stage. Comparisons that did not generate any significant GO-terms are omitted.</p>
<p>Next, to determine if our data could resolve large-scale expression
changes that occur during oogenesis we examined the expression of genes
in the GO-term meiotic cell cycle. Meiosis is initiated during the cyst
stages of differentiation and therefore we would expect genes in the
category, in general, to increase in expression in the &gt;<em>bam</em> RNAi;
hs-<em>bam</em> samples <span class="citation">(Carpenter, 1979; Tanneti, Landy, Joyce, &amp; McKim, 2011)</span>. We were surprised to find no
significant change in the mean mRNA expression of genes in this GO-term
in any of our enriched stages compared to enriched GSCs, though this
does not preclude gene expression changes for individual genes
(<strong>Figure 3.7A</strong>). However, this is consistent with the
observation that several factors that promote meiosis I are transcribed
in the GSCs and the cells that follow
<span class="citation">(McCarthy et al., 2021)</span>. This suggests that, in general,
a transition from a mitotic state to a meiotic state is not driven by
large changes in mRNA levels of meiotic genes.</p>
<p>As we did not see overall changes to mRNA levels of genes in the GO-term
meiotic cell cycle, we next examined the polysome-seq data of those
genes to determine if changes in expression might occur at the level of
translation. Polysome-seq uses polysome profiling to separate mRNAs that
are associated with polysomes which form by mRNAs engagement with
multiple ribosomes. To quantify the degree to which an mRNA is
associated with polysome fractions, we compared the relative abundance
of mRNAs from the polysome fractions to their relative expression using
corresponding input lysates to calculate a metric referred to as
translational efficiency (TE). Indeed, genes in the meiotic cell cycle
GO-term had a significant increase in translation efficiency in CBs and
a more dramatic increase in cysts despite no significant changes to the
overall mRNA level of these genes (<strong>Figure 3.7A-B</strong>).
Based on scRNA-seq data, the expression of meiotic cell cycle genes
increased slightly but significantly in the 4-CC cluster with a median
increase in expression of 1.25 fold (<strong>Figure 3.7C</strong>). This
suggests that some genes in the meiotic cell cycle GO-term may be
regulated at the mRNA level, but as a group this regulation is modest.
This is likely because genes in this GO-term are robustly expressed even
in GSCs as the median mRNA level of meiotic cell cycle genes in enriched
GSCs is 36.1 TPM, which exceeds the 70<sup>th</sup> expression percentile among
all genes in enriched GSCs.</p>
<div class="figure" style="text-align: center"><span id="fig:oosite-fig-6"></span>
<img src="figure/Oo_site/Figure4.png" alt="Ord expression is controlled post-transcriptionally." width="100%" />
<p class="caption">
Figure 3.6: <strong>Ord expression is controlled post-transcriptionally.</strong>
</p>
</div>
<p>(<strong>A-B</strong>) Visualization of expression of <em>ord</em> over germline development from (A) bulk RNA-seq of developmentally enriched stages and (B) polysome-seq of developmentally enriched stages indicates that the mRNA level of <em>ord</em> is consistent from GSCs to cysts, until decreasing in early egg chambers, but the translation efficiency of <em>ord</em> increases during the cyst stages compared to other stages. Color indicates (A) relative expression or (B) TE and values indicate the (A) mean TPM±standard error or (B) the log<sub>2</sub> mean TE±standard error (<strong>C-C''</strong>) Confocal images of ovaries expressing Ord::GFP with <em>in situ</em> hybridization of <em>gfp</em> mRNA (red, middle greyscale) and stained for GFP protein (green, right greyscale) and DAPI (blue) demonstrate that the mRNA level of Ord::GFP is consistent throughout the germarium. (<strong>D-D'</strong>) Quantification of normalized mean intensity of stainings (C-C''). X-axis represents the distance in microns from the niche, Y-axis represents mean intensity normalized to the maximum mean intensity per germarium of <em>ord</em> mRNA (D) or Ord protein (D). The line represents fit using a loess regression, shaded area represents the standard error of the fit. (n=8 germaria).</p>
<p>To validate this finding, we examined <em>orientation disrupter</em> (<em>ord)</em>
because it is a well-characterized gene, is required for sister
chromatid cohesion, and has previously been reported to peak in
expression as meiosis begins in <em>Drosophila</em>
<span class="citation">(S. E. Bickel, Wyman, Miyazaki, Moore, &amp; Orr-Weaver, 1996; Sharon E. Bickel, Wyman, &amp; Orr-Weaver, 1997; Khetani &amp; Bickel, 2007)</span>. Our Oo-site results suggested
that <em>ord</em> mRNA was expressed before meiosis, both from bulk RNA-seq
(<strong>Figure 3.6A</strong>) and scRNA-seq (<strong>Figure 3.7D</strong>) consistent
with reports that chromosome pairing initiates before meiotic entry
<span class="citation">(Christophorou, Rubin, &amp; Huynh, 2013; Joyce, Apostolopoulos, Beliveau, &amp; Wu, 2013)</span>. However, polysome-seq data were
consistent with the observation that Ord protein expression increases
during the cyst stages due to translation (<strong>Figure 3.6B</strong>). This led us
to predict that <em>ord</em> mRNA would be expressed before meiosis, and that
Ord protein expression would increase during the cyst stages as
previously observed, implying a change in the translation status of
<em>ord</em> mRNA. To test this, we performed fluorescent <em>in situ</em>
hybridization against GFP in a fly expressing Ord-GFP under the control
of the <em>ord</em> promoter and 5'UTR. We visualized both the GFP protein and
the mRNA and observed increased expression of Ord::GFP protein but
consistent <em>ord::GFP</em> mRNA expression, indicating that Ord is controlled
post-transcriptionally, likely at the level of translation based on our
polysome-seq data (<strong>Figure 3.6C-D'</strong>). This finding also underscores the
utility of Oo-site in exploring post-transcriptional gene expression
changes.</p>
<div class="figure" style="text-align: center"><span id="fig:oosite-fig-7"></span>
<img src="figure/Oo_site/Supplemental_Figure3.png" alt="Genes involved in meiotic cell cycle, including ord, may be controlled post-transcriptionally, related to Figure 3.6." width="100%" />
<p class="caption">
Figure 3.7: <strong>Genes involved in meiotic cell cycle, including <em>ord</em>, may be controlled post-transcriptionally, related to Figure 3.6.</strong>
</p>
</div>
<p>(<strong>A</strong>) Violin plots of gene expression from RNA-seq of genes in the GO-term category meiotic cell cycle. No significant overall change occurs to expression of these genes at any of the developmental milestones compared to GSCs. Values above plots represent Holm-Bonnferroni adjusted p-values from a Welch's t-test between the indicated genotypes. (<strong>B</strong>) Violin plots of TE from polysome-seq of genes in the GO-term category meiotic cell cycle. Overall TE increases in CBs and cysts significantly compared to GSCs indicating that meiotic entry may be partially controlled post-transcriptionally. Values above plots represent Holm-Bonnferroni adjusted p-values from a Welch's t-test between the indicated genotypes. (<strong>C</strong>) Violin plot of expression of genes in the GO category “meiotic cell cycle” from scRNA-seq. Overall expression of these genes increases in CBs, cysts, and young-WT ovaries compared to the GSC/CB/2CC cluster. Values above plots represent Holm-Bonnferroni adjusted p-values from a Welch's t-test between the indicated genotypes. (<strong>D</strong>) scRNA-seq data indicate that the mRNA level of <em>ord</em> is highest in the GSC/CB/2CC cluster, but remains relatively consistent in its expression starting in the 4-CC through 16-CC 2ab clusters and is dramatically decreased in early egg chambers. Color and values indicate the normalized expression of <em>ord</em> in each given stage.</p>
<p>To further determine if meiosis is regulated post-transcriptionally, we
examined the expression of genes in the GO-term “Double-strand break
repair,” which is known to occur during meiosis 1
<span class="citation">(Hughes, Miller, Miller, &amp; Hawley, 2018; Page &amp; Hawley, 2003)</span>. Double-stranded breaks are
resolved before egg chamber formation
<span class="citation">(Hughes, Miller, Miller, &amp; Hawley, 2018; Mehrotra &amp; McKim, 2006; Page &amp; Hawley, 2003)</span>. At the level of input mRNA,
we found no significant changes in the expression of genes in this
category compared to enriched GSCs (<strong>Figure 3.8A</strong>). From scRNA-seq data,
the median expression of double-strand break repair genes significantly
increases, but the median increase was only 1.05 fold in 4-CCs and 1.06
in 8-CCs compared to the GSC/CB/2CC group (<strong>Figure 3.8B</strong>). This suggests
that double-strand break repair gene transcription begins in GSC stages
and increases modestly during the cyst stages.</p>
<p>In contrast, we found a significant increase in the median translational
efficiency of double-strand break repair genes, with a 1.20 fold
increase in the median translational efficiency in enriched CBs and a
1.56 fold increase in enriched cysts compared to enriched GSCs (<strong>Figure
5C</strong>). In young-WT the median fold change in translational efficiency
decreased slightly but significantly compared to enriched GSCs at 0.95
fold. This is consistent with the observed progression of
double-stranded break repair that occurs <em>in vivo</em>. This demonstrates
that Oo-site can be used to derive insights into biological processes
that may be changing during early oogenesis
<span class="citation">(Mehrotra &amp; McKim, 2006; Page &amp; Hawley, 2003)</span>. That key processes related to
meiosis and differentiation are controlled post-transcriptionally is
consistent with the importance of proteins that regulate translation
such as Bam and Rbfox1 in differentiation and meiotic commitment during
<em>Drosophila</em> oogenesis <span class="citation">(Blatt, Martin, Breznak, &amp; Rangan, 2020; Carreira-Rosario et al., 2016; Flora, Wong-Deyrup, et al., 2018a; Kim-Ha, Kerr, &amp; Macdonald, 1995; Y. Li, Minor, Park, McKearin, &amp; Maines, 2009; Slaidina &amp; Lehmann, 2014; Tastan, Maines, Li, Mckearin, &amp; Buszczak, 2010)</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:oosite-fig-8"></span>
<img src="figure/Oo_site/Figure5.png" alt="Genes involved in double-strand break repair may be controlled post-transcriptionally." width="100%" />
<p class="caption">
Figure 3.8: <strong>Genes involved in double-strand break repair may be controlled post-transcriptionally.</strong>
</p>
</div>
<p>(<strong>A</strong>) Violin plot of expression of genes in the GO category “Double-strand break repair” from bulk RNA-seq. No significant overall change in expression of these genes occurs comparing each genetically enriched developmental stage to GSCs. (<strong>B</strong>) Violin plot of expression of genes in the GO category “Double-strand break repair” from scRNA-seq. Overall expression of these genes increases in CBs, cysts, and young-WT ovaries compared to the GSC/CB/2CC cluster. Values above plots represent Holm-Bonnferroni adjusted p-values from a Welch's t-test between the indicated genotypes (<strong>C</strong>) Violin plot of expression of genes in the GO category “Double-strand break repair” from polysome-seq. Overall expression of these genes increases in CBs, cysts, and young-WT ovaries compared to GSCs. Values above plots represent Holm-Bonnferroni adjusted p-values from a Welch's t-test between the indicated genotypes.</p>
</div>
<div id="discussion-1" class="section level2" number="3.4">
<h2><span class="header-section-number">3.4</span> Discussion</h2>
<p>We have developed an application that facilitates analysis of bulk
RNA-seq, sc RNA-seq, and polysome-seq data of early <em>Drosophila</em>
oogenesis that is accessible to non-bioinformaticians. We have
demonstrated its utility in representing expression at the mRNA and
translation level. Additionally, we have demonstrated that it can be
used to visualize the expression of groups of genes over development to
facilitate hypothesis development. As with all sequencing data, care
should be taken to validate findings from Oo-site as sequencing can be
influenced by a myriad of factors.</p>
<p>We have used Oo-site to discover that key meiosis regulators such as
proteins of the synaptonemal complex and proteins of the double-strand
break machinery are regulated at the level of translation. This adds to
our understanding of the mechanisms regulating the mitotic to meiotic
transition. In future work, identifying the factors mediating the
widespread post-transcriptional regulation of crucial meiotic genes and
mechanistically how it drives the mitotic to meiotic transition is of
high importance.</p>
<p>High-throughput sequencing has enabled researchers to generate more data
than ever before However, the development of analysis tools that are
usable without bioinformatics training that enable users to make sense
of these data to generate hypotheses and novel discoveries has lagged
<span class="citation">(Shachak, Shuval, &amp; Fine, 2007)</span>. Oo-site allows for
hypothesis generation and discovery using the powerful model system of
<em>Drosophila</em> oogenesis. We believe Oo-site might also have utility as a
teaching and demonstration tool to introduce students to the power of
genomics in developmental biology. The open-source nature of this
software facilitates future tool development, which will be crucial as
more researchers delve into more data-intensive scRNA-seq, where
visualization tools are limited and produce plots that may be difficult
to interpret for those not versed in bioinformatics. Oo-site can be
supplemented in the future to include additional data such as Cut and
Run for various chromatin marks, nascent mRNA transcription using
transient transcriptome sequencing or similar techniques, or protein
levels from mass-spectroscopy to further extend its utility in
hypothesis development.</p>
</div>
<div id="acknowledgements" class="section level2" number="3.5">
<h2><span class="header-section-number">3.5</span> Acknowledgements</h2>
<p>We thank the Drs. Ruth Lehmann and Maija Sladina for sharing scRNA-seq
data with us before publication of the manuscript. We are grateful to
all members of the Rangan laboratory for discussion and comments on the
manuscript. We thank Noor Kotb for naming the dashboard Oo-site. We also
thank Dr. Florence L. Marlow for critically reading and editing the
manuscript. P.R. is funded by the National Institutes of Health NIGMS
(RO1GM11177 and RO1GM135628).</p>
</div>
<div id="materials-and-methods-1" class="section level2" number="3.6">
<h2><span class="header-section-number">3.6</span> Materials and Methods</h2>
<p>The following RNAi stocks were used in this study; <em>ord-GFP</em> (Bickel
Lab), <em>Rps19b::GFP</em> <span class="citation">(McCarthy et al., 2021)</span>,
<em>UAS-Dcr2;nosGAL4</em> (Bloomington stock #25751), <em>bam</em> RNAi (Bloomington
#58178), <em>hs-bam</em>/TM3 (Bloomington #24637),</p>
<div id="sequencing-data" class="section level3" number="3.6.1">
<h3><span class="header-section-number">3.6.1</span> Sequencing data</h3>
<p>Polysome-seq data were obtained from previous studies conducted by the
Rangan lab. Data are available via the following GEO accession numbers:</p>
<p>&gt;UAS-<em>tkv</em> GSE171349</p>
<p>&gt;<em>bam</em> RNAi GSE171349, GSE166275</p>
<p>&gt;<em>bam</em> RNAi; hs-<em>bam</em> GSE143728, GSE195893</p>
<p>Young-WT GSE119458</p>
<p>Single-cell sequencing data were obtained from Slaidina <em>et al</em>., GEO
accession: GSE162192</p>
</div>
<div id="code-availability" class="section level3" number="3.6.2">
<h3><span class="header-section-number">3.6.2</span> Code Availability</h3>
<p>All code used in the preparation of this manuscript is available on
GitHub at
<a href="https://github.com/elliotmartin92/Developmental-Landscape/tree/master/Paper" class="uri">https://github.com/elliotmartin92/Developmental-Landscape/tree/master/Paper</a></p>
<p>The codebase underlying Oo-site is available on GitHub at
<a href="https://github.com/elliotmartin92/Developmental-Landscape/tree/master/ShinyExpresionMap" class="uri">https://github.com/elliotmartin92/Developmental-Landscape/tree/master/ShinyExpresionMap</a></p>
</div>
<div id="antibodies" class="section level3" number="3.6.3">
<h3><span class="header-section-number">3.6.3</span> Antibodies</h3>
<p>Mouse anti-1B1 1:20 (DSHB 1B1), rabbit anti-GFP 1:2000 (abcam, ab6556),
rabbit anti-Vasa 1:4000
<span class="citation">(Upadhyay et al., 2016)</span>, chicken anti-Vasa
1:4000 <span class="citation">(Upadhyay et al., 2016)</span></p>
</div>
<div id="polysome-seq-1" class="section level3" number="3.6.4">
<h3><span class="header-section-number">3.6.4</span> Polysome-seq</h3>
<p>Flies ready for heat shock were placed at 37°C for 2 hours, moved to
room temperature for 4 hours, and placed back into 37°C for 2 additional
hours. Flies were then left overnight at room temperature and the same
heat shocking procedure was repeated for a total of 2 days. Flies were
then dissected in 1x PBS. Polysome-seq was performed as previously
described <span class="citation">(McCarthy et al., 2021)</span>.</p>
</div>
<div id="polysome-seq-data-processing" class="section level3" number="3.6.5">
<h3><span class="header-section-number">3.6.5</span> Polysome-seq data processing</h3>
<p>Reads were mapped to the <em>Drosophila</em> genome (dm6.01) using STAR version
2.6.1c. Mapped reads were assigned to features also using STAR.
Translation efficiency was calculated as in (Flora et al., 2018) using
an R script which is available in the Oo-site Github repo. Briefly, TPMs
(transcripts per million) values were calculated The log<sub>2</sub> ratio of
TPMs between the polysome fraction and total mRNA was calculated as such
to prevent zero counts from overly influencing the data and to prevent
divide by zero errors:
<span class="math inline">\(\frac{\text{Polysome}_{\text{TPM}} + 1}{\text{Input}_{\text{TPM}} + 1}\)</span>
. This ratio represents TE, TE of each replicate was averaged and
standard error about the calculated average for each gene was
calculated.</p>
</div>
<div id="differential-expression" class="section level3" number="3.6.6">
<h3><span class="header-section-number">3.6.6</span> Differential Expression</h3>
<p>Differential expression analysis between all bulk RNA-seq samples in a
pairwise manner was performed using DEseq2
<span class="citation">(Love, Huber, &amp; Anders, 2014)</span>. Differential expression was
considered as Foldchange &gt; |4| fold, FDR &lt; 0.05.</p>
<p>Differential expression analysis between all polysome-seq samples in a
pairwise manner was performed using DEseq2
<span class="citation">(Love, Huber, &amp; Anders, 2014)</span> using the model ~ type + genotype
+ genotype:type with LRT (reduced = ~ type + genotype) to test for
changes in polysome counts controlling for input counts. Differential
expression was considered as (Foldchange &gt; |2| fold, pvalue &lt; 0.05)</p>
<p>Differentially expressed genes between all germline clusters from
scRNA-seq was determined using the FindAllMarkers function from Seurat
<span class="citation">(Hao et al., 2021)</span>. Cutoff was logfc.threshold =
0.75.</p>
<p>Differentially expressed genes between all germarium soma clusters from
scRNA-seq was determined using the FindAllMarkers function from Seurat
<span class="citation">(Hao et al., 2021)</span>. Cutoff was logfc.threshold =
0.75.</p>
</div>
<div id="go-term-heatmaps" class="section level3" number="3.6.7">
<h3><span class="header-section-number">3.6.7</span> GO term heatmaps</h3>
<p>GO-term enrichment analysis was performed using Panther (release
20210224) using the default settings for an Overrepresentation Test of
genes differentially expressed between Input samples. Top 5 GO-terms
based on fold enrichment of each category were plotted using ggplot2
<span class="citation">(Wickham, 2016)</span>.</p>
</div>
<div id="fluorescent-in-situ-hybridization" class="section level3" number="3.6.8">
<h3><span class="header-section-number">3.6.8</span> Fluorescent <em>in situ</em> hybridization</h3>
<p>A modified <em>in situ</em> hybridization procedure for Drosophila ovaries was
followed from Sarkar <em>et al</em>.
<span class="citation">(Sarkar et al., 2021)</span>. Probes were designed and
generated by LGC Biosearch Technologies using Stellaris® RNA FISH Probe
Designer, with specificity to target base pairs of target mRNAs. Ovaries
(3 pairs per sample) were dissected in RNase free 1X PBS and fixed in 1
mL of 5% formaldehyde for 10 minutes. The samples were then
permeabilized in 1mL of Permeabilization Solution (PBST+1% Triton X-100)
rotating in RT for 1 hour. Samples were then washed in wash buffer for 5
minutes (10% deionized formamide and 10% 20x SSC in RNase-free water).
Ovaries were covered and incubated overnight with 1ul of the probe in
hybridization solution (10% dextran sulfate, 1 mg/ml yeast tRNA, 2 mM
RNaseOUT, 0.02 mg/ml BSA, 5x SSC, 10% deionized formamide, and
RNase-free water) and primary antibody at 30°C. Samples were then washed
2 times in 1 mL wash buffer with 1ul of corresponding secondary antibody
for 30 minutes each and mounted in Vectashield (VectaLabs).</p>
</div>
<div id="quantification-of-stainings" class="section level3" number="3.6.9">
<h3><span class="header-section-number">3.6.9</span> Quantification of Stainings</h3>
<p>Stainings were quantified using the Fiji Measure tool. Images were
aligned and cropped to place the stem cell niche at x=0. Individual
cells were outlined within the germarium and Measure was used to
calculate the Mean intensity of staining within the cell as well as the
X coordinate of the centroid of the cell. Values were normalized to 1 by
dividing Mean Intensity values by the maximum of the Mean Intensity per
germarium. Data were plotted using ggplot2 and a fit line was added
using ggplot2 geom_smooth with a “loess” function with default settings.
The shaded area around the line represents standard error.</p>





</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.epub", "EPUB"], ["thesis.docx", "Word"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
