<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 4 Screen of post-translational modifying enzymes for their effects on translation | Translation control tunes Drosophila oogenesis</title>
  <meta name="description" content="Chapter 4 Screen of post-translational modifying enzymes for their effects on translation | Translation control tunes Drosophila oogenesis" />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 4 Screen of post-translational modifying enzymes for their effects on translation | Translation control tunes Drosophila oogenesis" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 4 Screen of post-translational modifying enzymes for their effects on translation | Translation control tunes Drosophila oogenesis" />
  
  
  

<meta name="author" content="Elliot T. Martin" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"/>
<link rel="next" href="5-conclusion.html"/>
<script src="libs/header-attrs-2.8/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="1" data-path="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html"><a href="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html"><i class="fa fa-check"></i><b>1</b> Post-transcriptional gene regulation instructs germline stem cell to oocyte transition during <em>Drosophila</em> oogenesis</a>
<ul>
<li class="chapter" data-level="1.1" data-path="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html"><a href="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html#abstract"><i class="fa fa-check"></i><b>1.1</b> Abstract</a></li>
<li class="chapter" data-level="1.2" data-path="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html"><a href="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html#introduction"><i class="fa fa-check"></i><b>1.2</b> Introduction</a></li>
<li class="chapter" data-level="1.3" data-path="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html"><a href="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html#alternative-splicing-ensures-accurate-production-of-critical-germline-mrnas-to-regulate-sex-determination-and-differentiation"><i class="fa fa-check"></i><b>1.3</b> Alternative splicing ensures accurate production of critical germline mRNAs to regulate sex determination and differentiation</a></li>
<li class="chapter" data-level="1.4" data-path="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html"><a href="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html#rna-modifications-direct-splicing-of-sex-determinants-and-translation-of-differentiation-promoting-genes-in-the-germline"><i class="fa fa-check"></i><b>1.4</b> RNA modifications direct splicing of sex determinants and translation of differentiation promoting genes in the germline</a></li>
<li class="chapter" data-level="1.5" data-path="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html"><a href="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html#production-of-ribosomes-is-finely-tuned-to-facilitate-differentiation"><i class="fa fa-check"></i><b>1.5</b> Production of ribosomes is finely tuned to facilitate differentiation</a></li>
<li class="chapter" data-level="1.6" data-path="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html"><a href="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html#hand-off-mechanisms-facilitated-by-combinatorial-rna-binding-proteins-dynamically-shape-the-translational-landscape-during-oogenesis"><i class="fa fa-check"></i><b>1.6</b> Hand off mechanisms facilitated by combinatorial RNA binding proteins dynamically shape the translational landscape during oogenesis</a></li>
<li class="chapter" data-level="1.7" data-path="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html"><a href="1-post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis.html#summary"><i class="fa fa-check"></i><b>1.7</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><i class="fa fa-check"></i><b>2</b> A translation control module coordinates germline stem cell differentiation with ribosome biogenesis during <em>Drosophila</em> oogenesis</a>
<ul>
<li class="chapter" data-level="2.1" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#summary-1"><i class="fa fa-check"></i><b>2.1</b> Summary:</a></li>
<li class="chapter" data-level="2.2" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#introduction-1"><i class="fa fa-check"></i><b>2.2</b> Introduction</a></li>
<li class="chapter" data-level="2.3" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#results"><i class="fa fa-check"></i><b>2.3</b> Results</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#three-conserved-rna-helicases-are-required-in-the-germline-for-gsc-differentiation"><i class="fa fa-check"></i><b>2.3.1</b> Three conserved RNA helicases are required in the germline for GSC differentiation</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#athos-aramis-and-porthos-are-required-for-ribosome-biogenesis"><i class="fa fa-check"></i><b>2.3.2</b> Athos, Aramis, and Porthos are required for ribosome biogenesis</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#aramis-promotes-cell-cycle-progression-via-p53-repression"><i class="fa fa-check"></i><b>2.3.3</b> Aramis promotes cell cycle progression via p53 repression</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#aramis-regulated-targets-contain-a-top-motif-in-their-5utr"><i class="fa fa-check"></i><b>2.3.4</b> Aramis-regulated targets contain a TOP motif in their 5'UTR</a></li>
<li class="chapter" data-level="2.3.5" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#larp-binds-top-sequences-in-drosophila"><i class="fa fa-check"></i><b>2.3.5</b> Larp binds TOP sequences in <em>Drosophila</em></a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#discussion"><i class="fa fa-check"></i><b>2.4</b> Discussion</a>
<ul>
<li class="chapter" data-level="2.4.1" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#aramis-athos-and-porthos-are-required-for-efficient-ribosome-biogenesis-in-drosophila"><i class="fa fa-check"></i><b>2.4.1</b> Aramis, Athos, and Porthos are required for efficient ribosome biogenesis in <em>Drosophila</em></a></li>
<li class="chapter" data-level="2.4.2" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#ribosome-biogenesis-defects-leads-to-cell-cycle-defects-mediated-by-p53"><i class="fa fa-check"></i><b>2.4.2</b> Ribosome biogenesis defects leads to cell cycle defects mediated by p53</a></li>
<li class="chapter" data-level="2.4.3" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#ribosome-biogenesis-defects-leads-to-repression-of-top-containing-mrna"><i class="fa fa-check"></i><b>2.4.3</b> Ribosome biogenesis defects leads to repression of TOP-containing mRNA</a></li>
<li class="chapter" data-level="2.4.4" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#larp-transduces-growth-status-to-ribosome-biogenesis-targets"><i class="fa fa-check"></i><b>2.4.4</b> Larp transduces growth status to ribosome biogenesis targets</a></li>
<li class="chapter" data-level="2.4.5" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#ribosome-biogenesis-in-stem-cell-differentiation-and-ribosomopathies"><i class="fa fa-check"></i><b>2.4.5</b> Ribosome biogenesis in stem cell differentiation and ribosomopathies</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html"><a href="2-a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis.html#materials-and-methods"><i class="fa fa-check"></i><b>2.5</b> Materials and Methods</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><i class="fa fa-check"></i><b>3</b> Oo-site: A dashboard to visualize gene expression during <em>Drosophila</em> oogenesis reveals meiotic entry is regulated post-transcriptionally</a>
<ul>
<li class="chapter" data-level="3.1" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#summary-2"><i class="fa fa-check"></i><b>3.1</b> Summary</a></li>
<li class="chapter" data-level="3.2" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#introduction-2"><i class="fa fa-check"></i><b>3.2</b> Introduction</a></li>
<li class="chapter" data-level="3.3" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#results-1"><i class="fa fa-check"></i><b>3.3</b> Results</a></li>
<li class="chapter" data-level="3.4" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#discussion-1"><i class="fa fa-check"></i><b>3.4</b> Discussion</a></li>
<li class="chapter" data-level="3.5" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#acknowledgements"><i class="fa fa-check"></i><b>3.5</b> Acknowledgements</a></li>
<li class="chapter" data-level="3.6" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#materials-and-methods-1"><i class="fa fa-check"></i><b>3.6</b> Materials and Methods</a>
<ul>
<li class="chapter" data-level="3.6.1" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#sequencing-data"><i class="fa fa-check"></i><b>3.6.1</b> Sequencing data</a></li>
<li class="chapter" data-level="3.6.2" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#code-availability"><i class="fa fa-check"></i><b>3.6.2</b> Code Availability</a></li>
<li class="chapter" data-level="3.6.3" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#antibodies"><i class="fa fa-check"></i><b>3.6.3</b> Antibodies</a></li>
<li class="chapter" data-level="3.6.4" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#polysome-seq-1"><i class="fa fa-check"></i><b>3.6.4</b> Polysome-seq</a></li>
<li class="chapter" data-level="3.6.5" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#polysome-seq-data-processing"><i class="fa fa-check"></i><b>3.6.5</b> Polysome-seq data processing</a></li>
<li class="chapter" data-level="3.6.6" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#differential-expression"><i class="fa fa-check"></i><b>3.6.6</b> Differential Expression</a></li>
<li class="chapter" data-level="3.6.7" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#go-term-heatmaps"><i class="fa fa-check"></i><b>3.6.7</b> GO term heatmaps</a></li>
<li class="chapter" data-level="3.6.8" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#fluorescent-in-situ-hybridization"><i class="fa fa-check"></i><b>3.6.8</b> Fluorescent <em>in situ</em> hybridization</a></li>
<li class="chapter" data-level="3.6.9" data-path="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html"><a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html#quantification-of-stainings"><i class="fa fa-check"></i><b>3.6.9</b> Quantification of Stainings</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html"><a href="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html"><i class="fa fa-check"></i><b>4</b> Screen of post-translational modifying enzymes for their effects on translation</a>
<ul>
<li class="chapter" data-level="4.1" data-path="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html"><a href="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html#abstract-1"><i class="fa fa-check"></i><b>4.1</b> Abstract</a></li>
<li class="chapter" data-level="4.2" data-path="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html"><a href="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html#introduction-3"><i class="fa fa-check"></i><b>4.2</b> Introduction</a></li>
<li class="chapter" data-level="4.3" data-path="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html"><a href="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html#results-and-conclusions"><i class="fa fa-check"></i><b>4.3</b> Results and Conclusions</a></li>
<li class="chapter" data-level="4.4" data-path="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html"><a href="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html#methods"><i class="fa fa-check"></i><b>4.4</b> Methods</a>
<ul>
<li class="chapter" data-level="4.4.1" data-path="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html"><a href="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html#flylines"><i class="fa fa-check"></i><b>4.4.1</b> Flylines</a></li>
<li class="chapter" data-level="4.4.2" data-path="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html"><a href="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html#cloning"><i class="fa fa-check"></i><b>4.4.2</b> Cloning</a></li>
<li class="chapter" data-level="4.4.3" data-path="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html"><a href="4-screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation.html#dual-luciferase-assay"><i class="fa fa-check"></i><b>4.4.3</b> Dual luciferase assay</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-conclusion.html"><a href="5-conclusion.html"><i class="fa fa-check"></i><b>5</b> Conclusion</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Translation control tunes <em>Drosophila</em> oogenesis</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="screen-of-post-translational-modifying-enzymes-for-their-effects-on-translation" class="section level1" number="4">
<h1><span class="header-section-number">Chapter 4</span> Screen of post-translational modifying enzymes for their effects on translation</h1>
<p>Elliot Martin, Siu Wah Wong-Deyrup, Gaby Fuchs, and Prashanth Rangan</p>
<div id="abstract-1" class="section level2" number="4.1">
<h2><span class="header-section-number">4.1</span> Abstract</h2>
<p>The ribosome has been demonstrated to play an active role in regulating
translation. We hypothesized that post-translational modifications on
ribosomal proteins might modulate the activity of the ribosome.
Therefore, we attempted to screen for enzymes known to deposit
post-translational modifications with the goal of identifying genes that
play a role in regulating translation using a dual-luciferase assay
which monitors the relative amount of Internal Ribosome Entry Site
(IRES) translation compared to cap-based translation. We identified
several genes that had a significant effect on the translation of the
dual luciferase reporter, however, these affects were modest. We also
found that ovaries enriched for stem cells and stem cell daughters had
significantly higher relative IRES translation, however, we did not
observe the expected result when we knocked down RpS25, which is known
to be required for efficient IRES translation. Due to these limitations,
we did not pursue these findings further.</p>
</div>
<div id="introduction-3" class="section level2" number="4.2">
<h2><span class="header-section-number">4.2</span> Introduction</h2>
<p>In the past, ribosomes have been considered as passive players in the
process of translation. However, evidence has begun to accumulate that
ribosomes can play a regulatory role in translation
<span class="citation">(Gościńska &amp; Topf, 2020)</span>. There are several mechanisms
through which ribosomes have been found to regulate translation. One
example of this is Rack1 which is a core ribosomal protein
<span class="citation">(Gerbasi, Weaver, Hill, Friedman, &amp; Link, 2004)</span>. Despite the role of Rack1 as a
core ribosomal protein, it is dispensable for cell viability, but
required for efficient Internal Ribosome Entry Site (IRES) translation
<span class="citation">(Coyle, Gilbert, &amp; Doudna, 2009; LaFontaine, Miller, Permaul, Martin, &amp; Fuchs, 2020; Majzoub et al., 2014)</span>. Furthermore, Rack1 activity
can be regulated through post-translational phosphorylation which can be
exploited during viral infection to allow for translation initiation on
polyA-leader sequences <span class="citation">(Jha et al., 2017)</span>.</p>
<p>Other mechanisms of translation control involving the ribosome have not
been conclusively demonstrated but show promise. For instance, under
stress conditions, ribosomal proteins have been shown to carry different
post-translational modifications than under basal conditions
<span class="citation">(Jha et al., 2017; Mukhopadhyay et al., 2008; Simsek &amp; Barna, 2017)</span>. The modifications in some cases can
alter the propensity of the ribosome to translate certain mRNAs, which
could allow the cell to better adapt to varying conditions
<span class="citation">(Jha et al., 2017)</span>.</p>
<p>There are two main modes of translation initiation, canonical cap-based
translation and cap-independent translation. Cap-based translation is a
complex many-stepped process in which the 43S ribosome pre-initiation
complex (43S PIC) is recruited to an mRNA through interacting with
cap-binding protein (eIF4E) as well as other components
<span class="citation">(Jackson, Hellen, &amp; Pestova, 2010)</span>. This initiates scanning
where the 43S PIC moves along the mRNA until a suitable start codon is
identified and translation begins once a 60S ribosomal subunit is
recruited <span class="citation">(Jackson, Hellen, &amp; Pestova, 2010)</span>. Translation
can also be initiated cap-independently
<span class="citation">(Jackson, Hellen, &amp; Pestova, 2010)</span>. Several mechanisms can
allow translation to occur without the presence of eIF4E, but of
interest to this work is IRES-based initiation
<span class="citation">(Shatsky, Terenin, Smirnova, &amp; Andreev, 2018)</span>. IRESs are secondary
structures of RNA of which several classes have been described which are
classified based on the initiation factors they require as well as their
secondary structure <span class="citation">(Mailliot &amp; Martin, 2018)</span>. IRESs
bypass at least the use of eIF4E and can bypass the use of any
initiation factors at all, requiring only the 40S and 60S ribosomal
subunits to perform initiation <span class="citation">(Mailliot &amp; Martin, 2018; Shatsky, Terenin, Smirnova, &amp; Andreev, 2018)</span>. In general, IRESs are used
at a higher rate under stress conditions such as those initiated by
viral infection, but also under other stress conditions
<span class="citation">(Mailliot &amp; Martin, 2018; Shatsky, Terenin, Smirnova, &amp; Andreev, 2018)</span>. However, how cellular
stress changes the usage of IRES translation vs cap-based translation is
not fully understood.</p>
<p>We performed a screen to attempt to identify enzymes that
post-translationally modify proteins and have the potential to alter the
translation landscape. To perform this screen, we used the <em>Drosophila</em>
ovary as it has several advantages. First, <em>Drosophila</em> have thousands
of available RNAi lines that can be used to knock down genes of interest
<span class="citation">(del Valle Rodríguez, Didiano, &amp; Desplan, 2012)</span>. Second, many of the genes we
chose to screen are histone modifying enzymes that may be required for
cell survival, by using the ovary as our tissue of interest, we were
able to deplete those genes in the germline without affecting the
viability of the flies. The last major advantage to working in the
<em>Drosophila</em> ovary in this case is that endogenous control elements have
been identified allowing for expression of a reporter specifically in
the germline <span class="citation">(Rørth, 1998; Serano, Cheung, Frank, &amp; Cohen, 1994)</span>. This in combination with
germline specific RNAi allowed us to assay for changes in translation
only in the germline tissue.</p>
</div>
<div id="results-and-conclusions" class="section level2" number="4.3">
<h2><span class="header-section-number">4.3</span> Results and Conclusions</h2>
<p>To assess the status of translation we adapted a dual luciferase assay
commonly used to monitor translation status. This reporter contains two
ORFs encoded in a single transcript which allowed us to assay for
changes in translation while eliminating confounding differences in
transcription. The reporter transcript was under the control of a Nos
promoter, which is only expressed in the germline. The reporter had a
Nos 5'UTR and a K10 3'UTR which have both been demonstrated to have a
neutral effect on translation <span class="citation">(Gavis &amp; Lehmann, 1992, 1992; Serano, Cheung, Frank, &amp; Cohen, 1994)</span>. The first ORF encodes
<em>Renilla</em> luciferse under the control of the Nos 5'UTR and its
production monitors for cap-based translation, this ORF is followed by
hairpins to prevent ribosomal read-through. The second ORF encodes a
firefly luciferase under the control of the Cricket Paralysis Virus
intergenic (CrPV) IRES . These luciferases can be monitored sequentially
as they require different conditions in order to produce luminescence
and in this way can be used to monitor changes in the relative amount of
IRES to cap-based translation.</p>
<div class="figure" style="text-align: center"><span id="fig:Screen-fig-1"></span>
<img src="figure/Screen/Construct.png" alt="Schematic of dual luciferase reporter used for screen." width="97%" />
<p class="caption">
Figure 4.1: <strong>Schematic of dual luciferase reporter used for screen.</strong>
</p>
</div>
<p>The dual luciferase reporter consists of a Nos promoter and 5’UTR controlling translation initaiation of <em>Renilla</em> luciferase followed by a read-though stop consisting of hairpins and the CrPV-IGR IRES which controls initiation of Firefly luciferase.</p>
<p>We performed a screen of 35 genes known to perform post-translational
modification to determine if any of them played a role in regulating
IRES vs cap-based translation. We also screened several control genes to
ensure our reporter system was working as intended. We knocked down
RpS25 which is known to be required for IRES, but not cap-based
translation, however, we did not find a statistically significant
difference between the knockdown and control ovaries
<span class="citation">(Hertz, Landry, Willis, Luo, &amp; Thompson, 2013; Landry, Hertz, &amp; Thompson, 2009)</span>. There are a several
explanations of this finding. First, knockdown of RpS25 had a severe
phenotype, which was unexpected as previous work indicated that RpS25 is
required for IRES, but not cap-based translation
<span class="citation">(Landry, Hertz, &amp; Thompson, 2009)</span>. Therefore, it could be that
developmental differences between the control and RpS25 RNAi masked any
change in relative IRES translation. Second, it is possible RpS25 is not
efficiently depleted or that the RNAi line used has off-target effects.
Because of this lack of a result from RpS25 RNAi, our positive control,
the remainder of our findings must be taken in context. Several genes
were found to have a significant affect on the amount of relative IRES
translation, including Not, HDAC3, Lsd1, nsl-1, and Set8.</p>
<p>We additionally included two developmental controls to determine if a
phenotype which enriched for stem cells (UAS-<em>tkv</em>) or stem cell
daughters (<em>bam</em> RNAi) would alter the relative IRES translation .
Indeed, we found that the relative IRES translation was slight, but
significantly elevated. This could suggest that IRES translation occurs
more in the stem cells and stem cell daughters of the ovary.</p>
<p>Lastly, we identified five genes that when knocked down resulted in
significantly lower relative IRES translation, including <em>Set8</em>,
<em>nsl-1</em>, <em>Lsd1</em>, <em>HDAC3</em>, and <em>Not</em>. However, the affect size observed
when these genes were knocked down was modest, with Set8 having the most
dramatic affect with a 23.6% reduction in the relative amount of IRES
translation. However, it should also be noted that as is widely
accepted, no multiple test-correction was performed as for screens
multiple-test correction leads to a high degree of type I error and
screens typically are followed by confirmatory studies
<span class="citation">(Gaus, 2015)</span>. In this case the
results have not been followed-up for the aforementioned limitations
regarding RpS25 as well as how the subtle changes in relative IRES
translation were.</p>
<div class="figure" style="text-align: center"><span id="fig:Screen-fig-2"></span>
<img src="figure/Screen/IRES_screen.png" alt="Screen of post-translational modification enzymes." width="95%" />
<p class="caption">
Figure 4.2: <strong>Screen of post-translational modification enzymes.</strong>
</p>
</div>
<p>Boxplot of relative IRES translation normalized to control ovaries resulting from knockdown of indicated target proteins. Boxes in green represent significant changes in relative IRES-mediated translation compared to control.</p>
<p>As one of the largest changes to relative IRES translation was in <em>bam</em>
RNAi, which increased relative IRES translation by 23%, it could indeed
be that IRES translation is regulated over the course of stem cell
differentiation. Future work should be aimed at this aspect of these
findings. A spatial version of this reporter could be made using
fluorescent reporters to assay the relative IRES translation over stem
cell differentiation in the ovary.</p>
</div>
<div id="methods" class="section level2" number="4.4">
<h2><span class="header-section-number">4.4</span> Methods</h2>
<div id="flylines" class="section level3" number="4.4.1">
<h3><span class="header-section-number">4.4.1</span> Flylines</h3>
<p><em>nosGAL4</em> (Bloomington stock 7303)</p>
<p>Generated by this work: <em>nosGAL4</em>; <em>dLuciferase</em></p>
<p>Protein targets screened and corresponding RNAi line number used:</p>
<table>
<thead>
<tr class="header">
<th align="left">Protein Target</th>
<th align="left">Line Number</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Gpp</td>
<td align="left">v110264</td>
</tr>
<tr class="even">
<td align="left">D12</td>
<td align="left">v29954</td>
</tr>
<tr class="odd">
<td align="left">Art8</td>
<td align="left">v100228</td>
</tr>
<tr class="even">
<td align="left">Su(z)12</td>
<td align="left">33402</td>
</tr>
<tr class="odd">
<td align="left">Sgf29</td>
<td align="left">36637</td>
</tr>
<tr class="even">
<td align="left">Atac2</td>
<td align="left">32890</td>
</tr>
<tr class="odd">
<td align="left">CG2051</td>
<td align="left">34730</td>
</tr>
<tr class="even">
<td align="left">chm</td>
<td align="left">32484</td>
</tr>
<tr class="odd">
<td align="left">Tip60</td>
<td align="left">28563</td>
</tr>
<tr class="even">
<td align="left">Set1</td>
<td align="left">33704</td>
</tr>
<tr class="odd">
<td align="left">chm</td>
<td align="left">32484</td>
</tr>
<tr class="even">
<td align="left">Atac2</td>
<td align="left">53918</td>
</tr>
<tr class="odd">
<td align="left">Lsd1</td>
<td align="left">V16235</td>
</tr>
<tr class="even">
<td align="left">Ada3 (dik)</td>
<td align="left">28905</td>
</tr>
<tr class="odd">
<td align="left">Caf1</td>
<td align="left">31714</td>
</tr>
<tr class="even">
<td align="left">CG10289</td>
<td align="left">35597</td>
</tr>
<tr class="odd">
<td align="left">Spt7 (comr)</td>
<td align="left">42552</td>
</tr>
<tr class="even">
<td align="left">E(z)</td>
<td align="left">36068</td>
</tr>
<tr class="odd">
<td align="left">Enok</td>
<td align="left">29518</td>
</tr>
<tr class="even">
<td align="left">faf</td>
<td align="left">35728</td>
</tr>
<tr class="odd">
<td align="left">Gcn5</td>
<td align="left">33981</td>
</tr>
<tr class="even">
<td align="left">Hdac3</td>
<td align="left">31633</td>
</tr>
<tr class="odd">
<td align="left">Jarid2</td>
<td align="left">32891</td>
</tr>
<tr class="even">
<td align="left">Ubp8 (not)</td>
<td align="left">28725</td>
</tr>
<tr class="odd">
<td align="left">nsl1 (wah)</td>
<td align="left">32561</td>
</tr>
<tr class="even">
<td align="left">Polybromo</td>
<td align="left">32840</td>
</tr>
<tr class="odd">
<td align="left">Set8</td>
<td align="left">35322</td>
</tr>
<tr class="even">
<td align="left">Su(var)205</td>
<td align="left">33400</td>
</tr>
<tr class="odd">
<td align="left">Sxc</td>
<td align="left">50909</td>
</tr>
<tr class="even">
<td align="left">Trr</td>
<td align="left">29563</td>
</tr>
<tr class="odd">
<td align="left">Wda</td>
<td align="left">31125</td>
</tr>
<tr class="even">
<td align="left">Rps25</td>
<td align="left">V101342</td>
</tr>
<tr class="odd">
<td align="left">LRRK</td>
<td align="left">32457</td>
</tr>
<tr class="even">
<td align="left">S6K</td>
<td align="left">41895</td>
</tr>
<tr class="odd">
<td align="left">S6KII</td>
<td align="left">41895</td>
</tr>
<tr class="even">
<td align="left">TKV</td>
<td align="left">36537</td>
</tr>
</tbody>
</table>
</div>
<div id="cloning" class="section level3" number="4.4.2">
<h3><span class="header-section-number">4.4.2</span> Cloning</h3>
<p>A gBlock corresponding to a codon optimized version of <em>Renilla</em> and
firefly luciferase as well as the readthrough block and CrPV IGR IRES
were purchased from IDT. This gBlock was cloned using Gibson cloning
according to manufacturer's instructions (#E2611S) into pCasper2
containing a Nos promoter and K10 3'UTR. Colonies were picked, cultured,
and plasmids purified and sequenced by Eton Bioscience Inc. to conform
the correct sequence had been cloned. A midi-prep scale of the plasmid
was prepared using standard methods and plasmids were sent to BestGene
Inc. for microinjection.</p>
</div>
<div id="dual-luciferase-assay" class="section level3" number="4.4.3">
<h3><span class="header-section-number">4.4.3</span> Dual luciferase assay</h3>
<p>Ovaries were dissected on 1X PBS in microcentrifuge tubes. Excess PBS
was aspirated and ovaries were stored at -20°C. Ovaries were lysed in 25
µl of passive lysis buffer (Promega) using a plastic pestle. 20 µl of
lysate was added to a white 96 well plate and luminance was measured
using a Glomax 20/20 luminometer (Promega) using the dual luciferase
assay reagent (Promega #E1910). At least three biological replicates
for Firefly/<em>Renilla</em> luminescence values of each RNAi line were
generated.</p>

</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="3-oo-site-a-dashboard-to-visualize-gene-expression-during-drosophila-oogenesis-reveals-meiotic-entry-is-regulated-post-transcriptionally.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="5-conclusion.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.epub", "EPUB"], ["thesis.docx", "Word"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
