% This is the Reed College LaTeX thesis template. Most of the work
% for the document class was done by Sam Noble (SN), as well as this
% template. Later comments etc. by Ben Salzberg (BTS). Additional
% restructuring and APA support by Jess Youngberg (JY).
% Your comments and suggestions are more than welcome; please email
% them to cus@reed.edu
%
% See https://www.reed.edu/cis/help/LaTeX/index.html for help. There are a
% great bunch of help pages there, with notes on
% getting started, bibtex, etc. Go there and read it if you're not
% already familiar with LaTeX.
%
% Any line that starts with a percent symbol is a comment.
% They won't show up in the document, and are useful for notes
% to yourself and explaining commands.
% Commenting also removes a line from the document;
% very handy for troubleshooting problems. -BTS

% As far as I know, this follows the requirements laid out in
% the 2002-2003 Senior Handbook. Ask a librarian to check the
% document before binding. -SN

%%
%% Preamble
%%
% \documentclass{<something>} must begin each LaTeX document
\documentclass[12pt,oneside]{reedthesis}
% Packages are extensions to the basic LaTeX functions. Whatever you
% want to typeset, there is probably a package out there for it.
% Chemistry (chemtex), screenplays, you name it.
% Check out CTAN to see: https://www.ctan.org/
%%
\usepackage{graphicx,latexsym}
\usepackage{amsmath}
\usepackage{amssymb,amsthm}
\usepackage{longtable,booktabs,setspace}
\usepackage{chemarr} %% Useful for one reaction arrow, useless if you're not a chem major
\usepackage[hyphens]{url}
% Added by CII
\usepackage{hyperref}
\usepackage{lmodern}
\usepackage{float}
\floatplacement{figure}{H}
% Thanks, @Xyv
\usepackage{calc}
% End of CII addition
\usepackage{rotating}

% Next line commented out by CII
%%% \usepackage{natbib}
% Comment out the natbib line above and uncomment the following two lines to use the new
% biblatex-chicago style, for Chicago A. Also make some changes at the end where the
% bibliography is included.
%\usepackage{biblatex-chicago}
%\bibliography{thesis}


% Added by CII (Thanks, Hadley!)
% Use ref for internal links
\renewcommand{\hyperref}[2][???]{\autoref{#1}}
\def\chapterautorefname{Chapter}
\def\sectionautorefname{Section}
\def\subsectionautorefname{Subsection}
% End of CII addition

% Added by CII
\usepackage{caption}
\captionsetup{width=5in}
% End of CII addition

% \usepackage{times} % other fonts are available like times, bookman, charter, palatino

% Syntax highlighting #22
  \usepackage{color}
  \usepackage{fancyvrb}
  \newcommand{\VerbBar}{|}
  \newcommand{\VERB}{\Verb[commandchars=\\\{\}]}
  \DefineVerbatimEnvironment{Highlighting}{Verbatim}{commandchars=\\\{\}}
  % Add ',fontsize=\small' for more characters per line
  \usepackage{framed}
  \definecolor{shadecolor}{RGB}{248,248,248}
  \newenvironment{Shaded}{\begin{snugshade}}{\end{snugshade}}
  \newcommand{\AlertTok}[1]{\textcolor[rgb]{0.94,0.16,0.16}{#1}}
  \newcommand{\AnnotationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
  \newcommand{\AttributeTok}[1]{\textcolor[rgb]{0.77,0.63,0.00}{#1}}
  \newcommand{\BaseNTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
  \newcommand{\BuiltInTok}[1]{#1}
  \newcommand{\CharTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
  \newcommand{\CommentTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textit{#1}}}
  \newcommand{\CommentVarTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
  \newcommand{\ConstantTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
  \newcommand{\ControlFlowTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{\textbf{#1}}}
  \newcommand{\DataTypeTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{#1}}
  \newcommand{\DecValTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
  \newcommand{\DocumentationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
  \newcommand{\ErrorTok}[1]{\textcolor[rgb]{0.64,0.00,0.00}{\textbf{#1}}}
  \newcommand{\ExtensionTok}[1]{#1}
  \newcommand{\FloatTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
  \newcommand{\FunctionTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
  \newcommand{\ImportTok}[1]{#1}
  \newcommand{\InformationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
  \newcommand{\KeywordTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{\textbf{#1}}}
  \newcommand{\NormalTok}[1]{#1}
  \newcommand{\OperatorTok}[1]{\textcolor[rgb]{0.81,0.36,0.00}{\textbf{#1}}}
  \newcommand{\OtherTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{#1}}
  \newcommand{\PreprocessorTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textit{#1}}}
  \newcommand{\RegionMarkerTok}[1]{#1}
  \newcommand{\SpecialCharTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
  \newcommand{\SpecialStringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
  \newcommand{\StringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
  \newcommand{\VariableTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
  \newcommand{\VerbatimStringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
  \newcommand{\WarningTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}

% To pass between YAML and LaTeX the dollar signs are added by CII
\title{Translation control tunes \emph{Drosophila} oogenesis}
\author{Elliot T. Martin}
% The month and year that you submit your FINAL draft TO THE LIBRARY (May or December)
\date{December 2021}
\division{Biology}
\advisor{Prashanth Rangan}
\institution{University at Albany}
\degree{Doctor of Philosophy}
%If you have two advisors for some reason, you can use the following
% Uncommented out by CII
\altadvisor{Gabriele Fuchs}
% End of CII addition

%%% Remember to use the correct department!
\department{Biology}
% if you're writing a thesis in an interdisciplinary major,
% uncomment the line below and change the text as appropriate.
% check the Senior Handbook if unsure.
%\thedivisionof{The Established Interdisciplinary Committee for}
% if you want the approval page to say "Approved for the Committee",
% uncomment the next line
%\approvedforthe{Committee}

% Added by CII
%%% Copied from knitr
%% maxwidth is the original width if it's less than linewidth
%% otherwise use linewidth (to make sure the graphics do not exceed the margin)
\makeatletter
\def\maxwidth{ %
  \ifdim\Gin@nat@width>\linewidth
    \linewidth
  \else
    \Gin@nat@width
  \fi
}
\makeatother

% From {rticles}
\newlength{\csllabelwidth}
\setlength{\csllabelwidth}{3em}
\newlength{\cslhangindent}
\setlength{\cslhangindent}{1.5em}
% for Pandoc 2.8 to 2.10.1
\newenvironment{cslreferences}%
  {\setlength{\parindent}{0pt}%
  \everypar{\setlength{\hangindent}{\cslhangindent}}\ignorespaces}%
  {\par}
% For Pandoc 2.11+
% As noted by @mirh [2] is needed instead of [3] for 2.12
\newenvironment{CSLReferences}[2] % #1 hanging-ident, #2 entry spacing
 {% don't indent paragraphs
  \setlength{\parindent}{0pt}
  % turn on hanging indent if param 1 is 1
  \ifodd #1 \everypar{\setlength{\hangindent}{\cslhangindent}}\ignorespaces\fi
  % set entry spacing
  \ifnum #2 > 0
  \setlength{\parskip}{#2\baselineskip}
  \fi
 }%
 {}
\usepackage{calc} % for calculating minipage widths
\newcommand{\CSLBlock}[1]{#1\hfill\break}
\newcommand{\CSLLeftMargin}[1]{\parbox[t]{\csllabelwidth}{#1}}
\newcommand{\CSLRightInline}[1]{\parbox[t]{\linewidth - \csllabelwidth}{#1}}
\newcommand{\CSLIndent}[1]{\hspace{\cslhangindent}#1}

\renewcommand{\contentsname}{Table of Contents}
% End of CII addition

\setlength{\parskip}{0pt}

% Added by CII

\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}

\Acknowledgements{
I want to thank a few people.
}

\Dedication{
You can have a dedication here if you wish.
}

\Preface{
This is an example of a thesis setup to use the reed thesis document class
(for LaTeX) and the R bookdown package, in general.
}

\Abstract{
All dynamic biological processes require control over transcription, translation, or post-translational products. Stem cells in particular require dynamic control of gene expression. My work has focused on characterizing this control, primarily at the translation level, to better understand how stem cell differentiation occurs. Stem cells are cells with the unique ability to develop into more specialized cell types in a process called differentiation. Some stem cell, including those focused on in my work, also have the ability to ``self-renew'', a process that allows one stem cell to copy itself giving rise to two stem cells. These processes must be carefully balanced as excess self-renewal will result in cells that do not give rise to differentiated cells necessary for further development or biological function. However, excess differentiation will result in the lack of an available pool of stem cells, preventing future differentiation and development.
The decision of a stem cell to either self renew or differentiate is controlled by specific cellular pathways that can act at the level of transcription, translation, or post-translation. To study the regulation of these pathways in-vivo I have used the female \emph{Drosophila} germline as a model system. The female \emph{Drosophila} germline is contained within two pairs of ovaries. Ovaries consist of two main types of tissue, soma and germline. Each ovary is made up of strands called ovarioles. Ovarioles represent an assembly line of successive development. At the anterior tip of each ovariole a structure called a germarium is present. At the anterior of the germarium two to three stem cells are housed in a somatic niche. These germline stem cells (GSCs) can self-renew, or differentiate giving rise to a daughter cell called a cystoblast (CB). The CB turns on a differentiation factor called bag of marbles (bam). This CB then undergoes four incomplete cellular divisions, resulting in interconnected cysts consisting of two, four, eight, and finally sixteen cells. One of these cells is designated as the oocyte while the rest of the cells will become nurse cells. The sixteen cell cyst is then encapsulated by somatic cells, forming egg chambers. Egg chambers successively grow in size in fourteen stages. During this time the nurse cells produce mRNAs and proteins that are transported to the oocyte. The oocyte continues to grow, while the nurse cells eventually die, dumping their contents into the oocyte. Once the oocyte reaches the final, 14th stage it is known as an egg.
Each of the steps from GSC to egg require changes in cellular pathways. These changes can occur at the level of transcription, translation, or post-translation. Decades of research has elucidated many of the changes that occur during oogenesis, however, many players in this process still remain mysterious. My work has helped to identify and characterize novel developmental mechanisms that are required for the successive developmental transitions that take place during oogenesis. I have leveraged RNAseq and polysome-seq to probe the global transcriptional and translational landscape over development. I have also used the power of \emph{Drosophila} genetics in concert with these sequencing techniques to identify and characterize misregulated pathways.
}

% End of CII addition
%%
%% End Preamble
%%
%
\begin{document}

% Everything below added by CII
  \maketitle

\frontmatter % this stuff will be roman-numbered
\pagestyle{empty} % this removes page numbers from the frontmatter
  \begin{acknowledgements}
    I want to thank a few people.
  \end{acknowledgements}
  \begin{preface}
    This is an example of a thesis setup to use the reed thesis document class
    (for LaTeX) and the R bookdown package, in general.
  \end{preface}
  \hypersetup{linkcolor=black}
  \setcounter{secnumdepth}{2}
  \setcounter{tocdepth}{2}
  \tableofcontents

  \listoftables

  \listoffigures
  \begin{abstract}
    All dynamic biological processes require control over transcription, translation, or post-translational products. Stem cells in particular require dynamic control of gene expression. My work has focused on characterizing this control, primarily at the translation level, to better understand how stem cell differentiation occurs. Stem cells are cells with the unique ability to develop into more specialized cell types in a process called differentiation. Some stem cell, including those focused on in my work, also have the ability to ``self-renew'', a process that allows one stem cell to copy itself giving rise to two stem cells. These processes must be carefully balanced as excess self-renewal will result in cells that do not give rise to differentiated cells necessary for further development or biological function. However, excess differentiation will result in the lack of an available pool of stem cells, preventing future differentiation and development.
    The decision of a stem cell to either self renew or differentiate is controlled by specific cellular pathways that can act at the level of transcription, translation, or post-translation. To study the regulation of these pathways in-vivo I have used the female \emph{Drosophila} germline as a model system. The female \emph{Drosophila} germline is contained within two pairs of ovaries. Ovaries consist of two main types of tissue, soma and germline. Each ovary is made up of strands called ovarioles. Ovarioles represent an assembly line of successive development. At the anterior tip of each ovariole a structure called a germarium is present. At the anterior of the germarium two to three stem cells are housed in a somatic niche. These germline stem cells (GSCs) can self-renew, or differentiate giving rise to a daughter cell called a cystoblast (CB). The CB turns on a differentiation factor called bag of marbles (bam). This CB then undergoes four incomplete cellular divisions, resulting in interconnected cysts consisting of two, four, eight, and finally sixteen cells. One of these cells is designated as the oocyte while the rest of the cells will become nurse cells. The sixteen cell cyst is then encapsulated by somatic cells, forming egg chambers. Egg chambers successively grow in size in fourteen stages. During this time the nurse cells produce mRNAs and proteins that are transported to the oocyte. The oocyte continues to grow, while the nurse cells eventually die, dumping their contents into the oocyte. Once the oocyte reaches the final, 14th stage it is known as an egg.
    Each of the steps from GSC to egg require changes in cellular pathways. These changes can occur at the level of transcription, translation, or post-translation. Decades of research has elucidated many of the changes that occur during oogenesis, however, many players in this process still remain mysterious. My work has helped to identify and characterize novel developmental mechanisms that are required for the successive developmental transitions that take place during oogenesis. I have leveraged RNAseq and polysome-seq to probe the global transcriptional and translational landscape over development. I have also used the power of \emph{Drosophila} genetics in concert with these sequencing techniques to identify and characterize misregulated pathways.
  \end{abstract}
  \begin{dedication}
    You can have a dedication here if you wish.
  \end{dedication}
\mainmatter % here the regular arabic numbering starts
\pagestyle{fancyplain} % turns page numbering back on

\hypertarget{introduction}{%
\chapter*{Introduction}\label{introduction}}
\addcontentsline{toc}{chapter}{Introduction}

















\hypertarget{post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis}{%
\chapter{\texorpdfstring{Post-transcriptional gene regulation instructs germline stem cell to oocyte transition during \emph{Drosophila} oogenesis}{Post-transcriptional gene regulation instructs germline stem cell to oocyte transition during Drosophila oogenesis}}\label{post-transcriptional-gene-regulation-instructs-germline-stem-cell-to-oocyte-transition-during-drosophila-oogenesis}}

Patrick Blatt, Elliot T. Martin, Shane M. Breznak, Prashanth Rangan*

Department of Biological Sciences/RNA Institute, University at Albany
SUNY, Albany, NY

University at Albany SUNY, 1400 Washington Avenue, Albany, 12222, USA

*Correspondence to: prangan@albany.edu

\hypertarget{abstract}{%
\section{Abstract}\label{abstract}}

During oogenesis, several developmental processes must be traversed to
ensure effective completion of gametogenesis including, stem cell
maintenance and asymmetric division, differentiation, mitosis and
meiosis, and production of maternally contributed mRNAs, making the germ
line a salient model for understanding how cell fate transitions are
mediated. Due to silencing of the genome during meiotic divisions, there
is little instructive transcription, barring a few examples, to mediate
these critical transitions. In \emph{Drosophila}, several layers of
post-transcriptional regulation ensure that the mRNAs required for these
processes are expressed in a timely manner and as needed during germline
differentiation. These layers of regulation include alternative
splicing, RNA modification, ribosome production, and translational
repression. Many of the molecules and pathways involved in these
regulatory activities are conserved from \emph{Drosophila} to humans making
the \emph{Drosophila} germline an elegant model for studying the role of
post-transcriptional regulation during stem cell differentiation and
meiosis.

\textbf{Key words}

{Splicing, Translation Control, RNA Modifications, Ribosome Biogenesis,
Oogenesis, Drosophila, Germline Stem Cell, RNA regulation, Germline
Differentiation, Gametogenesis, RNA Binding Proteins}

\hypertarget{introduction-1}{%
\section{Introduction}\label{introduction-1}}

Gametogenesis gives rise to eggs or sperm in all sexually-reproducing
organisms (Cinalli, Rangan, \& Lehmann, 2008; Ellis \& Kimble, 1994; Lesch \& Page, 2012; Seydoux \& Braun, 2006).
Thus, understanding how gametogenesis is regulated is critical to
comprehending this essential phenomenon that dictates fertility.
Post-fertilization, the zygote gives rise to an entire organism, thus
understanding how gametogenesis is regulated also has implications for
the field of regeneration (Lasko, 2012; Lee \& Lee, 2014; Magnúsdóttir \& Surani, 2014; Soldner \& Jaenisch, 2018; Tadros \& Lipshitz, 2009; Theunissen \& Jaenisch, 2017).
\emph{Drosophila melanogaster} has been one of the central organisms used to
study heritability and gametogenesis for nearly a century due to its
rapid generation time and genetic tractability (Mattox, Palmer, \& Baker, 1990; Spradling, 1993; Spradling et al., 1997; Spradling \& Rubin, 1981; Spradling, Fuller, Braun, \& Yoshida, 2011; Xie \& Li, 2007). These traits have facilitated the establishment of an
extensive collection of informative and useful mutant and transgenic
flies (Hales, Korey, Larracuente, \& Roberts, 2015). In addition, many of the gametogenic regulatory
factors described in the \emph{Drosophila} germ line are conserved to mammals
and also play critical roles in other tissues, such as neurons
(Goldstrohm, Hall, \& McKenney, 2018; Lin \& Spradling, 1997; Reichardt et al., 2018; Vessey et al., 2010; Zamore, Bartel, Lehmann, \& Williamson, 1999; Zhang \& Smith, 2015). While both male and female \emph{Drosophila}
undergo meiosis to give rise to gametes, here we focus on the female
germline as regulation of gametogenesis in males has been reviewed
elsewhere (Barreau, Benson, Gudmannsdottir, Newton, \& White-Cooper, 2008; Fuller, 1998; Spradling et al., 2011; Yamashita \& Fuller, 2005; Zhao \& Garbers, 2002).

The spatiotemporal stages of \emph{Drosophila} oogenesis are discrete and can
be easily identified by their morphology and molecular markers
(Gáspár \& Ephrussi, 2017; Jia, Xu, Xie, Mio, \& Deng, 2016; Spradling et al., 2011). At the anterior end of
the ovary, germline stem cells (GSCs) reside in a structure known as the
germarium and initiate differentiation to give rise to gametes
(Kai, Williams, \& Spradling, 2005; Twombly et al., 1996; Xie \& Li, 2007; Xie \& Spradling, 1998, 2000a). GSCs
are maintained by signaling from the surrounding somatic niche. GSCs
undergo asymmetric mitotic division, producing a stem cell daughter, or
cystoblast (CB) which will begin the process of differentiation by
expressing the essential differentiation factor \emph{bag of marbles} (\emph{bam})
(D. Chen \& McKearin, 2003b; McKearin \& Ohlstein, 1995). The differentiating CB then undergoes
four incomplete mitotic divisions, giving rise to an interconnected
16-cell cyst (McKearin \& Spradling, 1990; McKearin \& Ohlstein, 1995). In this cyst, one
cell is designated to become the oocyte and the other 15 cells take on
the role of nurse cells, which generate proteins and mRNAs that are
provided to the developing oocyte (Navarro, Puthalakath, Adams, Strasser, \& Lehmann, 2004; Spradling et al., 1997).
The specified oocyte and its associated nurse cells are then
encapsulated by somatic cells to form an egg chamber that buds off from
the germarium (Figure 1B) (Gilboa \& Lehmann, 2004; Margolis \& Spradling, 1995). The nurse
cells will enter into a unique state in which they undergo a modified
version of the cell cycle without undergoing mitosis, creating polyploid
nuclei capable of fulfilling the high transcriptional demand required to
transcribe all of the mRNAs necessary for the egg (Lilly \& Duronio, 2005; Royzman \& Orr-Weaver, 1998). As this process ensues, the egg chambers and oocyte
increase in size as the supply of mRNAs and proteins is created and
deposited into the mature egg (Figure 1A) (Lasko, 2012; Richter \& Lasko, 2011).

Oocyte development entails multiple processes that ensure effective
completion of gametogenesis and fertility. Among these are stem cell
maintenance and asymmetric division, differentiation, mitosis and
meiosis, and production of the maternal mRNA contribution, thus the germ
line is a salient model for understanding how cells navigate fate
transitions (D. Chen \& McKearin, 2003b; Fu et al., 2015; Harris, Pargett, Sutcliffe, Umulis, \& Ashe, 2011; Lasko, 2012).
During oogenesis, there is little instructive transcription, barring a
few examples, to mediate these critical transitions (Cinalli et al., 2008; Rangan, DeGennaro, \& Lehmann, 2008). Instead, the germline relies highly on
post-transcriptional regulatory mechanisms to coordinate gametogenesis
(Slaidina \& Lehmann, 2014a). These include: alternative splicing, RNA
modifications to modulate splicing, protein-RNA interactions, small RNA
biology, and organization of the translation machinery to control the
output of gene expression to mediate cell fate transitions. Here we
focus on post-transcriptional processing of germline mRNAs and
translational regulation both of which are required for successful
oogenesis.
\begin{figure}

{\centering \includegraphics[width=6 in,height=3.046676 in]{./figure/Book Chapter/Figure1} 

}

\caption[Schematic of \emph{Drosophila} ovariole.]{(A) Schematic of \emph{Drosophila} ovariole. \emph{Drosophila} females have two ovaries consisting of 16--20 ovarioles, which are assembly lines for producing mature eggs. The germarium, the structure that houses the germline stem cell (GSC), is present at anterior tip of the ovariole. The germline stem cell asymmetrically divides, giving rise to another GSC and a GSC daughter. The daughter cell then will undergo four incomplete rounds of mitosis, giving rise to a 16-cell cyst. Of the 16 cells one will be specified as the egg while the others serve as polyploid nurse cells that support oocyte and egg development. The surrounding somatic cells encapsulate the 16-cell cyst creating egg chambers. As development proceeds, the nurse cells provide mRNAs and proteins allowing the oocyte to grow in size and to eventually become a mature egg. (B) Inset of a germarium showing the developing germline, with the GSC located at the most anterior tip. Upon differentiation, the CB will undergo 4 incomplete mitotic divisions giving rise to a 16-cell cyst. Only one cell of the sixteen cells completes meiosis and is destined to become the oocyte.}\label{fig:unnamed-chunk-2}
\end{figure}
\hypertarget{alternative-splicing-ensures-accurate-production-of-critical-germline-mrnas-to-regulate-sex-determination-and-differentiation}{%
\section{Alternative splicing ensures accurate production of critical germline mRNAs to regulate sex determination and differentiation}\label{alternative-splicing-ensures-accurate-production-of-critical-germline-mrnas-to-regulate-sex-determination-and-differentiation}}

Splicing decisions are crucial during the generation of mature mRNAs
post-transcriptionally and significantly contribute to germline
development. Splicing is mediated by a large ribonucleoprotein catalytic
complex called the spliceosome, the core of which is made up of five
small nuclear RNAs (snRNA), U1, U2, U4, U5 and U6, that work with
spliceosomal proteins to form a small nuclear ribonucleoprotein complex
(snRNP) (Madhani, Bordonne, \& Guthrie, 1990; Wahl, Will, \& Lührmann, 2009; Will \& Lührmann, 2001, 2011). This
complex removes introns from newly synthesized pre-mRNAs and links
exonic sequences together (Wahl et al., 2009). Initially, U1 snRNP recognizes
the donor site, which is located at the 5' end of the intron, and U2
snRNP binds the branch site located at the 3' end, leading to structural
rearrangements of the complex and its associated substrate pre-mRNA
(Matera \& Wang, 2014). Catalytic actions of pre-mRNA splicing occur in two
main steps. Cleavage at the 5' splice site forms a lariat-like structure
such that a 2'-5' phosphodiester bond is created between the first
nucleotide of the donor site and a conserved adenosine residue at the
branch site (Rymond \& Rosbash, 1985). Next, a second cleavage event occurs at
the 3' splice site and is followed by ligation of flanking exons to
complete splicing (Umen \& Guthrie, 1995; Wahl et al., 2009).

Alternative splicing is a process by which a single locus can give rise
to many unique mRNA isoforms and their resulting protein variants
(Black, 2000). The selection of the splice sites is exquisitely
regulated to determine which exons will be included in the resulting
alternatively spliced transcripts (Wang et al., 2015). Alternative splicing
is highly regulated and is critical to germline development
(Hager \& Cline, 1997; Kalsotra \& Cooper, 2011). There are a myriad of RNA targets that
must be differentially spliced, and a complex web of interacting
proteins orchestrate production of their splice variants (Lee \& Rio, 2015).
One of the first described instances of alternative splicing in
\emph{Drosophila} females is the splicing of the sex determination gene
\emph{sex-lethal} (\emph{sxl}) (Bell, Maine, Schedl, \& Cline, 1988). \emph{sxl} is alternatively spliced to
generate isoforms that control sex determination in somatic tissues
(Chang, Dunham, Nuzhdin, \& Arbeitman, 2011). In females, an autoregulatory loop forms between Sxl
protein, U2AF splicing factor and U1 snRNP (Nagengast, Stitzinger, Tseng, Mount, \& Salz, 2003). In
\emph{Drosophila}, the protein component of the U1 and U2 snRNPs are encoded
by a gene called \emph{sans fille} (\emph{snf}) (Cline, Rudner, Barbash, Bell, \& Vutien, 1999). Loss of \emph{snf}
results in a sterility phenotype in females that specifically affects
germline \emph{sxl} splicing and leads to a tumor comprised of
undifferentiated cells (Johnson, Nagengast, \& Salz, 2010; Nagengast et al., 2003). When
correctly spliced, the resulting Sxl protein recognizes its own
pre-mRNAs by binding both upstream and downstream of Exon 3
(Penalva \& Sánchez, 2003). In addition, Sxl protein interacts with the U2AF and
U1 snRNP to block the recognition of splice sites at Exon 3
(Nagengast et al., 2003). As a result, exon 3 is spliced out of the pre-mRNA
in the final transcript that is capable of being translated into a fully
functional protein (Penalva \& Sánchez, 2003). In contrast, males include exon
three in the final \emph{sxl} transcript. Exon 3 contains a premature stop
codon within the \emph{sxl} transcript that results in a truncated protein
that lacks the activity of the female-specific variant (Inoue, Hoshijima, Sakamoto, \& Shimura, 1990).
Thus, \emph{sxl} is differentially expressed in the male and the female gonad
due to alternative splicing events.

In addition to control of \emph{sxl} via alternative splicing, \emph{sxl}
expression is controlled at the level of transcription by several
transcription factors, such as Ovo (Salles, Mével-Ninio, Vincent, \& Payre, 2002). Ovo is a zinc
finger DNA binding protein that is required in the germline for proper
gametogenesis(Andrews et al., 2000). \emph{ovo} is also alternatively spliced and
each of its isoforms have different implications for \emph{sxl} expression.
Ovo-A and Ovo-B where the first splice variants of \emph{ovo} shown to be
expressed in the female germline during oogenesis (Salles et al., 2002). In
addition to differences due to alternative exon usage, Ovo-A, unlike
Ovo-B, contains a 381 amino acid N-terminal extension which arises due
to alternative transcription start sites (Andrews et al., 2000). Use of
these promoters generates distinct Ova isoforms with unique temporal
requirements during oocyte development; Ovo-B was found to be necessary
and sufficient during early oogenesis and Ovo-A is critical in the later
stages of egg development for a fully functional egg. The \emph{ovo-B} gene
has two characterized isoforms, Ovo+2B and Ovo-2B, which were discovered
through a transposon insertion that disrupts exon splicing of \emph{ovo-B}.
This transposition event prevents inclusion of the exon 2b extension,
producing a nonfunctional protein that accumulates during oogenesis. In
the absence of retrotransposon insertion, the 178-amino acid extension
encoded by exon 2b is included forming a fully functional Ovo protein,
known as Ovo-+2B (Salles et al., 2002). Interestingly, Ovo-B promotes
transcription of \emph{ovarian tumor} (\emph{otu}), which enhances \emph{sxl}
expression (Figure 2) (Lu \& Oliver, 2001). The mechanism by which Otu regulates
sxl expression is unknown but various mutations in \emph{otu} lead to a
myriad of phenotypes such as loss of germ cell proliferation, and
inability to complete the differentiation process. The \emph{otu} gene
produces two cytoplasmic protein isoforms, a 104-kDA isoform (Otu-104)
and a 98-kDA isoform (Otu-98) (Tirronen, Lahti, Heino, \& Roos, 1995). Strikingly, only
Otu-104 is capable of rescuing all the \emph{otu} mutant phenotypes,
indicating its requirement during oogenesis, while Otu-98 is dispensable
during this process (Tirronen et al., 1995). Despite the lack of insight into
how the \emph{otu} splice forms regulate GSC development, its alternative
splicing is critical for oogenesis (Sass, Comer, \& Searles, 1995). Thus, a cascade of
alternative splicing events regulate production of Sxl in the female
germline to promote oogenesis (Figure 2).

Sxl expression in the female gonad regulates both sex determination as
well as differentiation (Chau, Kulnane, \& Salz, 2012). One critical task of Sxl is to
represses Tudor domain containing protein 5-like (\emph{tdrd5l})
(Primus, Pozmanter, Baxter, \& Van Doren, 2019). Tdrd5l is present in the cytoplasm of the male
germline, localizing to granules associated with RNA regulation, to
promote male identity and differentiation. Sxl expression the female
gonad represses translation of Tdrd5l to promote female identity
(Primus et al., 2019). In addition, female Sxl has been found to regulate
transcription of \emph{PHD finger protein 7 (phf7)}, a key regulator of male
identity (Yang, Baxter, \& Van Doren, 2012). Sxl was found to recruit SETDB1, a chromatin
writer, to deposit trimethylated H3K9 (H3K9me3) repressing transcription
of \emph{phf7} (Smolko, Shapiro-Kulnane, \& Salz, 2018). Thus, alternative splicing of \emph{sxl} results
in different sexes helps promote proper sex determination in the
germline (Figure 2). Sxl also fulfills additional functions outside of
sex determination. Sxl is required in the female germline for germline
stem cell GSC differentiation. Loss of Sxl protein causes an
accumulation of single cells and two cell cysts (Chau, Kulnane, \& Salz, 2009). It is
thought that Sxl binds \emph{nanos} (\emph{nos}) mRNA, an RNA binding protein that
is necessary for GSC self-renewal, using a canonical Sxl binding
sequence in the 3' UTR (Chau et al., 2012). Loss of Sxl leads to an
accumulation of excess of Nanos protein, which is thought to limit? GSC
differentiation (Boerner \& Becker, 2016; Chau et al., 2012; Li et al., 2013). While
regulation by Sxl is beginning to be deciphered, several aspects remain
to be discovered. For example, Sxl, a splicing factor, is predominantly
cytoplasmic in undifferentiated cells but becomes nuclear as
differentiation proceeds (Chau et al., 2009), yet, how it works as
translational regulator while in the cytoplasm and how it is transported
to the nucleus to function as splicing factor during differentiation are
not known.

Polypyrimidine tract binding proteins (PTBs) promote splicing by binding
polypyrimidine tracts that are \textasciitilde10nt long and bring splice sites
together by means of protein dimerization to promote alternative
splicing (Polydorides, Okano, Yang, Stefani, \& Darnell, 2000; Romanelli, Diani, \& Lievens, 2013). A PTB, \emph{half pint}
(\emph{hfp}), a homolog of human PUF60, is important for oogenesis
(Maniatis \& Tasic, 2002). Loss of \emph{hfp} results in missplicing of the \emph{otu}
transcripts described above (Van Buskirk \& Schüpbach, 2002). In addition, \emph{hfp}
also regulates alternative splicing of \emph{eukaryotic initiation factor 4E}
(\emph{eIF4E}) during development through 3' splice site selection
(Reyes \& Izquierdo, 2008). Hfp is required to increase the relative abundance of
the longer \emph{eIF4E} transcript (Van Buskirk \& Schüpbach, 2002). Lastly, \emph{hfp} also
regulates splicing of \emph{gurken}, a critical regulator of dorsal-ventral
patterning (Kalifa, Armenti, \& Gavis, 2009). Thus, sex determination, differentiation
and production of the determinants of embryonic patterning for the next
generation are all regulated by mechanisms involving alternative
splicing in the female germline.
\begin{figure}

{\centering \includegraphics[width=6.0 in,height=1.483636 in]{./figure/Book Chapter/Figure2} 

}

\caption[Schematic of the pathway that promotes alternative splicing of \emph{sxl}]{Schematic of the pathway that promotes alternative splicing of \emph{sxl} to generate the female \emph{sex determining} variant in the germline. Ovo-B promotes the transcription of \emph{otu,} which enhances splicing of \emph{sxl}. The female-specific splice form of \emph{sxl} is further enhanced by RNA modification by the m6A writer. Formation of the female-specific form generates a functional Sxl protein. Sxl represses Tdrd5l, a protein that promotes male identify. Additionally, Sxl post-transcriptionally represses \emph{nhp2} to promote cyst formation during differentiation.}\label{fig:unnamed-chunk-3}
\end{figure}
\hypertarget{rna-modifications-direct-splicing-of-sex-determinants-and-translation-of-differentiation-promoting-genes-in-the-germline}{%
\section{RNA modifications direct splicing of sex determinants and translation of differentiation promoting genes in the germline}\label{rna-modifications-direct-splicing-of-sex-determinants-and-translation-of-differentiation-promoting-genes-in-the-germline}}

Post transcriptional RNA modifications are abundant and conserved in all
branches of life (Yi \& Pan, 2011). There have been over 100 described RNA
modifications that can alter stability, function and splicing of RNAs
(Licht \& Jantsch, 2016; Roundtree, Evans, Pan, \& He, 2017). A well-known example of an mRNA
modification is the 5' methylguanosine cap that is added to all mRNAs to
promote their stability and aid in translation initiation
(Mitchell et al., 2010; Mukherjee et al., 2012). A variety of RNA modifications have
been linked to developmental transitions, such as those affecting GSC
fate (Batista et al., 2014; Roundtree et al., 2017). Specifically during oogenesis,
N6A-methyladenosine (m\textsuperscript{6}A) has been shown to be important for
differentiation of germline stem cell daughter cells in females by
ensuring proper female-specific splicing of \emph{sxl} (Haussmann et al., 2016).
Additionally, the H/ACA box complex, an RNP complex responsible for
depositing pseudouridine on rRNA, has been suggested to be regulated by
Sxl during the germline stem cell to daughter cell transition and is
required for proper cyst differentiation (Kiss, Fayet-Lebaron, \& Jády, 2010; Morita, Ota, \& Kobayashi, 2018).

m\textsuperscript{6}A is prevalent on mRNA and is mediated by a methyltransferase
complex that deposits a methyl-group at the sixth nitrogen on adenosine
(Yang, Hsu, Chen, \& Yang, 2018). In \emph{Drosophila,} m\textsuperscript{6}A is placed by a m\textsuperscript{6}A writer
complex consisting of Xio, Virilizer (Vir), Spenito (Nito), female
lethal d (fl(2)d), Methyltransferase like 3 (Mettl3) and
Methyltransferase like 14 (Mettl14) (Yan \& Perrimon, 2015). Some described roles
of m\textsuperscript{6}A involve modulating RNA-structure, facilitating mRNA
degradation, promoting translation initiation and mediating alternative
splicing (Roundtree et al., 2017). Interestingly, the m\textsuperscript{6}A writer complex
has been linked to \emph{sxl} splicing during \emph{Drosophila} oogenesis
(Kan et al., 2017). miCLIP data revealed that m\textsuperscript{6}A must be placed at
intergenic regions of the \emph{sxl} mRNA in order to produce the
female-specific isoform (Kan et al., 2017). Accordingly. loss of m\textsuperscript{6}A
complex members such as \emph{spenito} result in expression of the male
specific isoform of \emph{sxl}, and tumors of undifferentiated cells, similar
to loss of \emph{sxl} (Mattox et al., 1990); (Kan et al., 2017). This suggests that
m\textsuperscript{6}A enables proper splicing of female-specific \emph{sxl}, which allows for
proper differentiation of germline stem cells into cystoblast daughter
cells (Figure 2).

Pseudouridine is one of the most abundant RNA modifications
(Zhao \& He, 2015). Although most commonly found on tRNAs, pseudouridine is
also found on mRNAs as well as rRNA (Penzo \& Montanaro, 2018). Unlike the
canonical nucleoside uridine which is attached to the sugar via a
nitrogen-carbon bond, pseudouridine is a uridine isomer attached through
a carbon-carbon bond (Cohn, 1960). Pseudouridine can be placed by two
different classes of enzymes; either by a sequence specific
pseudouridine synthase or a small RNA guided complex called the box
H/ACA ribonucleoprotein (De Zoysa \& Yu, 2017). Depletion of the H/ACA box
complex member Nucleolar Protein Family A Member 2 (NHP2) in the
germline leads to an accumulation of 4- and 8- cell cysts that do not
transition to the 16-cell cyst stage (Morita et al., 2018). Interestingly,
the accumulation of single cells due to loss of \emph{sxl} is partially
rescued by loss of \emph{NHP2 indicating that this sxl phenotype is due to
excess NHP2} (Morita et al., 2018). Consistent with this notion, Sxl
interacts with \emph{nhp2} mRNA suggesting that Sxl may impose a regulatory
function, in this case likely repression of \emph{nhp2} to allow initiation
of the differentiation program (Figure 2) (Morita et al., 2018). Thus,
although it is clear that RNA modifications help to ensure proper
splicing of sex determination factors, but the pathway, mechanism, and
direct targets remain unresolved.

\hypertarget{production-of-ribosomes-is-finely-tuned-to-facilitate-differentiation}{%
\section{Production of ribosomes is finely tuned to facilitate differentiation}\label{production-of-ribosomes-is-finely-tuned-to-facilitate-differentiation}}

While splicing mediates proper mRNA production, access of the mature
mRNAs to ribosomes controls their translation. Once mRNAs are gated for
translation, proper ribosome levels control protein production. The
levels of ribosomes during early oogenesis are strictly regulated and
shockingly dynamic. Ribosome biogenesis is the process of transcribing
and processing the ribosomal RNA (rRNA) components, as well as
transcribing and translating the protein constituents of the ribosome
(Granneman \& Baserga, 2004; Nazar, 2004; Teng, Thomas, \& Mercer, 2013; Yelick \& Trainor, 2015). This
process is exquisitely regulated as ribosome biogenesis is one of the
most energy intensive tasks of maintaining cell homeostasis and is even
more crucial in proliferative cells (Phipps, Charette, \& Baserga, 2011). In addition to
the high energy requirement of ribosome biogenesis, all of the
components of the ribosome must be coordinated in their production. The
process of ribosome biogenesis involves a series of coordinated steps of
processing and assembly that involve dozens of non-coding RNAs and
proteins and the molecular details of this process have been thoroughly
covered in detail in several recent reviews (Granneman \& Baserga, 2004; Yelick \& Trainor, 2015; You, Park, \& Kim, 2015). Briefly, ribosomal DNA (rDNA) is present in
multicopy stretches within the genome; these areas of DNA are localized
to a subnuclear organelle called the nucleolus (Karpen, Schaefer, \& Laird, 1988; Ritossa \& Spiegelman, 1965; Schwarzacher \& Wachtler, 1993). rDNA is transcribed into rRNA in
the nucleolus and processing steps begin cotranscriptionally
(Koš \& Tollervey, 2010) to remove internal and external spacers found in immature
rRNA (Granneman \& Baserga, 2004; Granneman, Petfalski, Tollervey, \& Hurt, 2011; Schäfer, Strauß, Petfalski, Tollervey, \& Hurt, 2003; Tafforeau et al., 2013). As these processing steps occur, the rRNA is
covalently modified and ribosomal proteins begin to interact with the
partially processed rRNA (Agalarov, Sridhar, Funke, Stout, \& Williamson, 2000; Deshmukh, Tsay, Paulovich, \& Woolford, 1993; Gumienny et al., 2017; Jády \& Kiss, 2001; Kiss, Jady, Bertrand, \& Kiss, 2004). When the rRNA is mostly
mature it is exported from the nucleus to the cytoplasm where the small
and large subunits of the ribosome fully mature and assemble
(Lo et al., 2010; Schäfer et al., 2003; Sloan et al., 2017; Tschochner \& Hurt, 2003; Zemp \& Kutay, 2007). Errors at any of these steps can result in ribosome
biogenesis defects which in humans result in disease states known as
ribosomopathies (Armistead \& Triggs-Raine, 2014; Barlow et al., 2010; Brooks et al., 2014; Higa-Nakamine et al., 2012; Mills \& Green, 2017; Sloan et al., 2017).

Curiously, despite the presence of ribosomes across cell types and
sharing similar molecular origins, ribosomopathies manifest as tissue
specific defects rather than pleiotropic phenotypes (Brooks et al., 2014; Higa-Nakamine et al., 2012; Mills \& Green, 2017; Pereboom, van Weele, Bondt, \& MacInnes, 2011; Yelick \& Trainor, 2015).
The reasons behind the unique, tissue-specific manifestations are still
being investigated but in several cases it seems that stem cells may be
particularly sensitive to perturbations in ribosome biogenesis
(Brooks et al., 2014; Morgado-Palacin, Llanos, \& Serrano, 2012; Pereboom et al., 2011; Watanabe-Susaki et al., 2014). Indeed, a growing body of evidence is beginning
to suggest that \emph{Drosophila} GSCs not only have a specific requirement
for ribosome biogenesis, but also that ribosome biogenesis, as well as
global translation, vary greatly over the course of GSC differentiation
and are uncoupled during early oogenesis (Sanchez et al., 2016; Zhang, Shalaby, \& Buszczak, 2014). These attributes make \emph{Drosophila} oogenesis an
excellent system to address how perturbations of ribosome levels affects
stem cell differentiation.

In order to maintain stem cell fate, GSCs asymmetrically partition
factors required for ribosome biogenesis by retaining more of this
machinery than they pass on to daughter cells (Fichelson et al., 2009; Zhang et al., 2014). In particular, Underdeveloped (Udd), an rRNA
transcription factor segregates asymmetrically to the GSC during mitosis
and seems to promote a high rate of rRNA synthesis within the GSC
(Zhang et al., 2014). Furthermore, Wicked (Wcd), a U3 snoRNP complex member
required for rRNA maturation, is also asymmetrically partitioned to GSCs
and associates with the original spectrosome, an ER rich organelle found
in GSCs and CBs (Spradling et al., 1997), of the dividing GSC. How GSCs
carry out this specialized cellular division requires further
investigation, however, asymmetric stem cell division is crucial for
proper differentiation (D. Chen \& McKearin, 2003a, 2003b; Lin \& Spradling, 1997).Consistent with this loss of \emph{wcd} results in premature
differentiation of GSCs (Fichelson et al., 2009). Nascent rRNA production,
measured by BrUTP incorporation, and presumably ribosomes, are produced
at high levels in GSCs but this production drops in CBs and in
subsequent stages (Figure 3) (Zhang et al., 2014). Additionally, it has been
observed that certain ribosome biogenesis components are expressed at
high levels specifically in the germline (Kai et al., 2005). In particular,
RNA exonuclease 5 (Rexo5) is an RNA exonuclease that facilitates
ribosome biogenesis by trimming snoRNAs as well as rRNAs
(Gerstberger et al., 2017). Depletion of \emph{rexo5} in the germline results in
an accumulation of egg chambers that bud off from the germarium, but do
not grow in size, and causes defects in GSC proliferation
(Gerstberger et al., 2017). These observations suggest that the machinery for
ribosome biogenesis is not only critical for germline development but is
also dynamically regulated.

Sanchez et al.~demonstrated that the dynamic nature of rRNA
transcription during germline development is not simply a consequence of
the differentiation process. Instead, lowering ribosome biogenesis is
required for timely differentiation, but severe loss of ribosome
biogenesis causes formation of stem-cysts, a product of perturbed
cytokinesis of GSC daughters (Mathieu et al., 2013; Matias, Mathieu, \& Huynh, 2015; Sanchez et al., 2016) . Somewhat surprisingly, despite their increased
retention of ribosome biogenesis components, GSCs exhibit a lower rate
of translation compared to daughter cells and cyst stages (Figure 3).
This finding invokes the hypothesis that despite the GSCs elevated
capacity for ribosome biogenesis, GSCs do not intrinsically require
higher ribosome levels for translation. Instead, the data is suggestive
of the possibility that GSCs produce high levels of ribosomes in order
to pass them on to and facilitate differentiation of their daughter
cells. We thus hypothesize that a ribosome biogenesis checkpoint could
couple ribosome production to cell cycle progression to ensure a
sufficient ribosome concentration is passed from the GSC to the daughter
CB. Conversely, increasing ribosome biogenesis via overexpression of
TIF-IA, an RNA Pol I transcription initiation factor that is required
for rRNA synthesis (Grewal, Evans, \& Edgar, 2007), results in a failure of germ cells
to differentiate, causing a marked overproliferation of undifferentiated
GSC daughters (Zhang et al., 2014). This overproliferation may be caused by
bypassing or rapid progression through the proposed ribosome biogenesis
checkpoint such that the cell cycle is hastened in response to elevated
ribosome biogenesis. The overproliferation of undifferentiated germ
cells when ribosome levels are elevated is consistent with observations
that high ribosome levels lead to rapidly growing cancers
(Belin et al., 2009; Deisenroth \& Zhang, 2010; Vlachos \& Muir, 2010).

Although reducing ribosome biogenesis tends to result in the formation
of a stem-cyst as previously described, some factors that play a role in
ribosome biogenesis have a less severe phenotypes. For example, some
mutants of the ribosomal protein S2 (rps2) gene have a repeating
egg-chamber mid-oogenesis defect, wherein ovarian development halts at
stage 5 and successive egg chambers do not grow in size and eventually
die, resulting in sterility (Cramton \& Laski, 1994). This phenotype may be
the consequence of incomplete loss of function as the allele that
results in the repeating egg chamber phenotype reduces mRNA expression
of \emph{rps2}, incompletely, by 60-70\%, while other allelic combinations
result in embryonic lethality (Cramton \& Laski, 1994). Incomplete loss of
function alleles for another ribosomal protein, \emph{ribosomal protein S3},
result in a similar repeating egg chamber phenotype
{[}({\textbf{???}})¦bÃ¸e-Larssen1998{]}. These observations suggest that partial loss of
ribosome biogenesis during oogenesis may be tolerated during
differentiation but results in phenotypes at a later phase of egg
production, consistent with the model that high levels of biogenesis in
early stages supply the ribosomes for subsequent differentiation and
development.

Not only do ribosome levels vary but a class of ribosomal protein
paralogs are enriched specifically in early germ cells (Xue \& Barna, 2012).
Several variant ribosomal proteins such as \emph{ribosomal proteins S5b}
(\emph{rps5b}), \emph{s10a}, \emph{s19b}, and \emph{l22}-like are enriched in the germline
and others are enriched during early oogenesis (Kai et al., 2005). The role
of these ribosomal proteins has not been thoroughly explored, but their
presence indicates either a role for specialized ribosomes early during
germline development or as a way to further increase the availability of
ribosomal proteins to facilitate the high level of ribosome production
in GSCs. One of these ribosomal protein paralogs, RpS5b, has recently
been characterized (Kong et al., 2019). \emph{rps5b} is most highly expressed in
ovaries in contrast to its paralog, \emph{ribosomal protein S5a} (\emph{rps5a}),
which is expressed at high levels ubiquitously (Kong et al., 2019). Loss of
\emph{rps5a} in the germline does not cause a germline phenotype, however,
loss of \emph{rps5b} results in a mid-oogenesis defect that is further
exacerbated when \emph{rps5a} is depleted in a \emph{rps5b} mutant background
(Kong et al., 2019). This could suggest that RpS5a and RpS5b are functionally
similar and that the RpS5b phenotype results from lowering the overall
amount of RpS5 available during oogenesis. However, RpS5b was also found
to interact preferentially with mRNAs that encode proteins involved in
mitochondrial electron transport, in contrast to RpS5a which binds mRNAs
from a broad spectrum of gene categories (Kong et al., 2019). In accordance
with the binding data, \emph{rps5b} depleted ovaries expressed lower levels
of proteins involved in oxidative phosphorylation and mitochondrial
respiration (Kong et al., 2019). This evidence suggests that the expression
of ribosomal protein paralogs may be a part of specialized ribosomes
that translate specific groups of mRNAs; however, these ribosomal
protein paralogs must be carefully analyzed to determine if they make up
bonafide special ribosomes or instead have ribosome independent
functions (Dinman, 2016).

What regulates ribosome biogenesis to allow for it to be dynamic during
early \emph{Drosophila} germline development? The best understood regulator
of ribosome biogenesis is the Target of Rapamycin (TOR) pathway
(Chymkowitch, Aanes, Robertson, Klungland, \& Enserink, 2017; Magnuson, Ekim, \& Fingar, 2012; Wei \& Zheng, 2009; Yerlikaya et al., 2016)
TOR is a kinase that is part of two distinct subcomplexes, TOR complex 1
(TORC1) and TOR complex 2 (TORC2) (Wullschleger, Loewith, \& Hall, 2006). These
complexes have distinct biological roles. TORC2 has been shown to
function as an important regulator of the cytoskeleton
(Wullschleger et al., 2006). Whereas, TORC1 receives and integrates several
different signals including nutritional and growth factors and its
activity promotes pro-proliferative activities such as global
translation, ribosomal protein translation, and cell cycle progression
(Kim, Goraksha-Hicks, Li, Neufeld, \& Guan, 2008; Magnuson et al., 2012; Texada et al., 2019). TORC1 activity also helps
to coordinate the transcription and translation of the components
required for ribosome biogenesis (Grewal et al., 2007; Magnuson et al., 2012; Martin, Powers, \& Hall, 2006). In \emph{Drosophila,} TORC1 activity is high in GSCs through
the 4-cell cyst, but TORC1 activity dips in 8 and 16 cell cysts and
subsequently increases after the cyst stages (Wei, Bettedi, Kim, Ting, \& Lilly, 2019).
Interestingly, the landscape of TORC1 activity resembles the landscape
of ribosome biogenesis, but not global translation (Figure 3)
(Sanchez et al., 2016; Zhang et al., 2014). However, loss of TORC1 components does
not phenocopy perturbation of ribosome biogenesis (Sanchez et al., 2016).
This is possibly because TORC1 plays a broader role in early oogenesis
given the myriad of regulatory functions TORC1 is known to play in other
systems (Kim et al., 2008; S. Li et al., 2009; Moreno-Torres, Jaquenoud, \& De Virgilio, 2015; Noda, 2017; Wei \& Zheng, 2009). A downstream effector of mTORC1, La related protein 1
(Larp1) is known to silence ribosomal protein translation in mammals
through binding to terminal oligopyrimidine tracts in the 5'UTR of its
targets (Fonseca et al., 2015; Hong et al., 2017; Lahr et al., 2017; Tcherkezian et al., 2014); however, the same has yet to be
demonstrated for the \emph{Drosophila} ortholog, La related protein (Larp).
Tantalizingly, Larp is required for male and female fertility in
\emph{Drosophila}, but details of Larp's precise role in the female and
oogenesis are lacking (Blagden et al., 2009; Ichihara, Shimizu, Taguchi, Yamaguchi, \& Inoue, 2007). In contrast,
in males Larp is required for proper spindle pole formation as well as
proper cytokinesis (Blagden et al., 2009). Given the regulatory role Larp
plays in ribosome biogenesis in mammals and the data from \emph{Drosophila}
spermatogenesis, Larp could facilitate the dynamic nature of ribosome
biogenesis during GSC differentiation and meiosis. However, further
study is required to understand the role of Larp during GSC
differentiation and oogenesis to determine its function in this context.

The process of differentiation requires major cellular reprogramming.
Surprisingly, despite being required for cell viability ribosome
biogenesis and global translation are two key programs that are
modulated to shape GSC differentiation(Sanchez et al., 2016; Zhang et al., 2014).
When ribosome production is improperly modulated during GSC
differentiation it results in characteristic phenotypes, accumulation of
single cells if biogenesis components are overexpressed and formation of
a stem-like cyst if ribosome biogenesis components are knocked down in
the germline (Sanchez et al., 2016; Zhang et al., 2014). Additionally, several
ribosomal protein variants are highly enriched in ovaries and they may
perform special functions, however, these variants are just beginning to
be studied. Additionally, based on what we know of the mechanisms and
networks that control ribosome biogenesis in \emph{Drosophila} oocytes, the
dynamic nature of ribosome biogenesis seems likely to be conserved;
however, further investigation is required to determine and compare the
basis of ribosome biogenesis control.
\begin{figure}

{\centering \includegraphics[width=4.536585 in,height=6 in]{./figure/Book Chapter/Figure3} 

}

\caption[Global translation rate, rRNA transcription rate, and mTorc1 activity during development]{Schematic representing the germarium and plots representing relative changes in global translation rate, rRNA transcription rate, and mTorc1 activity during development at the developmental stages indicated. As germline stem cell differentiation occurs rRNA production decreases, while global translation initially increases as differentiation occurs then falls off post differentiation. A global regulator of both translation and rRNA production, mTorc1 activity decreases during differentiation and increases post differentiation.}\label{fig:unnamed-chunk-4}
\end{figure}
\hypertarget{hand-off-mechanisms-facilitated-by-combinatorial-rna-binding-proteins-dynamically-shape-the-translational-landscape-during-oogenesis}{%
\section{Hand off mechanisms facilitated by combinatorial RNA binding proteins dynamically shape the translational landscape during oogenesis}\label{hand-off-mechanisms-facilitated-by-combinatorial-rna-binding-proteins-dynamically-shape-the-translational-landscape-during-oogenesis}}

While some mRNAs are translated post-transcriptionally, other critical
mRNAs are translationally regulated. For efficient translation of mRNAs,
it is thought that the mRNAs must be circularized - bringing their 5'
cap and 3' poly A tail in close proximity to each other (Fukao et al., 2009; Martineau et al., 2008; Preiss \& Hentze, 1998). This interaction is mediated by cap
binding proteins such as eukaryotic initiation factor 4E (eIF4E) and the
poly-A binding protein (PABP)(Eichhorn et al., 2016; Kronja et al., 2014; Subtelny, Eichhorn, Chen, Sive, \& Bartel, 2014; Tarun Jr, Wells, Deardorff, \& Sachs, 1997). A longer poly-A tail and uninhibited
access to the 5' cap for eIF4E is believed to promote efficient
translation (Jalkanen, Coleman, \& Wilusz, 2014). A major mode of translational
regulation is that RNA binding proteins (RBPs) recognize cognate
sequences in the 3' UTRs of their target mRNAs (Harvey et al., 2018). The
binding of the RBP prevents circularization of the mRNA and inhibits
efficient translation initiation, leading to reduced translation
(Mazumder, Seshadri, Imataka, Sonenberg, \& Fox, 2001). RBP binding to the 3' UTR can mediate translation
inhibition by recruiting cofactors to inhibit circularization
(Szostak \& Gebauer, 2013).This inhibition of circularization can be achieved by
RBP binding to the cap and competing with eIF4E, removal of the cap by
the decapping machinery, or recruitment of factors such as the CCR4-Not
complex to shorten poly-A tail length (Rissland, 2017). In some
cases, RBPs can both block initiation as well as mediate shortening of
the poly-A tail (Neve, Patel, Wang, Louey, \& Furger, 2017).

As mentioned in the germline several developmental processes such as
stem cell maintenance, differentiation, mitosis and meiosis are
coordinated and successful transition through these diverse programs
relies on precise translational control (Figure 4) (Joshi, Riddle, Djabrayan, \& Rothman, 2010; Slaidina \& Lehmann, 2014a). As factors that interfere with translation such as
the decapping machinery and the poly-A tail shortening CCR4-Not complex
are expressed continuously during oogenesis, and cannot support dynamic
translational control on their own, a dynamic and diverse landscape of
translational regulators has evolved to allow for fine-scale temporal
control of mRNA translation (Eichhorn et al., 2016; Flora, Wong-Deyrup, et al., 2018). To add an
additional layer of complexity, the expression or abundance of several
RBPs that regulate translational control oscillate as oogenesis
progresses (Figure 4) (Flora, Wong-Deyrup, et al., 2018; Rangan et al., 2009; Richter \& Lasko, 2011). As
the levels of RBPs decrease, their bound mRNA targets are licensed for
translation (Flora, Wong-Deyrup, et al., 2018; Lasko, 2000; Linder \& Lasko, 2006). There are three
major themes that work to control mRNA translation: 1. RBPs collaborate
in a combinatorial manner to regulate mRNAs, 2. Target mRNAs are handed
off from one RBP complex to another as levels oscillate during oogenesis
to consistently repress or promote target mRNA translation, and 3.
Multiple feedback mechanisms operate to mediate each transition (Figure
4) (Flora, Wong-Deyrup, et al., 2018). The feedback mechanism has been extensively
reviewed elsewhere and is not the focus of this chapter (Flora, Wong-Deyrup, et al., 2018; Slaidina \& Lehmann, 2014a). Here, we outline how RBPs both collaborate as well
hand off mRNAs during the transition from GSC to mature oocyte.

GSCs rely on several factors to maintain self-renewal, two of the main
factors are Pumilio (Pum) and Nanos (Nos), which work in a combinatorial
fashion to repress the translation of differentiation-promoting mRNAs
(Figure 4) (Forbes \& Lehmann, 1998; Gilboa \& Lehmann, 2004; Joly, Chartier, Rojas-Rios, Busseau, \& Simonelig, 2013; Lin \& Spradling, 1997).
Pum, a member of the conserved Pum- and Fem-3-binding factor (PUF)
family of proteins, is present at high levels in the undifferentiated
germline cells of the ovary, including GSCs, CBs, and
early-differentiating cysts (Forbes \& Lehmann, 1998; Kai et al., 2005). Independent of
other factors, Pum can directly bind mRNA, but it requires the catalytic
activity of other proteins to regulate translation of its targets in the
\emph{Drosophila} germline (Sonoda \& Wharton, 1999; Tadauchi, Matsumoto, Herskowitz, \& Irie, 2001). Pum is known
to have dynamic interactions with two critical regulators, Nos in GSCs,
and Brain tumor (Brat) in CBs (Figure 4) (Arvola, Weidmann, Tanaka Hall, \& Goldstrohm, 2017; Goldstrohm et al., 2018; Harris et al., 2011; Reichardt et al., 2018; Sonoda \& Wharton, 1999, 2001). Nos, a well conserved RNA binding protein, has the
ability to bind mRNA, albeit at low affinity and requires the presence
of Pum to recognize its targets (Arvola et al., 2017). Nanos directly
interacts with Not1, a member of the CCR4-Not complex, recruiting it to
target mRNAs, such as \emph{meiotic P26} (\emph{mei-p26}) and \emph{brat}, to regulate
their translation (Bhandari, Raisch, Weichenrieder, Jonas, \& Izaurralde, 2014; Raisch et al., 2016; Temme, Simonelig, \& Wahle, 2014).
While in some systems Pum can directly recruit the CCR4-Not complex,
activity of \emph{nos} is required for this interaction in the \emph{Drosophila}
germline (Joly et al., 2013; Temme et al., 2014). Upon loss of Pum, Nanos or Twin,
GSCs fail to maintain stem cell fate and differentiate into stem cell
daughters, resulting in the inability to sustain oogenesis as outlined
below.

An example of distinct, stage-specific translational control by
Pum/Nos/CCR4-Not complex in the germline is the mechanism by which
\emph{polar granule component} (\emph{pgc}), a germline-specific transcriptional
repressor, is controlled (Figure 4) (Flora, Wong-Deyrup, et al., 2018). Pgc interacts with
the Positive Transcription Elongation Factor (P-TEFb) complex and
inhibits the phosphorylation of the Serine-2 residue that is critical
for transcriptional elongation, resulting in global transcriptional
silencing (Hanyu-Nakamura, Sonobe-Nojima, Tanigawa, Lasko, \& Nakamura, 2008). A single pulse of expression of Pgc
protein in the CB allows for epigenetic and transcriptomic reprogramming
during differentiation (Flora, Schowalter, et al., 2018). While \emph{pgc} mRNA is expressed
highly and ubiquitously throughout oogenesis, translation of \emph{pgc} mRNA
is tightly regulated to mitigate the effects of its potent
transcriptional silencing activity. The \emph{pgc} 3' UTR contains a
conserved consensus sequence that is transiently and sequentially bound
by multiple distinct, developmentally regulated RBPs (Flora, Wong-Deyrup, et al., 2018).
This 3' UTR sequence is required for post-transcriptional control of
\emph{pgc} as Pgc protein expression is restricted to the CB. In the GSCs,
Pum and Nos bind the \emph{pgc} 3' UTR and recruit Twin a component of the
CCR4-Not complex to deadenylate \emph{pgc} mRNA and inhibit its translation
(Figure 4) (Flora, Wong-Deyrup, et al., 2018). In addition to \emph{pgc}, Pum/Nos and Twin also
regulate Brain tumor (Brat) (Joly et al., 2013). Brat is a TRIM-NHL domain
protein expressed in the germline that represses translation by engaging
with d4EHP and competing with the cap-binding protein eIF4E to prevent
translation initiation (Figure 4) (Arvola et al., 2017; Harris et al., 2011; Sonoda \& Wharton, 2001). While \emph{brat} mRNA is expressed in the GSC, it is
specifically repressed by Nos and Pum . In addition to these targets,
several differentiation promoting mRNAs such as \emph{meiP26} are also
repressed (Joly et al., 2013). Thus, in the GSCs, a combination of Pum, Nos
and CCR4-Not complex are required for repressing translation of several
critical differentiation promoting mRNAs (Flora, Wong-Deyrup, et al., 2018; Lasko, 2000, 2012; Slaidina \& Lehmann, 2014a).

Subsequent differentiation of the GSC daughters relies on several
factors to repress expression of \emph{nos} mRNA (Lasko, 2000, 2012). Differentiation is initiated upon Bam expression in the
CB, where Bam and its binding partner benign gonial cell neoplasm (Bgcn)
act through a sequence in the \emph{nos} 3' UTR to its inhibit translation
(Figure 4) (Y. Li et al., 2009; McCarthy, Deiulio, Martin, Upadhyay, \& Rangan, 2018). This repression mechanism
includes deadenylation activity by Twin, which works in conjunction with
Bam and Bgcn (Fu et al., 2015). As Nos protein levels decrease in the CB,
\emph{pgc} and \emph{brat} mRNAs are translated (Flora, Wong-Deyrup, et al., 2018). The expressed
Brat protein now partners with Pum to repress translation of GSC
self-renewal genes (Figure 4) (Harris et al., 2011). In addition, expression
of Mei-P26 increases initiating interactions with Bam, Bgcn and Sxl.
Mei-P26 then promotes translational repression of GSC fate promoting
genes such as \emph{nos}, allowing for further differentiation by cooperating
with Bam and Bgcn (Li et al., 2013; Reichardt et al., 2018). As the CB
differentiates into 2-, 4-, 8- and 16- cell cysts, levels of Nanos
protein rebound. However, in spite of the presence of Nos, Pum partners
with Brat to suppress \emph{pgc} translation in the 4- to 16-cell cyst stages
(Figure 4) (Flora, Wong-Deyrup, et al., 2018). Thus, in CBs, absence of Nos allows for Pum
to complex with a different subset of proteins as well as license
expression of new translational regulators to promote differentiation.

After cyst differentiation, Pum protein levels decrease and expression
of another translational repressor, Bruno (Bru), increases
(Kim-Ha, Kerr, \& Macdonald, 1995; Schupbach \& Wieschaus, 1989, 1991; Webster, Liang, Berg, Lasko, \& Macdonald, 1997). Downregulation of Pum expression is
critical for the transition from GSC to an oocyte
(Carreira-Rosario et al., 2016; Forbes \& Lehmann, 1998). Rbfox1, an RBP whose
cytoplasmic isoform regulates the translation of specific mRNAs in the
germline is responsible for repressing Pum translation through binding
of a consensus sequence in the \emph{pum} 3' UTR (Figure 4)
(Carreira-Rosario et al., 2016). Loss of Rbfox1 leads to an expansion of Pum
protein expression and a disruption of differentiation
(Carreira-Rosario et al., 2016). Repression of Pum levels by Rbfox1 allows
for Bru expression (Carreira-Rosario et al., 2016). Surprisingly, Bru can
bind to a sequence in the 3' UTR that is very similar to Pum binding
sequence (Figure 4)(Reveal, Garcia, Ellington, \& Macdonald, 2011). Bruno blocks translation
initiation by interacting with Cup, a conserved eIF4E binding protein
(Kim et al., 2015; Nakamura, Sato, \& Hanyu-Nakamura, 2004). In fact, Bru binds the same sequence in
the \emph{pgc} 3' UTR as Nos/Pum to prevent \emph{pgc} translation
(Flora, Wong-Deyrup, et al., 2018). This mode of translation repression is not restricted
to \emph{pgc}, rather a cohort of maternal mRNAs are co-regulated by Pum and
Bru representing a hand-off mechanism for repression of maternal mRNAs
(Flora, Wong-Deyrup, et al., 2018).
\begin{figure}

{\centering \includegraphics[width=5 in,height=6.157635 in]{./figure/Book Chapter/Figure4} 

}

\caption[Schematic of combinatorial and dynamic translation regulation in the \emph{Drosophila} germarium.]{Schematic of combinatorial and dynamic translation regulation in the \emph{Drosophila} germarium. In the GSCs Nos, Pum and Twin form a complex to inhibit the translation of differentiation mRNAs such as \emph{pgc}, which increases throughout oogenesis. Expression of Bam in the CB initiates differentiation by interacting with its partner Bgcn and Mei-P26 to repress the translation of GSC-expressed mRNAs, specifically \emph{nos}. As Nos protein levels decrease in the CB, Pum is available to partner with Brat to repress the translation of self-renewal genes and \emph{pgc}. In cyst stages, Rbfox1 binds the \emph{pum} 3' UTR to inhibit its translation. Throughout oogenesis Bru and Cup continuously block translation of \emph{pgc}.}\label{fig:unnamed-chunk-5}
\end{figure}
\hypertarget{summary}{%
\section{Summary}\label{summary}}

Decades of work using elegant genetics has revealed several paradigms in
which splicing machinery, RNA modifying enzymes, ribosome levels, and
translational regulation mediates the transition from GSC to oocyte
fate. However, several critical details such as the direct targets and
mechanisms still need to be deciphered. Together the advent of
cost-effective sequencing technologies combined with the increasing
ability to easily create mutants in previously uncharacterized genes
will allow us to further elucidate the regulatory logic (underlying or
of) this critical transition.



\hypertarget{a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis}{%
\chapter{\texorpdfstring{A translation control module coordinates germline stem cell differentiation with ribosome biogenesis during \emph{Drosophila} oogenesis}{A translation control module coordinates germline stem cell differentiation with ribosome biogenesis during Drosophila oogenesis}}\label{a-translation-control-module-coordinates-germline-stem-cell-differentiation-with-ribosome-biogenesis-during-drosophila-oogenesis}}

Elliot T. Martin\textsuperscript{1}*, Patrick Blatt\textsuperscript{1}*, Elaine Ngyuen\textsuperscript{2}, Roni
Lahr\textsuperscript{2}, Sangeetha Selvam\textsuperscript{1}, Hyun Ah M. Yoon\textsuperscript{1,3}, Tyler
Pocchiari\textsuperscript{1,4}, Shamsi Emtenani\textsuperscript{5}, Daria E. Siekhaus\textsuperscript{5}, Andrea
Berman\textsuperscript{2}, Gabriele Fuchs\textsuperscript{1†} and Prashanth Rangan\textsuperscript{1†}

\textsuperscript{1}Department of Biological Sciences/RNA Institute, University at Albany
SUNY, Albany, NY 12202

\textsuperscript{2}Department of Biological Sciences, University of Pittsburgh,
Pittsburgh, PA 15260

\textsuperscript{3}Albany Medical College, Albany, NY 12208

\textsuperscript{4}SUNY Upstate Medical University, Syracuse, NY 13210-2375

\textsuperscript{5}Institute of Science and Technology Austria, Klosterneuburg, Austria

*These authors contributed equally to this work

\textsuperscript{†}Co-corresponding authors

Email: \href{mailto:gfuchs@albany.edu}{{gfuchs@albany.edu}},
{prangan@albany.edu}

\hypertarget{summary-1}{%
\section{Summary:}\label{summary-1}}

Ribosomal defects perturb stem cell differentiation, causing diseases called ribosomopathies. How ribosome levels control stem cell differentiation is not fully known. Here, we discovered three RNA helicases are required for ribosome biogenesis and for \emph{Drosophila} oogenesis. Loss of these helicases, which we named Aramis, Athos and Porthos, lead to aberrant stabilization of p53, cell cycle arrest and stalled GSC differentiation. Aramis controls cell cycle progression by regulating translation of a cohort of mRNAs containing a Terminal Oligo Pyrimidine (TOP) motif in their 5'-UTRs, which we find confers sensitivity to ribosome levels mediated by La-related protein (Larp). One such TOP-containing mRNA codes for Novel Nucleolar protein 1 (Non1), a conserved p53 destabilizing protein. When ribosome concentration is sufficient, Non1 is expressed and promotes GSC cell cycle progression via p53 degradation. Unexpectedly, Aramis is required for efficient translation of a cohort of mRNAs containing a 5'-Terminal-Oligo-Pyrimidine (TOP)-motif, including mRNAs that encode ribosomal proteins and a conserved p53 inhibitor, Novel Nucleolar protein 1 (Non1). The TOP-motif co-regulates the translation of growth-related mRNAs in mammals. As in mammals, the La-related protein co-regulates the translation of TOP-motif containing RNAs during \emph{Drosophila} oogenesis. Thus, a previously unappreciated TOP-motif in \emph{Drosophila} responds to reduced ribosome biogenesis to co-regulate the translation of ribosomal proteins and a p53 repressor, thus coupling ribosome biogenesis to GSC differentiation.

\hypertarget{introduction-2}{%
\section{Introduction}\label{introduction-2}}

All life depends on the ability of ribosomes to translate mRNAs into
proteins. Despite this universal requirement, ribosome biogenesis is not
universally equivalent. Stem cells, the unique cell type that underlies
the generation and expansion of tissues, in particular have a distinct
ribosomal requirement (Gabut, Bourdelais, \& Durand, 2020; Sanchez et al., 2016; Woolnough, Atwood, Liu, Zhao, \& Giles, 2016; Zahradkal, Larson, \& Sells, 1991; Zhang et al., 2014). Ribosome
production and levels are dynamically regulated to maintain higher
amounts in stem cells (Fichelson et al., 2009; Gabut et al., 2020; Sanchez et al., 2016; Woolnough et al., 2016; Zahradkal et al., 1991; Zhang et al., 2014). For
example, ribosome biogenesis components are often differentially
expressed, as observed during differentiation of embryonic stem cells,
osteoblasts, and myotubes (Gabut et al., 2020; Watanabe-Susaki et al., 2014; Zahradkal et al., 1991). In
some cases, such as during \emph{Drosophila} germline stem cell (GSC)
division, ribosome biogenesis factors asymmetrically segregate during
asymmetric cell division, such that a higher pool of ribosome biogenesis
factors is maintained in the stem cell compared to the daughter cell
(Blatt et al., 2020; Fichelson et al., 2009; Zhang et al., 2014). Reduction of ribosome levels in several stem cells
systems can cause differentiation defects
(Corsini et al., 2018; Fortier, MacRae, Bilodeau, Sargeant, \& Sauvageau, 2015; Khajuria et al., 2018; Zhang et al., 2014). In \emph{Drosophila,} perturbations that reduce ribosome
levels in the GSCs result in differentiation defects causing infertility
(Sanchez et al., 2016). Similarly, humans with reduced ribosome levels are
afflicted with clinically distinct diseases known as ribosomopathies,
such as Diamond-Blackfan anemia, that often result from loss of proper
differentiation of tissue-specific progenitor cells (Armistead \& Triggs-Raine, 2014; Barlow et al., 2010; Brooks et al., 2014; Higa-Nakamine et al., 2012; Lipton, Kudisch, Gross, \& Nathan, 1986; Mills \& Green, 2017). However, the
mechanisms by which ribosome biogenesis is coupled to proper stem cell
differentiation remain incompletely understood.

Ribosome production requires the transcription of ribosomal RNAs (rRNAs)
and of mRNAs encoding ribosomal proteins (Bousquet-Antonelli, Vanrobays, Gélugne, Caizergues-Ferrer, \& Henry, 2000; de la Cruz, Karbstein, \& Woolford, 2015; Granneman, Bernstein, Bleichert, \& Baserga, 2006; Granneman et al., 2011; Tafforeau et al., 2013; Venema, Cile Bousquet-Antonelli, Gelugne, Le Caizergues-Ferrer, \& Tollervey, 1997). Hundreds of factors,
including helicases and endonucleases, transiently associate with
maturing rRNAs to facilitate rRNA processing, modification, and folding
(Granneman et al., 2011; Sloan et al., 2017; Tafforeau et al., 2013; Watkins \& Bohnsack, 2012).
Ribosomal proteins are imported into the nucleus, where they assemble
with rRNA to form precursors to the 40S and large 60S ribosomal
subunits, which are then exported to the cytoplasm
(Baxter-Roshek, Petrov, \& Dinman, 2007; Decatur \& Fournier, 2002; Granneman et al., 2006, 2011; Koš \& Tollervey, 2010; Nerurkar et al., 2015; Tafforeau et al., 2013; Zemp \& Kutay, 2007). Loss of RNA Polymerase I transcription
factors, helicases, exonucleases, large or small subunit ribosomal
proteins, or other processing factors all compromise ribosome biogenesis
and trigger diverse stem cell-related phenotypes (Brooks et al., 2014; Calo et al., 2018; Mills \& Green, 2017; Sanchez et al., 2016; Yelick \& Trainor, 2015; Zhang et al., 2014).

Nutrient availability influences the demand for \emph{de novo} protein
synthesis and thus ribosome biogenesis
(Anthony, Anthony, Kimball, Vary, \& Jefferson, 2000; Hong, Mannan, \& Inoki, 2012; Mayer \& Grummt, 2006; Shu, Swanda, \& Qian, 2020). In mammals, nearly all of the mRNAs that
encode the ribosomal proteins contain a Terminal Oligo Pyrimidine (TOP)
motif within their 5' untranslated region (UTR), which regulates their
translation in response to nutrient levels (Fonseca et al., 2015; Hong et al., 2017; Lahr et al., 2017; Tcherkezian et al., 2014). Under growth-limiting
conditions, La related protein 1 (Larp1) binds to the TOP sequences and
to mRNA caps to inhibit translation of ribosomal proteins
(Fonseca et al., 2015; Jia et al., 2021; Lahr et al., 2017; Philippe, Vasseur, Debart, \& Thoreen, 2018). When growth conditions are
suitable, Larp1 is phosphorylated by the nutrient/redox/energy sensor
mammalian Target of rapamycin (mTOR) complex 1 (mTORC1), and does not
efficiently bind the TOP sequence, thus allowing for efficient
translation of ribosomal proteins (Fonseca et al., 2018, 2015; Hong et al., 2017; Jia et al., 2021). In some instances, Larp1 binding can also
stabilize TOP-containing mRNAs (Aoki et al., 2013; Berman et al., 2020; Gentilella et al., 2017; Ogami, Oishi, Nogimori, Sakamoto, \& Hoshino, 2020), linking mRNA translation
with mRNA stability to promote ribosome biogenesis
(Aoki et al., 2013; Berman et al., 2020; Fonseca et al., 2018, 2015; Hong et al., 2017; Lahr et al., 2017; Ogami et al., 2020; Philippe et al., 2018). Cellular nutrient levels are
known to affect stem cell differentiation and oogenesis in \emph{Drosophila}
(Hsu, LaFever, \& Drummond-Barbosa, 2008), however whether TOP motifs exist in
\emph{Drosophila} to coordinate ribosome protein synthesis is unclear. The
\emph{Drosophila} ortholog of Larp1, La related protein (Larp) is required
for proper cytokinesis and meiosis in \emph{Drosophila} testis as well as for
female fertility, but its targets remain undetermined (Blagden et al., 2009; Ichihara et al., 2007).

Germline depletion of ribosome biogenesis factors manifests as a
stereotypical GSC differentiation defect during \emph{Drosophila} oogenesis
(Sanchez et al., 2016). Female \emph{Drosophila} maintain 2-3 GSCs in the
germarium (\textbf{Figure 2.1A}) (Kai et al., 2005; Twombly et al., 1996; Xie \& Li, 2007; Xie \& Spradling, 1998, 2000a). Asymmetric cell division of GSCs produces a
self-renewing daughter GSC, and a differentiating daughter, called the
cystoblast (CB) (D. Chen \& McKearin, 2003b; McKearin \& Ohlstein, 1995). This asymmetric
division is unusual: following mitosis, the abscission of the GSC and CB
is not completed until the following G2 phase (\textbf{Figure 2.1A'})
(De Cuevas \& Spradling, 1998; Hsu et al., 2008). The GSC is marked by a
round structure called the spectrosome, which elongates and eventually
bridges the GSC and CB, similar to the fusomes that connect
differentiated cysts (\textbf{Figure 2.1A'}). During abscission the extended
spectrosome structure is severed and a round spectrosome is established
in the GSC and the CB (De Cuevas \& Spradling, 1998; Hsu et al., 2008).
Ribosome biogenesis defects result in failed GSC-CB abscission, causing
cells to accumulate as interconnected cysts marked by a fusome-like
structure called ``stem cysts'' (\textbf{Figure 2.1A'}) (Mathieu et al., 2013; Sanchez et al., 2016). In contrast with differentiated cysts
(McKearin \& Spradling, 1990; McKearin \& Ohlstein, 1995; Ohlstein \& McKearin, 1997), these stem cysts do not
express the differentiation factor Bag of Marbles (Bam), do not
differentiate, and typically die, resulting in sterility
(Sanchez et al., 2016). How proper ribosome biogenesis promotes GSC
abscission and differentiation is not known.

By characterizing three RNA helicases that promote ribosome biogenesis,
we identified a translational control module that is sensitive to proper
ribosome biogenesis and coordinates ribosome levels with GSC
differentiation. When ribosome biogenesis is optimal, ribosomal proteins
and a p53 repressor are both efficiently translated allowing for proper
GSC cell cycle progression and its differentiation. However, when
ribosome biogenesis is perturbed, we observe diminished translation of
both ribosomal proteins and the p53 repressor. As a consequence, p53 is
stabilized, cell cycle progression is blocked and GSC differentiation is
stalled. Thus, our work reveals an elegant tuning mechanism that links
ribosome biogenesis with a cell cycle progression checkpoint and thus
stem cell differentiation. Given that ribosome biogenesis defects in
humans result in ribosomopathies, which often result from stem cell
differentiation defects, our data lay the foundation for understanding
the etiology of developmental defects that arise due to ribosomopathies.

\hypertarget{results}{%
\section{Results}\label{results}}

\hypertarget{three-conserved-rna-helicases-are-required-in-the-germline-for-gsc-differentiation}{%
\subsection{Three conserved RNA helicases are required in the germline for GSC differentiation}\label{three-conserved-rna-helicases-are-required-in-the-germline-for-gsc-differentiation}}

We performed a screen to identify RNA helicases that are required for
female fertility in \emph{Drosophila}, and identified three predicted RNA
helicases with previously uncharacterized functions, \emph{CG5589}, \emph{CG4901,}
and \emph{CG9253} (\textbf{Figure 2.1B-C}) (\textbf{Supplemental Table 2.1})
(Blatt, Wong-Deyrup, et al., 2020). We named these candidate genes
\emph{aramis}, \emph{athos}, and \emph{porthos,} respectively, after Alexandre Dumas'
three musketeers who fought in service of their queen. We evaluated the
efficiency of RNAi in ovaries using qPCR and found that \emph{aramis},
\emph{athos}, and \emph{porthos} was significantly downregulated \textbf{(Figure 2.2A)}.
We additionally drove RNAi of \emph{aramis} and \emph{athos} in the germline of
flies expressing GFP::3XFLAG tagged versions of each gene respectively
and performed immunostaining and found that the expression of Aramis and
Athos was reduced specifically in the germline in contrast to the soma
\textbf{(Figure 2.2B-E)}. To further investigate how these helicases promote
fertility, we depleted \emph{aramis}, \emph{athos}, and \emph{porthos} in the germline
using the germline-driver \emph{nanos}-\emph{GAL4} (\emph{nosGAL4}) in combination with
RNAi lines. We detected the germline and spectrosomes/fusomes in ovaries
by immunostaining for Vasa and 1B1, respectively. In contrast to
controls, \emph{aramis}, \emph{athos}, and \emph{porthos} germline RNAi flies lacked
spectrosome-containing cells, and instead displayed cells with
fusome-like structures proximal to the self-renewal niche \textbf{(Figure 2.1D-H; Figure 2.2E-E'''}).
The cells in this cyst-like structurecontained ring canals,
a marker of cytoplasmic bridges, suggesting that
they are indeed interconnected (\textbf{Figure 2.2F-F'''}) (Zhang et al., 2014).
In addition to forming cysts in an aberrant location, the \emph{aramis},
\emph{athos}, and \emph{porthos} germline RNAi ovaries failed to form egg chambers
\textbf{(Figure 2.2G-G''')}.

Aberrant cyst formation proximal to the niche could reflect stem cysts
with GSCs that divide to give rise to CBs but fail to undergo
cytokinesis or differentiated cysts that initiate differentiation but
cannot progress further to form egg chambers. To discern between these
possibilities, first we examined the expression of a marker of GSCs,
phosphorylated Mothers against decapentaplegic (pMad). We observed pMad
expression in the cells closest to the niche, but not elsewhere in the
germline cysts of \emph{aramis}, \emph{athos}, and \emph{porthos} germline RNAi flies
(\textbf{Figure 2.2H-H')} (Kai \& Spradling, 2003). Additionally, none
of the cells connected to the GSCs in \emph{aramis}, \emph{athos}, and \emph{porthos}
germline RNAi flies expressed the differentiation reporter \emph{bamGFP}
(\textbf{Figure 2.1D-G'')} (McKearin \& Ohlstein, 1995). Thus, loss of \emph{aramis},
\emph{athos}, or \emph{porthos} in the germline results in the formation of stem
cysts, however with variable severity. This variability could be due to
a differential requirement for these genes or different RNAi
efficiencies. Overall, we infer that Aramis, Athos, and Porthos are
required for proper GSC cytokinesis to produce a stem cell and
differentiating daughter.

\hypertarget{athos-aramis-and-porthos-are-required-for-ribosome-biogenesis}{%
\subsection{Athos, Aramis, and Porthos are required for ribosome biogenesis}\label{athos-aramis-and-porthos-are-required-for-ribosome-biogenesis}}

We found that Aramis, Athos, and Porthos are conserved from yeast to
humans (\textbf{Figure 2.1B}). The closest orthologs of Aramis, Athos, and
Porthos are Rok1, Dhr2, and Rrp3 in yeast and DExD-Box Helicase 52
(DDX52), DEAH-Box Helicase 33 (DHX33), and DEAD-Box Helicase 47 (DDX47)
in humans, respectively (Hu et al., 2011). Both
the yeast and human orthologs have been implicated in rRNA biogenesis
(Bohnsack, Kos, \& Tollervey, 2008; Khoshnevis et al., 2016; Martin et al., 2014; O 'day, Chavanikamannil, \& Abelson, 1996; Sekiguchi, Hayano, Yanagida, Takahashi, \& Nishimoto, 2006; Tafforeau et al., 2013; Venema et al., 1997; Venema \& Tollervey, 1995; Vincent, Charette, \& Baserga, 2017; Zhang, Forys, Miceli, Gwinn, \& Weber, 2011). In addition, the
GSC-cytokinesis defect that we observed in \emph{aramis}, \emph{athos}, and
\emph{porthos} RNAi flies is a hallmark of reduced ribosome biogenesis in the
germline (Sanchez et al., 2016). Based on these observations, we
hypothesized that Aramis, Athos, and Porthos could enhance ribosome
biogenesis to promote proper GSC differentiation.

Many factors involved in rRNA biogenesis localize to the nucleolus and
interact with rRNA (Arabi et al., 2005; Grandori et al., 2005; Henras et al., 2008; Karpen et al., 1988). To detect
the subcellular localization of Aramis and Athos, we used available
lines that express Aramis::GFP::FLAG or Athos::GFP::FLAG fusion proteins
under endogenous control. For Porthos, we expressed a Porthos::FLAG::HA
fusion under the control of UASt promoter in the germline using a
previously described approach (DeLuca \& Spradling, 2018).
We found that in the germline, Aramis, Athos and Porthos colocalized
with Fibrillarin, which marks the nucleolus, the site of rRNA synthesis
(\textbf{Figure 2.3A-C'''}) (Ochs, Lischwe, Spohn, \& Busch, 1985). Aramis was
also in the cytoplasm of the germline and somatic cells of the gonad. To
determine if Aramis, Athos, and Porthos directly interact with rRNA, we
performed immunoprecipitation (IP) followed by RNA-seq. We found that
rRNA immunopurified with Aramis, Athos, and Porthos \textbf{(Figure 2.3D-D'', Figure 2.4A-A'')}.
Thus, Aramis, Athos, and Porthos are present in thenucleolus and interact with rRNA,
suggesting that they might regulate rRNA biogenesis.

Nucleolar size, and in particular nucleolar hypotrophy, is associated
with reduced ribosome biogenesis and nucleolar stress (Neumüller et al., 2008; Zhang et al., 2011). If Aramis, Athos, and Porthos
promote ribosome biogenesis, then their loss would be expected to cause
nucleolar stress and a reduction in mature ribosomes. Indeed,
immunostaining for Fibrillarin revealed hypotrophy of the nucleolus in
\emph{aramis, athos,} and \emph{porthos} germline RNAi flies compared to in
control flies, consistent with nucleolar stress (\textbf{Figure 2.3B-C'}).
Next, we used polysome profile analysis to evaluate the ribosomal
subunit ratio and translation status of ribosomes in S2 cells depleted
of \emph{aramis}, \emph{athos}, or \emph{porthos}
(Boamah, Kotova, Garabedian, Jarnik, \& Tulin, 2012; Õunap, Käsper, Kurg, \& Kurg, 2013).
We found that upon the depletion of all three helicases, the heights of
the polysome peaks were reduced (\textbf{Figure 2.3E-E''}). We found that
depletion of \emph{aramis} and \emph{porthos} diminished the height of the 40S
subunit peak compared to the 60S subunit peak, characteristic of
defective 40S ribosomal subunit biogenesis (\textbf{Figure 2.3E, E''}, \textbf{Figure 2.4D)} (Cheng et al., 2019),
whereas \emph{athos} depletion
diminished the height of the 60S subunit peak compared to the 40S peaks,
characteristic of a 60S ribosomal subunit biogenesis defect (\textbf{Figure 2.3E', Figure 2.4D'}) (Cheng et al., 2019).
Previous work indicates that the stem-cyst that arises from depletion of genes
involved in ribosome biogenesis in the germline genetically interacts
with Shrub (shrb) a member of the Escrt-III complex. To further
determine if \emph{aramis}, \emph{athos}, and \emph{porthos} regulates ribosome
biogenesis, we performed trans-heterozygous crosses between \emph{aramis},
\emph{athos}, and \emph{porthos} and \emph{shrb}. We observed the presence of stem-cyst
structures even heterozygotes mutants of \emph{shrb}, consistent with
previous observations (Matias et al., 2015; Sanchez et al., 2016), and in
\emph{aramis}, or \emph{athos}, \emph{porthos}. We found that in trans-heterozygous
germaria of a \emph{shrb} mutant our genes of interest result in a more
frequent occurrence of stem-cysts than in their respective heterozygous
background consistent with their role in ribosome biogenesis (\textbf{Figure
S2E-L}). Taken together our findings indicate that these helicases
promote ribosome biogenesis in \emph{Drosophila}.

\hypertarget{aramis-promotes-cell-cycle-progression-via-p53-repression}{%
\subsection{Aramis promotes cell cycle progression via p53 repression}\label{aramis-promotes-cell-cycle-progression-via-p53-repression}}

Our data so far indicate that Aramis, Athos and Porthos promote ribosome
biogenesis, which is known to be required for GSC abscission
(Sanchez et al., 2016). Yet the connections between ribosome biogenesis and
GSC abscission are poorly understood. To explore the connection, we
further examined the \emph{aramis} germline RNAi line, as its defect was
highly penetrant but maintained sufficient germline for analysis
(\textbf{Figure 2.1E, H}). First, we compared the mRNA profiles of \emph{aramis}
germline RNAi ovaries to \emph{bam} germline RNAi to determine if genes that
are known to be involved in GSC abscission have altered expression. We
used germline \emph{bam} depletion as a control because it leads to the
accumulation of CBs with no abscission defects (Flora, Schowalter, et al., 2018; Gilboa, Forbes, Tazuke, Fuller, \& Lehmann, 2003; McKearin \& Ohlstein, 1995; Ohlstein \& McKearin, 1997), whereas loss of \emph{aramis}
resulted in accumulation of CBs that do not abscise from the GSCs.

We performed RNA-seq and found that 607 RNAs were downregulated and 673
RNAs were upregulated in \emph{aramis} germline RNAi versus \emph{bam} germline
RNAi (cut-offs for differential gene expression were log\textsubscript{2}(foldchange)
\textgreater\textbar1.5\textbar, FDR \textless{} 0.05) (\textbf{Figure 2.10A, Supplemental Table 2.2}). Gene
Ontology (GO) analysis for biological processes on these genes encoding
these differentially expressed mRNAs
(Thomas et al., 2003) revealed that the genes that were
downregulated upon \emph{aramis} germline depletion were enriched for GO
terms related to the cell cycle, whereas the upregulated genes were
enriched for GO terms related to stress response (\textbf{Figure 2.5A, Figure 2.6B}). The downregulated genes included \emph{Cyclin A}, which is required
for cell cycle progression, \emph{Cyclin B} (\emph{CycB}) and \emph{aurora B}, which
are required for both cell cycle progression and cytokinesis; in
contrast the housekeeping gene \emph{Actin 5C} was unaffected (\textbf{Figure 2.5B-C, Figure 2.6C-C'}) (Mathieu et al., 2013; Matias et al., 2015). We confirmed that
CycB was reduced in the ovaries of \emph{aramis} germline RNAi flies compared
to \emph{bam} germline RNAi flies by immunofluorescence (\textbf{Figure 2.5D-F}).
These changes to genes that promote cell cycle and cytokinesis were also
seen \emph{aramis; bam} double depletions as the Biological Process GO-terms
we identified from targets downregulated in \emph{aramis} RNAi were very
similar to those we identified from in \emph{bam} RNAi; \emph{aramis} RNAi
compared to their controls (\textbf{Figure 2.5A, Supplemental Table 2.3}). Similarly,
all of the GO-terms we identified in from upregulated genes are also
enriched GO terms from the double-knockdown upregulated targets (\textbf{Figure 2.6B, Supplemental Table 2.3}).
Crucially, all the genes we refer to in the
manuscript such as \emph{CycB}, \emph{AurB,} and \emph{CycA} are also targets in \emph{bam}
RNAi; \emph{aramis} RNAi. (\textbf{Figure 2.6B, Supplemental Table 2.3}). These
results suggest that \emph{aramis} is required for the proper expression of
key regulators of GSC abscission.

CycB is expressed during G2 phase after asymmetric cell division to
promote GSC abscission (Flora, Schowalter, et al., 2018; Mathieu et al., 2013). To test if the
loss of germline \emph{aramis} leads to GSC abscission defects due to
diminished expression of CycB, we attempted to express a functional
CycB::GFP fusion protein in the germline under the control of a UAS/GAL4
system (\textbf{Figure 2.6D-D'}) (Mathieu et al., 2013). Unexpectedly, the
CycB::GFP fusion protein was not expressed in the \emph{aramis-}depleted
germline, unlike the wild type (WT) germline (\textbf{Figure 2.6E-E'})
(Glotzer, Murray, \& Kirschner, 1991; Mathieu et al., 2013; Zielke et al., 2014).
We considered the possibility that progression into G2 is blocked in the
absence of \emph{aramis}, precluding expression of CycB. To monitor the cell
cycle, we used the Fluorescence Ubiquitin-based Cell Cycle Indicator
(FUCCI) system. \emph{Drosophila} FUCCI utilizes a GFP-tagged degron from
E2f1 to mark G2, M, and G1 phases and an RFP-tagged degron from CycB to
mark S, G2, and M phases (Zielke et al., 2014). We observed cells in
different cell cycle stages in both WT and \emph{bam}-depleted germaria, but
the \emph{aramis}-depleted germaria expressed neither GFP nor RFP (\textbf{Figure 2.6F-H''}). Double negative reporter expression is thought to indicate
early S phase, when expression of E2f1 is low and CycB is not expressed
(Hinnant, Alvarez, \& Ables, 2017). The inability to express FPs is
not due to a defect in translation as \emph{aramis}-depleted germline can
express GFP that is not tagged with the degron (\textbf{Figure 2.6I-I'}).
Taken together, we infer that loss of \emph{aramis} blocks cell cycle
progression around late G1 phase/early S phase and prevents progression
to G2 phase, when GSCs abscise from CBs.

In mammals, cells defective for ribosome biogenesis stabilize p53, which
is known to impede the G1 to S transition (Agarwal, Agarwal, Taylor, \& Stark, 1995; Senturk \& Manfredi, 2013). Therefore, we hypothesized that the reduced
ribosome biogenesis in the \emph{aramis}-depleted germline leads to p53
stabilization in undifferentiated cells, driving cell cycle arrest and
GSC abscission defects. To test this hypothesis, we detected p53 and
Vasa in the germline by immunostaining. A hybrid dysgenic cross that
expresses p53 in undifferentiated cells was utilized as a positive
control, and \emph{p53} null flies were used as negative controls (\textbf{Figure 2.6J-K}) (Moon et al., 2018). In WT, we observed p53 expression in the
meiotic stages of germline but p53 expression in GSCs and CBs was
attenuated as previously reported (\textbf{Figure 2.5G-G''})
(Lu, Chapo, Roig, \& Abrams, 2010). However, compared to WT
GSCs/CBs, we observed p53 expression in the stem cysts of the
\emph{aramis-}depleted germline (\textbf{Figure 2.5G-I}). Similarly, we observed p53
expression in the stem cysts of \emph{athos-} and \emph{porthos-}depleted
germlines \textbf{(Figure 2.6L-M)}, further supporting that reduced ribosome
biogenesis stabilizes p53. To determine if p53 stabilization is required
for the cell cycle arrest in \emph{aramis}, \emph{athos}, and \emph{porthos-}depleted
germline cysts, we depleted \emph{aramis}, \emph{athos} and \emph{porthos} in the
germline of \emph{p53} mutants using germline specific knockdown. We observed
a partial but significant alleviation of the cyst phenotype, such that
spectrosomes were restored (\textbf{Figure 2.5J-L, Figure 2.6N-Q}). This finding
indicates that p53 contributes to cytokinesis failure upon loss of
\emph{aramis}, but that additional factors are also involved.

To determine if aberrant expression of p53 is sufficient to cause the
formation of stem-cysts we overexpressed p53 in the germline under the
control of a UAS/GAL4 system. While 84\% of germeria had a complete loss
of germline, in 12.7\% of germaria the germline cells were connected by a
fusome-like structure, phenocopying loss of \emph{aramis}, \emph{athos}, or
\emph{porthos} (\emph{Figure 2.5M-N}), in the remaining 3.6\% of the time we observed
several single germline cells (XX-refer to the quant). Taken together,
we find that \emph{aramis-}depleted germ cells display reduced ribosome
biogenesis, aberrant expression of p53 protein and a block in cell cycle
progression. Reducing p53 partially alleviates the cell cycle block and
GSC cytokinesis defect, while inappropriate p53 expression results in
loss of germline and cytokinesis defects.

\textbf{Aramis promotes translation of Non1, a negative regulator of p53,
linking ribosome biogenesis to the cell cycle}

Although p53 protein levels were elevated upon loss of \emph{aramis} in the
germline, \emph{p53} mRNA levels were not significantly altered (log\textsubscript{2} fold
change: -0.49; FDR: 0.49). Given that ribosome biogenesis is affected,
we considered that translation of p53 or one of its regulators was
altered in \emph{aramis}-depleted germlines. To test this hypothesis, we
performed polysome-seq of gonads enriched for GSCs or CBs as
developmental controls, as well as gonads depleted for \emph{aramis} in the
germline (Flora, Wong-Deyrup, et al., 2018). We plotted the ratios of polysome-associated
RNAs to total RNAs (\textbf{Figure 2.7A-A'', Supplemental Table 2.4}) and
identified 87 mRNAs with a reduced ratio upon depletion of \emph{aramis},
suggesting that they were translated less efficiently compared to
developmental controls. Loss of \emph{aramis} reduced the levels of these 87
downregulated transcripts in polysomes, without significantly affecting
their total mRNA levels (\textbf{Figure 2.7B, Figure 2.8A-A'}). These 87
transcripts encode proteins mostly associated with translation including
ribosomal proteins (\textbf{Figure 2.7C)}. To validate that Aramis regulates
translation of these target mRNAs, we utilized a reporter line for the
Aramis-regulated transcript encoding Ribosomal protein S2 (RpS2) that is
expressed in the context of the endogenous promoter and regulatory
sequences (Buszczak et al., 2007; Zhang et al., 2014). We
observed reduced levels of RpS2::GFP in germlines depleted of \emph{aramis}
but not in those depleted of \emph{bam} (\textbf{Figure 2.7D-F}). RpS2::GFP
expression is also reduced in \emph{bam} RNAi, \emph{aramis} RNAi double germline
knockdown compared to \emph{bam} RNAi (\textbf{Figure 2.8B-D}). To ensure that
reduced RpS2::GFP levels did not reflect a global decrease in
translation, we visualized nascent translation using
O-propargyl-puromycin (OPP). OPP is incorporated into nascent
polypeptides and can be detected using click-chemistry
(Sanchez et al., 2016). We observed that global translation in the germlines
of ovaries depleted of \emph{aramis} was not reduced compared to single cells
of control ovaries or \emph{bam} \textbf{(Figure 2.7G-J)}. Notably, the regulation
of these genes do not appear to be directly mediated by Aramis as none
of the target genes are enriched from Aramis::GFP::3XFLAG RNA IP-seq
(\textbf{Supplemental Table 2.5}). Thus, loss of \emph{aramis} results in reduced
translation of a subset of transcripts compared to the rest of the
transcriptome.

Previous work has indicated that excess ribosomal protein expression may
be deleterious to cellular functions and induce cell stress. We
hypothesized that the decrease in translation we observe may be the
result of a regulatory mechanism to mitigate such affects. To test if
overexpression of ribosomal proteins is deleterious to germline
development, we overexpressed several ribosomal proteins in the
germline. We found no obvious phenotype as a result of overexpression of
single ribosomal proteins (\textbf{Figure 2.8E-G'}). We reason that individual
overexpression may not be as detrimental as the excess expression of
nearly all ribosomal proteins that we observed in our experiments, which
is not technically possible to recapitulate through exogenous expression
lines.

None of these 87 translational targets have been implicated in directly
controlling abscission (Mathieu et al., 2013; Matias et al., 2015). However, we
noticed that the mRNA encoding Novel Nucleolar protein 1 (Non1/CG8801)
was reduced in polysomes upon loss of \emph{aramis} in the germline (\textbf{Figure 2.7C}).
The human ortholog of Non1 is GTP Binding Protein 4 (GTPBP4), and
these proteins are known to physically interact with p53 in both
\emph{Drosophila} and human cells and have been implicated in repressing p53
(mentioned as CG8801 in Lunardi et al.) (Li et al., 2018; Lunardi et al., 2010).
To determine if the protein level of Non1 is reduced upon depletion of
\emph{aramis,} we monitored the abundance of Non1::GFP, a transgene that is
under endogenous control (Sarov et al., 2016), and
found that Non1::GFP was expressed in the undifferentiated GSCs and CBs
(\textbf{Figure 2.9A-A'')}. Non1::GFP levels were reduced in the \emph{aramis},
\emph{athos} or \emph{porthos-}depleted stem cysts compared to the CBs that
accumulated upon \emph{bam}-depletion (\textbf{Figure 2.9B-D, Figure 2.10C-F}),
suggesting that efficient ribosome biogenesis promotes efficient
translation of Non1.

During normal oogenesis, p53 protein is expressed in cyst stages in
response to recombination-induced double strand breaks
(Lu et al., 2010). We found that Non1 was highly
expressed at undifferentiated stages and in two- and four-cell cysts
when p53 protein levels were low, whereas its expression was attenuated
at eight- and 16-cell cyst stages when p53 protein levels were high
(\textbf{Figure 2.9A-A''}, \textbf{Figure 2.10A-B'}). Non1 was highly expressed in egg
chambers, which express low levels of p53 protein suggesting that Non1
could regulate p53 protein levels. To determine if Non1 regulates GSC
differentiation and p53, we depleted \emph{Non1} in the germline. We found
that germline-depletion of \emph{Non1} results in stem cyst formation and
loss of later stages, as well as increased p53 expression, phenocopying
germline-depletion of \emph{aramis}, \emph{athos}, and \emph{porthos} (\textbf{Figure 2.9E-F, H, Figure 2.10G-I}). In addition, we found that loss of \emph{p53} from
\emph{Non1-}depleted germaria partially suppressed the phenotype (\textbf{Figure
5F-H}). Thus, \emph{Non1} is regulated by \emph{aramis} and is required for p53
suppression, cell cycle progression, and GSC abscission.

To determine if Aramis, Athos, and Porthos promotes GSC differentiation
via translation of Non1, we restored \emph{Non1} expression in germ cells
depleted of \emph{aramis}, \emph{athos}, or \emph{porthos}. Briefly, we cloned \emph{Non1}
with heterologous UTR elements under the control of the UAS/GAL4 system
(see Methods) (Rørth, 1998). We found that
restoring \emph{Non1} expression in the \emph{aramis}, \emph{athos}, or \emph{porthos
-}depleted germline significantly attenuated the stem cysts and
increased the number of cells with spectrosomes, however overexpression
of Non1 alone did not cause any noticeable defect (\textbf{Figure 2.9I-K, Figure 2.10J-N}).
Taken together, we conclude that Non1 can partially suppress
the cytokinesis defect caused by germline \emph{aramis} depletion.

\hypertarget{aramis-regulated-targets-contain-a-top-motif-in-their-5utr}{%
\subsection{Aramis-regulated targets contain a TOP motif in their 5'UTR}\label{aramis-regulated-targets-contain-a-top-motif-in-their-5utr}}

We next asked how \emph{aramis} and efficient ribosome biogenesis promote the
translation of a subset of mRNAs, including \emph{Non1}, to regulate GSC
differentiation. We hypothesized that the 87 mRNA targets share a
property that make them sensitive to rRNA and ribosome levels. To
identify shared characteristics, we performed \emph{de novo} motif discovery
of target genes compared to non-target genes (Bailey, Williams, Misleh, \& Li, 2006) and
identified a polypyrimidine motif in the 5'UTRs of most target genes
(UCUUU; E-value: 6.6e\textsuperscript{-094}). This motif resembles the previously
described TOP motif at the 5' end of mammalian transcripts
(Philippe et al., 2018; Thoreen et al., 2012). Although the existence of
TOP-containing mRNAs in \emph{Drosophila} has been speculated, to our
knowledge their presence has not been explicitly demonstrated
(Chen \& Steensel, 2017; Qin, Ahn, Speed, \& Rubin, 2007). This observation motivated us to
precisely determine the 5' end of transcripts, so we analyzed previously
published {c}ap {a}nalysis of {g}ene
{e}xpression sequencing (CAGE-seq) data that had determined
transcription start sites (TSS) in total mRNA from the ovary (\textbf{Figure
6A}) (Boley, Wan, Bickel, \& Celniker, 2014; Chen et al., 2014; dos Santos et al., 2015). Of the 87 target genes,
76 had sufficient expression in the CAGE-seq dataset to define their
TSS. We performed motif discovery using the CAGE-seq data and found that
72 of 76 Aramis-regulated mRNAs have a polypyrimidine motif that starts
within the first 50 nt of their TSS (\textbf{Figure 2.11B-C}). In mammals, it
was previously thought that the canonical TOP motif begins with an
invariant 'C' (Meyuhas, 2000; Philippe, van den Elzen, Watson, \& Thoreen, 2020). However, systematic
characterization of the sequence required in order for an mRNA to be
regulated as a TOP containing mRNA revealed that TOP mRNAs can start
with either a 'C' or a 'U' (Philippe et al., 2020). Thus,
mRNAs whose efficient translation is dependent on \emph{aramis} share a
terminal polypyrimidine-rich motif in their 5'UTR that resembles a TOP
motif.

In vertebrates, most canonical TOP-regulated mRNAs encode ribosomal
proteins and translation initiation factors that are coordinately
upregulated in response to growth cues mediated by the Target of
Rapamycin (TOR) pathway and the TOR complex 1 (mTORC1)
(Hornstein, Tang, \& Meyuhas, 2001; Iadevaia, Liu, \& Proud, 2014; Kim et al., 2008; Meyuhas \& Kahan, 2015; Pallares-Cartes, Cakan-Akdogan, \& Teleman, 2012) Indeed, 76 of the
87 Aramis targets were ribosomal proteins, and 9 were known or putative
translation factors, consistent with TOP-containing RNAs in vertebrates
(\textbf{Figure 2.7C, Supplemental Table 2.6}). To determine if the putative TOP
motifs that we identified are sensitive to TORC1 activity, we designed
``TOP reporter'' constructs. Specifically, the germline-specific \emph{nanos}
promoter was employed to drive expression of an mRNA with 1) the 5'UTR
of the \emph{aramis} target \emph{RpL30}, which contains a putative TOP motif, 2)
the coding sequence for a GFP-HA fusion protein and 3) a 3'UTR (K10)
that is not translationally repressed (Flora, Wong-Deyrup, et al., 2018; Serano, Cheung, Frank, \& Cohen, 1994), referred to as the WT-TOP
reporter (\textbf{Figure 2.11D}). As a control, we created a construct in which
the polypyrimidine sequence was mutated to a polypurine sequence
referred to as the Mut-TOP reporter (\textbf{Figure 2.11D}).

In \emph{Drosophila}, TORC1 activity increases during meiotic cyst stages
(Wei et al., 2019, 2014). We found that the WT-TOP reporter displayed
peak expression in 8-cell cysts, whereas the Mutant-TOP reporter did not
\textbf{(Figure 2.11E-F'')}, suggesting that the WT-TOP reporter is sensitive to
TORC1 activity. Moreover, depletion of \emph{Nitrogen permease regulator-like
3} (\emph{Nprl3}), an inhibitor of TORC1 (Wei et al., 2014), led to a significant
increase in expression of the WT-TOP reporter but not the Mutant-TOP
reporter \textbf{(Figure 2.12A-E)}. Additionally, to attenuate TORC1 activity,
we depleted \emph{raptor}, one of the subunits of the TORC1 complex
(Hong et al., 2012; Loewith \& Hall, 2011). Here we found that the WT-TOP
reporter had a significant decrease in reporter expression while the
Mutant-TOP reporter did not show a decrease in expression \textbf{(Figure 2.12F-J)}.
Taken together, our data suggest that Aramis-target
transcripts contain TOP motifs that are sensitive to TORC1 activity.
However, we note that our TOP reporter did not recapitulate the pattern
of Non1::GFP expression, suggesting that Non1 may have additional
regulators that modulate its protein levels in the cyst stages.

TOP mRNAs show increased translation in response to TOR signaling,
leading to increased ribosome biogenesis
(Jefferies et al., 1997; Jia et al., 2021; Powers \& Walter, 1999; Thoreen et al., 2012). However, to our knowledge,
whether reduced ribosome biogenesis can coordinately diminish the
translation of TOP mRNAs to balance and lower ribosome protein
production and thus balance the levels of the distinct components needed
for full ribosome assembly is not known. To address this question, we
crossed the transgenic flies carrying the WT-TOP reporter and Mutant-TOP
reporter into \emph{bam} and \emph{aramis}, \emph{athos}, and \emph{porthos} germline RNAi
backgrounds. We found that the expression from the WT-TOP reporter was
reduced by 2.9-fold in the germline of \emph{aramis} RNAi ovaries compared to
\emph{bam} RNAi ovaries \textbf{(Figure 2.11F-G, J)}. In contrast, the Mutant-TOP
reporter was only reduced by 1.6-fold in the germline of \emph{aramis} RNAi
ovaries compared to \emph{bam} RNAi ovaries \textbf{(Figure 2.11H-J)}. We observed
the same trend for \emph{athos} and \emph{porthos} (\textbf{Figure 2.12K-Q}). This
suggests that the TOP motif-containing mRNAs are sensitive to ribosome
biogenesis.

\hypertarget{larp-binds-top-sequences-in-drosophila}{%
\subsection{\texorpdfstring{Larp binds TOP sequences in \emph{Drosophila}}{Larp binds TOP sequences in Drosophila}}\label{larp-binds-top-sequences-in-drosophila}}

Next, we sought to determine how TOP-containing mRNAs are regulated
downstream of Aramis. In mammalian cells, Larp1 is a critical negative
regulator of TOP-containing RNAs during nutrient deprivation
(Berman et al., 2020; Fonseca et al., 2015; Hong et al., 2017; Philippe et al., 2020; Tcherkezian et al., 2014). Therefore, we
hypothesized that \emph{Drosophila} Larp reduces the translation of
TOP-containing mRNAs when rRNA biogenesis is reduced upon loss of
\emph{aramis}. First, using an available gene-trap line in which endogenous
Larp is tagged with GFP and 3XFLAG, we confirmed that Larp was robustly
expressed throughout all stages of oogenesis including in GSCs \textbf{(Figure 2.14A-A')}.

Next, we performed electrophoretic mobility shift assays (EMSA) to
examine protein-RNA interactions with purified \emph{Drosophila} Larp-DM15,
the conserved domain that binds to TOP sequences in vertebrates
(Lahr et al., 2017). As probes, we utilized capped 42-nt RNAs corresponding
to the 5'UTRs of \emph{RpL30} and \emph{Non1}, including their respective TOP
sequences. We observed a gel shift with these RNA oligos in the presence
of increasing concentrations of Larp-DM15 \textbf{(Figure 2.13A-A', Figure 2.14B)}, and this shift was abrogated when the TOP sequences were mutated
to purines \textbf{(Figure 2.14C-C').} To determine if Larp interacts with
TOP-containing mRNAs \emph{in vivo}, we immunopurified Larp::GFP::3XFLAG from
the ovaries of the gene-trap line and performed RNA-seq (\textbf{Figure 2.14D}). We uncovered 156 mRNAs that were bound to Larp, and 84 of these
were among the 87 \emph{aramis} translationally regulated targets, including
\emph{Non1}, \emph{RpL30}, and \emph{RpS2} (\textbf{Figure 2.13B-C, Supplemental Table 2.7}).
Thus, \emph{Drosophila} Larp binds to TOP sequences \emph{in vitro} and
TOP-containing mRNAs \emph{in vivo}.

To test our hypothesis that \emph{Drosophila} Larp inhibits the translation
of TOP-containing mRNAs upon loss of \emph{aramis}, we immunopurified
Larp::GFP::3XFLAG from germline \emph{bam} RNAi ovaries and germline \emph{aramis}
RNAi ovaries. Larp protein is not expressed at higher levels in \emph{aramis}
RNAi compared to developmental control \emph{bam} RNAi (\textbf{Figure 2.14E-G}).
Larp protein is also not expressed at higher levels in \emph{bam} RNAi;
\emph{aramis} RNAi germline knockdown compared to \emph{bam} RNAi as a control
(\textbf{Figure 2.14H-I}). We found that Larp binding to \emph{aramis} target mRNAs
\emph{Non1} and \emph{RpL30} was increased in \emph{aramis} RNAi ovaries compared to
\emph{bam} RNAi ovaries (\textbf{Figure 2.13D, Figure 2.14J}). In contrast, a
non-target mRNA that does not contain a TOP motif, \emph{alpha-tubulin} mRNA,
did not have a significant increase in binding to Larp in \emph{aramis} RNAi
ovaries compared to \emph{bam} RNAi ovaries. Overall, these data suggest that
reduced rRNA biogenesis upon loss of \emph{aramis} increases Larp binding to
the TOP-containing mRNAs \emph{Non1} and \emph{RpL30}.

If loss of \emph{aramis} inhibits the translation of TOP-containing mRNAs due
to increased binding of Larp to its targets, then overexpression of Larp
would be expected to phenocopy germline depletion of \emph{aramis}.
Unphosphorylated Larp binds to TOP motifs more efficiently, but the
precise phosphorylation sites of \emph{Drosophila} Larp, to our knowledge,
are currently unknown (Hong et al., 2017). To circumvent this issue, we
overexpressed the DM15 domain of Larp which we showed binds the \emph{RpL30}
and \emph{Non1} TOP motifs \emph{in vitro} (\textbf{Figure 2.13A-A'}), and, based on
homology to mammalian Larp1, lacks majority of the putative
phosphorylation sites (Jia et al., 2021; Lahr et al., 2017; Philippe et al., 2018). We found that overexpression of a
Larp-DM15::GFP fusion in the germline resulted in fusome-like structures
extending from the niche (\textbf{Figure 2.13E-F'})\textbf{.} Additionally, ovaries
overexpressing Larp-DM15 had 32-cell egg chambers, which were not
observed in control ovaries (\textbf{Figure 2.14K-K'}). The presence of 32-cell
egg chambers is emblematic of cytokinesis defects that occur during
early oogenesis (Mathieu et al., 2013; Matias et al., 2015; Sanchez et al., 2016). Our
findings indicate that these cells are delayed in cytokinesis and that
over expression of Larp partially phenocopies depletion of \emph{aramis}.

\hypertarget{discussion}{%
\section{Discussion}\label{discussion}}

During \emph{Drosophila} oogenesis, efficient ribosome biogenesis is required
in the germline for proper GSC cytokinesis and differentiation. The
outstanding questions that needed to be addressed were: 1) Why does
disrupted ribosome biogenesis impair GSC abscission and differentiation?
and 2) How does the GSC monitor and couple ribosome abundance to
differentiation? Our results suggest that germline ribosome biogenesis
defect stalls the cell cycle, resulting a loss of differentiation and
the formation of stem cysts. We discovered that proper ribosome
biogenesis is monitored through a translation control module that allows
for co-regulation of ribosomal proteins and a p53 repressor. Loss of
\emph{aramis}, \emph{athos} and \emph{porthos} reduces ribosome biogenesis and inhibits
translation of a p53 repressor, leading to p53 stabilization, cell cycle
arrest and loss of stem cell differentiation (\textbf{Figure 2.13G}).

\hypertarget{aramis-athos-and-porthos-are-required-for-efficient-ribosome-biogenesis-in-drosophila}{%
\subsection{\texorpdfstring{Aramis, Athos, and Porthos are required for efficient ribosome biogenesis in \emph{Drosophila}}{Aramis, Athos, and Porthos are required for efficient ribosome biogenesis in Drosophila}}\label{aramis-athos-and-porthos-are-required-for-efficient-ribosome-biogenesis-in-drosophila}}

We provide evidence that Aramis, Athos and Porthos play a role in
ribosome biogenesis in \emph{Drosophila}, similar to their orthologs in yeast
(Bohnsack et al., 2008; Granneman et al., 2006; Khoshnevis et al., 2016; O 'day et al., 1996) and
mammals (Sekiguchi et al., 2006; Tafforeau et al., 2013; Zhang et al., 2011). Their role in ribosome
biogenesis is likely a direct function of these helicases as they
physically interact with precursor rRNA. In yeast, Rok1, the ortholog of
Aramis, binds to several sites on pre-rRNA, predominantly in the 18S
region (Bohnsack et al., 2008; Khoshnevis et al., 2016; Martin et al., 2014). This is consistent with the
small subunit ribosome biogenesis defect we observe upon loss of
\emph{aramis} in \emph{Drosophila}. Rrp3, the yeast ortholog of Porthos, promotes
proper cleavage of pre-rRNA and is required for proper 18S rRNA
production (Granneman et al., 2006; O 'day et al., 1996). DDX47, the
mammalian ortholog of Porthos, binds to early rRNA precursors as well as
proteins involved in ribosome biogenesis
(Sekiguchi et al., 2006). Consistent with these findings,
we find that Aramis and Porthos promote 40S ribosome biogenesis. DHX33,
the mammalian ortholog of Athos, has been implicated in facilitating
rRNA synthesis (Zhang et al., 2011). In contrast,
we find that Athos promotes 60S ribosome biogenesis by directly
interacting with rRNA. However, we cannot rule out that Athos also
affects transcription of rRNA in \emph{Drosophila} as it does in mammals
(Zhang et al., 2011). Overall, we find that each
mammalian ortholog of Aramis, Athos, and Porthos has consistent ribosome
subunit defects, suggesting that the function of these helicases is
conserved from flies to mammals. Intriguingly, DDX52 (Aramis) is one of
the 15 genes deleted in 17q12 syndrome
(Hendrix, Clemens, Canavan, Surti, \& Rajkovic, 2012). 17q12 syndrome results in
delayed development, intellectual disability, and, more rarely,
underdevelopment of organs such as the uterus
(Bernardini et al., 2009; Hendrix et al., 2012). Our finding that Aramis
disrupts stem cell differentiation could explain some of the poorly
understood defects in 17q12 syndrome.

\hypertarget{ribosome-biogenesis-defects-leads-to-cell-cycle-defects-mediated-by-p53}{%
\subsection{Ribosome biogenesis defects leads to cell cycle defects mediated by p53}\label{ribosome-biogenesis-defects-leads-to-cell-cycle-defects-mediated-by-p53}}

Here we report that three RNA helicases, \emph{aramis}, \emph{athos}, and
\emph{porthos,} that promote proper ribosome biogenesis in \emph{Drosophila} are
required in the germline for fertility. Loss of \emph{aramis}, \emph{athos}, and
\emph{porthos} causes formation of a ``stem cyst'' and loss of later stage
oocytes. Stem cysts are a characteristic manifestation of ribosome
biogenesis deficiency wherein GSCs are unable to complete cytokinesis
and fail to express the differentiation factor Bam, which in GSCs is
initiated at G2 of the cell cycle (Sanchez et al., 2016; Zhang et al., 2014). Our
RNA seq and cell cycle analysis indicates that depletion of \emph{aramis}
blocks the cell cycle at G1, and that failure to progress to G2 prevents
abscission and expression of Bam. Thus, our results suggest that
ribosome biogenesis defects in the germline stall the cell cycle,
resulting in formation of stem cysts and sterility.

In most tissues in \emph{Drosophila,} p53 primarily activates apoptosis,
however, in the germline p53 is activated during meiosis and does not
cause cell death (Fan et al., 2010; Lu et al., 2010). Furthermore, p53 activation in
the germline is required for germline repopulation and GSC survival
after genetic insult, implicating p53 as a potential cell cycle
regulator (Ma et al., 2016; Tasnim \& Kelleher, 2018). Our
observation that reduction of \emph{p53} partially rescues a stem cyst defect
caused by ribosome deficiency due to germline depletion of \emph{aramis}
indicates that the G1 block in GSCs is, in part, mediated by p53
activation. Thus, in \emph{Drosophila} GSCs, p53 blocks the GSC cell cycle
and is sensitive to rRNA production. The developmental upregulation of
p53 during GSC differentiation concomitant with lower ribosome levels
parallels observations in disease states, such as ribosomopathies
(Calo et al., 2018; Deisenroth \& Zhang, 2010; Pereboom et al., 2011; Yelick \& Trainor, 2015).

We find that p53 levels in GSCs are regulated by conserved p53 regulator
Non1. In mammalian cells, increased free RpS7 protein due to nucleolar
stress binds and sequesters MDM2, a repressor of p53, freeing p53,
resulting in G1 cell cycle arrest (Deisenroth \& Zhang, 2010; Zhang \& Lu, 2009). \emph{Drosophila} have no identified
homolog to MDM2. It is not fully known how ribosome levels are monitored
in \emph{Drosophila} in the absence of MDM2 and how this contributes to cell
cycle progression. In \emph{Drosophila,} Non1 levels are high in the GSCs and
p53 is low, and reciprocally Non1 levels are low during meiosis, but p53
is expressed. Our finding that loss of Aramis leads to diminished Non1
and elevated p53, and that either loss of p53 or elevated Non1 suppress
differentiation defects caused by loss of Aramis, suggests that, in the
female germline, Non1 may fulfill the function of Mdm2 by promoting p53
degradation during \emph{Drosophila} oogenesis. While Non1 has been shown to
directly interact with p53, how it regulates p53 levels in both humans
and \emph{Drosophila} is not known (Li et al., 2018; Lunardi et al., 2010). Overall,
our data place Non1 downstream of ribosome biogenesis and upstream of
p53 in controlling cell cycle progression and GSC differentiation.
However, our data do not rule out that Non1 may also act upstream of or
in parallel to Aramis.

The vertebrate ortholog of Non1, GTPBP4, also controls p53 levels and is
upregulated in some cancers (Li et al., 2018; Lunardi et al., 2010; Yu, Jin, Zhang, \& Xu, 2016). This suggests that there may be
parallel pathways for monitoring ribosome levels via p53 in different
tissue types. Unlike \emph{Drosophila} Non1, its ortholog, GTPBP4 has not
been identified as a TOP mRNA, so if it similarly acts as a mediator
between ribosome biogenesis and the cell cycle it is likely activated in
a somewhat different manner (Philippe et al., 2020).
However, mammalian Larp1 is required for proper cell cycle progression
and cytokinesis (Burrows et al., 2010; Tcherkezian et al., 2014).
Excitingly several differentiation and cell cycle regulation genes in
mammals are TOP mRNAs regulated by Larp1, including Tumor Protein,
Translationally-Controlled 1 (TPT1) and Nucleosome Assembly Protein 1
Like 1 (NAP1L1) (Philippe et al., 2020). TPT1 is a cancer
associated factor that has been implicated in activating pluripotency
(Koziol, Garrett, \& Gurdon, 2007). Similarly, NAP1L1, a
nucleosome assembly protein, is required to maintain proper cell cycle
control as loss of NAP1L1 results in cell cycle exit and premature
differentiation (Qiao et al., 2018). Overall, although
the specific targets of Larp1 in mammals may differ from those in
\emph{Drosophila}, the mechanism by which Larp modulates cell cycle and
differentiation may be conserved.

\hypertarget{ribosome-biogenesis-defects-leads-to-repression-of-top-containing-mrna}{%
\subsection{Ribosome biogenesis defects leads to repression of TOP-containing mRNA}\label{ribosome-biogenesis-defects-leads-to-repression-of-top-containing-mrna}}

TOP-containing mRNAs are known to be coregulated to coordinate ribosome
production in response to nutrition or other environmental cues
(Kimball, 2002; Meyuhas \& Kahan, 2015; Tang et al., 2001). Surprisingly, our observation
that loss of \emph{aramis} reduces translation of a cohort of TOP-containing
mRNAs, including Non1, suggests that the TOP motif also sensitizes their
translation to lowered levels of rRNA. This notion is supported by TOP
reporter assays demonstrating that reduced translation upon loss of
\emph{aramis} requires the TOP motif. We hypothesize that limiting TOP mRNA
translation lowers ribosomal protein production to maintain a balance
with reduced rRNA production. This mechanism would prevent the
production of excess ribosomal proteins that cannot be integrated into
ribosomes and the ensuing harmful aggregates
(Tye et al., 2019). Additionally, it would
coordinate rRNA production and ribosomal protein translation during
normal germline development, where it is known that the level of
ribosome biogenesis and of global translation are dynamic
(Blatt et al., 2020; Fichelson et al., 2009; Sanchez et al., 2016; Zhang et al., 2014).

\hypertarget{larp-transduces-growth-status-to-ribosome-biogenesis-targets}{%
\subsection{Larp transduces growth status to ribosome biogenesis targets}\label{larp-transduces-growth-status-to-ribosome-biogenesis-targets}}

Recent work has shown that the translation and stability of
TOP-containing mRNAs are mediated by Larp1 and its phosphorylation
(Berman et al., 2020; Hong et al., 2017; Jia et al., 2021). We found that perturbing rRNA production
and thus ribosome biogenesis, without directly targeting ribosomal
proteins, similarly results in dysregulation of TOP mRNAs. Our data show
that \emph{Drosophila} Larp binds the \emph{RpL30} and \emph{Non1} 5'UTR in a
TOP-dependent manner \emph{in vitro} and to nearly all of the translation
targets we identified \emph{in vivo.} Together these data suggest that rRNA
production regulates TOP mRNAs via Larp. Furthermore, the cytokinesis
defect caused by overexpression of Larp-DM15 in the germline suggests
that Larp regulation could maintain the homeostasis of ribosome
biogenesis more broadly by balancing the expression of ribosomal protein
production with the rate of other aspects of ribosome biogenesis, such
as rRNA processing, during development.

Previous studies indicate that unphosphorylated Larp1 binds to and
represses its targets more efficiently than phosphorylated Larp1
(Fonseca et al., 2018; Hong et al., 2017; Jia et al., 2021). In mammalian systems Larp1 has been shown
to be phosphorylated by the TORC1 complex, while this has not been
demonstrated in \emph{Drosophila}, we have demonstrated that expression of
our TOP-reporter is dependent on Raptor and TOP-reporter expression is
repressed by Nprl3 (\textbf{Figure 2.12A-J}). These results are consistent with
a model where TORC1 monitors ribosome biogenesis status and in turn
regulates the activity of Larp. Thus, although we do not know the
identity of the kinase that phosphorylates Larp in \emph{Drosophila}
definitively, we hypothesize that Larp is not phosphorylated upon loss
of \emph{aramis, athos} and \emph{porthos}, when ribosome biogenesis is perturbed.
We propose that until ribosome biogenesis homeostasis is reached, this
kinase will remain inactive, continuously increasing the pool of
dephosphorylated Larp. In this scenario, as dephosphorylated Larp
accumulates, it begins to bind its targets. Initially, it will bind its
highest affinity targets, presumably encoding ribosomal proteins and
repress their translation to rebalance ribosomal protein production with
rRNA production. Consistent with this model, the TOP motif in \emph{RpL30} is
bound by Larp even more tightly with a nearly 9-fold higher affinity
compared to the \emph{Non1} TOP site (\textbf{Figure 2.14B}). We propose that such
differences in affinity may allow Larp to repress ribosomal protein
translation to facilitate cellular homeostasis without immediately
causing cell cycle arrest. However, if homeostasis cannot be achieved
and sufficient dephosphorylated Larp accumulates, Larp will also bind
and repress the translation of lower affinity targets. Repression of
Non1 in this manner would result in cell cycle arrest and block
differentiation as occurs upon \emph{aramis} depletion.

\hypertarget{ribosome-biogenesis-in-stem-cell-differentiation-and-ribosomopathies}{%
\subsection{Ribosome biogenesis in stem cell differentiation and ribosomopathies}\label{ribosome-biogenesis-in-stem-cell-differentiation-and-ribosomopathies}}

Ribosomopathies arise from defects in ribosomal components or ribosome
biogenesis and include a number of diseases such as Diamond-Blackfan
anemia, Treacher Collins syndrome, Shwachman-Diamond syndrome, and
5q-myelodysplastic syndrome (Armistead \& Triggs-Raine, 2014; Draptchinskaia et al., 1999; McGowan et al., 2011; Valdez, Henning, So, Dixon, \& Dixon, 2004; Warren, 2018).
Despite the ubiquitous requirement for ribosomes and translation,
ribosomopathies cause tissue-specific disease (Armistead \& Triggs-Raine, 2014). The
underlying mechanisms of tissue specificity remain unresolved.

In this study we demonstrate that loss of helicases involved in rRNA
processing lead to perturbed ribosome biogenesis and, ultimately, cell
cycle arrest. Given that \emph{Drosophila} germ cells undergo an atypical
cell cycle program as a normal part of their development it may be that
this underlying cellular program in the germline leads to the
tissue-specific symptom of aberrant stem cyst formation
(McKearin \& Spradling, 1990). This model implies that other tissues would
likewise exhibit unique tissue-specific manifestations of
ribosomopathies due to their underlying cell state and underscores the
need to further explore tissue-specific differentiation programs and
development to shed light not only on ribosomopathies but on other
tissue-specific diseases associated with ubiquitous processes. Although
it is also possible that phenotypic differences arise from a common
molecular cause, our data suggests two sources of potential tissue
specificity: 1) tissues express different cohorts of mRNAs, such as
\emph{Non1}, that are sensitive to ribosome levels. For example, we find that
in \emph{Drosophila} macrophages, RNAs that regulate the metabolic state of
macrophages and influence their migration require increased levels of
ribosomes for their translation (Emtenani et al., 2021).
2) p53 activation, as has been previously described, is differentially
tolerated in different developing tissues
(Bowen \& Attardi, 2019; Calo et al., 2018; Jones et al., 2008). Together, both
mechanisms could begin to explain the tissue-specific nature of
ribosomopathies and their link to differentiation.

\textbf{{Acknowledgements}}

We are grateful to all members of the Rangan and Fuchs labs for their
discussion and comments on the manuscript. We also thanks Dr.~Sammons,
Dr.~Marlow and Life Science Editors for their thoughts and comments the
manuscript Additionally, we thank the Bloomington Stock Center, the
Vienna \emph{Drosophila} Resource Center, the BDGP Gene Disruption Project,
and Flybase for fly stocks, reagents, and other resources. P.R. is
funded by the NIH/NIGMS (R01GM111779-06 and RO1GM135628-01), G.F. is
funded by NSF MCB-2047629 and NIH RO3 AI144839, D.E.S. was funded by
Marie Curie CIG 334077/IRTIM and the Austrian Science Fund (FWF) grant
ASI\_FWF01\_P29638S, and A.B is funded by NIH R01GM116889 and American
Cancer Society RSG-17-197-01-RMC.

\textbf{{[}Author Contributions\(\Delta\)}

Conceptualization, E.T.M., P.B., G.F., and P.R.; Methodology, E.T.M.,
P.B., G.F., and P.R.; Investigation, E.T.M., P.B., E.N., R.L., S.S.,
H.Y., T.P., and S.E.; Writing -- Original Draft, E.T.M., D.E.S., and
P.R.; Writing -- Review \& Editing, E.T.M., P.B., D.E.S, A.B., G.F., and
P.R.; Funding Acquisition, G.F. and P.R.; Visualization, E.T.M., E.N.;
Supervision, G.F. and P.R.
\begin{figure}

{\centering \includegraphics[width=6.5 in,height=8.9375 in]{./figure/Ribosome Biogenesis/Ribosome Biogenesis 1} 

}

\caption[test short caption]{test short caption}\label{fig:unnamed-chunk-6}
\end{figure}
\begin{figure}

{\centering \includegraphics[width=6.5 in,height=8.9375 in]{./figure/Ribosome Biogenesis/Ribosome Biogenesis 1S} 

}

\caption[test short caption]{test short caption}\label{fig:unnamed-chunk-7}
\end{figure}
\begin{figure}

{\centering \includegraphics[width=6.5 in,height=8.9375 in]{./figure/Ribosome Biogenesis/Ribosome Biogenesis 2} 

}

\caption[test short caption]{test short caption}\label{fig:unnamed-chunk-8}
\end{figure}
\begin{figure}

{\centering \includegraphics[width=6.5 in,height=8.9375 in]{./figure/Ribosome Biogenesis/Ribosome Biogenesis 2S} 

}

\caption[test short caption]{test short caption}\label{fig:unnamed-chunk-9}
\end{figure}
\begin{figure}

{\centering \includegraphics[width=6.5 in,height=8.9375 in]{./figure/Ribosome Biogenesis/Ribosome Biogenesis 3} 

}

\caption[test short caption]{test short caption}\label{fig:unnamed-chunk-10}
\end{figure}
\begin{figure}

{\centering \includegraphics[width=7 in,height=5.409091 in]{./figure/Ribosome Biogenesis/Ribosome Biogenesis 3S} 

}

\caption[test short caption]{test short caption}\label{fig:unnamed-chunk-11}
\end{figure}
\begin{figure}

{\centering \includegraphics[width=6.5 in,height=8.9375 in]{./figure/Ribosome Biogenesis/Ribosome Biogenesis 4} 

}

\caption[test short caption]{test short caption}\label{fig:unnamed-chunk-12}
\end{figure}
\begin{figure}

{\centering \includegraphics[width=6.5 in,height=8.9375 in]{./figure/Ribosome Biogenesis/Ribosome Biogenesis 4S} 

}

\caption[test short caption]{test short caption}\label{fig:unnamed-chunk-13}
\end{figure}
\begin{figure}

{\centering \includegraphics[width=6.5 in,height=8.9375 in]{./figure/Ribosome Biogenesis/Ribosome Biogenesis 5} 

}

\caption[test short caption]{test short caption}\label{fig:unnamed-chunk-14}
\end{figure}
\begin{figure}

{\centering \includegraphics[width=7 in,height=5.409091 in]{./figure/Ribosome Biogenesis/Ribosome Biogenesis 5S} 

}

\caption[test short caption]{test short caption}\label{fig:unnamed-chunk-15}
\end{figure}
\begin{figure}

{\centering \includegraphics[width=6.5 in,height=8.9375 in]{./figure/Ribosome Biogenesis/Ribosome Biogenesis 6} 

}

\caption[test short caption]{test short caption}\label{fig:unnamed-chunk-16}
\end{figure}
\begin{figure}

{\centering \includegraphics[width=6.5 in,height=8.9375 in]{./figure/Ribosome Biogenesis/Ribosome Biogenesis 6S} 

}

\caption[test short caption]{test short caption}\label{fig:unnamed-chunk-17}
\end{figure}
\begin{figure}

{\centering \includegraphics[width=6.5 in,height=8.9375 in]{./figure/Ribosome Biogenesis/Ribosome Biogenesis 7} 

}

\caption[test short caption]{test short caption}\label{fig:unnamed-chunk-18}
\end{figure}
\begin{figure}

{\centering \includegraphics[width=6.5 in,height=8.9375 in]{./figure/Ribosome Biogenesis/Ribosome Biogenesis 7S} 

}

\caption[test short caption]{test short caption}\label{fig:unnamed-chunk-19}
\end{figure}
\textbf{Figure 2.1: RNA helicases Aramis, Athos and Porthos are required for GSC differentiation.}
(\textbf{A}) Schematic of \emph{Drosophila} germarium.
Germline stem cells are attached to the somatic niche (dark red). The
stem cells divide and give rise to a stem cell and a cystoblast (CB)
that expresses the differentiation factor Bag-of-marbles (Bam). GSCs and
CBs are marked by spectrosomes. The CB undergoes four incomplete mitotic
divisions giving rise to a 16-cell cyst (blue). Cysts are marked by
branched spectrosome structures known as fusomes (red). One cell of the
16-cell cyst is specified as the oocyte. The 16-cell cyst is
encapsulated by the surrounding somatic cells giving rise to an egg
chamber. (\textbf{A'}) Ribosome biogenesis promotes GSC cytokinesis and
differentiation. Disruption of ribosome biogenesis results in
undifferentiated stem cyst accumulation. (\textbf{B}) Conservation of
\emph{aramis}, \emph{athos}, and \emph{porthos} between \emph{H. sapiens}, \emph{D.
melanogaster}, and \emph{S. cerevisiae} (left), trees are drawn to scale,
with branch lengths measured in the number of substitutions per site.
Representation of conserved protein domains for three RNA helicases in
\emph{Drosophila} compared to \emph{H. sapiens} and \emph{S. cerevisiae} orthologs
(right). Percentage values represent similarity to \emph{Drosophila}
orthologs. (\textbf{C}) Egg laying assay after germline RNAi knockdown of
\emph{aramis}, \emph{athos} or \emph{porthos} indicating a loss of fertility compared
to \emph{nosGAL4}, driver control (n=3 trials). *** = p \textless{} 0.001, Tukey's
post-hoc test after one-way ANOVA, p \textless{} 0.001. Error bars represent
standard error (SE). (\textbf{D-G''}) Confocal micrographs of ovaries from
control, \emph{UAS-Dcr2; nosGAL4; bam-GFP} (\textbf{D-D''}) and germline RNAi
depletion targeting (\textbf{E-E''}) \emph{aramis}, (\textbf{F-F''}) \emph{athos} or
(\textbf{G-G''}) \emph{porthos} stained for 1B1 (red, left grayscale), Vasa
(green), and Bam-GFP (blue, right grayscale). Depletion of these genes
(\textbf{E-G''}) results in a characteristic phenotype in which early germ
cells are connected marked by a 1B1 positive, fusome-like structure
highlighted by a yellow dotted line in contrast to the single cells
present in (\textbf{D-D''}) controls (white arrow) or differentiating cysts
(yellow dashed line). Bam expression, if present, is followed by loss of
the germline. (\textbf{H}) Phenotype quantification of ovaries depleted of
\emph{aramis}, \emph{athos} or \emph{porthos} compared to control ovaries (n=50
ovarioles, df=2, *** = p \textless{} 0.001, Fisher's exact tests with
Holm-Bonferroni correction). Scale bars are 15 micron.

\textbf{\hfill\break
}

\textbf{Figure 2.3. Athos, Aramis, and Porthos are required for efficient
ribosome biogenesis.} (\textbf{A-C''}) Confocal images of ovariole
immunostained for Fibrillarin (red, right grayscale), Vasa (blue),
(\textbf{A-A''}) Aramis::GFP, (\textbf{B-B''}) Athos::GFP and (\textbf{C-C''})
Porthos::HA (green, left grayscale). (\textbf{A'''-C'''}) Fluorescence
intensity plot generated from a box of averaged pixels centered around
the punctate of Fibrillarin in the white box. The white box also
indicates a nucleus, while the yellow dotted outline indicates the
divide between soma and germline, with the germline on the interior of
the outline and soma on the exterior. R values denote Spearman
correlation coefficients between GFP and Fibrillarin from plot profiles
generated using Fiji, taken from the nucleolus denoted by the white box.
Aramis, Athos and Porthos are expressed throughout oogenesis and
localize to the nucleolus. Aramis is also present in the cytoplasm.
(\textbf{D-D''}) RNA IP-seq of (\textbf{D}) Aramis, (\textbf{D'}) Athos, and (\textbf{D''})
Porthos aligned to rDNA displayed as genome browser tracks. Bar height
represents log scaled rRNA reads mapping to rDNA normalized to input and
spike-in. Grey boxes outline rRNA precursors that are significantly
enriched in the IP compared to the IgG control (bootstrapped paired
t-tests, n=3, * = p-value \textless{} 0.05). (\textbf{E-E''}) Polysome traces from
\emph{Drosophila} S2 cells treated with dsRNA targeting (\textbf{E}) \emph{aramis},
(\textbf{E'}) \emph{athos}, (\textbf{E''}) \emph{porthos} (red line) compared to a mock
transfection control (black line). \emph{aramis} and \emph{porthos} are required
to maintain a proper 40S/60S ribosomal subunit ratio compared to control
and have a smaller 40S/60S ratio. \emph{athos} is required to maintain a
proper 40S/60S ribosomal subunit ratio compared to control and has a
larger 40S/60S ratio. Additionally, \emph{aramis}, \emph{athos}, and \emph{porthos} are
required to maintain polysome levels. All three helicases are required
to maintain polysome levels. Scale bar for all images is 15 micron.

\textbf{\hfill\break
}

\textbf{Figure 2.5. Athos, Aramis, and Porthos are required for cell cycle
progression during early oogenesis.} (\textbf{A}) Bar plot representing the
most significant Biological Process GO terms of downregulated genes in
ovaries depleted of \emph{aramis} compared to \emph{bam} RNAi control (FDR = False
Discovery Rate from p-values using a Fisher's exact test). (\textbf{B-C})
Genome browser tracks representing the gene locus of (\textbf{B}) \emph{CycB} and
(\textbf{C}) a\emph{urora B} in ovaries depleted of \emph{aramis} compared to the
developmental control, \emph{bam} RNAi. Y-axis represents the number of reads
mapping to the locus in bases per million (BPM). (\textbf{D-E''}) Confocal
images of germaria stained for CycB (red, left grayscale) and Vasa
(blue, right grayscale) in (\textbf{D-D'''}) \emph{bam} RNAi control ovaries and
(\textbf{E-E''}) \emph{aramis} germline RNAi. (\textbf{F}) Boxplot of CycB intensity in
the germline normalized to Cyc B intensity in the soma in \emph{bam} RNAi and
\emph{aramis} RNAi (n=12-14 germaria per sample, *** = p\textless{} 0.001, Welch
t-test. (\textbf{G-H''}) Confocal images of germaria stained for p53 (red,
left grayscale), GFP (green), and Vasa (blue, right grayscale) in
(\textbf{G-G''}) \emph{nosGAL4}, driver control ovaries and (\textbf{H-H''}) germline
depletion of \emph{aramis}. Cells highlighted by a dashed yellow circle
represent cell shown in the inset. Driver control \emph{nosGAL4} ovaries
exhibit attenuated p53 expression in GSCs and CBs, but higher expression
in cyst stages as previously reported, while p53 punctate are visible in
the germline of \emph{aramis} RNAi in the undifferentiated cells. (\textbf{I}) Box
plot of percentage of pixel area exceeding the background threshold for
p53 in GSCs and CBs in driver control \emph{nosGAL4} ovaries and the germline
of \emph{aramis} RNAi indicates p53 expression is elevated in the germline
over the GSCs/CBs of control ovaries. (n=10 germaria per sample, ***
= p \textless{} 0.001, Welch's t-test. (\textbf{J-K''}) Confocal images of germaria
stained for 1B1 (red, left grayscale) and Vasa (blue, right grayscale)
in (\textbf{J-J''}) germline \emph{aramis} RNAi in a wild type background and
(\textbf{K-K''}) germline \emph{aramis} RNAi with a mutant, null, \emph{p53\textsuperscript{5-A-14}}
background showing presence of spectrosomes upon loss of p53. (\textbf{L})
Quantification of stem cyst phenotypes demonstrates a significant rescue
upon of loss of \emph{p53\textsuperscript{5-A-14}} in \emph{aramis} germline depletion compared to
the wild type control (n=43-55 germaria per genotype, df=2, Fisher's
exact test p\textless{} 0.05). (\textbf{M-N}) Confocal images of ovaries stained for
1B1 (red) and Vasa (blue) in \emph{nosGAL4} ovaries (\textbf{M}) and ovaries
overexpressing p53 in the germline (\textbf{N}). We observed that 84\% of
germaria from p53 overexpression ovaries had loss of germline while
12.7\% of germaria contained a cyst, marked by an extended spectrosome
structure connecting germline cells together, and an additional 3.6\% of
germaria contained an accumulation of single cells (n=55 germaria).
Scale bar for main images is 15 micron, scale bar for insets is 3.75
micron.

\textbf{\hfill\break
}

\textbf{Figure 2.7.} \textbf{Aramis is required for efficient translation of a subset
of mRNAs.} (\textbf{A-A''}) Biplots of poly(A)+ mRNA Input versus polysome
associated mRNA from (\textbf{A}) ovaries genetically enriched for GSCs
(\emph{UAS-tkv}), (\textbf{A'}) Undifferentiated GSC daughter cells (\emph{bam} RNAi)
or (\textbf{A''}) germline \emph{aramis} RNAi ovaries. (\textbf{B}) Boxplot of
translation efficiency of target genes in \emph{UAS-tkv}, \emph{bam} RNAi, and
\emph{aramis} RNAi samples (ANOVA p\textless0.001, post-hoc Welch's t-test, n=87,
*** = p \textless{} 0.001). (\textbf{C}) Summary of downregulated target genes
identified from polysome-seq. (\textbf{D-E'}) Confocal images of germaria
stained for 1B1 (red), RpS2::GFP (green, grayscale), and Vasa (blue) in
(\textbf{D-D'}) \emph{bam} RNAi control and (\textbf{E-E'}) \emph{aramis} RNAi (yellow
dashed line marks approximate region of germline used for
quantification). (\textbf{F}) A.U. quantification of germline RpS2::GFP
expression normalized to RpS2::GFP expression in the surrounding soma in
undifferentiated daughter cells of \emph{bam} RNAi compared to \emph{aramis} RNAi.
RpS2::GFP expression is significantly lower in \emph{aramis} RNAi compared to
control (n=14 germaria per sample, Welch's t-test, *** = p \textless{} 0.001).
(\textbf{G-I'}) Confocal images of germaria stained for 1B1 (red), OPP
(green, grayscale), and Vasa (blue) in (\textbf{G-G'}) \emph{nosGAL4}, (\textbf{H-H'})
\emph{bam} RNAi, and (\textbf{H-H'}) \emph{aramis} RNAi (yellow dashed line marks
approximate region of germline used for quantification). (\textbf{I}) A.U.
quantification of OPP intensity in single cells of nosGAL4 control
germeria and undifferentiated daughter cells in \emph{bam} RNAi as controls
and \emph{aramis} RNAi (n = 10 germaria per genotype, Welch's t-test, NS = p
\textgreater{} 0.05). OPP intensity is not significantly downregulated in \emph{aramis}
RNAi compared to either control. Scale bar for all images is 15 micron.

\textbf{\hfill\break
}

\textbf{Figure 2.9. Non1 represses p53 expression to allow for GSC
differentiation.} (\textbf{A-A'}) Confocal images of Non1::GFP germaria
stained for 1B1 (red), GFP (green, grayscale), and Vasa (blue).
(\textbf{A''}) Boxplot of Non1::GFP expression over germline development in
GSCs, CBs and Cyst (CC) stages (n=5-25 cysts of each type, * = p \textless{}
0.05, ** = p \textless{} 0.01, ANOVA with Welch's post-hoc tests). (\textbf{B-C'})
Confocal images of (\textbf{B-B'}) \emph{bam} RNAi and (\textbf{C-C'}) \emph{aramis} RNAi
germaria both carrying \emph{Non1::GFP} transgene stained for 1B1 (red), Vasa
(blue), and Non1::GFP (green, grayscale). Yellow dashed line marks
region of germline used for quantification. (\textbf{D}) Boxplot of Non1::GFP
expression in the germline normalized to somatic Non1::GFP expression in
\emph{bam} RNAi and \emph{aramis} RNAi (n=24 germaria per genotype, Welch's
t-test, *** = p \textless{} 0.001). Non1 expression is significantly lower in
the germline of \emph{aramis} RNAi compared to \emph{bam} RNAi control. (\textbf{E-G'})
Confocal images of germaria stained for 1B1 (red, grayscale), and Vasa
(blue) in (\textbf{E-E'}) \emph{nosGAL4}, driver control ovaries, (\textbf{F-F'})
germline \emph{Non1} RNAi, and (\textbf{G-G'}) germline \emph{Non1} RNAi in a
\emph{p53\textsuperscript{5-A-1-4}} background. Arrow marks the presence of a single cell
(\textbf{E, G}), yellow dashed line marks a stem cyst emanating from the
niche (\textbf{F-F'}) or the presence of proper cysts (\textbf{E-E'}). (\textbf{H})
Quantification of percentage of germaria with no defect (black),
presence of single cell (salmon), presence of a stem cyst emanating from
the niche (brown-red), or germline loss (dark red) demonstrates a
significant rescue of stem cyst formation upon of loss of \emph{Non1} in
\emph{p53\textsuperscript{5-A-14}} compared to the \emph{p53} wild type control (n=35-55 germaria
per genotype, df=3, Fisher's exact test with Holm-Bonferroni correction
** = p\textless{} 0.01, *** = p\textless{} 0.001). (\textbf{I-K'}) Confocal images of
germaria stained for 1B1 (red, grayscale), and Vasa (blue) in ovaries
with (\textbf{I-I'}) germline Non1 overexpression, (\textbf{J-J'}) \emph{aramis}
germline RNAi exhibiting stem cyst phenotype (yellow dashed line) and
(\textbf{K-K}') \emph{aramis} germline RNAi with \emph{Non1} overexpression exhibiting
single cells (arrow). (\textbf{L}) Phenotypic quantification of \emph{aramis} RNAi
with \emph{Non1} overexpression demonstrates a significant alleviation of the
stem cyst phenotype (n=33-57 germaria per genotype, df=2, Fisher's exact
test, ** = p\textless{} 0.01). Scale bar for all images is 15 micron.

\textbf{\hfill\break
}

\textbf{Figure 2.11. Aramis regulated mRNAs contain a TOP motif}. (\textbf{A}) Genome
browser tract of \emph{RpL30} locus in ovary CAGE-seq data showing the
proportion of transcripts that are produced from a given TSS (orange).
Predominant TSSs are shown in orange and putative TOP motif indicated
with a green box. The bottom blue and red graph represents sequence
conservation of the locus across \emph{Diptera}. The dominant TSS initiates
with a canonical TOP motif. (\textbf{B}) Sequence logo generated from \emph{de
novo} motif discovery on the first 200 bases downstream of CAGE derived
TSSs of \emph{aramis} translation target genes resembles a canonical TOP
motif. (\textbf{C}) Histogram representing the location of the first 5-mer
polypyrimidine sequence from each CAGE based TSS of \emph{aramis} translation
target genes demonstrates that the TOP motifs occur proximal to the TSS
(n=76 targets). (\textbf{D}) Diagram of the \emph{WT} and \emph{Mut-TOP-GFP} reporter
constructs indicating the TOP sequence that is mutated by transversion
in the Mutant reporter (blue). (\textbf{D-E''}) Confocal images and
quantifications of (\textbf{E-E'}) \emph{WT-TOP-GFP} and (\textbf{F-F'}) \emph{Mut-TOP-GFP}
reporter expression stained for 1B1 (red), GFP (green, grayscale), and
Vasa (blue). Yellow dotted-line marks increased reporter expression in
8-cell cysts of \emph{WT-TOP-GFP} but not in \emph{Mut-TOP-GFP}. Reporter
expression was quantified over germline development for (\textbf{E''})
\emph{WT-TOP-GFP} and (\textbf{F''}) \emph{Mut-TOP-GFP} reporter expression and
normalized to expression in the GSC reveals dynamic expression based on
the presence of a TOP motif. (\textbf{G-H'}) Confocal images of \emph{WT-TOP-GFP}
reporter ovarioles showing 1B1 (red), GFP (green, grayscale), and Vasa
(blue) in (\textbf{G-G'}) \emph{bam} germline depletion as a developmental control
and (\textbf{H-H'}) \emph{aramis} germline depleted ovaries. Yellow dotted lines
indicate germline. (\textbf{I-J'}) Confocal images of \emph{Mut-TOP-GFP} reporter
expression showing 1B1 (red), GFP (green, grayscale), and Vasa (blue) in
(\textbf{I-I'}) \emph{bam} RNAi and (\textbf{J-J'}) \emph{aramis} germline RNAi. Yellow
dotted lines indicate germline. (\textbf{K}) A.U. quantification of WT and
Mutant TOP reporter expression in undifferentiated daughter cells in
\emph{bam} RNAi compared \emph{aramis} RNAi demonstrates that the \emph{WT-TOP-GFP}
reporter shows significantly lower expression in \emph{aramis} RNAi than the
\emph{Mut-TOP-GFP} relative to the expression of the respective reporters in
\emph{bam} RNAi indicating that the presence of a TOP motif sensitizes
transcripts to regulation (n=17-25 germaria per genotype, with Welch's
t-test *** = p\textless0.001). Scale bar for all images is 15 micron.

\textbf{\hfill\break
}

\textbf{Figure 2.13. Larp binds to TOP mRNAs and binding is regulated by
Aramis.} (\textbf{A-A'}) EMSA of Larp-DM15 and the leading 42 nucleotides of
(\textbf{A}) \emph{RpL30} and (\textbf{A'}) \emph{Non1} with increasing concentrations of
Larp-DM15 from left to right indicates that both RNAs bind to Larp-DM15.
(\textbf{B}) Volcano plot of mRNAs in Larp::GFP::3XFLAG IP compared to input.
Blue points represent mRNAs significantly enriched in Larp::GFP::3XFLAG
compared to input, but not enriched in an IgG control compared to input.
(\textbf{C}) Venn diagram of overlapping Larp IP targets and \emph{aramis} RNAi
polysome seq targets indicates that Larp physically associates with
mRNAs that are translationally downregulated in germline \emph{aramis} RNAi
(p \textless{} 0.001, Hypergeometric Test). (\textbf{D}) Bar plot representing the
fold enrichment of mRNAs from Larp RNA IP in germline \emph{aramis} RNAi
relative to matched \emph{bam} RNAi ovaries as a developmental control
measured with qPCR (n=3, * = p\textless0.5, ** = p\textless0.01, NS =
nonsignificant, One-sample t-test, mu=1) indicates that more of two
\emph{aramis} translation targets \emph{Non1} and \emph{RpL30} are bound by Larp in
\emph{aramis} RNAi. (\textbf{E-F''}) Confocal images of (\textbf{E-E''}) \emph{nosGAL4},
driver control and (\textbf{F-F''}) ovaries overexpressing the DM15 region of
Larp in the germline ovaries stained for 1B1 (red, left grayscale), Vasa
(blue), and Larp-DM15::GFP (green, right grayscale). Overexpression of
Larp results in an accumulation of extended 1B1 structures (highlighted
with a dotted yellow line), marking interconnected cells when Larp-DM15
is overexpressed compared to \emph{nosGAL4}, driver control ovaries. (\textbf{G})
In conditions with normal ribosome biogenesis Non1 is efficiently
translated, downregulating p53 levels allowing for progression through
the cell cycle. When ribosome biogenesis is perturbed Non1 is not
translated to sufficient levels, resulting in the accumulation of p53
and cell cycle arrest. Scale bar for all images is 15 micron.

\textbf{\hfill\break
}

\textbf{Figure 2.2. Aramis, Athos, and Porthos are required for
proper cytokinesis and differentiation, related to Figure 2.1.} (\textbf{A})
qPCR targeting \emph{aramis}, \emph{athos}, \emph{porthos} relative to \emph{±-tub84B} to
measure knockdown efficiency of \emph{aramis}, \emph{athos}, and \emph{porthos} RNAi in
the germline (n=3, Welch's t-test with Holm-Bonferroni correction, *
=p\textless0.05, **=p\textless0.01). (\textbf{B-B'}) Confocal images of (\textbf{B})
Aramis::GFP::3XFLAG and (\textbf{B'}) Aramis::GFP::3XFLAG in conjunction with
\emph{aramis} germline knockdown stained for Vasa (blue), and GFP (green).
(\textbf{C-C'}) Confocal images of (\textbf{C}) Athos::GFP::3XFLAG and (\textbf{C'})
Athos::GFP::3XFLAG in conjunction with \emph{athos} germline knockdown
stained for Vasa (blue), and GFP (green). (\textbf{D}) Quantification of
germline expression of GFP normalized to somatic GFP expression in
Aramis::GFP::3XFLAG ovaries compared to Aramis::GFP::3XFLAG in
conjunction with \emph{aramis} germline knockdown and Athos::GFP::3XFLAG
ovaries compared to Athos::GFP::3XFLAG in conjunction with \emph{athos}
germline knockdown. (n=14-15, Welch's t-test with Holm-Bonferroni
correction, ***=p\textless0.001) (\textbf{E-E'''}) Confocal images of (\textbf{E})
\emph{nosGAL4}, driver control and germline RNAi knockdown using additional
RNAi lines for (\textbf{E'}) \emph{aramis} and (\textbf{A''}) \emph{athos} stained for 1B1
(red) and Vasa (green). (\textbf{E'''}) Quantification of percentage of
germaria with no defect (black), stem cysts (salmon), or germline loss
(dark red) in ovaries depleted of \emph{athos}, \emph{aramis}, or \emph{porthos}
compared to control ovaries recapitulates the phenotypes with
independent RNAi lines (n=50, df=2, *** = p\textless0.001, Fisher's exact
test with Holm-Bonferroni correction). (\textbf{F-F'''}) Confocal images of
germaria stained for 1B1 (red) and Phospho-tyrosine (green). Ring
canals, marked by Phosopho-tyrosine, connect differentiating cysts in
(\textbf{F}) control \emph{nosGAL4} ovaries and in between the interconnected
cells of ovaries depleted of (\textbf{F'}) \emph{athos}, (\textbf{F''}) \emph{aramis}, or
(\textbf{F'''}) \emph{porthos} with 1B1 positive structures going through the ring
canals. (\textbf{G-G'''}) Confocal images of ovarioles stained for 1B1 (red)
and Vasa (green). Control, \emph{nosGAL} ovaries have egg chambers while
ovaries depleted of (\textbf{G'}) \emph{athos}, (\textbf{G''}) \emph{aramis}, or (\textbf{G'''})
\emph{porthos} lack egg chambers. (\textbf{H-H'''}) Confocal images of germaria
stained for pMad (red, grayscale) and Vasa (green). In (\textbf{H}) control
ovaries nuclear pMad staining occurs in cells proximal to the niche
marking GSCs. Nuclear pMad staining in ovaries depleted of (\textbf{H'})
\emph{athos}, (\textbf{H''}) \emph{aramis}, and (\textbf{H'''}) \emph{porthos} demonstrates that
the observed cysts are not composed of GSCs. Scale bar for main images
is 15 micron, scale bar for insets is 3.75 micron.

\textbf{\hfill\break
}

\textbf{Figure 2.4. Athos, Aramis, and Porthos are required for
efficient ribosome biogenesis., related to Figure 2.3.} (\textbf{A-A''})
Western blots of immunoprecipitations from ovaries for FLAG-tagged
(\textbf{A}) Aramis, (\textbf{A'}) Athos, , and (\textbf{A''}) Porthos. (\textbf{B-B'''})
Confocal images of (\textbf{B}) \emph{nosGAL4}, driver control, (\textbf{B'}) \emph{aramis}
(\textbf{B''}) \emph{athos} and (\textbf{B'''}) \emph{porthos} germline RNAi germaria
stained for Fibrillarin (green), and Vasa (blue). (\textbf{C-C'})
Quantification of raw nucleolar volume (\textbf{C}) or normalized to cell
volume nucleolar volume (\textbf{C'}) in GSCs of \emph{aramis}, \emph{athos}, and
\emph{porthos} RNAi, compared to control indicates loss of each helicase
results in nucleolar stress. (n=12-16 GSCs per genotype, One-way ANOVA,
p\textless0.001, with Welch's t-test, Holm-Bonferroni corrected, * = p\textless0.05,
** = p\textless0.01, *** = p\textless0.001). (\textbf{D-D'}) Polysome preparations
from \emph{Drosophila} S2 cells in cells treated with dsRNA targeting (\textbf{D})
\emph{RpS19a} or (\textbf{D'}) \emph{RpL30} (red line) compared to a mock transfection
control (black line). (\textbf{E-K}) Confocal images of heterozygous (\textbf{E})
\emph{shrb}\textsuperscript{O3}/+, (\textbf{F}) \emph{aramis}\textsuperscript{f06152}/+ mutant ovaries, (\textbf{G})
\emph{shrb}\textsuperscript{O3}/+; \emph{aramis}\textsuperscript{f06152}/+ transheterozygous, (\textbf{J'}) \emph{athos}
deficiency/+ mutant ovaries,(\textbf{J''}) \emph{shrb}\textsuperscript{O3}\emph{/athos} deficiency
transheterozygous, heterozygous (\textbf{K}) \emph{shrb}\textsuperscript{O3}/+, (\textbf{K'})
\emph{porthos}\textsuperscript{KG05120}/+ mutant ovaries and (\textbf{K''})
\emph{shrb}\textsuperscript{O3}\emph{/porthos}\textsuperscript{KG05120} transheterozygous ovaries stained for 1B1
(red) and Vasa (blue). (\textbf{L}) Quantification of proportion of germaria
with presence of a stem-cyst or single cells from the indicated
genotypes (pairwise Fisher's tests, Holm-Bonferroni corrected, * =
p\textless0.05, ** = p\textless0.01, *** = p\textless0.001). Scale bar for main images
is 15 micron, scale bar for insets is 3.75 micron.

\textbf{\hfill\break
}

\textbf{Figure 2.6}. \textbf{Aramis is required for proper cell cycle
progression, related to Figure 2.5}. (\textbf{A}) Biplot of mRNA levels in
\emph{aramis} RNAi compared to \emph{bam} RNAi. Blue points represent mRNAs
significantly upregulated \emph{aramis} RNAi compared to \emph{bam} RNAi, red
points represent mRNAs significantly downregulated \emph{aramis} RNAi
compared to \emph{bam} RNAi. (\textbf{B}) Bar plot representing the most
significant Biological Process GO terms of upregulated genes in ovaries
depleted of \emph{aramis} compared to the developmental control, \emph{bam} RNAi.
(\textbf{C-C'}) Genome browser tracks of mRNA levels at the (\textbf{C}) \emph{Cyclin}
\emph{A} and (\textbf{C'}) \emph{Actin 5C} loci indicate that the RNAseq target gene
\emph{Cyclin A} expression is downregulated, while a non-target, \emph{Actin 5C}
is not downregulated. (\textbf{D-E'}) Confocal images of germaria stained for
1B1 (red), Vasa (blue), and Cyclin B::GFP (green, grayscale) in
(\textbf{D-D'}) control and (\textbf{E-E'}) germline depletion of \emph{aramis}
demonstrates that functional Cyclin B::GFP cannot be efficiently
expressed in germline depleted of \emph{aramis}. (\textbf{F-H''}) Confocal images
of germaria that express Fly-FUCCI in the germline stained for Vasa
(blue). GFP-E2f1\textsuperscript{degron} (green, right grayscale) and RFP-CycB\textsuperscript{degron}
(red, left grayscale) in (\textbf{F-F''}) \emph{nosGAL4}, driver control ovaries,
(\textbf{G-G''}) \emph{bam} RNAi as a developmental control, and (\textbf{H-H''})
ovaries with germline depletion \emph{of} \emph{aramis} demonstrates that the
germline of \emph{aramis} RNAi germline depleted ovaries are negative for
both G1 and G2 cell cycle markers. (\textbf{I-I'}) Confocal images of
\emph{aramis} germline RNAi expressing GFP in the germline, stained for 1B1
(red), Vasa (blue), and GFP (green, grayscale) indicates productive
translation of transgenes still occurs. (\textbf{J-M}) Confocal images of
germaria stained for p53 (red) and Vasa (blue) in (\textbf{J}) hybrid
dysgenic, Harwich, ovaries and (\textbf{K}) p53\textsuperscript{11-B-1} ovaries stained for
p53 (red) and Vasa (blue) demonstrate the expected p53 staining
patterns. (\textbf{L-M}) Confocal images of germaria stained for p53 (red)
and Vasa (blue) in ovaries depleted of (\textbf{L}) \emph{athos} or (\textbf{M})
\emph{porthos} in the germline exhibit p53 punctate staining. Cells
highlighted by a dashed yellow circle represent cells shown in the
inset. (\textbf{N-O}) Confocal images of germaria stained for 1B1 (red) and
Vasa (blue) in (\textbf{N}) germline \emph{athos} RNAi in a wild type background
and (\textbf{O}) germline \emph{athos} RNAi with a mutant, null, \emph{p53\textsuperscript{5-A-14}}
background showing presence of spectrosomes upon loss of p53. (\textbf{P-Q})
Confocal images of germaria stained for 1B1 (red) and Vasa (blue) in
(\textbf{P}) germline \emph{porthos} RNAi in a wild type background and (\textbf{Q})
germline \emph{porthos} RNAi with a mutant, heterozygous, \emph{p53\textsuperscript{5-A-14}}/+
background showing presence of spectrosomes upon reduction of p53. Scale
bar for main images is 15 micron, scale bar for insets is 3.75 micron.

\textbf{\hfill\break
}

\textbf{Figure 2.8. The mRNA levels of Aramis polysome-seq targets
are not significantly changing, related to Figure 2.7.} (\textbf{A-A'})
Volcano plot of mRNA levels from poly(A)+ mRNA input libraries in
germline \emph{aramis} RNAi compared to (\textbf{A}) germline driven \emph{UAS-tkv}
and (\textbf{A'}) \emph{bam} RNAi of targets identified from polysome-seq. No
target genes identified from polysome-seq meet the differential
expression cutoff for mRNA in \emph{UAS-tkv} compared to \emph{aramis} RNAi or
\emph{bam} RNAi compared to \emph{aramis} RNAi input libraries. (\textbf{B-C'})
Confocal images of germaria stained for 1B1 (red) RpS2::GFP (green,
greyscale) and Vasa (blue) in (\textbf{B-B'}) germline \emph{bam} RNAi as a
control and (\textbf{C-C'}) germline \emph{bam} RNAi; \emph{aramis} RNAi. Yellow dotted
outline denotes the germline on the interior of the outline with the
soma on the exterior of the outline. (\textbf{D}) Quantification of RpS2::GFP
expression in the germline normalized to the soma in germline \emph{bam} RNAi
compared to \emph{bam} RNAi; \emph{aramis} RNAi demonstrates a significant
decrease in RpS2::GFP expression when \emph{aramis} is knocked down in a
germline \emph{bam} RNAi background (Welch's t-test, *** = p\textless0.001,
n=15). (\textbf{E-G'}). Confocal images of germaria stained for 1B1 (red,
greyscale) and Vasa (blue) in (\textbf{E-E'}) driver control \emph{nosGAL4}
ovaries, and ovaries overexpressing (\textbf{F}) RpL10ab::GFP, or (\textbf{G})
RpL3 have no obvious phenotypic difference.

\textbf{\hfill\break
}

\textbf{Figure 2.10. Non1 and p53 are inversely expressed, related
to Figure 2.9.} (\textbf{A-A''}) Confocal images of ovarioles expressing
Non1::GFP stained for p53 (red, right grayscale), Vasa (blue), and
Non1::GFP (green, left grayscale). (\textbf{B-B'}) Quantifications of
staining, (\textbf{B}) peak Non1 expression in control ovaries occurs in
GSC-4 cell cyst stages and 16-cell cyst-region 2b stages where (\textbf{B'})
p53 expression is low. (\textbf{C-E'}) Confocal images of germline (\textbf{C-C'})
\emph{bam} RNAi, (\textbf{D-D'}) germline \emph{athos} RNAi, or (\textbf{E-E'}) germline
\emph{porthos} RNAi germaria stained for 1B1 (red), \emph{Non1::GFP} (green,
greyscale), and Vasa (blue). (\textbf{F}) Quantification of Non1:GFP
expression in germline \emph{athos, or porthos} RNAi compared to germline
\emph{bam} RNAi as a developmental control demonstrates there is
significantly lower Non1::GFP expression in \emph{athos, or porthos} RNAi
compared to \emph{bam} RNAi (n=10 germaria per genotype, Welch's t-test,
*** = p\textless0.001) (\textbf{G-H'}) Confocal images of (\textbf{G-G'}) \emph{nosGAL4},
driver control and germline (\textbf{H-H'}) \emph{Non1} RNAi germaria stained for
p53 (red, grayscale) and Vasa (blue). (\textbf{I}) Quantification of p53
punctate area above cutoff are markedly brighter in the germline of
\emph{Non1} RNAi depleted ovaries compared to the control (n=10 germaria per
genotype, Welch's t-test, * = p\textless0.05). Cells highlighted by a dashed
yellow circle represent cells shown in the inset. (\textbf{I-J'}) Confocal
images of germaria stained for 1B1 (red, grayscale), and Vasa (blue) in
(\textbf{I-I'}) \emph{aramis} germline RNAi exhibiting stem cyst phenotype (yellow
dashed line) and (\textbf{J-J}') \emph{aramis} germline RNAi with \emph{Non1}
overexpression exhibiting single cells (arrow). (\textbf{N}) Phenotypic
quantification of germline \emph{athos} RNAi or \emph{porthos} RNAi with \emph{Non1}
overexpression demonstrates a significant alleviation of the stem cyst
phenotype compared to germline \emph{athos} or \emph{porthos} RNAi alone (n=26-33
germaria per genotype, df=2, Fisher's exact test, * = p\textless{} 0.05, ** =
p\textless{} 0.01). Scale bar for main images is 15 micron, scale bar for insets
is 3.75 micron.

\textbf{\hfill\break
}

\textbf{Figure 2.12. TORC1 activity regulates TOP expression in the
germarium, related to Figure 2.11.} (\textbf{A-B}) Confocal images of \emph{WT-TOP}
reporter expression stained for 1B1 (red), GFP (green), and Vasa (blue)
in (\textbf{A}) \emph{nosGAL4}, driver control ovaries and (\textbf{B}) ovaries
depleted of \emph{Nprl3} in the germline. (\textbf{C-D}) Confocal images of
\emph{Mut-TOP-GFP} reporter expression stained for 1B1 (red), GFP (green,
grayscale), and Vasa (blue) in (\textbf{C}) \emph{nosGAL4}, driver control ovaries
and (\textbf{D}) ovaries depleted of \emph{Nprl3} in the germline. (\textbf{E}) A.U.
quantification of WT and Mutant TOP reporter expression in GSCs of
\emph{nosGAL4}, driver control ovaries and GSCs of \emph{Nprl3} germline depleted
ovaries normalized to Vasa expression indicate that the relative
expression of the \emph{WT-TOP-GFP} reporter is higher than the \emph{Mut-TOP-GFP}
reporter (n=9-11 germaria per genotype, Welch's t-test, * = p\textless0.05,
** = p\textless0.01, *** = p\textless0.001). (\textbf{F-G}) Confocal images of
\emph{WT-TOP} reporter expression stained for 1B1 (red), GFP (green), and
Vasa (blue) in (\textbf{F}) \emph{nosGAL4}, driver control ovaries and (\textbf{G})
ovaries depleted of \emph{raptor} in the germline. (\textbf{H-I}) Confocal images
of \emph{Mut-TOP-GFP} reporter expression stained for 1B1 (red), GFP (green),
and Vasa (blue) in (\textbf{H}) \emph{nosGAL4}, driver control ovaries and (\textbf{I})
ovaries depleted of \emph{raptor} in the germline. (\textbf{J}) A.U.
quantification of WT and Mutant TOP reporter expression in GSCs of
\emph{nosGAL4}, driver control ovaries and GSCs of \emph{raptor} germline depleted
ovaries normalized to Vasa expression indicate that the relative
expression of the \emph{WT-TOP-GFP} reporter is lower than the \emph{Mut-TOP-GFP}
reporter (n=10 germaria per genotype, Welch's t-test, * = p\textless0.05, **
= p\textless0.01). (\textbf{K-M'}) Confocal images of \emph{WT-TOP-GFP} reporter
ovarioles showing 1B1 (red), GFP (green, grayscale), and Vasa (blue) in
(\textbf{K-K'}) \emph{bam} germline depletion as a developmental control,
(\textbf{L-L'}) \emph{athos} germline depleted ovaries, and (\textbf{M-M'}) \emph{porthos}
germline depleted ovaries. Yellow dotted lines indicate germline.
(\textbf{N-P'}) Confocal images of \emph{Mut-TOP-GFP} reporter expression showing
1B1 (red), GFP (green, grayscale), and Vasa (blue) in (\textbf{N-N'}) \emph{bam}
RNAi, (\textbf{O-O'}) \emph{athos} germline RNAi, and (\textbf{P-P'}) \emph{porthos}
germline depleted ovaries. Yellow dotted lines indicate germline.
(\textbf{J}) A.U. quantification of WT and Mutant TOP reporter expression in
undifferentiated daughter cells in \emph{bam} RNAi compared \emph{athos} or
\emph{porthos} RNAi demonstrates that the \emph{WT-TOP-GFP} reporter shows
significantly lower expression in \emph{athos} and \emph{porthos} RNAi than the
\emph{Mut-TOP-GFP} relative to the expression of the respective reporters in
\emph{bam} RNAi indicating that the presence of a TOP motif sensitizes
transcripts to regulation (n=17-25 germaria per genotype, with Welch's
t-test with , ** = p\textless0.01, *** = p\textless0.001). Scale bar for images
is 15 micron.

\textbf{\hfill\break
}

\textbf{Figure 2.14. Larp binds specifically to TOP containing mRNAs
and regulates cytokinesis, related to Figure 2.13.} (\textbf{A-A'}) Confocal
images of germaria stained for 1B1 (red), Vasa (blue), and \emph{Larp
GFP-3XFLAG} (green, grayscale) indicates Larp is expressed throughout
early oogenesis. (\textbf{B}) Quantification of EMSAs and summary of K\textsubscript{d} of
the protein-RNA interactions. (\textbf{C-C'}) EMSA of Larp-DM15 and the
leading 42 nucleotides of (\textbf{B}) \emph{RpL30} and (\textbf{B'}) \emph{Non1} with their
TOP sequence mutated to purines as a negative control with increasing
concentrations of Larp-DM15 from left to right indicates that Larp-DM15
requires a leading TOP sequence for its binding. (\textbf{D}) Western of
representative IP of Larp::GFP::FLAG from ovary tissue used for RNA
IP-seq. (\textbf{E-F'}) Confocal images of \emph{Larp::GFP::FLAG} reporter
expression stained for 1B1 (red), GFP (green, grayscale), and Vasa
(blue) in (\textbf{E-E'}) \emph{bam} and (\textbf{F-F'}) \emph{aramis} depleted germaria.
(\textbf{G}) A.U. quantification of Larp::GFP::FLAG reporter expression in
the germline of \emph{bam} RNAi and \emph{aramis} RNAi normalized to germline Vasa
intensity demonstrates that the germline expression of Larp is not
elevated in \emph{aramis} germline RNAi compared to \emph{bam} germline RNAi as a
developmental control (n=10, NS = p\textgreater0.05, Welch's t-test). (\textbf{H-H'})
Confocal images of \emph{Larp::GFP::FLAG} reporter expression stained for 1B1
(red), GFP (green), and Vasa (blue) in (\textbf{H}) \emph{bam} and (\textbf{H'}) \emph{bam};
\emph{aramis} depleted germaria. (\textbf{G}) A.U. quantification of
Larp::GFP::FLAG reporter expression in the germline of \emph{bam} RNAi and
\emph{bam} RNAi ; \emph{aramis} RNAi demonstrates that the germline expression of
Larp normalized to somatic Larp expression is not elevated in \emph{bam};
\emph{aramis} germline RNAi compared to \emph{bam} germline RNAi as a
developmental control (n=10, NS = p\textgreater0.05, Welch's t-test). (\textbf{J})
Western of representative IP of Larp::GFP::FLAG from ovary tissue used
for RNA IP qPCR. (\textbf{K-K'}) Confocal images of (\textbf{K}) \emph{nosGAL4}, driver
control and (\textbf{K'}) ovaries overexpressing the DM15 region of Larp in
the germline ovaries stained for DAPI (green) and Vasa (blue).
Overexpression of Larp-DM15 results in the production of 32-cell egg
chambers which indicates it causes a cytokinesis defect. Scale bar for
all images is 15 micron.

\textbf{\hfill\break
}

\textbf{Supplemental Table 2.1. Results of germline helicase RNAi screen on ovariole morphology.}\\
Results of screen of RNA helicases depleted from the germline. Reported
is the majority phenotype from n=50 ovarioles.

\textbf{\hfill\break
}

\textbf{Supplemental Table 2.2. Differential expression analysis fromRNAseq of ovaries depleted of \emph{aramis} in the germline compared to a developmental control.}
DEseq2 output from RNAseq of ovaries depleted
of \emph{aramis} in the germline compared to ovaries depleted of bam in the
germline as a developmental control. Sheet 1 (Downregulated Genes)
contains genes and corresponding Deseq2 output meeting the cutoffs to be
considered downregulated in \emph{aramis} RNAi compared to bam RNAi. Sheet 2
(Upregulated Genes) contains genes and corresponding Deseq2 output
meeting the cutoffs to be considered upregulated in \emph{aramis} RNAi
compared to bam RNAi. Sheet 3 (All Genes) contains Deseq2 output for all
genes in the dm6 assembly.

\textbf{Supplemental Table 2.3. Differential expression analysis from RNAseq of ovaries depleted of \emph{bam} and \emph{aramis} in the germline compared to ovaries depleted of \emph{bam} in the germline.}
DEseq2 output from RNAseq of ovaries depleted of \emph{bam} and \emph{aramis} in the germline compared to
ovaries depleted of \emph{bam} in the germline as a control. Sheet 1
(Downregulated Genes) contains genes and corresponding DEseq2 output
meeting the cutoffs to be considered downregulated in \emph{bam}; \emph{aramis}
RNAi compared to \emph{bam} RNAi. Sheet 2 (Upregulated Genes) contains genes
and corresponding DEseq2 output meeting the cutoffs to be considered
upregulated in \emph{bam}; \emph{aramis} \emph{aramis} RNAi compared to \emph{bam} RNAi.
Sheet 3 (All Genes) contains DEseq2 output for all genes in the dm6
assembly. Sheet 4(BP GO terms - downregulated) contains the output of a
Biological Process PANTHER Overrepresentation Test of significantly
enriched GO terms performed on genes identified as downregulated targets
from Sheet 1. Sheet 5 (BP GO terms - upregulated) contains the output of
a Biological Process PANTHER Overrepresentation Test of significantly
enriched GO terms performed on genes identified as upregulated targets
from Sheet 2.

\textbf{\hfill\break
}

\textbf{Supplemental Table 2.4. Analysis of polysome-seq of ovaries depleted of \emph{aramis} in the germline compared to developmental controls.}
Results of polysome-seq from ovaries depleted of \emph{aramis} in the germline,
ovaries depleted of \emph{bam}, and ovaries overexpressing tkv in the
germline as developmental controls. Sheet 1 (Downregulated Genes)
contains genes and corresponding polysome/input ratio values and values
representing the difference in the polysome/input ratios between
\emph{aramis} RNAi and the developmental controls meeting the cutoffs to be
considered downregulated in \emph{aramis} RNAi. Sheet 2 (Upregulated Genes)
contains genes and corresponding polysome/input ratio values and values
representing the difference in the polysome/input ratios between
\emph{aramis} RNAi and the developmental controls meeting the cutoffs to be
considered upregulated in \emph{aramis} RNAi. Sheet 3 (All Genes) contains
DEseq2 output for all genes in the dm6 assembly.

\textbf{\hfill\break
}

\textbf{Supplemental Table 2.5. Enrichment analysis of Aramis RNA IPmRNA-seq.}
Results of Aramis::GFP::FLAG IP/IgG/Input mRNAseq. Each
sheet contains the output of results from DEseq2. Sheet 1 (aramis
polysome IP Enrichment) contains the enrichment value of all \emph{aramis}
polysome targets from Aramis IP. Sheet 2 (aramis polysome IgG
Enrichment) contains the enrichment value of all \emph{aramis} polysome
targets from Aramis IgG control. Sheet 3 (Aramis IP Targets) contains
Aramis IP targets as defined in methods. Sheet 4 (IP vs In Enriched)
contains genes significantly enriched in the Aramis IP samples compared
to the input samples. Sheet 5 (IgG vs In Enriched) contains genes
significantly enriched (see methods) in the IgG samples compared to the
input samples. Sheet 6 (IPvsIn All Genes) contains the DEseq2 output of
all genes in the Aramis IP samples compared to the input samples. Sheet
7 (IgG vs In All Genes) contains the DEseq2 output of all genes in the
IgG samples compared to the input samples.

\textbf{\hfill\break
}

\textbf{Supplemental Table 2.6. Aramis translation targets contain TOP sequences.}
Sheet 1 (aramisRNAi target CAGE 5'UTRs) contains the CAGE
corrected 5'UTRs of \emph{aramis} RNAi polysome downregulated targets with
leading TOP sequences and start codons annotated. Sheet 2 (TOP location)
contains a list of \emph{aramis} RNAi polysome downregulated targets and the
position and sequence of the first instance of a 5-mer pyrimidine
sequence downstream of the CAGE-defined TSS of each gene.

\textbf{\hfill\break
}

\textbf{Supplemental Table 2.7. Enrichment analysis of Larp RNA IP mRNA-seq.}
Results of Larp::GFP::FLAG IP/IgG/Input mRNAseq. Each sheet
contains the output of DEseq2. Sheet 1 (Larp Targets) contains Larp IP
targets as defined in methods. Sheet 2 (IP vs In Enriched) contains
genes significantly enriched in the Larp IP samples compared to the
input samples. Sheet 3 (IgG vs In Enriched) contains genes significantly
enriched (see methods) in the IgG samples compared to the input samples.
Sheet 4 (IPvsIn All Genes) contains the DEseq2 output of all genes in
the Larp IP samples compared to the input samples. Sheet 5 (IgG vs In
All Genes) contains the DEseq2 output of all genes in the IgG samples
compared to the input samples.

\hypertarget{materials-and-methods}{%
\section{Materials and Methods}\label{materials-and-methods}}

\textbf{Resource Availability}

\textbf{Lead Contact:}
\begin{quote}
Further information and requests for resources and reagents should be
directed to and will be fulfilled by the lead contact, Prashanth
Rangan (\href{mailto:prangan@albany.edu}{\nolinkurl{prangan@albany.edu}}).
\end{quote}
\textbf{Materials availability:}

Materials generated during this study are available upon request.

\textbf{Data and Code availability:}
\begin{quote}
Sequencing data generated during this study are available on GEO under
the accession
\href{https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE171350}{GSE171350},
using the following token: araliaokbdqbrqd. Other data generated
during this study are available from the lead contact.
\end{quote}
\textbf{Fly lines}

The following Bloomington Stock Center lines were used in this study:
\#25751 \emph{UAS-Dcr2;nosGAL4}, \#4442 \emph{nosGAL4;MKRS/TM6}, \#32334 Aramis
RNAi\#1 CG5589\textsuperscript{HMS00325}, \#56977 Athos RNAi\#1 CG4901\textsuperscript{HMC04417},
\#36589 Porthos RNAi\#1 CG9253\textsuperscript{GL00549}, \#36537 UAS-tkv.CA, \#33631 bam
RNAi\textsuperscript{HMS00029}, \#6815 p53\textsuperscript{5A-1-4}, \#4264 Harwich, \#6816 p53\textsuperscript{11-1B-1},
\#55101 FUCCI: UASp-GFP.E2f1.1-230, UASp-mRFP1.CycB.1-266/TM6B, \#5431
UAS-EGFP, \#18942 aramis\textsuperscript{f06152} PBac\{WH\}CG5589f06152/TM6B, Tb1, \#9503
athos deficiency Df(2L)BSC143/CyO, \#13988 porthos\textsuperscript{KG05120}
P\{SUPor-P\}CG9253\textsuperscript{KG05120}, \#58178 bam RNAi P\{TRiP.HMJ22155\}, \#78777
Non1 RNAi P\{TRiP.HMS05872\}, \#61790 Larp::GFP::3XFLAG
Mi\{PT-GFSTF.1\}larp\textsuperscript{MI06928-GFSTF.1}, \#8841 w\^{}1118; Df(3R)Hsp70A,
Df(3R)Hsp70B, \#55384 Nprl3 RNAi P\{TRiP.HMC04072\}attP40, \#34814 raptor
RNAi P\{TRiP.HMS00124\}attP2

The following Vienna Stock Center lines were used in this study: Aramis
RNAi\#2 CG5589\textsuperscript{v44322}, Athos RNAi\#2 CG4901\textsuperscript{v34905}, Aramis::GFP
PBac\{fTRG01033.sfGFP-TVPTBF\}VK00002, Athos::GFP
PBac\{fTRG01233.sfGFP-TVPTBF\}VK00033, Non1::GFP
PBac\{fTRG00617.sfGFP-TVPTBF\}VK00033

The following additional fly lines were used in the study:
UASp-CycB::GFP (\href{https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5830152/\#R25}{Mathieu et al.,
2013}),
\emph{UAS-Dcr2;nosGAL4;bamGFP, If/CyO;nosGAL4} (Lehmann Lab), w1118 (Lehmann
lab),
\emph{tjGAL4/CyO} (Tanentzapf, Devenport, Godt, \& Brown, 2007),
UASp-p53 ({\textbf{???}}), RpS2::GFP\textsuperscript{CB02294}
(Buszczak et al., 2007; Zhang et al., 2014), UASp-Non1 (this
study), UASp-Larp-DM15 (this study), WT-TOP-Reporter (this study),
Mutant-TOP-Reporter (this study).

\textbf{Antibodies IF}

The following antibodies were used for immunoflourenscence: mouse
anti-1B1 1:20 (DSHB 1B1), rabbit anti-Vasa 1:833-1:4000 (Rangan Lab),
chicken anti-Vasa 1:833-1:4000 (Rangan Lab)
(Upadhyay et al., 2016), rabbit anti-pTyr
1:500 (Sigma T1235), rabbit anti-pMad 1:200 (Abcam ab52903), rabbit
anti-GFP 1:2000 (abcam, ab6556), mouse anti-p53 1:200 (DSHB 25F4),
Rabbit anti-CycB 1:200 (Santa Cruz Biotechnology, 25764), Rabbit
anti-Fibrillarin 1:200 (Abcam ab5821), Mouse anti-Fibrillarin 1:50
(Fuchs Lab) (McCarthy et al., 2018). Alexa 488 (Molecular Probes), Cy3 and
Cy5 (Jackson Labs) were used at a dilution of 1:500.

\textbf{Antibodies Western/IP}

Mouse anti-FLAG-HRP 1:5000 (Sigma Aldrich, A8592)

Mouse anti-FLAG (Sigma Aldrich, F1804)

Anti-GAPDH-HRP 1:10,000 (Cell Signaling, 14C10)

Rabbit anti-DDX52 1:5000 (Bethyl, A303-053A)

Rabbit anti-DHX33 1:5000 (Bethyl, A300-800A)

Rabbit anti-DDX47 1:1000 (Bethyl, A302-977A)

\textbf{Protein Domain Analysis}

Protein domain figures were adapted from: The Pfam protein families
database in 2019: S. El-Gebali et al.~Nucleic Acids Research (2019).
Protein Similarity values were obtained from the DRSC/TRiP Functional
Genomics Resources.

\textbf{Protein Conservation Analysis}

Evolutionary trees were generated using MEGA. The evolutionary history
was inferred by using the Maximum Likelihood method and JTT matrix-based
model. The tree with the highest log likelihood is shown. Initial
tree(s) for the heuristic search were obtained automatically by applying
Neighbor-Join and BioNJ algorithms to a matrix of pairwise distances
estimated using a JTT model, and then selecting the topology with
superior log likelihood value. Trees are drawn to scale, with branch
lengths measured in the number of substitutions per site.

\textbf{TOP Reporter Cloning}

gBlocks (see primer list for details) were cloned into pCasper2
containing a Nos promoter, HA-tag, GFP-tag, and K10 3'UTR. PCR was used
in order to amplify the gBlock and to remove the 5'-end of the RpL30
5'UTR in order to generate the 5'-UTR discovered via CAGE-seq. In order
to clone the Nos promoter followed by the RpL30 5'UTR without an
intervening restriction site, the portion of the plasmid 5' of the 5'UTR
consisting of a portion of the plasmid backbone, a NotI restriction
site, and the Nos Promoter was amplified from the pCasper plasmid using
PCR. HiFi cloning was performed on the amplified fragments. The backbone
was cut with NotI and SpeI and HiFi cloning was performed according to
the manufactures' instructions except the HiFi incubation was performed
for 1 hour to increase cloning efficiency. Colonies were picked and
cultured and plasmids were purified using standard techniques.
Sequencing was performed by Eton Bioscience Inc.~to confirm the correct
sequence was present in the final plasmids. Midi-prep scale plasmid was
prepared using standard methods and plasmids were sent to BestGene Inc.
for microinjection.

\textbf{Gateway Cloning}

Gateway cloning was performed as described according to the
manufacture's manual. Briefly, primers containing the appropriate
Gateway \emph{attb} sequence on the 5'-ends and gene specific sequences on
the 3'-ends (see primer list for sequences) were used to PCR amplify
each gene of interest. PCR fragments were BP cloned into pEntr221 as
detailed in the Thermofisher Gateway Cloning Manual and used to
transform Invitrogen One Shot OmniMAX 2 T1 Phage-Resistant Cells.
Resulting clones were picked and used to perform LR cloning into either
pPGW or pPWG as appropriate. Cloning was carried out according to the
Thermofisher Gateway Cloning Manual except the LR incubation was carried
out up to 16 hours. Colonies were picked and cultured and plasmids were
purified using standard techniques. Sequencing was performed by Eton
Bioscience Inc.~to confirm the correct sequence was present in the final
plasmids. Midi-prep scale plasmid was prepared using standard methods
and plasmids were sent to BestGene Inc.~for microinjection.

\textbf{Egg Laying Test}

Newly eclosed flies were collected and fattened overnight on yeast. Six
female flies were crossed to 4 male controls and kept in cages at 25Â°C.
Flies were allowed to lay for three days, and plates were changed and
counted daily. Total number of eggs laid over the three day laying
periods were determined and averaged between three replicate crosses for
control and experimental crosses.

\textbf{Immunostaining}

Ovaries were dissected and teased apart with mounting needles in cold
PBS and kept on ice for subsequent dissections. All incubations were
performed with nutation. Ovaries were fixed for 10-15 min in 5\%
methanol-free formaldehyde in PBS. Ovaries were washed with PBT (1x PBS,
0.5\% Triton X-100, 0.3\% BSA) once quickly, twice for 5 min, and finally
for 15 min. Ovaries were incubated overnight, up to 72 hours in PBT with
the appropriate primary antibodies. Ovaries were again washed with PBT
once quickly, twice for 5 min, and finally for 15 min. Ovaries were then
incubated with the appropriate secondary antibodies in PBT overnight up
to 72 hours at 4°C. Ovaries were washed once quickly, twice for 5 min,
and finally for 15 min in PBST (1x PBS, 0.2\% Tween 20 Ovaries). Ovaries
were mounted with Vectashield with 4â€²,6-diamidino-2-phenylindole (DAPI)
(Vector Laboratories) and imaged on a Zeiss 710. All gain, laser power,
and other relevant settings were kept constant for any immunostainings
being compared. Image processing was performed in Fiji, gain was
adjusted, and images were cropped in Photoshop CC 2018.

\textbf{Florescent Imaging}

Tissues were visualized and imaged were acquired using a Zeiss LSM-710
confocal microscope under the 20x--- and 40x--- oil objectives.

\textbf{Measurement of global protein synthesis}

OPP (Thermo Fisher, C10456) treatment was performed as in McCarthy
(McCarthy et al., 2019). Briefly, ovaries were
dissected in Schneider's media (Thermo Fisher, 21720024) and incubated
in 50 µM of OPP reagent for 30 minutes. Tissue was washed in 1x PBS and
fixed for 10 minutes in 1x PBS plus 5\% methanol-free formaldehyde.
Tissue was permeabilized with 1\% Triton X-100 in 1x PBST (1x PBS, 0.2\%
Tween 20) for 30 minutes. Samples were washed with 1x PBS and incubated
with Click-iT reaction cocktail, washed with Click-iT reaction rinse
buffer according to manufacturer's instructions. Samples were then
immunostained according to previously described procedures.

\textbf{Image Quantifications}

All quantifications were performed on images using the same confocal
settings. A.U. quantifications were performed in Fiji on images taken
with identical settings using the ``Measure'' function. Intensities were
normalized as indicated in the figure legends, boxplots of A.U.
measurements were plotted using R and statistics were calculated using
R.

Quantification of nucleolar size was measured in Fiji by measuring the
diameter of the nucleolus using the measure tool in Fiji. Volumes were
calculated using the formula for a sphere.

Quantification of p53 area of expression was performed from control,
\emph{nosGAL4} and nosGAL4\textgreater{}\emph{aramis} RNAi germaria. A manual threshold was
set based off of qualitative assessment of a ``punctate''. For control
ovaries, cells proximal to the niche consisting of GSCs/CBs were
outlined and for \emph{aramis} RNAi the entire germline proximal to the niche
was outlined and a Fiji script was used to determine the number of
pixels above the threshold and the total number of pixels. Data from
each slice for each replicate was summed prior to plotting and
statistical analysis.

Colocalization analysis of helicases with Fibrillarin was performed in
Fiji using the Plot Profile tool. A selection box was drawn over a
Fibrillarin punctate of interest (indicated with a box in the images)
and Plot Profiles was acquired for each channel of interest. Data was
plotted and Spearman correlations calculated using R.

Quantification of Non1-GFP expression and p53 expression over
development was calculated in Fiji using the Auto Threshold tool with
the Yen method (Sezgin \& Sankur, 2004) to threshold
expression. Quantifications were performed on 3 merged slices and egg
chambers were cropped out of quantified images prior to thresholding to
prevent areas outside of the germarium from influencing the thresholding
algorithm. Areas of germline with ``high'' and ``low'' expression of
Non1-GFP were outlined manually and a custom Fiji script was used in
order to quantify the proportion of pixels in the selected marked as
positive for expression for either Non1-GFP or p53, staging was inferred
from the results of the Non1-GFP quantification performed using 1B1 to
determine the stages of peak Non1 expression. Percent area was plotted
with ggplot2 as boxplots in a custom R script.

\textbf{RNA Extraction from Ovaries}

RNA extraction was performed using standard methods. Ovaries were
dissected into PBS and transferred to microcentrifuge tubes. PBS was
removed and 100ul of Trizol was added and ovaries were flash frozen and
stored at -80 Â°C. Ovaries were lysed in the microcentrifuge tube using a
plastic disposable pestle. Trizol was added to 1 mL total volume and
sample was vigorously shaken and incubated for 5 min at RT. The samples
were centrifuged for x min at \textgreater13,000 g at 4 Â°C and the supernatant was
transferred to a fresh microcentrifuge tube. 500 ul of chloroform was
added and the samples were vigorously shaken and incubated for 5 minutes
at RT. Samples were spun at max speed for 10 minutes at 4 Â°C. The
supernatant was transferred to a fresh microcentrifuge tube and ethanol
precipitated. Sodium acetate was added equaling 10\% of the volume
transferred and 2-2.5 volumes of 100\% ethanol were added. The samples
were shaken thoroughly and left to precipitate at -20 Â°C overnight. The
samples were centrifuged at max speed at 4 Â°C for 15 min to pellet the
RNA. The supernatant was discarded and 500 ul of 75\% ethanol was added
to wash the pellet. The samples were vortexed to dislodge the pellet to
ensure thorough washing. The samples were spun at 4 Â°C for 5 min and the
supernatant was discarded. The pellets were left for 10-20 min until
dry. The pellets were resuspended in 20-50ul of RNAse free water and the
absorbance at 260 was measured on a nanodrop to measure the
concentration of each sample.

\textbf{S2 Cell RNAi}

DRSC-S2 cells (Stock \#181, DGRC) were cultured according to standard
methods in M3+BPYE media supplemented with 10\% heat-inactivated FBS.
dsRNA for RNAi was prepared as described by the SnapDragon manual.
Briefly, template was prepared from S2 cell cDNA using the appropriate
primers (see primer list) designed using SnapDragon
(\url{https://www.flyrnai.org/snapdragon}). Template was either used
directly for \emph{in-vitro} transcription or TA-cloned into the pCR2.1-TOPO
vector (K450002) followed by transformation into TOP-10 cells (K450002),
plasmid purified, and digested with \emph{Eco}R I prior to \emph{in-vitro}
transcription. For \emph{in-vitro} transcription the T7 Megascript kit
(AM1334) was used following manufacturer's instructions and in-vitro
transcriptions were incubated overnight at 37Â°C. The RNA was treated
with DNAse according to the T7 Megascript manual and the RNA was
purified using acid-phenol chloroform extraction and ethanol
precipitated. The resulting RNA was annealed by heating at 65Â°C for 5
minutes and slow cooling to 37Â°C for an hour. S2 cell RNAi was performed
essentially as previously described using Effectine
(Zhou, Mohr, Hannon, \& Perrimon, 2013). 1.0x10\textsuperscript{6} cells were seeded 30
minutes prior to transfection and allowed to attach. After 30 minutes,
just prior to transfection, the media was changed for 500 Âµl of fresh
media. 500 Âµl of transfection complexes using 1 Âµg of dsRNA was prepared
per well of a 6-well plate and pipetted dropwise onto seeded cells.
After 24 hours an additional 1 mL of media was added to each well. After
an additional 24 hours cells were passaged to 10 cm dishes. After an
additional 3 days cells were harvested for further analysis.

\textbf{Polysome-profiling}

Polysome-profiling in S2 cells was performed as in Fuchs et al.
(Fuchs, Diges, Kohlstaedt, Wehner, \& Sarnow, 2011) with minor modifications. S2
cells were resuspended by pipetting, pelleted by centrifugation at 800g
for one minute, and washed in cold PBS. Cells were again pelleted and
resuspended in 400 Âµl of lysis buffer (300 mM NaCl, 15 mM Tris-HCl, pH
7.5, 15 mM EDTA, 100 g/mL cycloheximide, 1\% Triton X-100). Cells were
then allowed to continue to lyse for 15 min on ice. Lysate was cleared
by centrifugation at 8500g for 5 min at 4Â°C. Cleared lysate was loaded
onto 10\%-50\% sucrose gradients (300 mM NaCl, 15 mM Tris-HCl, pH 7.5, 15
mM MgCl2, 100 g/mL cycloheximide) and centrifuged in an SW41 rotor at
35,000 RPM, for 3 hours. Gradients were fractionated on a Density
Gradient Fractionation System (Brandel, \#621140007) at 0.75 mL/min.
Data generated from gradients were plotted using R.

\textbf{Western Blot}

HeLa cells were harvested for Western by in RIPA buffer by scraping.
Western blotting were performed according to standard methods, briefly,
each sample was loaded onto a 4-20\% commercial, precast gels and run at
100V for 60-90m depending on the size of the protein of interest. Gels
were transferred to nitrocellulose membranes at 100V for 1hr at 4Â°C.
Blot was blocked in 1\% milk in PBS and washed 3 times with PBS-T for 5
minutes. Primary antibodies were diluted in PBS-T+5\% BSA and incubated
overnight. Blot was washed once quickly, once for 5m, and once for 10m
in PBS-T. Blot was subsequently imaged with ECL for conjugated
primaries. For unconjugated primaries, the appropriate secondary was
diluted 1:10,000 in 5\% milk and incubated for 2-4 hours at RT. Blot was
washed once quickly, once for 5m, and once for 10m in PBS-T and imaged.
Images were quantified using Fiji.

\textbf{mRNAseq Library Preparation and Analysis}

Libraries were prepared with the Biooscientific kit (Bioo Scientific
Corp., NOVA-5138-08) according to manufacturer's instructions with
minor modifications. Briefly, RNA was prepared with Turbo DNAse
according to manufacturer's instructions (TURBO DNA-free Kit, Life
Technologies, AM1907), and incubated at 37Â°C for 30 min. DNAse was
inactivated using the included DNAse Inactivation reagent and buffer
according to manufactures instructions. The RNA was centrifuged at 1000
g for 1.5 min and 19 µl of supernatant was transferred into a new 1.5 mL
tube. This tube was again centrifuged at 1000 g for 1.5 min and 18 µl of
supernatant was transferred to a new tube to minimize any Inactivation
reagent carry-over. RNA concentration was measured on a nanodrop. Poly-A
selection was performed on a normalized quantity of RNA dependent on the
lowest amount of RNA in a sample, but within the manufacturer's
specifications for starting material. Poly-A selection was performed
according to manufacturer's instructions (Bioo Scientific Corp., 710
NOVA-512991). Following Poly-A selection mRNA libraries were generated
according to manufactures instructions (Bioo Scientific Corp.,
NOVA-5138-08) except RNA was incubated for 13 min at 95Â°C to generate
optimal fragment sizes. Library quantity was assessed via Qubit
according to manufacturer's instructions and library quality was
assessed with a Bioanalyzer or Fragment Analyzer according to
manufacturer's instructions to assess the library size distribution.
Sequencing was performed on biological duplicates from each genotype on
an Illumina NextSeq500 by the Center for Functional Genomics (CFG) to
generate single end 75 base pair reads. Reads were aligned to the dm6.01
assembly of the Drosophila genome using HISAT v2.1.0. Reads were counted
using featureCounts v1.4.6.p5. UCSC genome browser tracks were generated
using the bam coverage module of deeptools v3.1.2.0.0. Differential
expression analysis was performed using DEseq2 (v1.24.0) and data was
plotted using R. Differentially expressed genes were those with
log\textsubscript{2}(foldchange) \textgreater{} \textbar1.5\textbar{} and FDR \textless{} 0.05 in the \emph{aramis} RNAi
versus \emph{bam} RNAi experiment and foldchange \textgreater{} \textbar1.5\textbar{} and FDR \textless{} 0.05
in the \emph{bam} RNAi; \emph{aramis} RNAi versus \emph{bam} RNAi experiment. GO-term
analysis of GO biological processes was performed on differentially
expressed genes using PANTHER via \url{http://geneontology.org/}. Fisher's
exact test was used to calculate significance and FDR was used to
correct for multiple testing. GO-term analysis results were plotted
using R.

\textbf{Polysome-seq}

Polysome-seq was performed as in Flora et al.~(Flora, Wong-Deyrup, et al., 2018) with
minor modifications. Ovaries were dissected in PBS and transferred to a
microcentrifuge tube in liquid nitrogen. Ovaries were lysed in 300 Âµl of
lysis buffer (300 mM NaCl, 15 mM Tris-HCl, pH 7.5, 15 mM EDTA, 100 µg/mL
cycloheximide, 1\% Triton X-100) and allowed to lyse for 15 min on ice.
Lysate was cleared by centrifugation at 8500g for 5 min at 4Â°C. 20\% of
the lysate was reserved as input, 1 mL of Trizol (Invitrogen, 15596026)
was added and RNA was stored at -80°C. Cleared lysate was loaded onto
10\%-50\% sucrose gradients (300 mM NaCl, 15 mM Tris-HCl, pH 7.5, 15 mM
MgCl2, 100 g/mL cycloheximide) and centrifuged in an SW41 rotor at
35,000 RPM, for 3 hours. Gradients were fractionated on a Density
Gradient Fractionation System (Brandel, \#621140007) at 0.75 mL/min, 20
µl of 20\% SDS, 8 µl of 0.5 M pH 8 EDTA, and 16 µl of proteinase K (NEB,
P8107S) was added to each polysome fraction. Fractions were incubated
for 30m at 37Â°C. Standard acid phenol chloroform purification followed
by ethanol precipitation was performed on each fraction. The RNA from
polysome fractions was pooled and RNAseq libraries were prepared.

\textbf{Polysome-seq Data Analysis}

Reads were checked for quality using FastQC. Reads were mapped to the
Drosophila genome (dm6.01) using Hisat version 2.1.0. Mapped reads were
assigned to features using featureCount version v1.6.4. Translation
efficiency was calculated as in (Flora et al., 2018) using an R script.
Briefly, TPMs (transcripts per million) values were calculated. Any gene
having zero reads in any library was discarded from further analysis.
The log\textsubscript{2} ratio of CPMs between the polysome fraction and total mRNA
was calculated and averaged between replicates. This ratio represents
TE, TE of each replicate was averaged. Targets were defined as
transcripts falling greater or less than two standard deviations from
the median TE in aramis RNAi, but not in either of the two developmental
controls (NosGAL4\textgreater UAS-\emph{tkv} or NosGAL4\textgreater{}\emph{bam} RNAi). Additionally,
genes were only considered targets if their mean TE value in
NosGAL4\textgreater{}\emph{aramis} RNAi was higher (for upregulated targets) or lower
(for downregulated targets) their mean TE values in both of the two
developmental controls. Finally, only targets meeting a conservative
expression cutoff of log\textsubscript{2}(TPM) expression greater than five were
considered to exclude more lowly expressed genes as they are highly
influenced by noise in polysome-seq.

\textbf{CAGE-seq Tracks}

CAGE-seq tracks were visualized using the UCSC Genome Browser after
adding the publicly available track hub 'EPD Viewer Hub'.

\textbf{CAGE-seq Data Reanalysis}

Publicly available genome browser tracks were obtained of CAGE-seq data
(generated by Chen et al.~(2014) and viewed through the UCSC Genome
Browser. The original CAGE-seq data from ovaries was obtained from SRA
under the accession number SRR488282. Reads were aligned to the dm6.01
assembly of the \emph{Drosophila} genome using HISAT v2.1.0. cageFightR was
used to determine the dominant TSS for every gene with sufficient
expression in from the aligned dataset according to its documentation
with default parameters excepting the following: For getCTSS, a
mappingQualityThreshold of 10 was used. For normalizeTagCount the method
used was ``simpleTPM''. For clusterCTSS the following parameters were
used; threshold = 1, thresholdIsTPM = TRUE, nrPassThreshold = 1, method
= "paraclu", maxDist = 20, removeSingletons = TRUE,
keepSingletonsAbove = 5. R was used to obtain genome sequence
information downstream of the TSS of each gene identified.

To generate a table of \emph{aramis} polysome-seq target 5'UTRs adjusted
using CAGE-seq data, bigwig files of CAGE-seq from ovaries were obtained
from EPD Viewer Hub. The most highly expressed TSS within a CAGE cluster
(obtained as described in this section) was used to determine the new
5'-end coordinate associated with each \emph{aramis} polysome-seq target gene
at the transcript level. These coordinates were used to obtain the
corrected 5'UTR using R and transcripts with identical sequences were
discarded.

\textbf{Motif Enrichment Analysis}

Motif enrichment analysis was performed using Homer
(Heinz et al., 2010) using the
findmotifs.pl module, supplying Homer with the first 200 nucleotides
downstream of the TSS as determined by CAGE-seq for polysome-seq targets
and non-targets as a background control with the following parameters
``-rna -nogo -p 6 -len 6''. Only motifs not marked as potential false
positives were considered. The position of the putative TOP motifs was
determined using a custom R script by searching for the first instance
of any five pyrimidines in a row within the first 200 nucleotides of the
TSS using the Biostrings package
(Pagès, Aboyoun, Gentleman, \& DebRoy, 2019). Results were plotted as
a histogram in R.

\textbf{RNA Immunoprecipitation (RNA IP)}*

All RIPs were performed with biological triplicates. 50-60 ovary pairs
were dissected for each sample in RNase free PBS and dissected ovaries
were kept on ice during subsequent dissections. After dissection,
ovaries were washed with 500 Âµl of PBS to remove any debris. This PBS
was removed, and ovaries were lysed in 100 Âµl of RIPA buffer (10 mM
Tris-Cl Buffer (pH 8.0), 1 mM EDTA, 1\% Triton X-100,0.1\% Sodium
deoxycholate, 0.1\% SDS, 140 mM NaCl, 1 mM PMSF, 1 cOmplete, EDTA-free
Protease Inhibitor/10mLbuffer (Roche, 11873580001), RNase free H2O)
supplemented with 8 Âµl of RNase Out. Following lysis an additional 180
Âµl of RIPA was added to each sample. Lysate was cleared with
centrifugation at 14,000g for 20m at 4Â°C. Cleared lysate was transferred
to a new 1.5 mL tube. 10\% of this lysate was reserved for RNA input and
5\% was reserved as a protein input. To the RNA input 100 Âµl of Trizol
was added and the input was stored at -80Â°C. To the protein input SDS
loading buffer was added to a 1X working concentration and the sample
was heated at 95Â°C for 5m and stored at -20Â°C. The remaining lysate was
equally divided into two new 1.5 mL tubes. To one tube 3 Âµg of mouse
anti-FLAG antibody was added and to the other tube 3 Âµg of mouse IgG was
added. These samples were incubated for 3 hours with nutation at 4Â°C.
NP40 buffer was diluted to a 1X working concentration from a 10X stock
(10x NP40 Buffer: 50 mM Tris-Cl Buffer (pH 8.0), 150 mM NaCl, 10\% NP-40,
1 cOmplete, EDTA-free Protease Inhibitor Cocktail Pill/10mL buffer,
RNase free H2O). 30 Âµl of Protein-G beads per RIP were pelleted on a
magnetic stand and supernatant was discarded. 500 Âµl of 1X NP40 buffer
was used to resuspend Protein-G beads by nutation. Once beads were
resuspended, they were again pelleted on the magnetic stand. This
washing process was repeated a total of 5 times. Washed Protein-G beads
were added to each lysate and incubated overnight. The next day fresh 1X
NP40 buffer was prepared. Lysates were pelleted on a magnetic stand at
4Â°C and supernatant was discarded. 300 Âµl of 1X NP40 buffer was added to
each sample and samples were resuspended by nutation at 4Â°C. Once
samples were thoroughly resuspended, they were pelleted on a magnetic
stand. These washing steps were repeated 6 times. Following the final
washing steps, beads were resuspended in 25 ul of 1X NP40 Buffer. 5 Âµl
of beads were set aside for Western and the remaining beads were stored
at -80Â°C in 100 Âµl of Trizol. SDS loading buffer was added was added to
a 1X working concentration and the sample was heated at 95Â°C for 5m and
stored at -20Â°C or used for Western (refer to Western Blot section).

\textbf{Helicase RNA IPseq}

RNA was purified as previously described. RNA yield was quantified using
Qubit or nanodrop according to manufactures instructions. RNA was run on
a Fragment Analyzer according to manufactures instructions to assess
quality. Inputs were diluted 1:50 to bring them into a similar range as
the IgG and IP samples. To each sample 0.5 ng of Promega Luciferase
Control RNA was added as a spike-in. Libraries were prepared as
previously described except Poly(A) selection steps were skipped and
library preparation was started with between 1-100 ng of total RNA.
Reads were mapped to the M21017.1 NCBI \emph{Drosophila} rRNA sequence record
and the sequence of Luciferase obtained from Promega. All further
analysis was performed using custom R scripts. Reads were assigned to
features using featureCounts based off of a custom GTF file assembled
based off of the Flybase record of rRNA sequences. Reads mapping to rRNA
were normalized to reads mapping to the Luciferase spike-in control.
Reads were further normalized to the reads from the corresponding input
library to account for differences in input rRNA concentration between
replicates and replicates were subsequently averaged. Tracks were
visualized using the R package 'ggplot2', with additional formatting
performed using 'scales' and 'egg'. The rRNA GTF was read into R using
'rtracklayer' and visualized using 'gggenes'. Average reads mapping to
rRNA from IgG control and IP was plotted and a one-sided bootstrapped
paired t-test for was performed on regions on rRNA that appeared to be
enriched in the IP samples compared to the IgG control as it is a
non-parametric test suitable for use with low n using R with 100,000
iterations.

\hypertarget{larp-gel-shifts}{%
\subsubsection{Larp Gel Shifts}\label{larp-gel-shifts}}

\textbf{Cloning, Protein expression and purification}

The Larp-DM15 protein expression construct (amino acids 1330-1481
corresponding to isoform D) was cloned into a modified pET28a vector by
PCR using cDNA corresponding to accession ID NP\_733244.5. The resulting
fusion protein has an N-fHis\textsubscript{10}-maltose binding protein (MBP)-tobacco
etch virus (TEV) protease recognition site tag. Protein expression and
purification were performed as described previously
(Lahr et al., 2015). Briefly, plasmid was
transformed into BL21(DE3) \emph{E. coli} cells and plated onto
kanamycin-supplemented agar plates. A confluent plate was used to
inoculate 500 mL of autoinduction media
(Studier, 2005). Cells were grown for
three hours at 37ÂºC and induced overnight at 18ÂºC. Cells were harvested,
flash frozen, and stored at -80ÂºC.

Cells were resuspended in lysis buffer (50 mM Tris, pH 8, 400 mM NaCl,
10 mM imidazole, 10\% glycerol) supplemented with aprotinin (Gold Bio),
leupeptin (RPI Research), and PMSF (Sigma) protease inhibitors. Cells
were lysed via homogenization. Lysate was clarified by centrifugation
and incubated with Ni-NTA resin (ThermoScientific) for batch
purification. Resin was washed with lysis buffer supplemented with 35 mM
imidazole to remove non-specific interactions. His\textsubscript{10}-MBP-DM15 was
eluted with 250 mM imidazole. The tag was removed via proteolysis using
TEV protease and simultaneously dialyzed overnight (3 mg TEV to 40 mL
protein elution). Larp-DM15 was further purified by tandem anion (GE
HiTrap Q) and cation exchange (GE HiTrap SP) chromatography using an
AKTA Pure (GE) to remove nucleic acid and protein contaminants. The
columns were washed with in buffer containing 50 mM Tris, pH 7, 175 mM
NaCl, 0.5 mM EDTA, and 10\% glycerol and eluted with a gradient of the
same buffer containing higher salt (1 M NaCl). Fractions containing
Larp-DM15 were pooled, and 3 M ammonium sulfate was added to a final
concentration of 1 M. A butyl column (GE HiTrap Butyl HP) was run to
remove TEV contamination. The wash buffer contained 50 mM Tris, pH 7, 1
M ammonium sulfate, and 5\% glycerol, and the elution buffer contained 50
mM Tris pH 7 and 2 mM DTT. Fractions containing Larp-DM15 were buffer
exchanged into storage buffer (50 mM Tris pH, 7.5, 250 mM NaCl, 2 mM
DTT, 25\% glycerol), flash frozen in liquid nitrogen, and stored at
-80Â°C. The purification scheme and buffer conditions were the same as
with \emph{Hs}DM15 (Lahr et al., 2015), except cation and
anion exchange buffers were at pH 7, as noted above.

\textbf{RNA preparation}

5'-triphosphorylated \emph{RpL30} and \emph{Non1} 42-mers were synthesized
(ChemGenes). Purine-substituted controls were synthesized by \emph{in vitro}
transcription using homemade P266L T7 RNAP polymerase
(Guillerez, Lopez, Proux, Launay, \& Dreyfus, 2005). The transcription reaction containing
40 mM Tris, pH 8, 10 mM DTT, 5 mM spermidine, 2 mM NTPs, and 10-15 mM
MgCl\textsubscript{2} was incubated at 37Â°C for 4 hours. Transcripts were subsequently
purified from an 8\% polyacrylamide/6M urea/1XTBE denaturing gel, eluted
passively using 10 mM sodium cacodylate, pH 6.5, and concentrated using
spin concentrators (Millipore Amicon). All oligos were radioactively
capped using Vaccinia virus capping system (NEB) and \(\alpha^{32}\)-GTP
(Perkin-Elmer). Labelled oligos were purified using a 10\%
polyacrylamide/6M urea/1XTBE denaturing gel, eluted with 10 mM sodium
cacodylate, pH 6.5, and concentrated by ethanol precipitation.

The RNA sequences used were:

RpL30: \textbf{CUUUU}GCCAUUGUCAGCCGACGAAGUGCUUUAACCCAAACUA

Non1: \textbf{CUUUUU}GGAAUACGAAGCUGACACCGCGUGGUGUUUUUGCUU

*Purine-substituted RPL30 control:
\textbf{GAAAAG}CCAUUGUCAGCCGACGAAGUGCUUUAACCCAAACUA

*Purine-substituted Non1 control:
\textbf{GAAAAAG}GAAUACGAAGCUGACACCGCGUGGUGUUUUUGCUU

Oligos used for run-off transcription
\begin{longtable}[]{@{}ll@{}}
\toprule
\begin{minipage}[b]{0.46\columnwidth}\raggedright
\textbf{DNA oligo}\strut
\end{minipage} & \begin{minipage}[b]{0.46\columnwidth}\raggedright
\textbf{Sequence (5' to 3')}\strut
\end{minipage}\tabularnewline
\midrule
\endhead
\begin{minipage}[t]{0.46\columnwidth}\raggedright
**RpL30 control gene block
(with 3' HDV)\strut
\end{minipage} & \begin{minipage}[t]{0.46\columnwidth}\raggedright
GCGCGCGAATTCTAATACGACTCAC
\textbar{} TATA\textbf{GAAAAGCCATTGTCAGCC}
\textbf{GACGAAGTGCTTTAACCCAAACTA}
GGGTCGGCATGGCATCTCCACCT
CCTCGCGGTCCGACCTGGGCTAC
TTCGGTAGGCTAAGGGAGAAGCTT
GGCACTGGCCGTCGTTT\strut
\end{minipage}\tabularnewline
\begin{minipage}[t]{0.46\columnwidth}\raggedright
Non1 control

Forward\strut
\end{minipage} & \begin{minipage}[t]{0.46\columnwidth}\raggedright
GCGCGCGAATTCTAATACGAC
TCACTATA\textbf{GGAAAAAGGA}
\textbf{ATACGAAGCTGACA}\strut
\end{minipage}\tabularnewline
\begin{minipage}[t]{0.46\columnwidth}\raggedright
Non1 control

Reverse\strut
\end{minipage} & \begin{minipage}[t]{0.46\columnwidth}\raggedright
\textbf{AAGCAAAAACACCACGCGGTG}
\textbf{TCAGCTTCGTATTCCTTTTTC}
CTATAGTGAG\strut
\end{minipage}\tabularnewline
\begin{minipage}[t]{0.46\columnwidth}\raggedright
5' GEN amp\strut
\end{minipage} & \begin{minipage}[t]{0.46\columnwidth}\raggedright
GCGCGCGAATTCTAATACGACTCA\strut
\end{minipage}\tabularnewline
\begin{minipage}[t]{0.46\columnwidth}\raggedright
RpL30 amp Reverse\strut
\end{minipage} & \begin{minipage}[t]{0.46\columnwidth}\raggedright
TAGTTTGGGTTAAAGCACTTCGTCGGC\strut
\end{minipage}\tabularnewline
\begin{minipage}[t]{0.46\columnwidth}\raggedright
Non1 amp Reverse\strut
\end{minipage} & \begin{minipage}[t]{0.46\columnwidth}\raggedright
AAGCAAAAACACCACGCGGTGTCA\strut
\end{minipage}\tabularnewline
\bottomrule
\end{longtable}
* These RNAs were synthesized using run-off transcription.

{[}Electrophoretic mobility shift assays (EMSAs)\(\Delta\)

Each binding reaction contained 125 total radioactive counts with final
reaction conditions of: 20 mM Tris-HCl, pH 8, 150 mM NaCl, 10\% glycerol,
1 mM DTT, 0.5 µg tRNA (Ambion), 1 µg BSA (Invitrogen), and \textless90 pM RNA.
To anneal RNA, oligos were snap-cooled by heating at 95Â°C for 1 min and
cooled on ice for 1 hour. For capped RpL30 shifts and capped
purine-substituted controls, final concentrations of 0, 0.001, 0.003,
0.01, 0.03, 0.1, 0.3, 1, 3, 10, 30, and 100 nM Larp-DM15 were titrated.
For capped Non1 shifts, final concentrations of 0, 0.01, 0.03, 0.1, 0.3,
1, 3, 10, 30, 100, 300, and 1000 nM Larp-DM15 were titrated. Native 7\%
polyacrylamide 0.5X TBE gels were pre-run on ice at 120 V for 30 min.
Binding reactions were run at 120 V on ice for 45-52 min. Gels were
dried for 30 min and allowed to expose overnight using a phosphor screen
(GE). Screens were imaged using GE Amersham Typhoon. Bands were
quantified using ImageQuant TL (GE). Background subtraction was first
done using the rolling ball method and then subtracting the signal from
the zero-protein lane from each of the shifted bands. Fraction shifted
was determined by dividing the background-corrected intensity of the
shifted band by total intensity of bands in each lane. Three independent
experiments were done for each oligo, with the average plotted and
standard deviation shown.

\textbf{mRNA IPseq}

IPs of Larp and Aramis were performed as described in the RNA IP-seq
section above in triplicate. mRNA libraries were prepared as described
in mRNAseq Library Preparation and Data Processing using a constant
volume of RNA from each sample with input samples having been diluted
1:50. Data was processed as described as in the mRNAseq Library
Preparation and Data Processing section. Targets are defined as genes
with \textgreater2 fold enrichment and an adjusted p-value \textless0.05 in the Larp-IP
libraries compared to input libraries, but not meeting those criteria in
the IgG libraries compared to input.

\textbf{Larp RNA IP qPCR}

Larp RNA IP was performed as described in the Larp RNA IPseq section
with the following modifications. As the ovaries used were small, they
were flash frozen in order to accumulate 40-50 ovaries for each
biological replicate. Additionally, 5\% input was taken for both RNA and
protein samples. Once RNA was purified all of the RNA was treated with
Turbo DNAse as in the {} mRNAseq Library Preparation and
Analysis section. Reverse transcription (RT) was performed using
Superscript II according to the manufacture's protocol with equivalent
volumes of RNA for each sample. cDNA was diluted 1:8 before performing
qPCR using Syber Green. Each reaction consisted of 5ul Syber Green
master mix, 0.4 ul water, 0.3 ul of each primer, and 4 ul of diluted
cDNA. For each sample 3 biological and 3 technical replicates were
performed. Oulier values of technical replicates were removed using a
Dixon test with a cutoff of p\textless0.05. Remaining technical replicates were
averaged, and the IP Input Ct value, the log\textsubscript{2} of the Input dilution
(20) was also subtracted to account for the Input being 5\% of the total
sample as follows:

\(\Delta Ct[normalized\ IP] = (Average\ Ct[IP]/(Average\ Ct[Input] - log~2~(Input\ Dilution\ Factor))\)

Next, RNA recovery was normalized using the spike-in control for each
sample as follows:

\(\Delta \Delta Ct = \Delta Ct[normalized\ IP]-\Delta\ Ct[Luciferase]\)

Next, Each sample was normalized to it's matched \emph{bam} RNAi control as
follows:

\(bam\ RNAi\ normalized Ct= \Delta \Delta Ct[aramis\ RNAi\ IP] - \Delta \Delta Ct[bam\ RNAi\ IP]\)

Finally, fold increase of IP from \emph{aramis} RNAi over \emph{bam} RNAi was
calculated as follows:

\(Fold Enrichment = 2^{-bam\ RNAi\ normalized\ Ct}\)

Fold enrichment was plotted and One-sample t-test performed on \emph{aramis}
RNAi samples in R using a mu of 1.

\hypertarget{conclusion}{%
\chapter*{Conclusion}\label{conclusion}}
\addcontentsline{toc}{chapter}{Conclusion}

If we don't want Conclusion to have a chapter number next to it, we can add the \texttt{\{-\}} attribute.

\textbf{More info}

And here's some other random info: the first paragraph after a chapter title or section head \emph{shouldn't be} indented, because indents are to tell the reader that you're starting a new paragraph. Since that's obvious after a chapter or section title, proper typesetting doesn't add an indent there.

\appendix

\hypertarget{the-first-appendix}{%
\chapter{The First Appendix}\label{the-first-appendix}}

This first appendix includes all of the R chunks of code that were hidden throughout the document (using the \texttt{include\ =\ FALSE} chunk tag) to help with readibility and/or setup.

\textbf{In the main Rmd file}
\begin{Shaded}
\begin{Highlighting}[]
\CommentTok{\# This chunk ensures that the thesisdown package is}
\CommentTok{\# installed and loaded. This thesisdown package includes}
\CommentTok{\# the template files for the thesis.}
\ControlFlowTok{if}\NormalTok{ (}\OperatorTok{!}\KeywordTok{require}\NormalTok{(remotes)) \{}
  \ControlFlowTok{if}\NormalTok{ (params}\OperatorTok{$}\StringTok{\textasciigrave{}}\DataTypeTok{Install needed packages for \{thesisdown\}}\StringTok{\textasciigrave{}}\NormalTok{) \{}
    \KeywordTok{install.packages}\NormalTok{(}\StringTok{"remotes"}\NormalTok{, }\DataTypeTok{repos =} \StringTok{"https://cran.rstudio.com"}\NormalTok{)}
\NormalTok{  \} }\ControlFlowTok{else}\NormalTok{ \{}
    \KeywordTok{stop}\NormalTok{(}
      \KeywordTok{paste}\NormalTok{(}\StringTok{\textquotesingle{}You need to run install.packages("remotes")",}
\StringTok{            "first in the Console.\textquotesingle{}}\NormalTok{)}
\NormalTok{    )}
\NormalTok{  \}}
\NormalTok{\}}
\ControlFlowTok{if}\NormalTok{ (}\OperatorTok{!}\KeywordTok{require}\NormalTok{(thesisdown)) \{}
  \ControlFlowTok{if}\NormalTok{ (params}\OperatorTok{$}\StringTok{\textasciigrave{}}\DataTypeTok{Install needed packages for \{thesisdown\}}\StringTok{\textasciigrave{}}\NormalTok{) \{}
\NormalTok{    remotes}\OperatorTok{::}\KeywordTok{install\_github}\NormalTok{(}\StringTok{"ismayc/thesisdown"}\NormalTok{)}
\NormalTok{  \} }\ControlFlowTok{else}\NormalTok{ \{}
    \KeywordTok{stop}\NormalTok{(}
      \KeywordTok{paste}\NormalTok{(}
        \StringTok{"You need to run"}\NormalTok{,}
        \StringTok{\textquotesingle{}remotes::install\_github("ismayc/thesisdown")\textquotesingle{}}\NormalTok{,}
        \StringTok{"first in the Console."}
\NormalTok{      )}
\NormalTok{    )}
\NormalTok{  \}}
\NormalTok{\}}
\KeywordTok{library}\NormalTok{(thesisdown)}
\CommentTok{\# Set how wide the R output will go}
\KeywordTok{options}\NormalTok{(}\DataTypeTok{width =} \DecValTok{70}\NormalTok{)}
\end{Highlighting}
\end{Shaded}
\textbf{In Chapter \ref{ref-labels}:}

\hypertarget{the-second-appendix-for-fun}{%
\chapter{The Second Appendix, for Fun}\label{the-second-appendix-for-fun}}

\backmatter

\hypertarget{references}{%
\chapter*{References}\label{references}}
\addcontentsline{toc}{chapter}{References}

\markboth{References}{References}

\noindent

\setlength{\parindent}{-0.20in}
\setlength{\leftskip}{0.20in}
\setlength{\parskip}{8pt}

\hypertarget{refs}{}
\begin{cslreferences}
\leavevmode\hypertarget{ref-Agalarov2000}{}%
Agalarov, S. C., Sridhar, G., Funke, P. M., Stout, C. D., \& Williamson, J. R. (2000). Structure of the S15, S6, S18-rRNA complex: Assembly of the 30S ribosome central domain. \emph{Science}, \emph{288}(5463), 107--112.

\leavevmode\hypertarget{ref-Agarwal1995a}{}%
Agarwal, M. L., Agarwal, A., Taylor, W. R., \& Stark, G. R. (1995). P53 controls both the G2/M and the G1 cell cycle checkpoints and mediates reversible growth arrest in human fibroblasts. \emph{Proceedings of the National Academy of Sciences}, \emph{92}(18), 8493--8497.

\leavevmode\hypertarget{ref-Andrews2000a}{}%
Andrews, J., Garcia-Estefania, D., Delon, I., Lu, J., Mével-Ninio, M., Spierer, A., \ldots{} Oliver, B. (2000). OVO transcription factors function antagonistically in the Drosophila female germline. \emph{Development (Cambridge, England)}, \emph{127}(4), 881--892.

\leavevmode\hypertarget{ref-anthonyOrallyAdministeredLeucine2000}{}%
Anthony, J. C., Anthony, T. G., Kimball, S. R., Vary, T. C., \& Jefferson, L. S. (2000). Orally Administered Leucine Stimulates Protein Synthesis in Skeletal Muscle of Postabsorptive Rats in Association with Increased eIF4F Formation. \emph{The Journal of Nutrition}, \emph{130}(2), 139--145. \url{http://doi.org/10.1093/jn/130.2.139}

\leavevmode\hypertarget{ref-aokiLARP1SpecificallyRecognizes2013}{}%
Aoki, K., Adachi, S., Homoto, M., Kusano, H., Koike, K., \& Natsume, T. (2013). LARP1 specifically recognizes the 3\({'}\) terminus of poly(A) mRNA. \emph{FEBS Letters}, \emph{587}(14), 2173--2178. \url{http://doi.org/10.1016/j.febslet.2013.05.035}

\leavevmode\hypertarget{ref-arabiCMycAssociatesRibosomal2005}{}%
Arabi, A., Wu, S., Ridderstråle, K., Bierhoff, H., Shiue, C., Fatyol, K., \ldots{} Wright, A. P. H. (2005). C-Myc associates with ribosomal DNA and activates RNA polymerase I transcription. \emph{Nature Cell Biology}, \emph{7}(3), 303--310. \url{http://doi.org/10.1038/ncb1225}

\leavevmode\hypertarget{ref-Armistead2014a}{}%
Armistead, J., \& Triggs-Raine, B. (2014). Diverse diseases from a ubiquitous process: The ribosomopathy paradox. \emph{FEBS Letters}, \emph{588}(9), 1491--1500. \url{http://doi.org/10.1016/j.febslet.2014.03.024}

\leavevmode\hypertarget{ref-Arvola2017n}{}%
Arvola, R. M., Weidmann, C. A., Tanaka Hall, T. M., \& Goldstrohm, A. C. (2017). Combinatorial control of messenger RNAs by Pumilio, Nanos and Brain Tumor Proteins. \emph{RNA Biology}, \emph{14}(11), 1445--1456. \url{http://doi.org/10.1080/15476286.2017.1306168}

\leavevmode\hypertarget{ref-Bailey2006a}{}%
Bailey, T. L., Williams, N., Misleh, C., \& Li, W. W. (2006). MEME: Discovering and analyzing DNA and protein sequence motifs. \emph{Nucleic Acids Research}, \emph{34}(suppl\_2), W369--W373.

\leavevmode\hypertarget{ref-Barlow2010a}{}%
Barlow, J. L., Drynan, L. F., Trim, N. L., Erber, W. N., Warren, A. J., \& Mckenzie, A. N. J. (2010). Cell Cycle New insights into 5q-syndrome as a ribosomopathy. \emph{Cell Cycle}, \emph{9}, 4286--4293. \url{http://doi.org/10.4161/cc.9.21.13742}

\leavevmode\hypertarget{ref-Barreau2008d}{}%
Barreau, C., Benson, E., Gudmannsdottir, E., Newton, F., \& White-Cooper, H. (2008). Post-meiotic transcription in Drosophila testes. \emph{Development}, \emph{135}(11), 1897--1902.

\leavevmode\hypertarget{ref-Batista2014}{}%
Batista, P. J., Molinie, B., Wang, J., Qu, K., Zhang, J., Li, L., \ldots{} Daneshvar, K. (2014). m6A RNA modification controls cell fate transition in mammalian embryonic stem cells. \emph{Cell Stem Cell}, \emph{15}(6), 707--719.

\leavevmode\hypertarget{ref-Baxter-Roshek2007f}{}%
Baxter-Roshek, J. L., Petrov, A. N., \& Dinman, J. D. (2007). Optimization of ribosome structure and function by rRNA base modification. \emph{PLoS ONE}, \emph{2}(1), e174. \url{http://doi.org/10.1371/journal.pone.0000174}

\leavevmode\hypertarget{ref-Belin2009a}{}%
Belin, S., Beghin, A., Solano-Gonzàlez, E., Bezin, L., Brunet-Manquat, S., Textoris, J., \ldots{} Diaz, J.-J. (2009). Dysregulation of ribosome biogenesis and translational capacity is associated with tumor progression of human breast cancer cells. \emph{PloS One}, \emph{4}(9), e7147.

\leavevmode\hypertarget{ref-Bell1988}{}%
Bell, L. R., Maine, E. M., Schedl, P., \& Cline, T. W. (1988). Sex-lethal, a Drosophila sex determination switch gene, exhibits sex-specific RNA splicing and sequence similarity to RNA binding proteins. \emph{Cell}, \emph{55}(6), 1037--1046.

\leavevmode\hypertarget{ref-bermanControversiesFunctionLARP12020}{}%
Berman, A. J., Thoreen, C. C., Dedeic, Z., Chettle, J., Roux, P. P., \& Blagden, S. P. (2020). Controversies around the function of LARP1. \emph{RNA Biology}, 1--11. \url{http://doi.org/10.1080/15476286.2020.1733787}

\leavevmode\hypertarget{ref-bernardiniRecurrentMicrodeletion17q122009}{}%
Bernardini, L., Gimelli, S., Gervasini, C., Carella, M., Baban, A., Frontino, G., \ldots{} Dallapiccola, B. (2009). Recurrent microdeletion at 17q12 as a cause of Mayer-Rokitansky-Kuster-Hauser (MRKH) syndrome: Two case reports. \emph{Orphanet Journal of Rare Diseases}, \emph{4}(1), 25. \url{http://doi.org/10.1186/1750-1172-4-25}

\leavevmode\hypertarget{ref-Bhandari2014h}{}%
Bhandari, D., Raisch, T., Weichenrieder, O., Jonas, S., \& Izaurralde, E. (2014). Structural basis for the Nanos-mediated recruitment of the CCR4-NOT complex and translational repression. \emph{Genes \& Development}, \emph{28}(8), 888--901. \url{http://doi.org/10.1101/gad.237289.113}

\leavevmode\hypertarget{ref-Black2000}{}%
Black, D. L. (2000). Protein diversity from alternative splicing: A challenge for bioinformatics and post-genome biology. \emph{Cell}, \emph{103}(3), 367--370.

\leavevmode\hypertarget{ref-Blagden2009f}{}%
Blagden, S. P., Gatt, M. K., Archambault, V., Lada, K., Ichihara, K., Lilley, K. S., \ldots{} Glover, D. M. (2009). Drosophila Larp associates with poly (A)-binding protein and is required for male fertility and syncytial embryo development. \emph{Developmental Biology}, \emph{334}(1), 186--197. \url{http://doi.org/10.1016/J.YDBIO.2009.07.016}

\leavevmode\hypertarget{ref-blattPosttranscriptionalGeneRegulation2020}{}%
Blatt, P., Martin, E. T., Breznak, S. M., \& Rangan, P. (2020). Post-transcriptional gene regulation regulates germline stem cell to oocyte transition during Drosophila oogenesis. In \emph{Current Topics in Developmental Biology} (Vol. 140, pp. 3--34). Elsevier. \url{http://doi.org/10.1016/bs.ctdb.2019.10.003}

\leavevmode\hypertarget{ref-blattRNADegradationSculpts2020}{}%
Blatt, P., Wong-Deyrup, S. W., McCarthy, A., Breznak, S., Hurton, M. D., Upadhyay, M., \ldots{} Rangan, P. (2020). RNA degradation sculpts the maternal transcriptome during Drosophila oogenesis. \emph{bioRxiv}, 2020.06.30.179986. \url{http://doi.org/10.1101/2020.06.30.179986}

\leavevmode\hypertarget{ref-boamahPolyADPRibosePolymerase2012}{}%
Boamah, E. K., Kotova, E., Garabedian, M., Jarnik, M., \& Tulin, A. V. (2012). Poly(ADP-Ribose) Polymerase 1 (PARP-1) Regulates Ribosomal Biogenesis in Drosophila Nucleoli. \emph{PLoS Genetics}, \emph{8}(1). \url{http://doi.org/10.1371/journal.pgen.1002442}

\leavevmode\hypertarget{ref-Boerner2016}{}%
Boerner, K., \& Becker, P. B. (2016). Splice variants of the SWR1-type nucleosome remodeling factor Domino have distinct functions during Drosophila melanogaster oogenesis. \emph{Development}, \emph{143}(17), 3154--3167.

\leavevmode\hypertarget{ref-bohnsackQuantitativeAnalysisSnoRNA2008}{}%
Bohnsack, M. T., Kos, M., \& Tollervey, D. (2008). Quantitative analysis of snoRNA association with pre-ribosomes and release of snR30 by Rok1 helicase. \emph{EMBO Reports}, \emph{9}(12), 1230--1236. \url{http://doi.org/10.1038/embor.2008.184}

\leavevmode\hypertarget{ref-boleyNavigatingMiningModENCODE2014}{}%
Boley, N., Wan, K. H., Bickel, P. J., \& Celniker, S. E. (2014). Navigating and Mining modENCODE Data. \emph{Methods (San Diego, Calif.)}, \emph{68}(1), 38--47. \url{http://doi.org/10.1016/j.ymeth.2014.03.007}

\leavevmode\hypertarget{ref-Bousquet-Antonelli2000a}{}%
Bousquet-Antonelli, C. C., Vanrobays, E., Gélugne, J.-P., Caizergues-Ferrer, M., \& Henry, Y. (2000). Rrp8p is a yeast nucleolar protein functionally linked to Gar1p and involved in pre-rRNA cleavage at site A2. \emph{Rna}, \emph{6}(6), 826--843.

\leavevmode\hypertarget{ref-bowenRoleP53Developmental2019}{}%
Bowen, M. E., \& Attardi, L. D. (2019). The role of p53 in developmental syndromes. \emph{Journal of Molecular Cell Biology}, \emph{11}(3), 200--211. \url{http://doi.org/10.1093/jmcb/mjy087}

\leavevmode\hypertarget{ref-Brooks2014b}{}%
Brooks, S. S., Wall, A. L., Golzio, C., Reid, D. W., Kondyles, A., Willer, J. R., \ldots{} Davis, E. E. (2014). A novel ribosomopathy caused by dysfunction of RPL10 disrupts neurodevelopment and causes X-linked microcephaly in humans. \emph{Genetics}, \emph{198}(2), 723--33. \url{http://doi.org/10.1534/genetics.114.168211}

\leavevmode\hypertarget{ref-burrowsRNABindingProtein2010}{}%
Burrows, C., Abd Latip, N., Lam, S.-J., Carpenter, L., Sawicka, K., Tzolovsky, G., \ldots{} Blagden, S. P. (2010). The RNA binding protein Larp1 regulates cell division, apoptosis and cell migration. \emph{Nucleic Acids Research}, \emph{38}(16), 5542--5553. \url{http://doi.org/10.1093/nar/gkq294}

\leavevmode\hypertarget{ref-buszczakCarnegieProteinTrap2007}{}%
Buszczak, M., Paterno, S., Lighthouse, D., Bachman, J., Planck, J., Owen, S., \ldots{} Spradling, A. C. (2007). The Carnegie Protein Trap Library: A Versatile Tool for Drosophila Developmental Studies. \emph{Genetics}, \emph{175}(3), 1505--1531. \url{http://doi.org/10.1534/genetics.106.065961}

\leavevmode\hypertarget{ref-Calo2018a}{}%
Calo, E., Gu, B., Bowen, M. E., Aryan, F., Zalc, A., Liang, J., \ldots{} Attardi, L. D. (2018). Tissue-selective effects of nucleolar stress and rDNA damage in developmental disorders. \emph{Nature}, \emph{554}(7690), 112.

\leavevmode\hypertarget{ref-Carreira-Rosario2016e}{}%
Carreira-Rosario, A., Bhargava, V., Hillebrand, J., Kollipara, R. K. K., Ramaswami, M., \& Buszczak, M. (2016). Repression of Pumilio Protein Expression by Rbfox1 Promotes Germ Cell Differentiation. \emph{Developmental Cell}, \emph{36}(5), 562--571. \url{http://doi.org/10.1016/j.devcel.2016.02.010}

\leavevmode\hypertarget{ref-Chang2011}{}%
Chang, P. L., Dunham, J. P., Nuzhdin, S. V., \& Arbeitman, M. N. (2011). Somatic sex-specific transcriptome differences in Drosophila revealed by whole transcriptome sequencing. \emph{BMC Genomics}, \emph{12}(1), 364.

\leavevmode\hypertarget{ref-Chau2009}{}%
Chau, J., Kulnane, L. S., \& Salz, H. K. (2009). Sex-lethal facilitates the transition from germline stem cell to committed daughter cell in the Drosophila ovary. \emph{Genetics}, \emph{182}(1), 121--132.

\leavevmode\hypertarget{ref-Chau2012}{}%
Chau, J., Kulnane, L. S., \& Salz, H. K. (2012). Sex-lethal enables germline stem cell differentiation by down-regulating Nanos protein levels during Drosophila oogenesis. \emph{Proceedings of the National Academy of Sciences}, \emph{109}(24), 9465--9470.

\leavevmode\hypertarget{ref-Chen2003o}{}%
Chen, D., \& McKearin, D. (2003a). Dpp Signaling Silences bam Transcription Directly to Establish Asymmetric Divisions of Germline Stem Cells. \emph{Current Biology}, \emph{13}(20), 1786--1791. \url{http://doi.org/10.1016/J.CUB.2003.09.033}

\leavevmode\hypertarget{ref-Chen2003q}{}%
Chen, D., \& McKearin, D. M. (2003b). A discrete transcriptional silencer in the bam gene determines asymmetric division of the Drosophila germline stem cell. \emph{Development}, \emph{130}(6), 1159--1170. \url{http://doi.org/10.1242/dev.00325}

\leavevmode\hypertarget{ref-chenComprehensiveAnalysisNucleocytoplasmic2017}{}%
Chen, T., \& Steensel, B. van. (2017). Comprehensive analysis of nucleocytoplasmic dynamics of mRNA in Drosophila cells. \emph{PLOS Genetics}, \emph{13}(8), e1006929. \url{http://doi.org/10.1371/journal.pgen.1006929}

\leavevmode\hypertarget{ref-chenComparativeValidationMelanogaster2014}{}%
Chen, Z.-X., Sturgill, D., Qu, J., Jiang, H., Park, S., Boley, N., \ldots{} Richards, S. (2014). Comparative validation of the D. Melanogaster modENCODE transcriptome annotation. \emph{Genome Research}, \emph{24}(7), 1209--1223. \url{http://doi.org/10.1101/gr.159384.113}

\leavevmode\hypertarget{ref-chengSmallLargeRibosomal2019}{}%
Cheng, Z., Mugler, C. F., Keskin, A., Hodapp, S., Chan, L. Y.-L., Weis, K., \ldots{} Brar, G. A. (2019). Small and Large Ribosomal Subunit Deficiencies Lead to Distinct Gene Expression Signatures that Reflect Cellular Growth Rate. \emph{Molecular Cell}, \emph{73}(1), 36--47.e10. \url{http://doi.org/10.1016/j.molcel.2018.10.032}

\leavevmode\hypertarget{ref-Chymkowitch2017a}{}%
Chymkowitch, P., Aanes, H., Robertson, J., Klungland, A., \& Enserink, J. M. (2017). TORC1-dependent sumoylation of Rpc82 promotes RNA polymerase III assembly and activity. \emph{Proceedings of the National Academy of Sciences}, \emph{114}(5), 1039--1044.

\leavevmode\hypertarget{ref-Cinalli2008d}{}%
Cinalli, R. M., Rangan, P., \& Lehmann, R. (2008). Germ Cells Are Forever. \emph{Cell}, \emph{132}(4), 559--562. \url{http://doi.org/10.1016/j.cell.2008.02.003}

\leavevmode\hypertarget{ref-Cline1999}{}%
Cline, T. W., Rudner, D. Z., Barbash, D. A., Bell, M., \& Vutien, R. (1999). Functioning of the Drosophila integral U1/U2 protein Snf independent of U1 and U2 small nuclear ribonucleoprotein particles is revealed by snf+ gene dose effects. \emph{Proceedings of the National Academy of Sciences}, \emph{96}(25), 14451--14458.

\leavevmode\hypertarget{ref-Cohn1960}{}%
Cohn, W. E. (1960). Pseudouridine, a carbon-carbon linked ribonucleoside in ribonucleic acids: Isolation, structure, and chemical characteristics. \emph{Journal of Biological Chemistry}, \emph{235}(5), 1488--1498.

\leavevmode\hypertarget{ref-corsiniCoordinatedControlMRNA2018}{}%
Corsini, N. S., Peer, A. M., Moeseneder, P., Roiuk, M., Burkard, T. R., Theussl, H.-C., \ldots{} Knoblich, J. A. (2018). Coordinated Control of mRNA and rRNA Processing Controls Embryonic Stem Cell Pluripotency and Differentiation. \emph{Cell Stem Cell}, \emph{22}(4), 543--558.e12. \url{http://doi.org/10.1016/j.stem.2018.03.002}

\leavevmode\hypertarget{ref-Cramton1994a}{}%
Cramton, S. E., \& Laski, F. A. (1994). String of pearls encodes Drosophila ribosomal protein S2, has Minute-like characteristics, and is required during oogenesis. \emph{Genetics}, \emph{137}(4), 1039--1048.

\leavevmode\hypertarget{ref-Decatur2002b}{}%
Decatur, W. A., \& Fournier, M. J. (2002). rRNA modifications and ribosome function. \emph{Trends in Biochemical Sciences}, \emph{27}(7), 344--351. \url{http://doi.org/10.1016/S0968-0004(02)02109-6}

\leavevmode\hypertarget{ref-DeCuevas1998f}{}%
De Cuevas, M., \& Spradling, A. C. (1998). Morphogenesis of the Drosophila fusome and its implications for oocyte specification. \emph{Development}, \emph{125}(15), 2781 LP--2789.

\leavevmode\hypertarget{ref-Deisenroth2010e}{}%
Deisenroth, C., \& Zhang, Y. (2010). Ribosome biogenesis surveillance: Probing the ribosomal protein-Mdm2-p53 pathway. \emph{Oncogene}, \emph{29}(30), 4253--4260. \url{http://doi.org/10.1038/onc.2010.189}

\leavevmode\hypertarget{ref-delacruzFunctionsRibosomalProteins2015}{}%
de la Cruz, J., Karbstein, K., \& Woolford, J. L. (2015). Functions of ribosomal proteins in assembly of eukaryotic ribosomes in vivo. \emph{Annual Review of Biochemistry}, \emph{84}, 93--129. \url{http://doi.org/10.1146/annurev-biochem-060614-033917}

\leavevmode\hypertarget{ref-delucaEfficientExpressionGenes2018}{}%
DeLuca, S. Z., \& Spradling, A. C. (2018). Efficient Expression of Genes in the \emph{Drosophila} Germline Using a UAS Promoter Free of Interference by Hsp70 piRNAs. \emph{Genetics}, \emph{209}(2), 381--387. \url{http://doi.org/10.1534/genetics.118.300874}

\leavevmode\hypertarget{ref-Deshmukh1993a}{}%
Deshmukh, M., Tsay, Y. F., Paulovich, A. G., \& Woolford, J. L. (1993). Yeast ribosomal protein L1 is required for the stability of newly synthesized 5S rRNA and the assembly of 60S ribosomal subunits. \emph{Molecular and Cellular Biology}, \emph{13}(5), 2835--2845.

\leavevmode\hypertarget{ref-DeZoysa2017}{}%
De Zoysa, M. D., \& Yu, Y.-T. (2017). Posttranscriptional RNA pseudouridylation. In \emph{The Enzymes} (Vol. 41, pp. 151--167). Elsevier.

\leavevmode\hypertarget{ref-Dinman2016a}{}%
Dinman, J. D. (2016). Pathways to specialized ribosomes: The Brussels lecture. \emph{Journal of Molecular Biology}, \emph{428}(10), 2186--2194.

\leavevmode\hypertarget{ref-dossantosFlyBaseIntroductionDrosophila2015}{}%
dos Santos, G., Schroeder, A. J., Goodman, J. L., Strelets, V. B., Crosby, M. A., Thurmond, J., \ldots{} Gelbart, W. M. (2015). FlyBase: Introduction of the Drosophila melanogaster Release 6 reference genome assembly and large-scale migration of genome annotations. \emph{Nucleic Acids Research}, \emph{43}(Database issue), D690--D697. \url{http://doi.org/10.1093/nar/gku1099}

\leavevmode\hypertarget{ref-draptchinskaiaGeneEncodingRibosomal1999}{}%
Draptchinskaia, N., Gustavsson, P., Andersson, B., Pettersson, M., Willig, T.-N., Dianzani, I., \ldots{} Dahl, N. (1999). The gene encoding ribosomal protein S19 is mutated in Diamond-Blackfan anaemia. \emph{Nature Genetics}, \emph{21}(2), 169--175. \url{http://doi.org/10.1038/5951}

\leavevmode\hypertarget{ref-Eichhorn2016n}{}%
Eichhorn, S. W., Subtelny, A. O., Kronja, I., Kwasnieski, J. C., Orr-Weaver, T. L., \& Bartel, D. P. (2016). mRNA poly(A)-tail changes specified by deadenylation broadly reshape translation in Drosophila oocytes and early embryos. \emph{eLife}, \emph{5}, e16955. \url{http://doi.org/10.7554/eLife.16955}

\leavevmode\hypertarget{ref-Ellis1994d}{}%
Ellis, R. E., \& Kimble, J. (1994). Control of germ cell differentiation in Caenorhabditis elegans. \emph{Ciba Foundation Symposium}, \emph{182}, 179--192.

\leavevmode\hypertarget{ref-emtenaniGeneticProgramBoosts2021}{}%
Emtenani, S., Martin, E. T., Gyoergy, A., Bicher, J., Genger, J.-W., Hurd, T. R., \ldots{} Siekhaus, D. E. (2021). A genetic program boosts mitochondrial function to power macrophage tissue invasion. \emph{bioRxiv}, 2021.02.18.431643. \url{http://doi.org/10.1101/2021.02.18.431643}

\leavevmode\hypertarget{ref-fanDualRolesDrosophila2010}{}%
Fan, Y., Lee, T. V., Xu, D., Chen, Z., Lamblin, A.-F., Steller, H., \& Bergmann, A. (2010). Dual roles of Drosophila p53 in cell death and cell differentiation. \emph{Cell Death \& Differentiation}, \emph{17}(6), 912--921. \url{http://doi.org/10.1038/cdd.2009.182}

\leavevmode\hypertarget{ref-Fichelson2009a}{}%
Fichelson, P., Moch, C., Ivanovitch, K., Martin, C., Sidor, C. M., Lepesant, J.-A., \ldots{} Huynh, J.-R. (2009). Live-imaging of single stem cells within their niche reveals that a U3snoRNP component segregates asymmetrically and is required for self-renewal in Drosophila. \emph{Nature Cell Biology}, \emph{11}(6), 685.

\leavevmode\hypertarget{ref-Flora2018l}{}%
Flora, P., Schowalter, S., Wong-Deyrup, S., DeGennaro, M., Nasrallah, M. A., \& Rangan, P. (2018). Transient transcriptional silencing alters the cell cycle to promote germline stem cell differentiation in Drosophila. \emph{Developmental Biology}, \emph{434}(1), 84--95. \url{http://doi.org/10.1016/j.ydbio.2017.11.014}

\leavevmode\hypertarget{ref-Flora2018k}{}%
Flora, P., Wong-Deyrup, S. W., Martin, E. T., Palumbo, R. J., Nasrallah, M., Oligney, A., \ldots{} Rangan, P. (2018). Sequential Regulation of Maternal mRNAs through a Conserved cis-Acting Element in Their 3' UTRs. \emph{Cell Reports}, \emph{25}(13), 3828--3843.e9. \url{http://doi.org/10.1016/j.celrep.2018.12.007}

\leavevmode\hypertarget{ref-fonsecaLARP1MajorPhosphorylation2018}{}%
Fonseca, B. D., Jia, J.-J., Hollensen, A. K., Pointet, R., Hoang, H.-D., Niklaus, M. R., \ldots{} Alain, T. (2018). \emph{LARP1 is a major phosphorylation substrate of mTORC1} (Preprint). Biochemistry.

\leavevmode\hypertarget{ref-Fonseca2015a}{}%
Fonseca, B. D., Zakaria, C., Jia, J.-J., Graber, T. E., Svitkin, Y., Tahmasebi, S., \ldots{} Diao, I. T. (2015). La-related protein 1 (LARP1) represses terminal oligopyrimidine (TOP) mRNA translation downstream of mTOR complex 1 (mTORC1). \emph{Journal of Biological Chemistry}, \emph{290}(26), 15996--16020.

\leavevmode\hypertarget{ref-Forbes1998g}{}%
Forbes, A., \& Lehmann, R. (1998). Nanos and Pumilio have critical roles in the development and function of Drosophila germline stem cells. \emph{Development}, \emph{125}(4), 679 LP--690.

\leavevmode\hypertarget{ref-fortierHaploinsufficiencyScreenHighlights2015}{}%
Fortier, S., MacRae, T., Bilodeau, M., Sargeant, T., \& Sauvageau, G. (2015). Haploinsufficiency screen highlights two distinct groups of ribosomal protein genes essential for embryonic stem cell fate. \emph{Proceedings of the National Academy of Sciences}, \emph{112}(7), 2127--2132. \url{http://doi.org/10.1073/pnas.1418845112}

\leavevmode\hypertarget{ref-Fu2015h}{}%
Fu, Z., Geng, C., Wang, H., Yang, Z., Weng, C., Li, H., \ldots{} Xie, T. (2015). Twin Promotes the Maintenance and Differentiation of Germline Stem Cell Lineage through Modulation of Multiple Pathways. \emph{Cell Reports}, \emph{13}(7), 1366--1379. \url{http://doi.org/10.1016/j.celrep.2015.10.017}

\leavevmode\hypertarget{ref-fuchsProteomicAnalysisRibosomes2011}{}%
Fuchs, G., Diges, C., Kohlstaedt, L. A., Wehner, K. A., \& Sarnow, P. (2011). Proteomic Analysis of Ribosomes: Translational Control of mRNA Populations by Glycogen Synthase GYS1. \emph{Journal of Molecular Biology}, \emph{410}(1), 118--130. \url{http://doi.org/10.1016/J.JMB.2011.04.064}

\leavevmode\hypertarget{ref-Fukao2009c}{}%
Fukao, A., Sasano, Y., Imataka, H., Inoue, K., Sakamoto, H., Sonenberg, N., \ldots{} Fujiwara, T. (2009). The ELAV Protein HuD Stimulates Cap-Dependent Translation in a Poly(A)- and eIF4A-Dependent Manner. \emph{Molecular Cell}, \emph{36}(6), 1007--1017. \url{http://doi.org/10.1016/j.molcel.2009.11.013}

\leavevmode\hypertarget{ref-Fuller1998c}{}%
Fuller, M. T. (1998). Genetic control of cell proliferation and differentiation inDrosophilaspermatogenesis. In \emph{Seminars in cell \& developmental biology} (Vol. 9, pp. 433--444). Elsevier.

\leavevmode\hypertarget{ref-gabutRibosomeTranslationalControl2020}{}%
Gabut, M., Bourdelais, F., \& Durand, S. (2020). Ribosome and Translational Control in Stem Cells. \emph{Cells}, \emph{9}(2), 497. \url{http://doi.org/10.3390/cells9020497}

\leavevmode\hypertarget{ref-Gaspar2017b}{}%
Gáspár, I., \& Ephrussi, A. (2017). RNA localization feeds translation. \emph{Science}, \emph{357}(6357), 1235 LP--1236. \url{http://doi.org/10.1126/science.aao5796}

\leavevmode\hypertarget{ref-gentilellaAutogenousControlTOP2017}{}%
Gentilella, A., Morón-Duran, F. D., Fuentes, P., Zweig-Rocha, G., Riaño-Canalias, F., Pelletier, J., \ldots{} Thomas, G. (2017). Autogenous Control of 5\({'}\)TOP mRNA Stability by 40S Ribosomes. \emph{Molecular Cell}, \emph{67}(1), 55--70.e4. \url{http://doi.org/10.1016/j.molcel.2017.06.005}

\leavevmode\hypertarget{ref-Gerstberger2017}{}%
Gerstberger, S., Meyer, C., Benjamin-Hong, S., Rodriguez, J., Briskin, D., Bognanni, C., \ldots{} Tuschl, T. (2017). The conserved RNA exonuclease Rexo5 is required for 3\({'}\) end maturation of 28S rRNA, 5S rRNA, and snoRNAs. \emph{Cell Reports}, \emph{21}(3), 758--772.

\leavevmode\hypertarget{ref-gilboaGermLineStem2003}{}%
Gilboa, L., Forbes, A., Tazuke, S. I., Fuller, M. T., \& Lehmann, R. (2003). Germ line stem cell differentiation in Drosophila requires gap junctions and proceeds via an intermediate state. \emph{Development}, \emph{130}(26), 6625--6634. \url{http://doi.org/10.1242/dev.00853}

\leavevmode\hypertarget{ref-Gilboa2004a}{}%
Gilboa, L., \& Lehmann, R. (2004). Repression of Primordial Germ Cell Differentiation Parallels Germ Line Stem Cell Maintenance. \emph{Current Biology}, \emph{14}(11), 981--986. \url{http://doi.org/10.1016/j.cub.2004.05.049}

\leavevmode\hypertarget{ref-glotzerCyclinDegradedUbiquitin1991}{}%
Glotzer, M., Murray, A. W., \& Kirschner, M. W. (1991). Cyclin is degraded by the ubiquitin pathway. \emph{Nature}, \emph{349}(6305), 132--138. \url{http://doi.org/10.1038/349132a0}

\leavevmode\hypertarget{ref-Goldstrohm2018c}{}%
Goldstrohm, A. C., Hall, T. M. T., \& McKenney, K. M. (2018). Post-transcriptional Regulatory Functions of Mammalian Pumilio Proteins. \emph{Trends in Genetics}, \emph{34}(12), 972--990. \url{http://doi.org/10.1016/j.tig.2018.09.006}

\leavevmode\hypertarget{ref-grandoriCMycBindsHuman2005}{}%
Grandori, C., Gomez-Roman, N., Felton-Edkins, Z. A., Ngouenet, C., Galloway, D. A., Eisenman, R. N., \& White, R. J. (2005). C-Myc binds to human ribosomal DNA and stimulates transcription of rRNA genes by RNA polymerase I. \emph{Nature Cell Biology}, \emph{7}(3), 311--318. \url{http://doi.org/10.1038/ncb1224}

\leavevmode\hypertarget{ref-Granneman2004a}{}%
Granneman, S., \& Baserga, S. J. (2004). Ribosome biogenesis: Of knobs and RNA processing. \emph{Experimental Cell Research}, \emph{296}(1), 43--50.

\leavevmode\hypertarget{ref-Granneman2006}{}%
Granneman, S., Bernstein, K. A., Bleichert, F., \& Baserga, S. J. (2006). Comprehensive Mutational Analysis of Yeast DEXD/H Box RNA Helicases Required for Small Ribosomal Subunit Synthesis Downloaded from. \emph{MOLECULAR AND CELLULAR BIOLOGY}, \emph{26}(4), 1183--1194. \url{http://doi.org/10.1128/MCB.26.4.1183-1194.2006}

\leavevmode\hypertarget{ref-Granneman2011}{}%
Granneman, S., Petfalski, E., Tollervey, D., \& Hurt, E. C. (2011). A cluster of ribosome synthesis factors regulate pre-rRNA folding and 5.8S rRNA maturation by the Rat1 exonuclease. \emph{The EMBO Journal}, \emph{30}(19), 4006--19. \url{http://doi.org/10.1038/emboj.2011.256}

\leavevmode\hypertarget{ref-Grewal2007c}{}%
Grewal, S. S., Evans, J. R., \& Edgar, B. A. (2007). Drosophila TIF-IA is required for ribosome synthesis and cell growth and is regulated by the TOR pathway. \emph{Journal of Cell Biology}, \emph{179}(6), 1105--1113. \url{http://doi.org/10.1083/jcb.200709044}

\leavevmode\hypertarget{ref-guillerezMutationT7RNA2005}{}%
Guillerez, J., Lopez, P. J., Proux, F., Launay, H., \& Dreyfus, M. (2005). A mutation in T7 RNA polymerase that facilitates promoter clearance. \emph{Proceedings of the National Academy of Sciences}, \emph{102}(17), 5958--5963. \url{http://doi.org/10.1073/pnas.0407141102}

\leavevmode\hypertarget{ref-Gumienny2017c}{}%
Gumienny, R., Jedlinski, D. J., Schmidt, A., Gypas, F., Martin, G., Vina-Vilaseca, A., \& Zavolan, M. (2017). High-throughput identification of C/D box snoRNA targets with CLIP and RiboMeth-seq. \emph{Nucleic Acids Research}, \emph{45}(5), 2341--2353. \url{http://doi.org/10.1093/nar/gkw1321}

\leavevmode\hypertarget{ref-Hager1997}{}%
Hager, J. H., \& Cline, T. W. (1997). Induction of female Sex-lethal RNA splicing in male germ cells: Implications for Drosophila germline sex determination. \emph{Development}, \emph{124}(24), 5033--5048.

\leavevmode\hypertarget{ref-Hales2015a}{}%
Hales, K. G., Korey, C. A., Larracuente, A. M., \& Roberts, D. M. (2015). Genetics on the Fly: A Primer on the Drosophila Model System. \emph{Genetics}, \emph{201}(3), 815--842. \url{http://doi.org/10.1534/genetics.115.183392}

\leavevmode\hypertarget{ref-Hanyu-Nakamura2008g}{}%
Hanyu-Nakamura, K., Sonobe-Nojima, H., Tanigawa, A., Lasko, P., \& Nakamura, A. (2008). Drosophila Pgc protein inhibits P-TEFb recruitment to chromatin in primordial germ cells. \emph{Nature}, \emph{451}(7179), 730--733. \url{http://doi.org/10.1038/nature06498}

\leavevmode\hypertarget{ref-Harris2011i}{}%
Harris, R. E., Pargett, M., Sutcliffe, C., Umulis, D., \& Ashe, H. L. (2011). Brat promotes stem cell differentiation via control of a bistable switch that restricts BMP signaling. \emph{Developmental Cell}, \emph{20}(1), 72--83. \url{http://doi.org/10.1016/j.devcel.2010.11.019}

\leavevmode\hypertarget{ref-Harvey2018f}{}%
Harvey, R. F., Smith, T. S., Mulroney, T., Queiroz, R. M. L., Pizzinga, M., Dezi, V., \ldots{} Willis, A. E. (2018). Trans-acting translational regulatory RNA binding proteins. \emph{Wiley Interdisciplinary Reviews. RNA}, \emph{9}(3), e1465--e1465. \url{http://doi.org/10.1002/wrna.1465}

\leavevmode\hypertarget{ref-Haussmann2016}{}%
Haussmann, I. U., Bodi, Z., Sanchez-Moran, E., Mongan, N. P., Archer, N., Fray, R. G., \& Soller, M. (2016). M 6 A potentiates Sxl alternative pre-mRNA splicing for robust Drosophila sex determination. \emph{Nature}, \emph{540}(7632), 301.

\leavevmode\hypertarget{ref-heinzSimpleCombinationsLineagedetermining2010}{}%
Heinz, S., Benner, C., Spann, N., Bertolino, E., Lin, Y. C., Laslo, P., \ldots{} Glass, C. K. (2010). Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities. \emph{Molecular Cell}, \emph{38}(4), 576--589. \url{http://doi.org/10.1016/j.molcel.2010.05.004}

\leavevmode\hypertarget{ref-hendrixPrenatallyDiagnosed17q122012}{}%
Hendrix, N. W., Clemens, M., Canavan, T. P., Surti, U., \& Rajkovic, A. (2012). Prenatally Diagnosed 17q12 Microdeletion Syndrome with a Novel Association with Congenital Diaphragmatic Hernia. \emph{Fetal Diagnosis and Therapy}, \emph{31}(2), 129--133. \url{http://doi.org/10.1159/000332968}

\leavevmode\hypertarget{ref-Henras2008c}{}%
Henras, A. K., Soudet, J., Gérus, M., Lebaron, S., Caizergues-Ferrer, M., Mougin, A., \& Henry, Y. (2008). The post-transcriptional steps of eukaryotic ribosome biogenesis. \emph{Cellular and Molecular Life Sciences}, \emph{65}(15), 2334--2359. \url{http://doi.org/10.1007/s00018-008-8027-0}

\leavevmode\hypertarget{ref-Higa-Nakamine2012o}{}%
Higa-Nakamine, S., Suzuki, T. T., Uechi, T., Chakraborty, A., Nakajima, Y., Nakamura, M., \ldots{} Kenmochi, N. (2012). Loss of ribosomal RNA modification causes developmental defects in zebrafish. \emph{Nucleic Acids Research}, \emph{40}(1), 391--398. \url{http://doi.org/10.1093/nar/gkr700}

\leavevmode\hypertarget{ref-hinnantTemporalRemodelingCell2017}{}%
Hinnant, T. D., Alvarez, A. A., \& Ables, E. T. (2017). Temporal remodeling of the cell cycle accompanies differentiation in the Drosophila germline. \emph{Developmental Biology}, \emph{429}(1), 118--131. \url{http://doi.org/10.1016/j.ydbio.2017.07.001}

\leavevmode\hypertarget{ref-Hong2017a}{}%
Hong, S., Freeberg, M. A., Han, T., Kamath, A., Yao, Y., Fukuda, T., \ldots{} Inoki, K. (2017). LARP1 functions as a molecular switch for mTORC1-mediated translation of an essential class of mRNAs. \emph{Elife}, \emph{6}, e25237.

\leavevmode\hypertarget{ref-hongEvaluationNutrientSensingMTOR2012}{}%
Hong, S., Mannan, A. M., \& Inoki, K. (2012). Evaluation of the Nutrient-Sensing mTOR Pathway. In T. Weichhart (Ed.), \emph{mTOR: Methods and Protocols} (pp. 29--44). Totowa, NJ: Humana Press. \url{http://doi.org/10.1007/978-1-61779-430-8_3}

\leavevmode\hypertarget{ref-Hornstein2001a}{}%
Hornstein, E., Tang, H., \& Meyuhas, O. (2001). Mitogenic and nutritional signals are transduced into translational efficiency of TOP mRNAs. In \emph{Cold Spring Harbor symposia on quantitative biology} (Vol. 66, pp. 477--484). Cold Spring Harbor Laboratory Press.

\leavevmode\hypertarget{ref-hsuDietControlsNormal2008}{}%
Hsu, H.-J., LaFever, L., \& Drummond-Barbosa, D. (2008). Diet controls normal and tumorous germline stem cells via insulin-dependent and -independent mechanisms in Drosophila. \emph{Developmental Biology}, \emph{313}(2), 700--712. \url{http://doi.org/10.1016/j.ydbio.2007.11.006}

\leavevmode\hypertarget{ref-huIntegrativeApproachOrtholog2011}{}%
Hu, Y., Flockhart, I., Vinayagam, A., Bergwitz, C., Berger, B., Perrimon, N., \& Mohr, S. E. (2011). An integrative approach to ortholog prediction for disease-focused and other functional studies. \emph{BMC Bioinformatics}, \emph{12}(1), 357. \url{http://doi.org/10.1186/1471-2105-12-357}

\leavevmode\hypertarget{ref-iadevaiaMTORC1SignalingControls2014}{}%
Iadevaia, V., Liu, R., \& Proud, C. G. (2014). mTORC1 signaling controls multiple steps in ribosome biogenesis. \emph{Seminars in Cell \& Developmental Biology}, \emph{36}, 113--120. \url{http://doi.org/10.1016/j.semcdb.2014.08.004}

\leavevmode\hypertarget{ref-Ichihara2007a}{}%
Ichihara, K., Shimizu, H., Taguchi, O., Yamaguchi, M., \& Inoue, Y. H. (2007). A Drosophila orthologue of larp protein family is required for multiple processes in male meiosis. \emph{Cell Structure and Function}, 710190003.

\leavevmode\hypertarget{ref-Inoue1990}{}%
Inoue, K., Hoshijima, K., Sakamoto, H., \& Shimura, Y. (1990). Binding of the Drosophila sex-lethal gene product to the alternative splice site of transformer primary transcript. \emph{Nature}, \emph{344}(6265), 461.

\leavevmode\hypertarget{ref-Jalkanen2014h}{}%
Jalkanen, A. L., Coleman, S. J., \& Wilusz, J. (2014). Determinants and implications of mRNA poly(A) tail size--does this protein make my tail look big? \emph{Seminars in Cell \& Developmental Biology}, \emph{34}, 24--32. \url{http://doi.org/10.1016/j.semcdb.2014.05.018}

\leavevmode\hypertarget{ref-Jady2001c}{}%
Jády, B. E., \& Kiss, T. (2001). A small nucleolar guide RNA functions both in 2\({'}\)-O-ribose methylation and pseudouridylation of the U5 spliceosomal RNA. \emph{EMBO Journal}, \emph{20}(3), 541--551. \url{http://doi.org/10.1093/emboj/20.3.541}

\leavevmode\hypertarget{ref-jefferiesRapamycinSuppressesTOP1997}{}%
Jefferies, H. B. J., Fumagalli, S., Dennis, P. B., Reinhard, C., Pearson, R. B., \& Thomas, G. (1997). Rapamycin suppresses 5\({'}\)TOP mRNA translation through inhibition of p70s6k. \emph{The EMBO Journal}, \emph{16}(12), 3693--3704. \url{http://doi.org/10.1093/emboj/16.12.3693}

\leavevmode\hypertarget{ref-Jia2016b}{}%
Jia, D., Xu, Q., Xie, Q., Mio, W., \& Deng, W.-M. (2016). Automatic stage identification of Drosophila egg chamber based on DAPI images. \emph{Scientific Reports}, \emph{6}, 18850. \url{http://doi.org/10.1038/srep18850}

\leavevmode\hypertarget{ref-jiaMTORC1PromotesTOP2021}{}%
Jia, J.-J., Lahr, R. M., Solgaard, M. T., Moraes, B. J., Pointet, R., Yang, A.-D., \ldots{} Fonseca, B. D. (2021). mTORC1 promotes TOP mRNA translation through site-specific phosphorylation of LARP1. \emph{Nucleic Acids Research}. \url{http://doi.org/10.1093/nar/gkaa1239}

\leavevmode\hypertarget{ref-Johnson2010}{}%
Johnson, M. L., Nagengast, A. A., \& Salz, H. K. (2010). PPS, a large multidomain protein, functions with sex-lethal to regulate alternative splicing in Drosophila. \emph{PLoS Genetics}, \emph{6}(3), e1000872.

\leavevmode\hypertarget{ref-Joly2013f}{}%
Joly, W., Chartier, A., Rojas-Rios, P., Busseau, I., \& Simonelig, M. (2013). The CCR4 deadenylase acts with Nanos and Pumilio in the fine-tuning of Mei-P26 expression to promote germline stem cell self-renewal. \emph{Stem Cell Reports}, \emph{1}(5), 411--424. \url{http://doi.org/10.1016/j.stemcr.2013.09.007}

\leavevmode\hypertarget{ref-jonesPreventionNeurocristopathyTreacher2008}{}%
Jones, N. C., Lynn, M. L., Gaudenz, K., Sakai, D., Aoto, K., Rey, J.-P., \ldots{} Trainor, P. A. (2008). Prevention of the neurocristopathy Treacher Collins syndrome through inhibition of p53 function. \emph{Nature Medicine}, \emph{14}(2), 125--133. \url{http://doi.org/10.1038/nm1725}

\leavevmode\hypertarget{ref-Joshi2010l}{}%
Joshi, P. M., Riddle, M. R., Djabrayan, N. J. V., \& Rothman, J. H. (2010). Caenorhabditis elegans as a model for stem cell biology. \emph{Developmental Dynamics : An Official Publication of the American Association of Anatomists}, \emph{239}(5), 1539--1554. \url{http://doi.org/10.1002/dvdy.22296}

\leavevmode\hypertarget{ref-kaiEmptyDrosophilaStem2003}{}%
Kai, T., \& Spradling, A. (2003). An empty Drosophila stem cell niche reactivates the proliferation of ectopic cells.

\leavevmode\hypertarget{ref-Kai2005}{}%
Kai, T., Williams, D., \& Spradling, A. C. (2005). The expression profile of purified Drosophila germline stem cells. \emph{Developmental Biology}, \emph{283}(2), 486--502.

\leavevmode\hypertarget{ref-Kalifa2009}{}%
Kalifa, Y., Armenti, S. T., \& Gavis, E. R. (2009). Glorund interactions in the regulation of gurken and oskar mRNAs. \emph{Developmental Biology}, \emph{326}(1), 68--74.

\leavevmode\hypertarget{ref-Kalsotra2011}{}%
Kalsotra, A., \& Cooper, T. A. (2011). Functional consequences of developmentally regulated alternative splicing. \emph{Nature Reviews Genetics}, \emph{12}(10), 715.

\leavevmode\hypertarget{ref-Kan2017}{}%
Kan, L., Grozhik, A. V., Vedanayagam, J., Patil, D. P., Pang, N., Lim, K.-S., \ldots{} Despic, V. (2017). The m 6 A pathway facilitates sex determination in Drosophila. \emph{Nature Communications}, \emph{8}, 15737.

\leavevmode\hypertarget{ref-Karpen1988a}{}%
Karpen, G. H., Schaefer, J. E., \& Laird, C. D. (1988). A Drosophila rRNA gene located in euchromatin is active in transcription and nucleolus formation. \emph{Genes \& Development}, \emph{2}(12b), 1745--1763.

\leavevmode\hypertarget{ref-Khajuria2018a}{}%
Khajuria, R. K., Munschauer, M., Ulirsch, J. C., Fiorini, C., Ludwig, L. S., McFarland, S. K., \ldots{} Sankaran, V. G. (2018). Ribosome levels selectively regulate translation and lineage commitment in human hematopoiesis. \emph{Cell}, \emph{173}(1), 90--103. \url{http://doi.org/10.1016/J.CELL.2018.02.036}

\leavevmode\hypertarget{ref-khoshnevisDEADboxProteinRok12016}{}%
Khoshnevis, S., Askenasy, I., Johnson, M. C., Dattolo, M. D., Young-Erdos, C. L., Stroupe, M. E., \& Karbstein, K. (2016). The DEAD-box Protein Rok1 Orchestrates 40S and 60S Ribosome Assembly by Promoting the Release of Rrp5 from Pre-40S Ribosomes to Allow for 60S Maturation. \emph{PLOS Biology}, \emph{14}(6), e1002480. \url{http://doi.org/10.1371/journal.pbio.1002480}

\leavevmode\hypertarget{ref-Kim2008b}{}%
Kim, E., Goraksha-Hicks, P., Li, L., Neufeld, T. P., \& Guan, K.-L. (2008). Regulation of TORC1 by Rag GTPases in nutrient response. \emph{Nature Cell Biology}, \emph{10}(8), 935.

\leavevmode\hypertarget{ref-Kim2015m}{}%
Kim, G., Pai, C.-I., Sato, K., Person, M. D., Nakamura, A., \& Macdonald, P. M. (2015). Region-Specific Activation of oskar mRNA Translation by Inhibition of Bruno-Mediated Repression. \emph{PLOS Genetics}, \emph{11}(2), e1004992.

\leavevmode\hypertarget{ref-kimballRegulationGlobalSpecific2002}{}%
Kimball, S. R. (2002). Regulation of Global and Specific mRNA Translation by Amino Acids. \emph{The Journal of Nutrition}, \emph{132}(5), 883--886. \url{http://doi.org/10.1093/jn/132.5.883}

\leavevmode\hypertarget{ref-Kim-Ha1995i}{}%
Kim-Ha, J., Kerr, K., \& Macdonald, P. M. (1995). Translational regulation of oskar mRNA by Bruno, an ovarian RNA-binding protein, is essential. \emph{Cell}, \emph{81}(3), 403--412. \url{http://doi.org/10.1016/0092-8674(95)90393-3}

\leavevmode\hypertarget{ref-Kiss2004c}{}%
Kiss, A. M., Jady, B. E., Bertrand, E., \& Kiss, T. (2004). Human Box H/ACA Pseudouridylation Guide RNA Machinery. \emph{Molecular and Cellular Biology}, \emph{24}(13), 5797--5807. \url{http://doi.org/10.1128/MCB.24.13.5797-5807.2004}

\leavevmode\hypertarget{ref-Kiss2010}{}%
Kiss, T., Fayet-Lebaron, E., \& Jády, B. E. (2010). Box H/ACA small ribonucleoproteins. \emph{Molecular Cell}, \emph{37}(5), 597--606.

\leavevmode\hypertarget{ref-Kong2019}{}%
Kong, J., Han, H., Bergalet, J., Bouvrette, L. P. B., Hernández, G., Moon, N.-S., \ldots{} Lasko, P. (2019). Drosophila ribosomal protein S5b is essential for oogenesis and interacts with distinct RNAs. \emph{bioRxiv}, 600502.

\leavevmode\hypertarget{ref-Kos2010}{}%
Koš, M., \& Tollervey, D. (2010). Yeast pre-rRNA processing and modification occur cotranscriptionally. \emph{Molecular Cell}, \emph{37}(6), 809--820.

\leavevmode\hypertarget{ref-koziolTpt1ActivatesTranscription2007}{}%
Koziol, M. J., Garrett, N., \& Gurdon, J. B. (2007). Tpt1 Activates Transcription of oct4 and nanog in Transplanted Somatic Nuclei. \emph{Current Biology}, \emph{17}(9), 801--807. \url{http://doi.org/10.1016/j.cub.2007.03.062}

\leavevmode\hypertarget{ref-Kronja2014}{}%
Kronja, I., Yuan, B., Eichhorn, S. W. W., Dzeyk, K., Krijgsveld, J., Bartel, D. P. P., \& Orr-Weaver, T. L. L. (2014). Widespread Changes in the Posttranscriptional Landscape at the Drosophila Oocyte-to-Embryo Transition. \emph{Cell Reports}, \emph{7}(5), 1495--1508. \url{http://doi.org/10.1016/j.celrep.2014.05.002}

\leavevmode\hypertarget{ref-Lahr2017b}{}%
Lahr, R. M., Fonseca, B. D., Ciotti, G. E., Al-Ashtal, H. A., Jia, J.-J., Niklaus, M. R., \ldots{} Berman, A. J. (2017). La-related protein 1 (LARP1) binds the mRNA cap, blocking eIF4F assembly on TOP mRNAs. \emph{Elife}, \emph{6}, e24146.

\leavevmode\hypertarget{ref-lahrLarelatedProtein1specific2015}{}%
Lahr, R. M., Mack, S. M., Héroux, A., Blagden, S. P., Bousquet-Antonelli, C., Deragon, J.-M., \& Berman, A. J. (2015). The La-related protein 1-specific domain repurposes HEAT-like repeats to directly bind a 5\({'}\)TOP sequence. \emph{Nucleic Acids Research}, \emph{43}(16), 8077--8088. \url{http://doi.org/10.1093/nar/gkv748}

\leavevmode\hypertarget{ref-Lasko2000}{}%
Lasko, P. (2000). The Drosophila melanogaster genome: Translation factors and RNA binding proteins. \emph{Journal of Cell Biology}, \emph{150}(2), 51--56. \url{http://doi.org/10.1083/jcb.150.2.F51}

\leavevmode\hypertarget{ref-Lasko2012a}{}%
Lasko, P. (2012). mRNA localization and translational control in Drosophila oogenesis. \emph{Cold Spring Harbor Perspectives in Biology}, \emph{4}(10), a012294. \url{http://doi.org/10.1101/cshperspect.a012294}

\leavevmode\hypertarget{ref-Lee2014c}{}%
Lee, K.-A., \& Lee, W.-J. (2014). Drosophila as a model for intestinal dysbiosis and chronic inflammatory diseases. \emph{Developmental \& Comparative Immunology}, \emph{42}(1), 102--110. \url{http://doi.org/10.1016/j.dci.2013.05.005}

\leavevmode\hypertarget{ref-Lee2015}{}%
Lee, Y., \& Rio, D. C. (2015). Mechanisms and regulation of alternative pre-mRNA splicing. \emph{Annual Review of Biochemistry}, \emph{84}, 291--323.

\leavevmode\hypertarget{ref-Lesch2012b}{}%
Lesch, B. J., \& Page, D. C. (2012). Genetics of germ cell development. \emph{Nature Reviews Genetics}, \emph{13}, 781.

\leavevmode\hypertarget{ref-Li2018a}{}%
Li, L., Pang, X., Zhu, Z., Lu, L., Yang, J., Cao, J., \& Fei, S. (2018). GTPBP4 Promotes Gastric Cancer Progression via Regulating P53 Activity. \emph{Cellular Physiology and Biochemistry}, \emph{45}(2), 667--676.

\leavevmode\hypertarget{ref-Li2009n}{}%
Li, S., Zhang, C., Takemori, H., Zhou, Y., \& Xiong, Z.-Q. (2009). TORC1 regulates activity-dependent CREB-target gene transcription and dendritic growth of developing cortical neurons. \emph{Journal of Neuroscience}, \emph{29}(8), 2334--2343.

\leavevmode\hypertarget{ref-Li2009h}{}%
Li, Y., Minor, N. T., Park, J. K., McKearin, D. M., \& Maines, J. Z. (2009). Bam and Bgcn antagonize \&lt;Em\&gt;Nanos\&lt;/em\&gt;-dependent germ-line stem cell maintenance. \emph{Proceedings of the National Academy of Sciences}, \emph{106}(23), 9304 LP--9309. \url{http://doi.org/10.1073/pnas.0901452106}

\leavevmode\hypertarget{ref-Li2013h}{}%
Li, Y., Zhang, Q., Carreira-Rosario, A., Maines, J. Z., McKearin, D. M., \& Buszczak, M. (2013). Mei-P26 Cooperates with Bam, Bgcn and Sxl to Promote Early Germline Development in the Drosophila Ovary. \emph{PLOS ONE}, \emph{8}(3), e58301.

\leavevmode\hypertarget{ref-Licht2016}{}%
Licht, K., \& Jantsch, M. F. (2016). Rapid and dynamic transcriptome regulation by RNA editing and RNA modifications. \emph{J Cell Biol}, \emph{213}(1), 15--22.

\leavevmode\hypertarget{ref-Lilly2005d}{}%
Lilly, M. A., \& Duronio, R. J. (2005). New insights into cell cycle control from the Drosophila endocycle. \emph{Oncogene}, \emph{24}(17), 2765--2775. \url{http://doi.org/10.1038/sj.onc.1208610}

\leavevmode\hypertarget{ref-Lin1997b}{}%
Lin, H., \& Spradling, A. C. (1997). A novel group of pumilio mutations affects the asymmetric division of germline stem cells in the Drosophila ovary. \emph{Development (Cambridge, England)}, \emph{124}(12), 2463--2476.

\leavevmode\hypertarget{ref-Linder2006}{}%
Linder, P., \& Lasko, P. (2006). Bent out of Shape: RNA Unwinding by the DEAD-Box Helicase Vasa. \emph{Cell}, \emph{125}(2), 219--221. \url{http://doi.org/10.1016/j.cell.2006.03.030}

\leavevmode\hypertarget{ref-liptonDefectiveErythroidProgenitor1986}{}%
Lipton, J. M., Kudisch, M., Gross, R., \& Nathan, D. G. (1986). Defective Erythroid Progenitor Differentiation System in Congenital Hypoplastic (Diamond-Blackfan) Anemia. \emph{Blood}, \emph{67}(4), 962--968. \url{http://doi.org/10.1182/blood.V67.4.962.962}

\leavevmode\hypertarget{ref-Lo2010a}{}%
Lo, K.-Y., Li, Z., Bussiere, C., Bresson, S., Marcotte, E. M., \& Johnson, A. W. (2010). Defining the pathway of cytoplasmic maturation of the 60S ribosomal subunit. \emph{Molecular Cell}, \emph{39}(2), 196--208.

\leavevmode\hypertarget{ref-loewithTargetRapamycinTOR2011}{}%
Loewith, R., \& Hall, M. N. (2011). Target of Rapamycin (TOR) in Nutrient Signaling and Growth Control. \emph{Genetics}, \emph{189}(4), 1177--1201. \url{http://doi.org/10.1534/genetics.111.133363}

\leavevmode\hypertarget{ref-Lu2001}{}%
Lu, J., \& Oliver, B. (2001). Drosophila OVO regulates ovarian tumor transcription by binding unusually near the transcription start site. \emph{Development}, \emph{128}(9), 1671--1686.

\leavevmode\hypertarget{ref-luMeioticRecombinationProvokes2010}{}%
Lu, W.-J., Chapo, J., Roig, I., \& Abrams, J. M. (2010). Meiotic Recombination Provokes Functional Activation of the p53 Regulatory Network. \emph{Science}, \emph{328}(5983), 1278--1281. \url{http://doi.org/10.1126/science.1185640}

\leavevmode\hypertarget{ref-Lunardi2010a}{}%
Lunardi, A., Di Minin, G., Provero, P., Dal Ferro, M., Carotti, M., Del Sal, G., \& Collavin, L. (2010). A genome-scale protein interaction profile of Drosophila p53 uncovers additional nodes of the human p53 network. \emph{Proceedings of the National Academy of Sciences}, \emph{107}(14), 6322--6327.

\leavevmode\hypertarget{ref-maDNADamageinducedLok2016}{}%
Ma, X., Han, Y., Song, X., Do, T., Yang, Z., Ni, J., \& Xie, T. (2016). DNA damage-induced Lok/CHK2 activation compromises germline stem cell self-renewal and lineage differentiation. \emph{Development}, \emph{143}(23), 4312--4323. \url{http://doi.org/10.1242/dev.141069}

\leavevmode\hypertarget{ref-Madhani1990}{}%
Madhani, H. D., Bordonne, R., \& Guthrie, C. (1990). Multiple roles for U6 snRNA in the splicing pathway. \emph{Genes \& Development}, \emph{4}(12b), 2264--2277.

\leavevmode\hypertarget{ref-Magnuson2012c}{}%
Magnuson, B., Ekim, B., \& Fingar, D. C. (2012). Regulation and function of ribosomal protein S6 kinase (S6K) within mTOR signalling networks. \emph{Biochem. J}, \emph{441}, 1--21. \url{http://doi.org/10.1042/BJ20110892}

\leavevmode\hypertarget{ref-Magnusdottir2014d}{}%
Magnúsdóttir, E., \& Surani, M. A. (2014). How to make a primordial germ cell. \emph{Development}, \emph{141}(2), 245 LP--252. \url{http://doi.org/10.1242/dev.098269}

\leavevmode\hypertarget{ref-Maniatis2002}{}%
Maniatis, T., \& Tasic, B. (2002). Alternative pre-mRNA splicing and proteome expansion in metazoans. \emph{Nature}, \emph{418}(6894), 236.

\leavevmode\hypertarget{ref-Margolis1995a}{}%
Margolis, J., \& Spradling, A. (1995). Identification and behavior of epithelial stem cells in the Drosophila ovary. \emph{Development}, \emph{121}(11), 3797 LP--3807.

\leavevmode\hypertarget{ref-Martin2006a}{}%
Martin, D. E., Powers, T., \& Hall, M. N. (2006). Regulation of ribosome biogenesis: Where is TOR? \emph{Cell Metabolism}, \emph{4}(4), 259--260.

\leavevmode\hypertarget{ref-martinPreribosomalRNAInteraction2014}{}%
Martin, R., Hackert, P., Ruprecht, M., Simm, S., Brüning, L., Mirus, O., \ldots{} Bohnsack, M. T. (2014). A pre-ribosomal RNA interaction network involving snoRNAs and the Rok1 helicase. \emph{RNA}, \emph{20}(8), 1173--1182. \url{http://doi.org/10.1261/rna.044669.114}

\leavevmode\hypertarget{ref-Martineau2008c}{}%
Martineau, Y., Derry, M. C., Wang, X., Yanagiya, A., Berlanga, J. J., Shyu, A.-B., \ldots{} Sonenberg, N. (2008). Poly(A)-binding protein-interacting protein 1 binds to eukaryotic translation initiation factor 3 to stimulate translation. \emph{Molecular and Cellular Biology}, \emph{28}(21), 6658--6667. \url{http://doi.org/10.1128/MCB.00738-08}

\leavevmode\hypertarget{ref-Matera2014}{}%
Matera, A. G., \& Wang, Z. (2014). A day in the life of the spliceosome. \emph{Nature Reviews Molecular Cell Biology}, \emph{15}(2), 108.

\leavevmode\hypertarget{ref-Mathieu2013d}{}%
Mathieu, J., Cauvin, C., Moch, C., Radford, S. J. J., Sampaio, P., Perdigoto, C. N., \ldots{} Huynh, J.-R. (2013). Aurora B and cyclin B have opposite effects on the timing of cytokinesis abscission in Drosophila germ cells and in vertebrate somatic cells. \emph{Developmental Cell}, \emph{26}(3), 250--265. \url{http://doi.org/10.1016/J.DEVCEL.2013.07.005}

\leavevmode\hypertarget{ref-Matias2015g}{}%
Matias, N. R., Mathieu, J., \& Huynh, J.-R. (2015). Abscission is regulated by the ESCRT-III protein shrub in Drosophila germline stem cells. \emph{PLoS Genetics}, \emph{11}(2), e1004653. \url{http://doi.org/10.1371/journal.pgen.1004653}

\leavevmode\hypertarget{ref-Mattox1990}{}%
Mattox, W., Palmer, M. J., \& Baker, B. S. (1990). Alternative splicing of the sex determination gene transformer-2 is sex-specific in the germ line but not in the soma. \emph{Genes \& Development}, \emph{4}(5), 789--805.

\leavevmode\hypertarget{ref-Mayer2006a}{}%
Mayer, C., \& Grummt, I. (2006). Ribosome biogenesis and cell growth: mTOR coordinates transcription by all three classes of nuclear RNA polymerases. \emph{Oncogene}, \emph{25}(48), 6384--6391. \url{http://doi.org/10.1038/sj.onc.1209883}

\leavevmode\hypertarget{ref-Mazumder2001k}{}%
Mazumder, B., Seshadri, V., Imataka, H., Sonenberg, N., \& Fox, P. L. (2001). Translational silencing of ceruloplasmin requires the essential elements of mRNA circularization: Poly(A) tail, poly(A)-binding protein, and eukaryotic translation initiation factor 4G. \emph{Molecular and Cellular Biology}, \emph{21}(19), 6440--6449. \url{http://doi.org/10.1128/mcb.21.19.6440-6449.2001}

\leavevmode\hypertarget{ref-McCarthy2018h}{}%
McCarthy, A., Deiulio, A., Martin, E. T., Upadhyay, M., \& Rangan, P. (2018). Tip60 complex promotes expression of a differentiation factor to regulate germline differentiation in female Drosophila. \emph{Molecular Biology of the Cell}, \emph{29}(24), 2933--2945. \url{http://doi.org/10.1091/mbc.E18-06-0385}

\leavevmode\hypertarget{ref-mccarthyMSL3CoordinatesTranscriptional2019}{}%
McCarthy, A., Sarkar, K., Martin, E. T., Upadhyay, M., James, J. R., Lin, J. M., \ldots{} Rangan, P. (2019). MSL3 coordinates a transcriptional and translational meiotic program in female Drosophila. \emph{bioRxiv}, 2019.12.18.879874. \url{http://doi.org/10.1101/2019.12.18.879874}

\leavevmode\hypertarget{ref-mcgowanReducedRibosomalProtein2011}{}%
McGowan, K. A., Pang, W. W., Bhardwaj, R., Perez, M. G., Pluvinage, J. V., Glader, B. E., \ldots{} Barsh, G. S. (2011). Reduced ribosomal protein gene dosage and p53 activation in low-risk myelodysplastic syndrome. \emph{Blood}, \emph{118}(13), 3622--3633. \url{http://doi.org/10.1182/blood-2010-11-318584}

\leavevmode\hypertarget{ref-McKearin1990e}{}%
McKearin, D. M., \& Spradling, A. C. (1990). Bag-of-marbles: A Drosophila gene required to initiate both male and female gametogenesis. \emph{Genes \& Development}, \emph{4}(12b), 2242--2251. \url{http://doi.org/10.1101/gad.4.12b.2242}

\leavevmode\hypertarget{ref-McKearin1995b}{}%
McKearin, D., \& Ohlstein, B. (1995). A role for the Drosophila bag-of-marbles protein in the differentiation of cystoblasts from germline stem cells. \emph{Development}, \emph{121}(9), 2937 LP--2947.

\leavevmode\hypertarget{ref-meyuhasSynthesisTranslationalApparatus2000}{}%
Meyuhas, O. (2000). Synthesis of the translational apparatus is regulated at the translational level. \emph{European Journal of Biochemistry}, \emph{267}(21), 6321--6330. \url{http://doi.org/10.1046/j.1432-1327.2000.01719.x}

\leavevmode\hypertarget{ref-meyuhasRaceDecipherTop2015}{}%
Meyuhas, O., \& Kahan, T. (2015). The race to decipher the top secrets of TOP mRNAs. \emph{Biochimica et Biophysica Acta (BBA) - Gene Regulatory Mechanisms}, \emph{1849}(7), 801--811. \url{http://doi.org/10.1016/j.bbagrm.2014.08.015}

\leavevmode\hypertarget{ref-Mills2017c}{}%
Mills, E. W., \& Green, R. (2017). Ribosomopathies: There's strength in numbers. \emph{Science}, \emph{358}(6363), eaan2755. \url{http://doi.org/10.1126/SCIENCE.AAN2755}

\leavevmode\hypertarget{ref-Mitchell2010}{}%
Mitchell, S. F., Walker, S. E., Algire, M. A., Park, E.-H., Hinnebusch, A. G., \& Lorsch, J. R. (2010). The 5\({'}\)-7-methylguanosine cap on eukaryotic mRNAs serves both to stimulate canonical translation initiation and to block an alternative pathway. \emph{Molecular Cell}, \emph{39}(6), 950--962.

\leavevmode\hypertarget{ref-Moon2018a}{}%
Moon, S., Cassani, M., Lin, Y. A., Wang, L., Dou, K., \& Zhang, Z. Z. Z. (2018). A Robust Transposon-Endogenizing Response from Germline Stem Cells. \emph{Developmental Cell}, \emph{47}(5), 660--671.

\leavevmode\hypertarget{ref-Moreno-Torres2015}{}%
Moreno-Torres, M., Jaquenoud, M., \& De Virgilio, C. (2015). TORC1 controls G 1S cell cycle transition in yeast via Mpk1 and the greatwall kinase pathway. \emph{Nature Communications}, \emph{6}, 8256.

\leavevmode\hypertarget{ref-Morgado-Palacin2012a}{}%
Morgado-Palacin, L., Llanos, S., \& Serrano, M. (2012). Ribosomal stress induces L11-and p53-dependent apoptosis in mouse pluripotent stem cells. \emph{Cell Cycle}, \emph{11}(3), 503--510.

\leavevmode\hypertarget{ref-Morita2018}{}%
Morita, S., Ota, R., \& Kobayashi, S. (2018). Downregulation of NHP 2 promotes proper cyst formation in Drosophila ovary. \emph{Development, Growth \& Differentiation}, \emph{60}(5), 248--259.

\leavevmode\hypertarget{ref-Mukherjee2012}{}%
Mukherjee, C., Patil, D. P., Kennedy, B. A., Bakthavachalu, B., Bundschuh, R., \& Schoenberg, D. R. (2012). Identification of cytoplasmic capping targets reveals a role for cap homeostasis in translation and mRNA stability. \emph{Cell Reports}, \emph{2}(3), 674--684.

\leavevmode\hypertarget{ref-Nagengast2003}{}%
Nagengast, A. A., Stitzinger, S. M., Tseng, C.-H., Mount, S. M., \& Salz, H. K. (2003). Sex-lethal splicing autoregulation in vivo: Interactions between SEX-LETHAL, the U1 snRNP and U2AF underlie male exon skipping. \emph{Development}, \emph{130}(3), 463--471.

\leavevmode\hypertarget{ref-Nakamura2004g}{}%
Nakamura, A., Sato, K., \& Hanyu-Nakamura, K. (2004). Drosophila Cup Is an eIF4E Binding Protein that Associates with Bruno and Regulates oskar mRNA Translation in Oogenesis. \emph{Developmental Cell}, \emph{6}(1), 69--78. \url{http://doi.org/10.1016/S1534-5807(03)00400-3}

\leavevmode\hypertarget{ref-Navarro2004b}{}%
Navarro, C., Puthalakath, H., Adams, J. M., Strasser, A., \& Lehmann, R. (2004). Egalitarian binds dynein light chain to establish oocyte polarity and maintain oocyte fate. \emph{Nature Cell Biology}, \emph{6}(5), 427--435. \url{http://doi.org/10.1038/ncb1122}

\leavevmode\hypertarget{ref-Nazar2004a}{}%
Nazar, R. (2004). Ribosomal RNA Processing and Ribosome Biogenesis in Eukaryotes. \emph{IUBMB Life (International Union of Biochemistry and Molecular Biology: Life)}, \emph{56}(8), 457--465. \url{http://doi.org/10.1080/15216540400010867}

\leavevmode\hypertarget{ref-nerurkarEukaryoticRibosomeAssembly2015}{}%
Nerurkar, P., Altvater, M., Gerhardy, S., Schütz, S., Fischer, U., Weirich, C., \& Panse, V. G. (2015). Eukaryotic ribosome assembly and nuclear export. \emph{International Review of Cell and Molecular Biology}, \emph{319}, 107--40. \url{http://doi.org/10.1016/bs.ircmb.2015.07.002}

\leavevmode\hypertarget{ref-Neumuller2008}{}%
Neumüller, R. A., Betschinger, J., Fischer, A., Bushati, N., Poernbacher, I., Mechtler, K., \ldots{} Knoblich, J. A. (2008). Mei-P26 regulates microRNAs and cell growth in the Drosophila ovarian stem cell lineage. \emph{Nature}, \emph{454}(7201), 241--5. \url{http://doi.org/10.1038/nature07014}

\leavevmode\hypertarget{ref-Neve2017i}{}%
Neve, J., Patel, R., Wang, Z., Louey, A., \& Furger, A. M. (2017). Cleavage and polyadenylation: Ending the message expands gene regulation. \emph{RNA Biology}, \emph{14}(7), 865--890. \url{http://doi.org/10.1080/15476286.2017.1306171}

\leavevmode\hypertarget{ref-Noda2017}{}%
Noda, T. (2017). Regulation of autophagy through TORC1 and mTORC1. \emph{Biomolecules}, \emph{7}(3), 52.

\leavevmode\hypertarget{ref-ochsFibrillarinNewProtein1985}{}%
Ochs, R. L., Lischwe, M. A., Spohn, W. H., \& Busch, H. (1985). Fibrillarin: A new protein of the nucleolus identified by autoimmune sera. \emph{Biology of the Cell}, \emph{54}(2), 123--133. \url{http://doi.org/10.1111/j.1768-322X.1985.tb00387.x}

\leavevmode\hypertarget{ref-oday8SRRNAProcessing1996}{}%
O 'day, C. L., Chavanikamannil, F., \& Abelson, J. (1996). 8S rRNA processing requires the RNA helicase-like protein Rrp3. \emph{Nucleic Acids Research}, \emph{24}(16).

\leavevmode\hypertarget{ref-ogamiLARP1FacilitatesTranslational2020}{}%
Ogami, K., Oishi, Y., Nogimori, T., Sakamoto, K., \& Hoshino, S.-i. (2020). LARP1 facilitates translational recovery after amino acid refeeding by preserving long poly(A)-tailed TOP mRNAs. \emph{bioRxiv}, 716217. \url{http://doi.org/10.1101/716217}

\leavevmode\hypertarget{ref-ohlsteinEctopicExpressionDrosophila1997}{}%
Ohlstein, B., \& McKearin, D. (1997). Ectopic expression of the Drosophila Bam protein eliminates oogenic germline stem cells. \emph{Development}, \emph{124}(18), 3651--3662.

\leavevmode\hypertarget{ref-ounapHumanWBSCR22Protein2013}{}%
Õunap, K., Käsper, L., Kurg, A., \& Kurg, R. (2013). The Human WBSCR22 Protein Is Involved in the Biogenesis of the 40S Ribosomal Subunits in Mammalian Cells. \emph{PLoS ONE}, \emph{8}(9). \url{http://doi.org/10.1371/journal.pone.0075686}

\leavevmode\hypertarget{ref-pagesBiostringsEfficientManipulation2019}{}%
Pagès, H., Aboyoun, P., Gentleman, R., \& DebRoy, S. (2019). Biostrings: Efficient manipulation of biological strings.

\leavevmode\hypertarget{ref-Pallares-Cartes2012a}{}%
Pallares-Cartes, C., Cakan-Akdogan, G., \& Teleman, A. A. (2012). Tissue-specific coupling between insulin/IGF and TORC1 signaling via PRAS40 in Drosophila. \emph{Developmental Cell}, \emph{22}(1), 172--182.

\leavevmode\hypertarget{ref-Penalva2003}{}%
Penalva, L. O. F., \& Sánchez, L. (2003). RNA binding protein sex-lethal (Sxl) and control of Drosophila sex determination and dosage compensation. \emph{Microbiol. Mol. Biol. Rev.}, \emph{67}(3), 343--359.

\leavevmode\hypertarget{ref-Penzo2018}{}%
Penzo, M., \& Montanaro, L. (2018). Turning uridines around: Role of rRNA pseudouridylation in ribosome biogenesis and ribosomal function. \emph{Biomolecules}, \emph{8}(2), 38.

\leavevmode\hypertarget{ref-Pereboom2011a}{}%
Pereboom, T. C., van Weele, L. J., Bondt, A., \& MacInnes, A. W. (2011). A zebrafish model of dyskeratosis congenita reveals hematopoietic stem cell formation failure resulting from ribosomal protein-mediated p53 stabilization. \emph{Blood}, \emph{118}(20), 5458--5465.

\leavevmode\hypertarget{ref-philippeGlobalAnalysisLARP12020}{}%
Philippe, L., van den Elzen, A. M. G., Watson, M. J., \& Thoreen, C. C. (2020). Global analysis of LARP1 translation targets reveals tunable and dynamic features of 5\({'}\) TOP motifs. \emph{Proceedings of the National Academy of Sciences}, \emph{117}(10), 5319--5328. \url{http://doi.org/10.1073/pnas.1912864117}

\leavevmode\hypertarget{ref-philippeLarelatedProteinLARP12018}{}%
Philippe, L., Vasseur, J.-J., Debart, F., \& Thoreen, C. C. (2018). La-related protein 1 (LARP1) repression of TOP mRNA translation is mediated through its cap-binding domain and controlled by an adjacent regulatory region. \emph{Nucleic Acids Research}, \emph{46}(3), 1457--1469. \url{http://doi.org/10.1093/nar/gkx1237}

\leavevmode\hypertarget{ref-Phipps2011a}{}%
Phipps, K. R., Charette, J. M., \& Baserga, S. J. (2011). The small subunit processome in ribosome biogenesisProgress and prospects. \emph{Wiley Interdisciplinary Reviews: RNA}, \emph{2}(1), 1--21.

\leavevmode\hypertarget{ref-Polydorides2000}{}%
Polydorides, A. D., Okano, H. J., Yang, Y. Y. L., Stefani, G., \& Darnell, R. B. (2000). A brain-enriched polypyrimidine tract-binding protein antagonizes the ability of Nova to regulate neuron-specific alternative splicing. \emph{Proceedings of the National Academy of Sciences}, \emph{97}(12), 6350--6355.

\leavevmode\hypertarget{ref-powersRegulationRibosomeBiogenesis1999}{}%
Powers, T., \& Walter, P. (1999). Regulation of ribosome biogenesis by the rapamycin-sensitive TOR-signaling pathway in Saccharomyces cerevisiae. \emph{Molecular Biology of the Cell}, \emph{10}(4), 987--1000.

\leavevmode\hypertarget{ref-Preiss1998g}{}%
Preiss, T., \& Hentze, M. W. (1998). Dual function of the messenger RNA cap structure in poly(A)-tail-promoted translation in yeast. \emph{Nature}, \emph{392}(6675), 516--520. \url{http://doi.org/10.1038/33192}

\leavevmode\hypertarget{ref-Primus2019}{}%
Primus, S., Pozmanter, C., Baxter, K., \& Van Doren, M. (2019). Tudor-domain containing protein 5-like promotes male sexual identity in the Drosophila germline and is repressed in females by Sex lethal. \emph{PLoS Genetics}, \emph{15}(7), e1007617.

\leavevmode\hypertarget{ref-qiaoNap1l1ControlsEmbryonic2018}{}%
Qiao, H., Li, Y., Feng, C., Duo, S., Ji, F., \& Jiao, J. (2018). Nap1l1 Controls Embryonic Neural Progenitor Cell Proliferation and Differentiation in the Developing Brain. \emph{Cell Reports}, \emph{22}(9), 2279--2293. \url{http://doi.org/10.1016/j.celrep.2018.02.019}

\leavevmode\hypertarget{ref-qinGlobalAnalysesMRNA2007a}{}%
Qin, X., Ahn, S., Speed, T. P., \& Rubin, G. M. (2007). Global analyses of mRNA translational control during early Drosophila embryogenesis. \emph{Genome Biology}, \emph{8}(4), R63. \url{http://doi.org/10.1186/gb-2007-8-4-r63}

\leavevmode\hypertarget{ref-Raisch2016h}{}%
Raisch, T., Bhandari, D., Sabath, K., Helms, S., Valkov, E., Weichenrieder, O., \& Izaurralde, E. (2016). Distinct modes of recruitment of the CCR4-NOT complex by Drosophila and vertebrate Nanos. \emph{The EMBO Journal}, \emph{35}(9), 974--990. \url{http://doi.org/10.15252/embj.201593634}

\leavevmode\hypertarget{ref-Rangan2009}{}%
Rangan, P., DeGennaro, M., Jaime-Bustamante, K., Coux, R.-X. X., Martinho, R. G., \& Lehmann, R. (2009). Temporal and Spatial Control of Germ-Plasm RNAs. \emph{Current Biology}, \emph{19}(1), 72--77. \url{http://doi.org/10.1016/j.cub.2008.11.066}

\leavevmode\hypertarget{ref-Rangan2008}{}%
Rangan, P., DeGennaro, M., \& Lehmann, R. (2008). Regulating Gene Expression in the Drosophila Germ Line. \emph{Cold Spring Harbor Symposia on Quantitative Biology}, \emph{73}, 1--8. \url{http://doi.org/10.1101/sqb.2008.73.057}

\leavevmode\hypertarget{ref-Reichardt2018d}{}%
Reichardt, I., Bonnay, F., Steinmann, V., Loedige, I., Burkard, T. R., Meister, G., \& Knoblich, J. A. (2018). The tumor suppressor Brat controls neuronal stem cell lineages by inhibiting Deadpan and Zelda. \emph{EMBO Reports}, \emph{19}(1), 102--117. \url{http://doi.org/10.15252/embr.201744188}

\leavevmode\hypertarget{ref-Reveal2011j}{}%
Reveal, B., Garcia, C., Ellington, A., \& Macdonald, P. M. (2011). Multiple RNA binding domains of Bruno confer recognition of diverse binding sites for translational repression. \emph{RNA Biology}, \emph{8}(6), 1047--1060. \url{http://doi.org/10.4161/rna.8.6.17542}

\leavevmode\hypertarget{ref-Reyes2008}{}%
Reyes, R., \& Izquierdo, J. M. (2008). Half pint couples transcription and splicing of eIF4E-1, 2 gene during fly development. \emph{Biochemical and Biophysical Research Communications}, \emph{374}(4), 758--762.

\leavevmode\hypertarget{ref-Richter2011j}{}%
Richter, J. D., \& Lasko, P. (2011). Translational control in oocyte development. \emph{Cold Spring Harbor Perspectives in Biology}, \emph{3}(9), a002758--a002758. \url{http://doi.org/10.1101/cshperspect.a002758}

\leavevmode\hypertarget{ref-Rissland2017k}{}%
Rissland, O. S. (2017). The organization and regulation of mRNAProtein complexes. \emph{Wiley Interdisciplinary Reviews: RNA}, \emph{8}(1), e1369. \url{http://doi.org/10.1002/wrna.1369}

\leavevmode\hypertarget{ref-Ritossa1965a}{}%
Ritossa, F. M., \& Spiegelman, S. (1965). Localization of DNA complementary to ribosomal RNA in the nucleolus organizer region of Drosophila melanogaster. \emph{Proceedings of the National Academy of Sciences of the United States of America}, \emph{53}(4), 737.

\leavevmode\hypertarget{ref-Romanelli2013}{}%
Romanelli, M., Diani, E., \& Lievens, P. (2013). New insights into functional roles of the polypyrimidine tract-binding protein. \emph{International Journal of Molecular Sciences}, \emph{14}(11), 22906--22932.

\leavevmode\hypertarget{ref-Roundtree2017}{}%
Roundtree, I. A., Evans, M. E., Pan, T., \& He, C. (2017). Dynamic RNA modifications in gene expression regulation. \emph{Cell}, \emph{169}(7), 1187--1200.

\leavevmode\hypertarget{ref-Royzman1998}{}%
Royzman, I., \& Orr-Weaver, T. L. (1998). S phase and differential DNA replication during Drosophila oogenesis. \emph{Genes to Cells}, \emph{3}(12), 767--776. \url{http://doi.org/10.1046/j.1365-2443.1998.00232.x}

\leavevmode\hypertarget{ref-Rymond1985}{}%
Rymond, B. C., \& Rosbash, M. (1985). Cleavage of 5\({'}\) splice site and lariat formation are independent of 3\({'}\) splice site in yeast mRNA splicing. \emph{Nature}, \emph{317}(6039), 735.

\leavevmode\hypertarget{ref-rorthGal4DrosophilaFemale1998}{}%
Rørth, P. (1998). Gal4 in the Drosophila female germline. \emph{Mechanisms of Development}, \emph{78}(1), 113--118. \url{http://doi.org/10.1016/S0925-4773(98)00157-9}

\leavevmode\hypertarget{ref-Salles2002}{}%
Salles, C., Mével-Ninio, M., Vincent, A., \& Payre, F. (2002). A germline-specific splicing generates an extended ovo protein isoform required for Drosophila oogenesis. \emph{Developmental Biology}, \emph{246}(2), 366--376.

\leavevmode\hypertarget{ref-Sanchez2016h}{}%
Sanchez, C. G., Teixeira, F. K., Czech, B., Preall, J. B., Zamparini, A. L., Seifert, J. R. K., \ldots{} Lehmann, R. (2016). Regulation of Ribosome Biogenesis and Protein Synthesis Controls Germline Stem Cell Differentiation. \emph{Cell Stem Cell}, \emph{18}(2), 276--290. \url{http://doi.org/10.1016/J.STEM.2015.11.004}

\leavevmode\hypertarget{ref-sarovGenomewideResourceAnalysis2016}{}%
Sarov, M., Barz, C., Jambor, H., Hein, M. Y., Schmied, C., Suchold, D., \ldots{} Schnorrer, F. (2016). A genome-wide resource for the analysis of protein localisation in Drosophila. \emph{eLife}, \emph{5}, e12068. \url{http://doi.org/10.7554/eLife.12068}

\leavevmode\hypertarget{ref-Sass1995}{}%
Sass, G. L., Comer, A. R., \& Searles, L. L. (1995). The ovarian tumor protein isoforms of Drosophila melanogaster exhibit differences in function, expression, and localization. \emph{Developmental Biology}, \emph{167}(1), 201--212.

\leavevmode\hypertarget{ref-Schafer2003a}{}%
Schäfer, T., Strauß, D., Petfalski, E., Tollervey, D., \& Hurt, E. (2003). The path from nucleolar 90S to cytoplasmic 40S pre-ribosomes. \emph{The EMBO Journal}, \emph{22}(6), 1370--1380.

\leavevmode\hypertarget{ref-Schupbach1989c}{}%
Schupbach, T., \& Wieschaus, E. (1989). Female sterile mutations on the second chromosome of Drosophila melanogaster. I. Maternal effect mutations. \emph{Genetics}, \emph{121}(1), 101--117.

\leavevmode\hypertarget{ref-Schupbach1991f}{}%
Schupbach, T., \& Wieschaus, E. (1991). Female sterile mutations on the second chromosome of Drosophila melanogaster. II. Mutations blocking oogenesis or altering egg morphology. \emph{Genetics}, \emph{129}(4), 1119--1136.

\leavevmode\hypertarget{ref-Schwarzacher1993}{}%
Schwarzacher, H. G., \& Wachtler, F. (1993). The nucleolus. \emph{Anatomy and Embryology}, \emph{188}(6), 515--536.

\leavevmode\hypertarget{ref-sekiguchiNOP132RequiredProper2006}{}%
Sekiguchi, T., Hayano, T., Yanagida, M., Takahashi, N., \& Nishimoto, T. (2006). NOP132 is required for proper nucleolus localization of DEAD-box RNA helicase DDX47. \emph{Nucleic Acids Research}, \emph{34}(16), 4593--4608. \url{http://doi.org/10.1093/nar/gkl603}

\leavevmode\hypertarget{ref-senturkP53CellCycle2013}{}%
Senturk, E., \& Manfredi, J. J. (2013). P53 and Cell Cycle Effects After DNA Damage. \emph{Methods in Molecular Biology (Clifton, N.J.)}, \emph{962}, 49--61. \url{http://doi.org/10.1007/978-1-62703-236-0_4}

\leavevmode\hypertarget{ref-seranoElementTransformationVectors1994}{}%
Serano, T. L., Cheung, H.-K., Frank, L. H., \& Cohen, R. S. (1994). P element transformation vectors for studying Drosophila melanogaster oogenesis and early embryogenesis. \emph{Gene}, \emph{138}(1), 181--186. \url{http://doi.org/10.1016/0378-1119(94)90804-4}

\leavevmode\hypertarget{ref-Seydoux2006}{}%
Seydoux, G., \& Braun, R. E. (2006). Pathway to Totipotency: Lessons from Germ Cells. \emph{Cell}, \emph{127}(5), 891--904. \url{http://doi.org/10.1016/j.cell.2006.11.016}

\leavevmode\hypertarget{ref-sezginSurveyImageThresholding2004}{}%
Sezgin, M., \& Sankur, B. (2004). Survey over image thresholding techniques and quantitative performance evaluation. \emph{Journal of Electronic Imaging}, \emph{13}(1), 146--166.

\leavevmode\hypertarget{ref-shuNutrientControlMRNA2020}{}%
Shu, X. E., Swanda, R. V., \& Qian, S.-B. (2020). Nutrient Control of mRNA Translation. \emph{Annual Review of Nutrition}, \emph{40}(1), 51--75. \url{http://doi.org/10.1146/annurev-nutr-120919-041411}

\leavevmode\hypertarget{ref-Slaidina2014h}{}%
Slaidina, M., \& Lehmann, R. (2014a). Translational control in germline stem cell development. \emph{The Journal of Cell Biology}, \emph{207}(1), 13 LP--21. \url{http://doi.org/10.1083/jcb.201407102}

\leavevmode\hypertarget{ref-Slaidina2014h}{}%
Slaidina, M., \& Lehmann, R. (2014a). Translational control in germline stem cell development. \emph{The Journal of Cell Biology}, \emph{207}(1), 13 LP--21. \url{http://doi.org/10.1083/jcb.201407102}

\leavevmode\hypertarget{ref-Sloan2017e}{}%
Sloan, K. E., Warda, A. S., Sharma, S., Entian, K. D., Lafontaine, D. L. J., \& Bohnsack, M. T. (2017). Tuning the ribosome: The influence of rRNA modification on eukaryotic ribosome biogenesis and function. \emph{RNA Biology}, \emph{14}(9), 1138--1152. \url{http://doi.org/10.1080/15476286.2016.1259781}

\leavevmode\hypertarget{ref-Smolko2018}{}%
Smolko, A. E., Shapiro-Kulnane, L., \& Salz, H. K. (2018). The H3K9 methyltransferase SETDB1 maintains female identity in Drosophila germ cells. \emph{Nature Communications}, \emph{9}(1), 4155.

\leavevmode\hypertarget{ref-Soldner2018d}{}%
Soldner, F., \& Jaenisch, R. (2018). Stem Cells, Genome Editing, and the Path to Translational Medicine. \emph{Cell}, \emph{175}(3), 615--632. \url{http://doi.org/10.1016/j.cell.2018.09.010}

\leavevmode\hypertarget{ref-Sonoda1999a}{}%
Sonoda, J., \& Wharton, R. P. (1999). Recruitment of Nanos to hunchback mRNA by Pumilio. \emph{Genes \& Development}, \emph{13}(20), 2704--2712. \url{http://doi.org/10.1101/gad.13.20.2704}

\leavevmode\hypertarget{ref-Sonoda2001d}{}%
Sonoda, J., \& Wharton, R. P. (2001). Drosophila Brain Tumor is a translational repressor. \emph{Genes \& Development}, \emph{15}(6), 762--773. \url{http://doi.org/10.1101/gad.870801}

\leavevmode\hypertarget{ref-Spradling1993b}{}%
Spradling, A. C. (1993). Germline cysts: Communes that work. \emph{Cell}, \emph{72}(5), 649--651. \url{http://doi.org/10.1016/0092-8674(93)90393-5}

\leavevmode\hypertarget{ref-Spradling1997e}{}%
Spradling, A. C., De Cuevas, M., Drummond-Barbosa, D., Keyes, L., Lilly, M., Pepling, M., \& Xie, T. (1997). The Drosophila Germarium: Stem Cells, Germ Line Cysts, and Oocytes. \emph{Cold Spring Harbor Symposia on Quantitative Biology}, \emph{62}, 25--34. \url{http://doi.org/10.1101/SQB.1997.062.01.006}

\leavevmode\hypertarget{ref-Spradling1981b}{}%
Spradling, A. C., \& Rubin, G. M. (1981). DROSOPHILA GENOME ORGANIZATION: CONSERVED AND DYNAMIC ASPECTS. \emph{Annual Review of Genetics}, \emph{15}(1), 219--264. \url{http://doi.org/10.1146/annurev.ge.15.120181.001251}

\leavevmode\hypertarget{ref-Spradling2011f}{}%
Spradling, A., Fuller, M. T., Braun, R. E., \& Yoshida, S. (2011). Germline stem cells. \emph{Cold Spring Harbor Perspectives in Biology}, \emph{3}(11), a002642. \url{http://doi.org/10.1101/cshperspect.a002642}

\leavevmode\hypertarget{ref-studierProteinProductionAutoinduction2005}{}%
Studier, F. W. (2005). Protein production by auto-induction in high-density shaking cultures. \emph{Protein Expression and Purification}, \emph{41}(1), 207--234. \url{http://doi.org/10.1016/j.pep.2005.01.016}

\leavevmode\hypertarget{ref-Subtelny2014a}{}%
Subtelny, A. O., Eichhorn, S. W., Chen, G. R., Sive, H., \& Bartel, D. P. (2014). Poly(A)-tail profiling reveals an embryonic switch in translational control. \emph{Nature}, \emph{508}, 66.

\leavevmode\hypertarget{ref-Szostak2013l}{}%
Szostak, E., \& Gebauer, F. (2013). Translational control by 3'-UTR-binding proteins. \emph{Briefings in Functional Genomics}, \emph{12}(1), 58--65. \url{http://doi.org/10.1093/bfgp/els056}

\leavevmode\hypertarget{ref-Tadauchi2001a}{}%
Tadauchi, T., Matsumoto, K., Herskowitz, I., \& Irie, K. (2001). Post-transcriptional regulation through the HO 3'-UTR by Mpt5, a yeast homolog of Pumilio and FBF. \emph{The EMBO Journal}, \emph{20}(3), 552--561. \url{http://doi.org/10.1093/emboj/20.3.552}

\leavevmode\hypertarget{ref-Tadros2009c}{}%
Tadros, W., \& Lipshitz, H. D. (2009). The maternal-to-zygotic transition: A play in two acts. \emph{Development}, \emph{136}(18), 3033 LP--3042. \url{http://doi.org/10.1242/dev.033183}

\leavevmode\hypertarget{ref-Tafforeau2013a}{}%
Tafforeau, L., Zorbas, C., Langhendries, J.-L., Mullineux, S.-T., Stamatopoulou, V., Mullier, R., \ldots{} Lafontaine, D. L. J. (2013). The Complexity of Human Ribosome Biogenesis Revealed by Systematic Nucleolar Screening of Pre-rRNA Processing Factors. \emph{Molecular Cell}, \emph{51}(4), 539--551. \url{http://doi.org/10.1016/J.MOLCEL.2013.08.011}

\leavevmode\hypertarget{ref-tanentzapfIntegrindependentAnchoringStemcell2007}{}%
Tanentzapf, G., Devenport, D., Godt, D., \& Brown, N. H. (2007). Integrin-dependent anchoring of a stem-cell niche. \emph{Nature Cell Biology}, \emph{9}(12), 1413--1418. \url{http://doi.org/10.1038/ncb1660}

\leavevmode\hypertarget{ref-tangAminoAcidInducedTranslation2001}{}%
Tang, H., Hornstein, E., Stolovich, M., Levy, G., Livingstone, M., Templeton, D., \ldots{} Meyuhas, O. (2001). Amino Acid-Induced Translation of TOP mRNAs Is Fully Dependent on Phosphatidylinositol 3-Kinase-Mediated Signaling, Is Partially Inhibited by Rapamycin, and Is Independent of S6K1 and rpS6 Phosphorylation. \emph{Molecular and Cellular Biology}, \emph{21}(24), 8671--8683. \url{http://doi.org/10.1128/MCB.21.24.8671-8683.2001}

\leavevmode\hypertarget{ref-TarunJr1997l}{}%
Tarun Jr, S. Z., Wells, S. E., Deardorff, J. A., \& Sachs, A. B. (1997). Translation initiation factor eIF4G mediates in vitro poly(A) tail-dependent translation. \emph{Proceedings of the National Academy of Sciences of the United States of America}, \emph{94}(17), 9046--9051. \url{http://doi.org/10.1073/pnas.94.17.9046}

\leavevmode\hypertarget{ref-Tasnim2018a}{}%
Tasnim, S., \& Kelleher, E. S. (2018). P53 is required for female germline stem cell maintenance in P-element hybrid dysgenesis. \emph{Developmental Biology}, \emph{434}(2), 215--220.

\leavevmode\hypertarget{ref-Tcherkezian2014b}{}%
Tcherkezian, J., Cargnello, M., Romeo, Y., Huttlin, E. L., Lavoie, G., Gygi, S. P., \& Roux, P. P. (2014). Proteomic analysis of cap-dependent translation identifies LARP1 as a key regulator of 5\({'}\)TOP mRNA translation. \emph{Genes and Development}, \emph{28}(4), 357--371. \url{http://doi.org/10.1101/gad.231407.113}

\leavevmode\hypertarget{ref-Temme2014j}{}%
Temme, C., Simonelig, M., \& Wahle, E. (2014). Deadenylation of mRNA by the CCR4-NOT complex in Drosophila: Molecular and developmental aspects. \emph{Frontiers in Genetics}, \emph{5}, 143. \url{http://doi.org/10.3389/fgene.2014.00143}

\leavevmode\hypertarget{ref-Teng2013}{}%
Teng, T., Thomas, G., \& Mercer, C. A. (2013). Growth control and ribosomopathies. \emph{Current Opinion in Genetics \& Development}, \emph{23}(1), 63--71. \url{http://doi.org/10.1016/J.GDE.2013.02.001}

\leavevmode\hypertarget{ref-Texada2019}{}%
Texada, M. J., Jørgensen, A. F., Christensen, C. F., Koyama, T., Malita, A., Smith, D. K., \ldots{} Rewitz, K. F. (2019). A fat-tissue sensor couples growth to oxygen availability by remotely controlling insulin secretion. \emph{Nature Communications}, \emph{10}(1). \url{http://doi.org/10.1038/s41467-019-09943-y}

\leavevmode\hypertarget{ref-Theunissen2017b}{}%
Theunissen, T. W., \& Jaenisch, R. (2017). Mechanisms of gene regulation in human embryos and pluripotent stem cells. \emph{Development}, \emph{144}(24), 4496 LP--4509. \url{http://doi.org/10.1242/dev.157404}

\leavevmode\hypertarget{ref-thomasPANTHERLibraryProtein2003}{}%
Thomas, P. D., Campbell, M. J., Kejariwal, A., Mi, H., Karlak, B., Daverman, R., \ldots{} Narechania, A. (2003). PANTHER: A Library of Protein Families and Subfamilies Indexed by Function. \emph{Genome Research}, \emph{13}(9), 2129--2141. \url{http://doi.org/10.1101/gr.772403}

\leavevmode\hypertarget{ref-thoreenUnifyingModelMTORC1mediated2012}{}%
Thoreen, C. C., Chantranupong, L., Keys, H. R., Wang, T., Gray, N. S., \& Sabatini, D. M. (2012). A unifying model for mTORC1-mediated regulation of mRNA translation. \emph{Nature}, \emph{485}(7396), 109--113. \url{http://doi.org/10.1038/nature11083}

\leavevmode\hypertarget{ref-Tirronen1995}{}%
Tirronen, M., Lahti, V.-P., Heino, T. I., \& Roos, C. (1995). Two otu transcripts are selectively localised in Drosophila oogenesis by a mechanism that requires a function of the otu protein. \emph{Mechanisms of Development}, \emph{52}(1), 65--75.

\leavevmode\hypertarget{ref-Tschochner2003a}{}%
Tschochner, H., \& Hurt, E. (2003). Pre-ribosomes on the road from the nucleolus to the cytoplasm. \emph{Trends in Cell Biology}, \emph{13}(5), 255--263.

\leavevmode\hypertarget{ref-Twombly1996d}{}%
Twombly, V., Blackman, R. K., Jin, H., Graff, J. M., Padgett, R. W., \& Gelbart, W. M. (1996). The TGF-beta signaling pathway is essential for Drosophila oogenesis. \emph{Development}, \emph{122}(5), 1555 LP--1565.

\leavevmode\hypertarget{ref-tyeProteotoxicityAberrantRibosome2019}{}%
Tye, B. W., Commins, N., Ryazanova, L. V., Wühr, M., Springer, M., Pincus, D., \& Churchman, L. S. (2019). Proteotoxicity from aberrant ribosome biogenesis compromises cell fitness. \emph{eLife}, \emph{8}, e43002. \url{http://doi.org/10.7554/eLife.43002}

\leavevmode\hypertarget{ref-Umen1995}{}%
Umen, J. G., \& Guthrie, C. (1995). The second catalytic step of pre-mRNA splicing. \emph{Rna}, \emph{1}(9), 869.

\leavevmode\hypertarget{ref-upadhyayTransposonDysregulationModulates2016}{}%
Upadhyay, M., Martino Cortez, Y., Wong-Deyrup, S., Tavares, L., Schowalter, S., Flora, P., \ldots{} Rangan, P. (2016). Transposon Dysregulation Modulates dWnt4 Signaling to Control Germline Stem Cell Differentiation in Drosophila. \emph{PLOS Genetics}, \emph{12}(3), e1005918. \url{http://doi.org/10.1371/journal.pgen.1005918}

\leavevmode\hypertarget{ref-valdezTreacherCollinsSyndrome2004}{}%
Valdez, B. C., Henning, D., So, R. B., Dixon, J., \& Dixon, M. J. (2004). The Treacher Collins syndrome (TCOF1) gene product is involved in ribosomal DNA gene transcription by interacting with upstream binding factor. \emph{Proceedings of the National Academy of Sciences}, \emph{101}(29), 10709--10714. \url{http://doi.org/10.1073/pnas.0402492101}

\leavevmode\hypertarget{ref-VanBuskirk2002}{}%
Van Buskirk, C., \& Schüpbach, T. (2002). Half pint regulates alternative splice site selection in Drosophila. \emph{Developmental Cell}, \emph{2}(3), 343--353.

\leavevmode\hypertarget{ref-Venema1997}{}%
Venema, J., Cile Bousquet-Antonelli, C., Gelugne, J.-P., Le Caizergues-Ferrer, M., \& Tollervey, D. (1997). Rok1p Is a Putative RNA Helicase Required for rRNA Processing, \emph{17}(6), 3398--3407.

\leavevmode\hypertarget{ref-venemaProcessingPreribosomalRNA1995}{}%
Venema, J., \& Tollervey, D. (1995). Processing of pre-ribosomal RNA inSaccharomyces cerevisiae. \emph{Yeast}, \emph{11}(16), 1629--1650. \url{http://doi.org/10.1002/yea.320111607}

\leavevmode\hypertarget{ref-Vessey2010b}{}%
Vessey, J. P., Schoderboeck, L., Gingl, E., Luzi, E., Riefler, J., Di Leva, F., \ldots{} Macchi, P. (2010). Mammalian Pumilio 2 regulates dendrite morphogenesis and synaptic function. \emph{Proceedings of the National Academy of Sciences}, \emph{107}(7), 3222 LP--3227. \url{http://doi.org/10.1073/pnas.0907128107}

\leavevmode\hypertarget{ref-vincentSSUProcessomeInteractome2017}{}%
Vincent, N. G., Charette, J. M., \& Baserga, S. J. (2017). The SSU processome interactome in Saccharomyces cerevisiae reveals potential new protein subcomplexes. \emph{RNA}, rna.062927.117. \url{http://doi.org/10.1261/rna.062927.117}

\leavevmode\hypertarget{ref-Vlachos2010a}{}%
Vlachos, A., \& Muir, E. (2010). How I treat Diamond-Blackfan anemia, \emph{116}, 3715--3723. \url{http://doi.org/10.1182/blood-2010-02-251090}

\leavevmode\hypertarget{ref-Wahl2009}{}%
Wahl, M. C., Will, C. L., \& Lührmann, R. (2009). The spliceosome: Design principles of a dynamic RNP machine. \emph{Cell}, \emph{136}(4), 701--718.

\leavevmode\hypertarget{ref-Wang2015a}{}%
Wang, Y., Liu, J., Huang, B. O., Xu, Y.-M., Li, J., Huang, L.-F., \ldots{} Yang, W.-M. (2015). Mechanism of alternative splicing and its regulation. \emph{Biomedical Reports}, \emph{3}(2), 152--158.

\leavevmode\hypertarget{ref-warrenMolecularBasisHuman2018}{}%
Warren, A. J. (2018). Molecular basis of the human ribosomopathy Shwachman-Diamond syndrome. \emph{Advances in Biological Regulation}, \emph{67}, 109--127. \url{http://doi.org/10.1016/j.jbior.2017.09.002}

\leavevmode\hypertarget{ref-WatanabeSusaki2014a}{}%
Watanabe-Susaki, K., Takada, H., Enomoto, K., Miwata, K., Ishimine, H., Intoh, A., \ldots{} Asashima, M. (2014). Biosynthesis of ribosomal RNA in nucleoli regulates pluripotency and differentiation ability of pluripotent stem cells. \emph{Stem Cells}, \emph{32}(12), 3099--3111.

\leavevmode\hypertarget{ref-Watkins2012b}{}%
Watkins, N. J., \& Bohnsack, M. T. (2012). The box C/D and H/ACA snoRNPs: Key players in the modification, processing and the dynamic folding of ribosomal RNA. \emph{Wiley Interdisciplinary Reviews: RNA}, \emph{3}(3), 397--414. \url{http://doi.org/10.1002/wrna.117}

\leavevmode\hypertarget{ref-Webster1997a}{}%
Webster, P. J., Liang, L., Berg, C. A., Lasko, P., \& Macdonald, P. M. (1997). Translational repressor bruno plays multiple roles in development and is widely conserved. \emph{Genes \& Development}, \emph{11}(19), 2510--2521. \url{http://doi.org/10.1101/gad.11.19.2510}

\leavevmode\hypertarget{ref-Wei2018a}{}%
Wei, Y., Bettedi, L., Kim, K., Ting, C.-Y., \& Lilly, M. (2019). The GATOR complex regulates an essential response to meiotic double-stranded breaks in Drosophila. \emph{eLife}, \emph{8}, e42149. \url{http://doi.org/10.7554/eLife.42149}

\leavevmode\hypertarget{ref-Wei2014b}{}%
Wei, Y., Reveal, B., Reich, J., Laursen, W. J., Senger, S., Akbar, T., \ldots{} Lilly, M. A. (2014). TORC1 regulators Iml1/GATOR1 and GATOR2 control meiotic entry and oocyte development in Drosophila. \emph{Proceedings of the National Academy of Sciences}, \emph{111}(52), E5670--E5677.

\leavevmode\hypertarget{ref-Wei2009a}{}%
Wei, Y., \& Zheng, X. F. S. (2009). Sch9 partially mediates TORC1 signaling to control ribosomal RNA synthesis. \emph{Cell Cycle}, \emph{8}(24), 4085--4090.

\leavevmode\hypertarget{ref-Will2001}{}%
Will, C. L., \& Lührmann, R. (2001). Spliceosomal UsnRNP biogenesis, structure and function. \emph{Current Opinion in Cell Biology}, \emph{13}(3), 290--301.

\leavevmode\hypertarget{ref-Will2011d}{}%
Will, C. L., \& Lührmann, R. (2011). Spliceosome structure and function. \emph{Cold Spring Harbor Perspectives in Biology}, \emph{3}(7), a003707.

\leavevmode\hypertarget{ref-woolnoughRegulationRRNAGene2016}{}%
Woolnough, J. L., Atwood, B. L., Liu, Z., Zhao, R., \& Giles, K. E. (2016). The Regulation of rRNA Gene Transcription during Directed Differentiation of Human Embryonic Stem Cells. \emph{PLOS ONE}, \emph{11}(6), e0157276. \url{http://doi.org/10.1371/journal.pone.0157276}

\leavevmode\hypertarget{ref-Wullschleger2006b}{}%
Wullschleger, S., Loewith, R., \& Hall, M. N. (2006). TOR signaling in growth and metabolism. \emph{Cell}, \emph{124}(3), 471--484.

\leavevmode\hypertarget{ref-Xie2007a}{}%
Xie, T., \& Li, L. (2007). Stem cells and their niche: An inseparable relationship. \emph{Development}, \emph{134}(11), 2001 LP--2006. \url{http://doi.org/10.1242/dev.002022}

\leavevmode\hypertarget{ref-Xie1998d}{}%
Xie, T., \& Spradling, A. C. (1998). Decapentaplegic Is Essential for the Maintenance and Division of Germline Stem Cells in the Drosophila Ovary. \emph{Cell}, \emph{94}(2), 251--260. \url{http://doi.org/10.1016/S0092-8674(00)81424-5}

\leavevmode\hypertarget{ref-Xie2000b}{}%
Xie, T., \& Spradling, A. C. (2000a). A Niche Maintaining Germ Line Stem Cells in the Drosophila Ovary. \emph{Science}, \emph{290}(5490), 328--330. \url{http://doi.org/10.1126/science.290.5490.328}

\leavevmode\hypertarget{ref-Xie2000b}{}%
Xie, T., \& Spradling, A. C. (2000a). A Niche Maintaining Germ Line Stem Cells in the Drosophila Ovary. \emph{Science}, \emph{290}(5490), 328--330. \url{http://doi.org/10.1126/science.290.5490.328}

\leavevmode\hypertarget{ref-Xue2012}{}%
Xue, S., \& Barna, M. (2012). Specialized ribosomes: A new frontier in gene regulation and organismal biology. \emph{Nature Reviews Molecular Cell Biology}, \emph{13}(6), 355--369. \url{http://doi.org/10.1038/nrm3359}

\leavevmode\hypertarget{ref-Yamashita2005d}{}%
Yamashita, Y. M., \& Fuller, M. T. (2005). Asymmetric stem cell division and function of the niche in the Drosophila male germ line. \emph{International Journal of Hematology}, \emph{82}(5), 377--380. \url{http://doi.org/10.1532/IJH97.05097}

\leavevmode\hypertarget{ref-Yan2015}{}%
Yan, D., \& Perrimon, N. (2015). Spenito is required for sex determination in Drosophila melanogaster. \emph{Proceedings of the National Academy of Sciences}, \emph{112}(37), 11606--11611.

\leavevmode\hypertarget{ref-Yang2012}{}%
Yang, S. Y., Baxter, E. M., \& Van Doren, M. (2012). Phf7 controls male sex determination in the Drosophila germline. \emph{Developmental Cell}, \emph{22}(5), 1041--1051.

\leavevmode\hypertarget{ref-Yang2018}{}%
Yang, Y.-G. Y., Hsu, P. J., Chen, Y.-S., \& Yang, Y.-G. Y. (2018). Dynamic transcriptomic m 6 A decoration: Writers, erasers, readers and functions in RNA metabolism. \emph{Cell Research}, \emph{28}(6), 616.

\leavevmode\hypertarget{ref-Yelick2015a}{}%
Yelick, P. C., \& Trainor, P. A. (2015). Ribosomopathies: Global process, tissue specific defects. \emph{Rare Diseases}, \emph{3}(1), e1025185. \url{http://doi.org/10.1080/21675511.2015.1025185}

\leavevmode\hypertarget{ref-Yerlikaya2016a}{}%
Yerlikaya, S., Meusburger, M., Kumari, R., Huber, A., Anrather, D., Costanzo, M., \ldots{} Loewith, R. (2016). TORC1 and TORC2 work together to regulate ribosomal protein S6 phosphorylation in Saccharomyces cerevisiae. \emph{Molecular Biology of the Cell}, \emph{27}(2), 397--409.

\leavevmode\hypertarget{ref-Yi2011}{}%
Yi, C., \& Pan, T. (2011). Cellular dynamics of RNA modification. \emph{Accounts of Chemical Research}, \emph{44}(12), 1380--1388.

\leavevmode\hypertarget{ref-You2015}{}%
You, K. T., Park, J., \& Kim, V. N. (2015). Role of the small subunit processome in the maintenance of pluripotent stem cells. \emph{Genes \& Development}, \emph{29}(19), 2004--9. \url{http://doi.org/10.1101/gad.267112.115}

\leavevmode\hypertarget{ref-yuUpregulationGTPBP4Colorectal2016}{}%
Yu, H., Jin, S., Zhang, N., \& Xu, Q. (2016). Up-regulation of GTPBP4 in colorectal carcinoma is responsible for tumor metastasis. \emph{Biochemical and Biophysical Research Communications}, \emph{480}(1), 48--54. \url{http://doi.org/10.1016/j.bbrc.2016.10.010}

\leavevmode\hypertarget{ref-zahradkalRegulationRibosomeBiogenesis1991}{}%
Zahradkal, P., Larson, D., \& Sells, B. (1991). Regulation of ribosome biogenesis in differentiated rat myotubes. \emph{Molecular and Cellular Biochemistry}, \emph{104}(1-2). \url{http://doi.org/10.1007/BF00229819}

\leavevmode\hypertarget{ref-Zamore1999b}{}%
Zamore, P. D., Bartel, D. P., Lehmann, R., \& Williamson, J. R. (1999). The PUMILIO-RNA interaction: A single RNA-binding domain monomer recognizes a bipartite target sequence. \emph{Biochemistry}, \emph{38}(2), 596--604. \url{http://doi.org/10.1021/bi982264s}

\leavevmode\hypertarget{ref-Zemp2007}{}%
Zemp, I., \& Kutay, U. (2007). Nuclear export and cytoplasmic maturation of ribosomal subunits. \emph{FEBS Letters}, \emph{581}(15), 2783--2793.

\leavevmode\hypertarget{ref-Zhang2015c}{}%
Zhang, K., \& Smith, G. W. (2015). Maternal control of early embryogenesis in mammals. \emph{Reproduction, Fertility, and Development}, \emph{27}(6), 880--896. \url{http://doi.org/10.1071/RD14441}

\leavevmode\hypertarget{ref-Zhang2014d}{}%
Zhang, Q., Shalaby, N. A., \& Buszczak, M. (2014). Changes in rRNA transcription influence proliferation and cell fate within a stem cell lineage. \emph{Science}, \emph{343}(6168), 298--301.

\leavevmode\hypertarget{ref-zhangIdentificationDHX33Mediator2011}{}%
Zhang, Y., Forys, J. T., Miceli, A. P., Gwinn, A. S., \& Weber, J. D. (2011). Identification of DHX33 as a Mediator of rRNA Synthesis and Cell Growth. \emph{Molecular and Cellular Biology}, \emph{31}(23), 4676--4691. \url{http://doi.org/10.1128/MCB.05832-11}

\leavevmode\hypertarget{ref-zhangSignalingP53Ribosomal2009}{}%
Zhang, Y., \& Lu, H. (2009). Signaling to p53: Ribosomal Proteins Find Their Way. \emph{Cancer Cell}, \emph{16}(5), 369--377. \url{http://doi.org/10.1016/j.ccr.2009.09.024}

\leavevmode\hypertarget{ref-Zhao2015}{}%
Zhao, B. S., \& He, C. (2015). Pseudouridine in a new era of RNA modifications. \emph{Cell Research}, \emph{25}(2), 153.

\leavevmode\hypertarget{ref-Zhao2002d}{}%
Zhao, G.-Q., \& Garbers, D. L. (2002). Male germ cell specification and differentiation. \emph{Developmental Cell}, \emph{2}(5), 537--547.

\leavevmode\hypertarget{ref-zhouInducingRNAiDrosophila2013}{}%
Zhou, R., Mohr, S., Hannon, G. J., \& Perrimon, N. (2013). Inducing RNAi in Drosophila Cells by Transfection with dsRNA. \emph{Cold Spring Harbor Protocols}, \emph{2013}(5), pdb.prot074351--pdb.prot074351. \url{http://doi.org/10.1101/pdb.prot074351}

\leavevmode\hypertarget{ref-Zielke2014a}{}%
Zielke, N., Korzelius, J., van Straaten, M., Bender, K., Schuhknecht, G. F. P., Dutta, D., \ldots{} Edgar, B. A. (2014). Fly-FUCCI: A versatile tool for studying cell proliferation in complex tissues. \emph{Cell Reports}, \emph{7}(2), 588--598.
\end{cslreferences}

% Index?

\end{document}
